# abcrf

<details>

* Version: 1.8.1
* GitHub: NA
* Source code: https://github.com/cran/abcrf
* Date/Publication: 2019-11-05 14:40:02 UTC
* Number of recursive dependencies: 35

Run `revdep_details(, "abcrf")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.6Mb
      sub-directories of 1Mb or more:
        data   2.6Mb
        libs   3.8Mb
    ```

# AlpsNMR

<details>

* Version: 3.0.3
* GitHub: NA
* Source code: https://github.com/cran/AlpsNMR
* Date/Publication: 2020-11-04
* Number of recursive dependencies: 163

Run `revdep_details(, "AlpsNMR")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘zip’
      All declared Imports should be used.
    ```

# AMARETTO

<details>

* Version: 1.6.0
* GitHub: NA
* Source code: https://github.com/cran/AMARETTO
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 147

Run `revdep_details(, "AMARETTO")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.2Mb
      sub-directories of 1Mb or more:
        data   1.5Mb
        doc    2.3Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    License components with restrictions not permitted:
      Apache License (== 2.0) + file LICENSE
    Package listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘knitr’
    A package should be listed in only one of these fields.
    'LinkingTo' field is unused: package has no 'src' directory
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    AMARETTO_HTMLreport: no visible binding for global variable ‘Type’
    AMARETTO_HTMLreport: no visible binding for global variable ‘Color’
    AMARETTO_HTMLreport: no visible binding for global variable ‘Modules’
    AMARETTO_HTMLreport: no visible binding for global variable
      ‘dt_gensesetsall’
    GeneSetDescription: no visible binding for global variable
      ‘MsigdbMapping’
    GeneSetDescription : <anonymous>: no visible binding for global
      variable ‘MsigdbMapping’
    GmtFromModules: no visible binding for global variable ‘value’
    GmtFromModules: no visible binding for global variable ‘variable’
    GmtFromModules: no visible binding for global variable ‘GeneNames’
    HyperGTestGeneEnrichment: no visible binding for global variable ‘i’
    HyperGTestGeneEnrichment: no visible binding for global variable ‘j’
    read_gct: no visible binding for global variable ‘Description’
    Undefined global functions or variables:
      Color Description GeneNames Genes Geneset Geneset_length ModuleNr
      Modules MsigdbMapping NumberGenes Overlapping_genes RegulatorIDs
      TargetIDs Testset Type Weights dt_gensesetsall i j moduleNr
      n_Overlapping overlap_perc p.value p_value padj q.value value
      variable
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 152 marked UTF-8 strings
    ```

# amplican

<details>

* Version: 1.12.0
* GitHub: https://github.com/valenlab/amplican
* Source code: https://github.com/cran/amplican
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 108

Run `revdep_details(, "amplican")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 16.0Mb
      sub-directories of 1Mb or more:
        doc   13.8Mb
        libs   1.2Mb
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘CrispRVariants’
    ```

# antiProfiles

<details>

* Version: 1.30.0
* GitHub: https://github.com/HCBravoLab/antiProfiles
* Source code: https://github.com/cran/antiProfiles
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 7

Run `revdep_details(, "antiProfiles")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    ev: no visible global function definition for ‘predict’
    ev: no visible global function definition for ‘smoothScatter’
    ev : f1: no visible global function definition for ‘predict’
    ev: no visible global function definition for ‘curve’
    Undefined global functions or variables:
      curve predict smoothScatter
    Consider adding
      importFrom("graphics", "curve", "smoothScatter")
      importFrom("stats", "predict")
    to your NAMESPACE file.
    ```

# aroma.core

<details>

* Version: 3.2.1
* GitHub: https://github.com/HenrikBengtsson/aroma.core
* Source code: https://github.com/cran/aroma.core
* Date/Publication: 2020-02-04 15:20:21 UTC
* Number of recursive dependencies: 47

Run `revdep_details(, "aroma.core")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      'sfit', 'expectile', 'HaarSeg', 'mpcbs'
    ```

# aroma.light

<details>

* Version: 3.20.0
* GitHub: https://github.com/HenrikBengtsson/aroma.light
* Source code: https://github.com/cran/aroma.light
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 6

Run `revdep_details(, "aroma.light")` for more info

</details>

## In both

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      inst/rsp/.rspPlugins
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

# BASiCS

<details>

* Version: 2.2.0
* GitHub: https://github.com/catavallejos/BASiCS
* Source code: https://github.com/cran/BASiCS
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 134

Run `revdep_details(, "BASiCS")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 12.0Mb
      sub-directories of 1Mb or more:
        data   2.0Mb
        doc    1.4Mb
        libs   7.8Mb
    ```

# BatchQC

<details>

* Version: 1.18.0
* GitHub: https://github.com/mani2012/BatchQC
* Source code: https://github.com/cran/BatchQC
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 149

Run `revdep_details(, "BatchQC")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    gene_plot: no visible global function definition for ‘boxplot’
    gene_plot: no visible binding for global variable ‘batch’
    gls.series.C: no visible global function definition for ‘lm.fit’
    sample_plot: no visible global function definition for ‘boxplot’
    sample_plot: no visible binding for global variable ‘batch’
    Undefined global functions or variables:
      batch boxplot lm.fit
    Consider adding
      importFrom("graphics", "boxplot")
      importFrom("stats", "lm.fit")
    to your NAMESPACE file.
    ```

# baystability

<details>

* Version: 0.1.0
* GitHub: NA
* Source code: https://github.com/cran/baystability
* Date/Publication: 2018-03-13 15:55:34 UTC
* Number of recursive dependencies: 109

Run `revdep_details(, "baystability")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘ggfortify’ ‘ggplot2’ ‘matrixStats’ ‘reshape2’ ‘scales’
      All declared Imports should be used.
    ```

# bdynsys

<details>

* Version: 1.3
* GitHub: NA
* Source code: https://github.com/cran/bdynsys
* Date/Publication: 2014-12-08 07:01:51
* Number of recursive dependencies: 91

Run `revdep_details(, "bdynsys")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    ...
    phaseportmod: no visible binding for global variable ‘dev.off’
    plot_data: no visible global function definition for ‘dev.set’
    plot_data: no visible global function definition for ‘postscript’
    plot_data: no visible global function definition for ‘matplot’
    plot_data: no visible global function definition for ‘points’
    plot_data: no visible global function definition for ‘legend’
    plot_data: no visible global function definition for ‘dev.off’
    polyfitbayes: no visible global function definition for ‘runif’
    polyfitbayes: no visible global function definition for ‘var’
    polyfitreg: no visible global function definition for ‘lm’
    polyfitreg: no visible binding for global variable ‘na.exclude’
    polyfitreg: no visible global function definition for ‘var’
    Undefined global functions or variables:
      dev.off dev.set grid legend lm matplot na.exclude points postscript
      runif sd var write.table
    Consider adding
      importFrom("grDevices", "dev.off", "dev.set", "postscript")
      importFrom("graphics", "grid", "legend", "matplot", "points")
      importFrom("stats", "lm", "na.exclude", "runif", "sd", "var")
      importFrom("utils", "write.table")
    to your NAMESPACE file.
    ```

# bigPint

<details>

* Version: 1.6.0
* GitHub: https://github.com/lindsayrutter/bigPint
* Source code: https://github.com/cran/bigPint
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 166

Run `revdep_details(, "bigPint")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    ...
    > ret <- plotLitre(data = soybean_ir_sub,
    +     dataMetrics = soybean_ir_sub_metrics, threshVal = 1e-10,
    +     saveFile = FALSE)
    > length(ret)
    [1] 61
    > names(ret)[1]
    [1] "N_P_Glyma.19G168700.Wm82.a2.v1"
    > ret[[1]]
    > 
    > # Example 2: Create litre plots for each of the five most significant genes
    > # (low FDR values). View plot for gene "N_P_Glyma.19G168700.Wm82.a2.v1".
    > 
    > geneList = soybean_ir_sub_metrics[["N_P"]][1:5,]$ID
    > ret <- plotLitre(data = soybean_ir_sub, geneList = geneList,
    +     pointColor = "deeppink")
    Warning in jpeg(filename = paste0(outDir, "/", group1, "_", group2, "_",  :
      unable to open connection to X11 display ''
    Error in .External2(C_X11, paste0("jpeg::", quality, ":", filename), g$width,  : 
      unable to start device JPEG
    Calls: plotLitre -> lapply -> FUN -> lapply -> FUN -> jpeg
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  7.6Mb
      sub-directories of 1Mb or more:
        data             2.0Mb
        doc              2.3Mb
        shiny-examples   3.0Mb
    ```

# bingat

<details>

* Version: 1.3
* GitHub: NA
* Source code: https://github.com/cran/bingat
* Date/Publication: 2017-07-05 18:30:37 UTC
* Number of recursive dependencies: 34

Run `revdep_details(, "bingat")` for more info

</details>

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘genalg’
    ```

# biscuiteer

<details>

* Version: 1.4.0
* GitHub: https://github.com/trichelab/biscuiteer
* Source code: https://github.com/cran/biscuiteer
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 205

Run `revdep_details(, "biscuiteer")` for more info

</details>

## In both

*   checking whether package ‘biscuiteer’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘BiocParallel::bpstart’ by ‘QDNAseq::bpstart’ when loading ‘biscuiteer’
    See ‘/c4/home/henrik/repositories/matrixStats/revdep/checks/biscuiteer/new/biscuiteer.Rcheck/00install.out’ for details.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.8Mb
      sub-directories of 1Mb or more:
        data      1.1Mb
        extdata   3.4Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported object imported by a ':::' call: ‘QDNAseq:::expectedVariance’
      See the note in ?`:::` about the use of this operator.
    ```

# bnbc

<details>

* Version: 1.12.0
* GitHub: https://github.com/hansenlab/bnbc
* Source code: https://github.com/cran/bnbc
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 87

Run `revdep_details(, "bnbc")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.1Mb
      sub-directories of 1Mb or more:
        data   1.7Mb
        libs   1.2Mb
    ```

# bnclassify

<details>

* Version: 0.4.5
* GitHub: https://github.com/bmihaljevic/bnclassify
* Source code: https://github.com/cran/bnclassify
* Date/Publication: 2020-03-12 17:40:02 UTC
* Number of recursive dependencies: 94

Run `revdep_details(, "bnclassify")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  8.9Mb
      sub-directories of 1Mb or more:
        libs   8.2Mb
    ```

# brms

<details>

* Version: 2.14.4
* GitHub: https://github.com/paul-buerkner/brms
* Source code: https://github.com/cran/brms
* Date/Publication: 2020-11-03 06:40:22 UTC
* Number of recursive dependencies: 210

Run `revdep_details(, "brms")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘cmdstanr’
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.6Mb
      sub-directories of 1Mb or more:
        R     3.0Mb
        doc   2.7Mb
    ```

# BSgenome

<details>

* Version: 1.58.0
* GitHub: https://github.com/Bioconductor/BSgenome
* Source code: https://github.com/cran/BSgenome
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 92

Run `revdep_details(, "BSgenome")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    ...
    Loading required package: GenomicFeatures
    Loading required package: AnnotationDbi
    Loading required package: Biobase
    Welcome to Bioconductor
    
        Vignettes contain introductory material; view with
        'browseVignettes()'. To cite Bioconductor, see
        'citation("Biobase")', and for packages 'citation("pkgname")'.
    
    > txdb <- TxDb.Hsapiens.UCSC.hg38.knownGene
    > my_cds <- cds(txdb)
    > seqlevels(my_cds, pruning.mode="coarse") <- c("chr22", "chrM")
    > seqlevelsStyle(my_cds)  # UCSC
    [1] "UCSC"
    > seqlevelsStyle(snps)    # NCBI
    [1] "NCBI"
    > seqlevelsStyle(my_cds) <- seqlevelsStyle(snps)
    Error in .make_assembly_report_URL(assembly_accession) : 
      don't know where to find assembly report for GCF_000001405.38
    Calls: seqlevelsStyle<- ... .get_NCBI_chrom_info_from_accession -> fetch_assembly_report -> .make_assembly_report_URL
    Execution halted
    ```

*   checking for missing documentation entries ... WARNING
    ```
    ...
        'MaskedBSgenomeDataPkgSeed'
      generic 'forgeMaskedBSgenomeDataPkg' and siglist 'character'
      generic 'forgeMaskedBSgenomeDataPkg' and siglist 'list'
      generic 'length' and siglist 'OnDiskNamedSequences'
      generic 'names' and siglist 'FastaNamedSequences'
      generic 'names' and siglist 'TwobitNamedSequences'
      generic 'rowids' and siglist 'OnDiskLongTable'
      generic 'rowids' and siglist 'OnDiskLongTable_old'
      generic 'seqinfo' and siglist 'FastaNamedSequences'
      generic 'seqinfo' and siglist 'RdaNamedSequences'
      generic 'seqinfo' and siglist 'RdsNamedSequences'
      generic 'seqinfo' and siglist 'TwobitNamedSequences'
      generic 'seqnames' and siglist 'OnDiskNamedSequences'
      generic 'show' and siglist 'OnDiskLongTable'
      generic 'show' and siglist 'OnDiskLongTable_old'
      generic 'show' and siglist 'OnDiskNamedSequences'
      generic 'spatialIndex' and siglist 'OnDiskLongTable'
    All user-level objects in a package (including S4 classes and methods)
    should have documentation entries.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      'BiocGenerics', 'S4Vectors', 'IRanges', 'GenomeInfoDb',
      'GenomicRanges', 'Biostrings', 'rtracklayer'
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking for portable file names ... NOTE
    ```
    Found the following non-portable file path:
      BSgenome/inst/extdata/GentlemanLab/BSgenome.Gmellonella.NCBI.ASM364042v2-tools/fasta_to_sorted_2bit.R
    
    Tarballs are only required to store paths of up to 100 bytes and cannot
    store those of more than 256 bytes, with restrictions including to 100
    bytes for the final component.
    See section ‘Package structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  7.8Mb
      sub-directories of 1Mb or more:
        R         1.7Mb
        extdata   5.5Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Packages listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘methods’ ‘BiocGenerics’ ‘S4Vectors’ ‘IRanges’ ‘GenomeInfoDb’ ‘GenomicRanges’ ‘Biostrings’ ‘rtracklayer’
    A package should be listed in only one of these fields.
    ```

*   checking dependencies in R code ... NOTE
    ```
    ':::' calls which should be '::':
      ‘S4Vectors:::makeClassinfoRowForCompactPrinting’
      ‘S4Vectors:::makePrettyMatrixForCompactPrinting’
      See the note in ?`:::` about the use of this operator.
    Unexported objects imported by ':::' calls:
      ‘GenomeInfoDb:::compactPrintNamedAtomicVector’
      ‘GenomicRanges:::get_out_of_bound_index’ ‘IRanges:::.normargSEW’
      ‘IRanges:::unlist_as_integer’ ‘S4Vectors:::anyMissing’
      ‘S4Vectors:::anyMissingOrOutside’ ‘S4Vectors:::decodeRle’
      ‘S4Vectors:::diffWithInitialZero’
      ‘S4Vectors:::make_zero_col_DataFrame’ ‘S4Vectors:::quick_unlist’
      ‘S4Vectors:::quick_unsplit’ ‘S4Vectors:::recycleVector’
      ‘rtracklayer:::.DNAString_to_twoBit’ ‘rtracklayer:::.TwoBits_export’
      ‘rtracklayer:::checkArgFormat’ ‘rtracklayer:::twoBitPath’
      See the note in ?`:::` about the use of this operator.
    ```

# bumphunter

<details>

* Version: 1.32.0
* GitHub: https://github.com/ririzarr/bumphunter
* Source code: https://github.com/cran/bumphunter
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 104

Run `revdep_details(, "bumphunter")` for more info

</details>

## In both

*   checking tests ...
    ```
    ...
      ── Warning (test_annotation.R:103:1): (code run outside of `test_that()`) ──────
      'fetchExtendedChromInfoFromUCSC' is deprecated.
      Use 'getChromInfoFromUCSC' instead.
      See help("Deprecated")
      
      ── ERROR (test_annotation.R:103:1): (code run outside of `test_that()`) ────────
      Error: don't know where to find assembly report for GCF_000001405.26
      Backtrace:
          █
       1. └─GenomeInfoDb::fetchExtendedChromInfoFromUCSC("hg38") test_annotation.R:103:0
       2.   └─GenomeInfoDb:::FUN(...)
       3.     └─GenomeInfoDb:::fetch_assembly_report(assembly_accession, AssemblyUnits = AssemblyUnits)
       4.       └─GenomeInfoDb:::.make_assembly_report_URL(assembly_accession)
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      Warning (test_annotation.R:103:1): (code run outside of `test_that()`)
      ERROR (test_annotation.R:103:1): (code run outside of `test_that()`)
      
      [ FAIL 1 | WARN 1 | SKIP 0 | PASS 13 ]
      Error: Test failures
      Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      'S4Vectors', 'IRanges', 'GenomeInfoDb', 'GenomicRanges', 'foreach',
      'iterators', 'parallel', 'locfit'
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported object imported by a ':::' call: ‘doParallel:::.options’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    bumphunterEngine: no visible binding for global variable ‘bootstraps’
    Undefined global functions or variables:
      bootstraps
    ```

# calmate

<details>

* Version: 0.12.1
* GitHub: https://github.com/HenrikBengtsson/calmate
* Source code: https://github.com/cran/calmate
* Date/Publication: 2015-10-27 08:09:53
* Number of recursive dependencies: 22

Run `revdep_details(, "calmate")` for more info

</details>

## In both

*   checking tests ...
    ```
    ...
      +   stopifnot(length(dim(dummy)) == 3);
      + }
      Warning message:
      In rlm.default(t(TR), naiveGenoDiff, maxit = maxIter, weights = coeffs) :
        'rlm' failed to converge in 50 steps
      > 
      > # Create plot
      > Clim <- c(-0.2,4);
      > 
      > if (interactive()) {
      +   devNew(type="x11", aspectRatio=1.9);
      + } else {
      +   devNew(type="png", "test-calmateByTheta.png", aspectRatio=1.9);
      + }
      Error in .External2(C_X11, paste0("png::", filename), g$width, g$height,  : 
        unable to start device PNG
      Calls: devNew -> do.call -> png
      In addition: Warning message:
      In png("test-calmateByTheta.png", width = 480, height = 912) :
        unable to open connection to X11 display ''
      Execution halted
    ```

# CARBayesST

<details>

* Version: 3.1
* GitHub: https://github.com/duncanplee/CARBayesST
* Source code: https://github.com/cran/CARBayesST
* Date/Publication: 2020-03-09 15:10:11 UTC
* Number of recursive dependencies: 103

Run `revdep_details(, "CARBayesST")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.7Mb
      sub-directories of 1Mb or more:
        libs   4.4Mb
    ```

# CATALYST

<details>

* Version: 1.14.0
* GitHub: https://github.com/HelenaLC/CATALYST
* Source code: https://github.com/cran/CATALYST
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 234

Run `revdep_details(, "CATALYST")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 14.2Mb
      sub-directories of 1Mb or more:
        data   3.1Mb
        doc    9.8Mb
    ```

# celda

<details>

* Version: 1.6.1
* GitHub: https://github.com/campbio/celda
* Source code: https://github.com/cran/celda
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 243

Run `revdep_details(, "celda")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.3Mb
      sub-directories of 1Mb or more:
        libs   5.1Mb
    ```

# cellWise

<details>

* Version: 2.2.1
* GitHub: NA
* Source code: https://github.com/cran/cellWise
* Date/Publication: 2020-11-11 06:40:14 UTC
* Number of recursive dependencies: 88

Run `revdep_details(, "cellWise")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 12.7Mb
      sub-directories of 1Mb or more:
        data   3.2Mb
        doc    1.8Mb
        libs   7.5Mb
    ```

# CEMiTool

<details>

* Version: 1.14.0
* GitHub: NA
* Source code: https://github.com/cran/CEMiTool
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 162

Run `revdep_details(, "CEMiTool")` for more info

</details>

## In both

*   checking whether package ‘CEMiTool’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘ggplot2::annotate’ by ‘ggpmisc::annotate’ when loading ‘CEMiTool’
    See ‘/c4/home/henrik/repositories/matrixStats/revdep/checks/CEMiTool/new/CEMiTool.Rcheck/00install.out’ for details.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  8.8Mb
      sub-directories of 1Mb or more:
        data      3.1Mb
        doc       2.3Mb
        extdata   2.8Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    plot_mean_var,CEMiTool: no visible binding for global variable
      ‘..rr.label..’
    plot_ora,CEMiTool : <anonymous>: no visible global function definition
      for ‘head’
    plot_qq,CEMiTool: no visible binding for global variable ‘data’
    plot_sample_tree,CEMiTool: no visible global function definition for
      ‘dist’
    plot_sample_tree,CEMiTool: no visible global function definition for
      ‘dev.off’
    save_plots,CEMiTool : <anonymous>: no visible global function
      definition for ‘dev.off’
    save_plots,CEMiTool: no visible global function definition for
      ‘dev.off’
    Undefined global functions or variables:
      ..eq.label.. ..rr.label.. := Mean Variance as.dist data dev.off dist
      head modules num_genes setNames tail var
    Consider adding
      importFrom("grDevices", "dev.off")
      importFrom("stats", "as.dist", "dist", "setNames", "var")
      importFrom("utils", "data", "head", "tail")
    to your NAMESPACE file.
    ```

# ChAMP

<details>

* Version: 2.20.1
* GitHub: NA
* Source code: https://github.com/cran/ChAMP
* Date/Publication: 2020-11-03
* Number of recursive dependencies: 244

Run `revdep_details(, "ChAMP")` for more info

</details>

## In both

*   checking whether package ‘ChAMP’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Note: possible error in 'dmrcate(myannotation, ': unused argument (mc.cores = cores) 
    See ‘/c4/home/henrik/repositories/matrixStats/revdep/checks/ChAMP/new/ChAMP.Rcheck/00install.out’ for details.
    Information on the location(s) of code generating the ‘Note’s can be
    obtained by re-running with environment variable R_KEEP_PKG_SOURCE set
    to ‘yes’.
    ```

*   checking Rd \usage sections ... WARNING
    ```
    Duplicated \argument entries in documentation object 'champ.SVD':
      ‘Rplot’
    
    Functions with \usage entries need to have the appropriate \alias
    entries, and all their arguments documented.
    The \usage entries must correspond to syntactically valid R code.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      'minfi', 'ChAMPdata', 'DMRcate', 'Illumina450ProbeVariants.db',
      'IlluminaHumanMethylationEPICmanifest', 'DT', 'RPMM'
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.9Mb
      sub-directories of 1Mb or more:
        doc       3.6Mb
        extdata   1.9Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Package listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘rmarkdown’
    A package should be listed in only one of these fields.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: 'prettydoc'
      All declared Imports should be used.
    Package in Depends field not imported from: 'DT'
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    champ.DMR: possible error in dmrcate(myannotation, min.cpgs =
      minProbes, lambda = lambda, C = C, mc.cores = cores): unused argument
      (mc.cores = cores)
    champ.ebGSEA : gseaWTfn: no visible global function definition for
      'wilcox.test'
    champ.ebGSEA: no visible global function definition for 'stopCluster'
    champ.norm: no visible global function definition for 'stopCluster'
    champ.process: no visible global function definition for 'champ.EpiMod'
    Undefined global functions or variables:
      champ.EpiMod stopCluster wilcox.test
    Consider adding
      importFrom("stats", "wilcox.test")
    to your NAMESPACE file.
    ```

# Chicago

<details>

* Version: 1.18.0
* GitHub: NA
* Source code: https://github.com/cran/Chicago
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 163

Run `revdep_details(, "Chicago")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘PCHiCdata’
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘PCHiCdata’
    ```

# ChIPpeakAnno

<details>

* Version: 3.24.1
* GitHub: NA
* Source code: https://github.com/cran/ChIPpeakAnno
* Date/Publication: 2020-10-30
* Number of recursive dependencies: 183

Run `revdep_details(, "ChIPpeakAnno")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    ...
    Welcome to Bioconductor
    
        Vignettes contain introductory material; view with
        'browseVignettes()'. To cite Bioconductor, see
        'citation("Biobase")', and for packages 'citation("pkgname")'.
    
    Loading required package: AnnotationFilter
    
    Attaching package: 'ensembldb'
    
    The following object is masked from 'package:stats':
    
        filter
    
    >     library(EnsDb.Hsapiens.v75)
    >     data("myPeakList")
    >     annoGR <- toGRanges(EnsDb.Hsapiens.v75)
    Error in .make_assembly_report_URL(assembly_accession) : 
      don't know where to find assembly report for GCF_000001405.25
    Calls: toGRanges ... .get_NCBI_chrom_info_from_accession -> fetch_assembly_report -> .make_assembly_report_URL
    Execution halted
    ```

*   checking tests ...
    ```
    ...
       20.                     └─(function (UCSC_chrom_info, assembly_accession, AssemblyUnits = NULL, ...
       21.                       └─GenomeInfoDb::getChromInfoFromNCBI(assembly_accession, assembly.units = AssemblyUnits)
       22.                         └─GenomeInfoDb:::.get_NCBI_chrom_info_from_accession(...)
       23.                           └─GenomeInfoDb:::fetch_assembly_report(accession)
       24.                             └─GenomeInfoDb:::.make_assembly_report_URL(assembly_accession)
      
      ── Skip (test_write2FASTA.R:1:1): write2FASTA works not correct ────────────────
      Reason: empty test
      
      ── Skipped tests  ──────────────────────────────────────────────────────────────
      ● empty test (5)
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      Warning (test_annotatePeakInBatch.R:427:5): annotatePeakInBatch works not correct
      ERROR (test_binOverFeature.R:26:5): binOverFeature works not correct
      ERROR (test_getAllPeakSequence.R:7:5): getAllPeakSequence works not correct
      ERROR (test_toGRanges.R:34:5): toGRanges works not correct
      
      [ FAIL 3 | WARN 1 | SKIP 5 | PASS 195 ]
      Error: Test failures
      Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 32.8Mb
      sub-directories of 1Mb or more:
        data     20.5Mb
        doc       6.0Mb
        extdata   5.2Mb
    ```

# cliqueMS

<details>

* Version: 1.4.0
* GitHub: https://github.com/osenan/cliqueMS
* Source code: https://github.com/cran/cliqueMS
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 138

Run `revdep_details(, "cliqueMS")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.4Mb
      sub-directories of 1Mb or more:
        libs   5.8Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘MSnbase’
      All declared Imports should be used.
    ```

# Clomial

<details>

* Version: 1.26.0
* GitHub: NA
* Source code: https://github.com/cran/Clomial
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 2

Run `revdep_details(, "Clomial")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    Clomial: no visible global function definition for ‘runif’
    Clomial.generate.data: no visible global function definition for
      ‘runif’
    Clomial.generate.data: no visible global function definition for
      ‘rbinom’
    Clomial.likelihood: no visible global function definition for ‘dbinom’
    Phi: no visible global function definition for ‘dbinom’
    choose.best: no visible global function definition for ‘tail’
    compute.P.reparam : update.Wj: no visible global function definition
      for ‘optim’
    compute.q: no visible global function definition for ‘dbinom’
    Undefined global functions or variables:
      dbinom optim rbinom runif tail
    Consider adding
      importFrom("stats", "dbinom", "optim", "rbinom", "runif")
      importFrom("utils", "tail")
    to your NAMESPACE file.
    ```

# clusterExperiment

<details>

* Version: 2.10.0
* GitHub: https://github.com/epurdom/clusterExperiment
* Source code: https://github.com/cran/clusterExperiment
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 178

Run `revdep_details(, "clusterExperiment")` for more info

</details>

## In both

*   R CMD check timed out
    

*   checking contents of ‘data’ directory ... WARNING
    ```
    Output for data("rsecFluidigm", package = "clusterExperiment"):
      Search path was changed
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 17.5Mb
      sub-directories of 1Mb or more:
        R      1.3Mb
        data   3.6Mb
        doc   10.3Mb
        libs   1.9Mb
    ```

*   checking foreign function calls ... NOTE
    ```
    Foreign function calls to a different package:
      .C(ape::node_depth, ...)
      .C(ape::node_depth_edgelength, ...)
      .C(ape::node_height, ...)
      .C(ape::node_height_clado, ...)
    See chapter ‘System and foreign language interfaces’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘scRNAseq’, ‘ConsensusClusterPlus’
    ```

# cmapR

<details>

* Version: 1.2.0
* GitHub: https://github.com/cmap/cmapR
* Source code: https://github.com/cran/cmapR
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 71

Run `revdep_details(, "cmapR")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘prada’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# cna

<details>

* Version: 3.0.1
* GitHub: NA
* Source code: https://github.com/cran/cna
* Date/Publication: 2020-11-06 11:10:03 UTC
* Number of recursive dependencies: 64

Run `revdep_details(, "cna")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.6Mb
      sub-directories of 1Mb or more:
        libs   5.6Mb
    ```

# cola

<details>

* Version: 1.6.0
* GitHub: https://github.com/jokergoo/cola
* Source code: https://github.com/cran/cola
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 204

Run `revdep_details(, "cola")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘cola-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: top_rows_heatmap-matrix-method
    > ### Title: Heatmap of top rows from different top-value methods
    > ### Aliases: top_rows_heatmap,matrix-method
    > 
    > ### ** Examples
    > 
    > set.seed(123)
    > mat = matrix(rnorm(1000), nrow = 100)
    > top_rows_heatmap(mat, top_n = 25)
    Warning in png(file_name, width = image_width, height = image_height) :
      unable to open connection to X11 display ''
    Error in .External2(C_X11, paste0("png::", filename), g$width, g$height,  : 
      unable to start device PNG
    Calls: top_rows_heatmap -> top_rows_heatmap -> .local -> png
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 10.6Mb
      sub-directories of 1Mb or more:
        data   3.7Mb
        doc    3.3Mb
        libs   1.4Mb
    ```

*   checking compiled code ... NOTE
    ```
    File ‘cola/libs/cola.so’:
      Found ‘rand’, possibly from ‘rand’ (C)
        Object: ‘pdist.o’
    
    Compiled code should not call entry points which might terminate R nor
    write to stdout/stderr instead of to the console, nor use Fortran I/O
    nor system RNGs.
    
    See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
    ```

# collapse

<details>

* Version: 1.4.2
* GitHub: https://github.com/SebKrantz/collapse
* Source code: https://github.com/cran/collapse
* Date/Publication: 2020-11-10 15:10:12 UTC
* Number of recursive dependencies: 95

Run `revdep_details(, "collapse")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.7Mb
      sub-directories of 1Mb or more:
        doc    1.9Mb
        libs   1.9Mb
    ```

# ComplexHeatmap

<details>

* Version: 2.6.2
* GitHub: https://github.com/jokergoo/ComplexHeatmap
* Source code: https://github.com/cran/ComplexHeatmap
* Date/Publication: 2020-11-12
* Number of recursive dependencies: 102

Run `revdep_details(, "ComplexHeatmap")` for more info

</details>

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘magick’
    ```

# CopywriteR

<details>

* Version: 2.22.0
* GitHub: https://github.com/PeeperLab/CopywriteR
* Source code: https://github.com/cran/CopywriteR
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 62

Run `revdep_details(, "CopywriteR")` for more info

</details>

## In both

*   checking running R code from vignettes ...
    ```
      ‘CopywriteR.Rnw’... failed
     ERROR
    Errors in running code in vignettes:
    when running code in ‘CopywriteR.Rnw’
      ...
    Plotting to file /scratch/henrik/RtmpH5Jeja/file685e2cfe3dfe/vignettes/CNAprofiles/qc/read.counts.compensated.T43_4.bam.png 
    Warning in png(plot, width = 700, height = 1400) :
      unable to open connection to X11 display ''
    
      When sourcing ‘CopywriteR.R’:
    Error: The GC-content and mappability normalization did not work due to a
      failure to calculate loesses. This can generally be solved by using
      larger bin sizes. Stopping execution of the remaining part of the
      script...
    Execution halted
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      definition for ‘text’
    plotCNA : <anonymous> : <anonymous>: no visible global function
      definition for ‘axis’
    plotCNA : <anonymous> : <anonymous>: no visible global function
      definition for ‘abline’
    plotCNA : <anonymous> : <anonymous>: no visible global function
      definition for ‘dev.off’
    preCopywriteR: no visible global function definition for ‘as’
    Undefined global functions or variables:
      abline as axis dev.off dpois ecdf getClass lines loess packageVersion
      par pdf png points ppois predict read.table rgb segments text
      write.table
    Consider adding
      importFrom("grDevices", "dev.off", "pdf", "png", "rgb")
      importFrom("graphics", "abline", "axis", "lines", "par", "points",
                 "segments", "text")
      importFrom("methods", "as", "getClass")
      importFrom("stats", "dpois", "ecdf", "loess", "ppois", "predict")
      importFrom("utils", "packageVersion", "read.table", "write.table")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# corrcoverage

<details>

* Version: 1.2.1
* GitHub: https://github.com/annahutch/corrcoverage
* Source code: https://github.com/cran/corrcoverage
* Date/Publication: 2019-12-06 23:20:12 UTC
* Number of recursive dependencies: 73

Run `revdep_details(, "corrcoverage")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.2Mb
      sub-directories of 1Mb or more:
        extdata   3.8Mb
        libs      1.2Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘data.table’
      All declared Imports should be used.
    ```

# crlmm

<details>

* Version: 1.48.0
* GitHub: NA
* Source code: https://github.com/cran/crlmm
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 72

Run `revdep_details(, "crlmm")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.7Mb
      sub-directories of 1Mb or more:
        R      1.2Mb
        data   2.7Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Versioned 'LinkingTo' value for ‘preprocessCore’ is only usable in R >= 3.0.2
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' call to ‘RUnit’ in package code.
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    Namespace in Imports field not imported from: ‘splines’
      All declared Imports should be used.
    Unexported object imported by a ':::' call: ‘Biobase:::assayDataEnvLock’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    .test: no visible global function definition for ‘defineTestSuite’
    .test: no visible global function definition for ‘runTestSuite’
    .test: no visible global function definition for ‘printTextProtocol’
    calculateRBafCNSet : processByChromosome: no visible global function
      definition for ‘position’
    genotypeInf: no visible binding for global variable ‘anno’
    krlmm: no visible binding for global variable ‘VGLMparameters’
    Undefined global functions or variables:
      VGLMparameters anno defineTestSuite position printTextProtocol
      runTestSuite
    ```

# crossmeta

<details>

* Version: 1.16.1
* GitHub: NA
* Source code: https://github.com/cran/crossmeta
* Date/Publication: 2020-11-02
* Number of recursive dependencies: 159

Run `revdep_details(, "crossmeta")` for more info

</details>

## In both

*   checking dependencies in R code ... WARNING
    ```
    '::' or ':::' import not declared from: ‘tibble’
    Namespace in Imports field not imported from: ‘statmod’
      All declared Imports should be used.
    Unexported object imported by a ':::' call: ‘GEOquery:::parseGSEMatrix’
      See the note in ?`:::` about the use of this operator.
    There are ::: calls to the package's namespace in its code. A package
      almost never needs to use ::: for its own objects:
      ‘getDirListing’ ‘getGEO’ ‘getGEOSuppFiles’
    ```

*   checking for code/documentation mismatches ... WARNING
    ```
    Codoc mismatches from documentation object 'add_sources':
    add_sources
      Code: function(diff_exprs, data_dir = getwd(), postfix = NULL)
      Docs: function(diff_exprs, data_dir = getwd())
      Argument names in code not in docs:
        postfix
    ```

*   checking Rd \usage sections ... WARNING
    ```
    ...
    Undocumented arguments in documentation object 'ch2_subset'
      ‘prev_anal’
    
    Undocumented arguments in documentation object 'get_sva_mods'
      ‘pdata’
    Documented arguments not in \usage in documentation object 'get_sva_mods':
      ‘eset’
    
    Undocumented arguments in documentation object 'iqr_replicates'
      ‘eset’
    Documented arguments not in \usage in documentation object 'iqr_replicates':
      ‘mod’ ‘svobj’
    
    Undocumented arguments in documentation object 'run_select_contrasts'
      ‘port’
    
    Functions with \usage entries need to have the appropriate \alias
    entries, and all their arguments documented.
    The \usage entries must correspond to syntactically valid R code.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    get_palette: no visible global function definition for ‘head’
    is_invertible: no visible global function definition for ‘is’
    load_agil_plat : <anonymous>: no visible global function definition for
      ‘capture.output’
    load_raw: no visible global function definition for ‘setNames’
    symbol_annot: no visible binding for global variable ‘SYMBOL_9606’
    to_eset: no visible global function definition for ‘as’
    to_ma: no visible global function definition for ‘new’
    which_max_iqr: no visible binding for global variable ‘iqrange’
    xls_to_txt: no visible global function definition for ‘write.table’
    Undefined global functions or variables:
      SYMBOL_9606 as capture.output download.file head html iqrange is
      model.matrix new read.csv read.table select_contrasts setNames
      toggleClass write.csv write.table
    Consider adding
      importFrom("methods", "as", "is", "new")
      importFrom("stats", "model.matrix", "setNames")
      importFrom("utils", "capture.output", "download.file", "head",
                 "read.csv", "read.table", "write.csv", "write.table")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘GeneMeta’
    ```

# cSEM

<details>

* Version: 0.3.0
* GitHub: https://github.com/M-E-Rademaker/cSEM
* Source code: https://github.com/cran/cSEM
* Date/Publication: 2020-10-12 16:40:03 UTC
* Number of recursive dependencies: 119

Run `revdep_details(, "cSEM")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘Rdpack’
      All declared Imports should be used.
    ```

# cytomapper

<details>

* Version: 1.2.0
* GitHub: https://github.com/BodenmillerGroup/cytomapper
* Source code: https://github.com/cran/cytomapper
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 111

Run `revdep_details(, "cytomapper")` for more info

</details>

## In both

*   checking tests ...
    ```
    ...
      Warning (test_shiny_reactive.R:74:5): Gates can be set
      Warning (test_shiny_reactive.R:129:5): Gates can be set
      Warning (test_shiny_reactive.R:129:5): Gates can be set
      Warning (test_shiny_ui.R:30:5): Sidebar is correctly rendered
      Warning (test_shiny_ui.R:30:5): Sidebar is correctly rendered
      Warning (test_shiny_ui.R:30:5): Sidebar is correctly rendered
      Warning (test_shiny_ui.R:30:5): Sidebar is correctly rendered
      Warning (test_shiny_ui.R:30:5): Sidebar is correctly rendered
      Warning (test_shiny_ui.R:30:5): Sidebar is correctly rendered
      Warning (test_shiny_ui.R:55:5): Plots in tab 1 are correctly rendered
      Warning (test_shiny_ui.R:55:5): Plots in tab 1 are correctly rendered
      Warning (test_shiny_ui.R:55:5): Plots in tab 1 are correctly rendered
      Warning (test_shiny_ui.R:55:5): Plots in tab 1 are correctly rendered
      Warning (test_shiny_ui.R:55:5): Plots in tab 1 are correctly rendered
      Warning (test_shiny_ui.R:55:5): Plots in tab 1 are correctly rendered
      
      [ FAIL 2 | WARN 17 | SKIP 0 | PASS 1316 ]
      Deleting unused snapshots:
      * _snaps/shiny_snapshots.md
      Error: Test failures
      Execution halted
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .BBSoptions
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.2Mb
      sub-directories of 1Mb or more:
        R     1.1Mb
        doc   3.8Mb
    ```

# CytoTree

<details>

* Version: 1.0.3
* GitHub: https://github.com/JhuangLab/CytoTree
* Source code: https://github.com/cran/CytoTree
* Date/Publication: 2020-11-08
* Number of recursive dependencies: 240

Run `revdep_details(, "CytoTree")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.7Mb
      sub-directories of 1Mb or more:
        doc       3.2Mb
        figures   1.2Mb
        libs      1.1Mb
    ```

# DatabionicSwarm

<details>

* Version: 1.1.3
* GitHub: https://github.com/Mthrun/DatabionicSwarm
* Source code: https://github.com/cran/DatabionicSwarm
* Date/Publication: 2020-02-03 14:00:02 UTC
* Number of recursive dependencies: 138

Run `revdep_details(, "DatabionicSwarm")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  9.7Mb
      sub-directories of 1Mb or more:
        doc    3.4Mb
        libs   6.1Mb
    ```

# DeepBlueR

<details>

* Version: 1.16.0
* GitHub: NA
* Source code: https://github.com/cran/DeepBlueR
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 160

Run `revdep_details(, "DeepBlueR")` for more info

</details>

## In both

*   checking Rd files ... NOTE
    ```
    prepare_Rd: deepblue_enrich_regions_fast.Rd:35-38: Dropping empty section \examples
    ```

# DelayedMatrixStats

<details>

* Version: 1.12.0
* GitHub: https://github.com/PeteHaitch/DelayedMatrixStats
* Source code: https://github.com/cran/DelayedMatrixStats
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 80

Run `revdep_details(, "DelayedMatrixStats")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘BiocParallel’ ‘HDF5Array’
      All declared Imports should be used.
    Unexported objects imported by ':::' calls:
      ‘DelayedArray:::.get_ans_type’
      ‘DelayedArray:::.reduce_array_dimensions’
      ‘DelayedArray:::RleArraySeed’ ‘DelayedArray:::get_Nindex_lengths’
      ‘DelayedArray:::set_dim’ ‘DelayedArray:::set_dimnames’
      ‘DelayedArray:::subset_by_Nindex’ ‘DelayedArray:::to_linear_index’
      See the note in ?`:::` about the use of this operator.
    ```

# DeMixT

<details>

* Version: 1.6.0
* GitHub: NA
* Source code: https://github.com/cran/DeMixT
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 78

Run `revdep_details(, "DeMixT")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      'parallel', 'Rcpp', 'SummarizedExperiment', 'knitr', 'KernSmooth',
      'matrixcalc'
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  7.3Mb
      sub-directories of 1Mb or more:
        data   3.8Mb
        libs   2.7Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    DeMixT_GS: no visible global function definition for ‘qchisq’
    simulate_2comp: no visible global function definition for ‘SimpleList’
    simulate_2comp: no visible global function definition for ‘DataFrame’
    simulate_3comp: no visible global function definition for ‘SimpleList’
    simulate_3comp: no visible global function definition for ‘DataFrame’
    Undefined global functions or variables:
      DataFrame SimpleList qchisq
    Consider adding
      importFrom("stats", "qchisq")
    to your NAMESPACE file.
    ```

*   checking for unstated dependencies in vignettes ... NOTE
    ```
    'library' or 'require' call not declared from: ‘calibrate’
    ```

# DepecheR

<details>

* Version: 1.6.0
* GitHub: NA
* Source code: https://github.com/cran/DepecheR
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 108

Run `revdep_details(, "DepecheR")` for more info

</details>

## In both

*   checking compiled code ... NOTE
    ```
    File ‘DepecheR/libs/DepecheR.so’:
      Found ‘rand’, possibly from ‘rand’ (C)
        Object: ‘Clusterer.o’
      Found ‘srand’, possibly from ‘srand’ (C)
        Objects: ‘Clusterer.o’, ‘InterfaceUtils.o’
    
    Compiled code should not call entry points which might terminate R nor
    write to stdout/stderr instead of to the console, nor use Fortran I/O
    nor system RNGs.
    
    See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
    ```

# DEqMS

<details>

* Version: 1.8.0
* GitHub: https://github.com/yafeng/DEqMS
* Source code: https://github.com/cran/DEqMS
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 117

Run `revdep_details(, "DEqMS")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    ...
    
    The following objects are masked from ‘package:base’:
    
        Filter, Find, Map, Position, Reduce, anyDuplicated, append,
        as.data.frame, basename, cbind, colnames, dirname, do.call,
        duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
        lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,
        pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,
        tapply, union, unique, unsplit, which.max, which.min
    
    Loading required package: AnnotationHub
    Loading required package: BiocFileCache
    Loading required package: dbplyr
    > eh = ExperimentHub(localHub=TRUE)
    using temporary cache /scratch/henrik/Rtmpzkp1pT/BiocFileCache
    Error in .updateHubDB(hub_bfc, .class, url, proxy, localHub) : 
      Invalid Cache: sqlite file
      Hub has not been added to cache
      Run again with 'localHub=FALSE'
    Calls: ExperimentHub -> .Hub -> .create_cache -> .updateHubDB
    Execution halted
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Title field: should not end in a period.
    ```

*   checking R code for possible problems ... NOTE
    ```
    Residualplot: no visible global function definition for ‘fitted’
    Residualplot: no visible global function definition for ‘residuals’
    VarianceScatterplot: no visible global function definition for ‘fitted’
    peptideProfilePlot: no visible binding for global variable ‘variable’
    peptideProfilePlot: no visible binding for global variable ‘value’
    peptideProfilePlot: no visible binding for global variable ‘PSM_id’
    peptideProfilePlot: no visible binding for global variable ‘Peptide’
    spectraCounteBayes: no visible global function definition for ‘fitted’
    Undefined global functions or variables:
      PSM_id Peptide fitted residuals value variable
    Consider adding
      importFrom("stats", "fitted", "residuals")
    to your NAMESPACE file.
    ```

# detrendr

<details>

* Version: 0.6.9
* GitHub: https://github.com/rorynolan/detrendr
* Source code: https://github.com/cran/detrendr
* Date/Publication: 2020-09-03 05:20:03 UTC
* Number of recursive dependencies: 105

Run `revdep_details(, "detrendr")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  8.1Mb
      sub-directories of 1Mb or more:
        libs   7.0Mb
    ```

*   checking for GNU extensions in Makefiles ... NOTE
    ```
    GNU make is a SystemRequirements.
    ```

# DHS.rates

<details>

* Version: 0.8.0
* GitHub: NA
* Source code: https://github.com/cran/DHS.rates
* Date/Publication: 2020-10-11 23:30:06 UTC
* Number of recursive dependencies: 53

Run `revdep_details(, "DHS.rates")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘rlang’
      All declared Imports should be used.
    ```

# diffloop

<details>

* Version: 1.18.0
* GitHub: https://github.com/aryeelab/diffloop
* Source code: https://github.com/cran/diffloop
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 139

Run `revdep_details(, "diffloop")` for more info

</details>

## In both

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

# DiscoRhythm

<details>

* Version: 1.6.0
* GitHub: https://github.com/matthewcarlucci/DiscoRhythm
* Source code: https://github.com/cran/DiscoRhythm
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 155

Run `revdep_details(, "DiscoRhythm")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    ...
      ordinary text without R code
    
    label: unnamed-chunk-5 (with options) 
    List of 2
     $ code: language readLines(batchscript)
     $ eval: logi FALSE
    
       inline R code fragments
    
    label: unnamed-chunk-6
      ordinary text without R code
    
    label: unnamed-chunk-7
      ordinary text without R code
    
    label: unnamed-chunk-8
    Quitting from lines 116-118 (DiscoRhythm_report.Rmd) 
    Error in .External2(C_X11, paste0("png::", filename), g$width, g$height,  : 
      unable to start device PNG
    Calls: discoBatch ... <Anonymous> -> ggplotly.ggplot -> gg2list -> dev_fun
    Execution halted
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘data.table’
      All declared Imports should be used.
    ```

*   checking R code for possible problems ... NOTE
    ```
    discoApp: no visible binding for global variable ‘.discorhythm_ncores’
    discoApp: no visible binding for global variable ‘.discorhythm_local’
    discoBatch: no visible binding for global variable ‘discoODAres’
    Undefined global functions or variables:
      .discorhythm_local .discorhythm_ncores discoODAres
    ```

# dmrseq

<details>

* Version: 1.10.0
* GitHub: NA
* Source code: https://github.com/cran/dmrseq
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 164

Run `revdep_details(, "dmrseq")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    ...
    Error in get(txdb_name) : 
      object 'TxDb.Hsapiens.UCSC.hg19.knownGene' not found
    
    Trying again (4 attempts remaining)
    Error in get(txdb_name) : 
      object 'TxDb.Hsapiens.UCSC.hg19.knownGene' not found
    
    Trying again (3 attempts remaining)
    Error in get(txdb_name) : 
      object 'TxDb.Hsapiens.UCSC.hg19.knownGene' not found
    
    Trying again (2 attempts remaining)
    Error in get(txdb_name) : 
      object 'TxDb.Hsapiens.UCSC.hg19.knownGene' not found
    
    Trying again (1 attempts remaining)
    Error in get(txdb_name) : 
      object 'TxDb.Hsapiens.UCSC.hg19.knownGene' not found
    
    Error in getAnnot("hg19") : Annotation could not be retrieved.
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 10.3Mb
      sub-directories of 1Mb or more:
        data   8.8Mb
        doc    1.3Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    plotDMRs: warning in rank(as.numeric(rowSums(cov.unique)), ties =
      "first"): partial argument match of 'ties' to 'ties.method'
    plotEmpiricalDistribution: no visible binding for global variable ‘wt’
    Undefined global functions or variables:
      wt
    ```

# Doscheda

<details>

* Version: 1.12.0
* GitHub: NA
* Source code: https://github.com/cran/Doscheda
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 161

Run `revdep_details(, "Doscheda")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.9Mb
      sub-directories of 1Mb or more:
        data             2.1Mb
        doc              1.6Mb
        shiny-examples   1.9Mb
    ```

# dplR

<details>

* Version: 1.7.1
* GitHub: https://github.com/AndyBunn/dplR
* Source code: https://github.com/cran/dplR
* Date/Publication: 2020-03-19 13:20:02 UTC
* Number of recursive dependencies: 91

Run `revdep_details(, "dplR")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    ...
    > nrad <- 16
    > minrad <- maxrad / nrad
    > Rad <- seq(maxrad, by = (minrad - maxrad) / (nrad - 1), length.out=nrad)
    > xmar2 <- xmar + maxrad
    > ymar2 <- (xmar2 / xrange) * pin[1] / pin[2] * yrange
    > expr <- quote({
    +     rect(xleft, usr[4] - 1.5 * ysize, xright, usr[4] - ymar,
    +          col = rainbow(8), border = NA)
    +     symbols(rep(usr[2] - xmar2, nrad), rep(usr[3] + ymar2, nrad),
    +             circles = Rad, inches = FALSE, add = TRUE, fg = NA,
    +             bg = gray.colors(nrad + 1, 1, 0)[-1])
    +     points(y)
    +     lines(spl)
    + })
    > rasterPlot(expr, res = 50)
    Warning in png(fname, width = pngWidthHeight[1], height = pngWidthHeight[2],  :
      unable to open connection to X11 display ''
    Error in .External2(C_X11, paste0("png::", filename), g$width, g$height,  : 
      unable to start device PNG
    Calls: rasterPlot -> png
    Execution halted
    ```

# EasyqpcR

<details>

* Version: 1.31.0
* GitHub: NA
* Source code: https://github.com/cran/EasyqpcR
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 61

Run `revdep_details(, "EasyqpcR")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘gWidgetsRGtk2’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

*   checking package namespace information ... NOTE
    ```
      Namespace with empty importFrom: ‘gWidgetsRGtk2’
    ```

# ecospat

<details>

* Version: 3.1
* GitHub: https://github.com/ecospat/ecospat
* Source code: https://github.com/cran/ecospat
* Date/Publication: 2020-03-25 15:20:03 UTC
* Number of recursive dependencies: 177

Run `revdep_details(, "ecospat")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘maps’
      All declared Imports should be used.
    ```

# EMDomics

<details>

* Version: 2.20.0
* GitHub: NA
* Source code: https://github.com/cran/EMDomics
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 68

Run `revdep_details(, "EMDomics")` for more info

</details>

## In both

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Authors@R field gives more than one person with maintainer role:
      Sadhika Malladi <contact@sadhikamalladi.com> [aut, cre]
      Daniel Schmolze <emd@schmolze.com> [aut, cre]
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    calculate_cvm : <anonymous>: no visible global function definition for
      ‘median’
    calculate_cvm_gene: no visible global function definition for ‘combn’
    calculate_emd: no visible global function definition for ‘combn’
    calculate_emd : <anonymous>: no visible global function definition for
      ‘median’
    calculate_emd_gene: no visible global function definition for ‘combn’
    calculate_ks: no visible global function definition for ‘combn’
    calculate_ks : <anonymous>: no visible global function definition for
      ‘p.adjust’
    calculate_ks : <anonymous>: no visible global function definition for
      ‘median’
    calculate_ks_gene: no visible global function definition for ‘combn’
    calculate_ks_gene: no visible global function definition for ‘ks.test’
    Undefined global functions or variables:
      combn hist ks.test median p.adjust
    Consider adding
      importFrom("graphics", "hist")
      importFrom("stats", "ks.test", "median", "p.adjust")
      importFrom("utils", "combn")
    to your NAMESPACE file.
    ```

# EnrichedHeatmap

<details>

* Version: 1.20.0
* GitHub: https://github.com/jokergoo/EnrichedHeatmap
* Source code: https://github.com/cran/EnrichedHeatmap
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 89

Run `revdep_details(, "EnrichedHeatmap")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  9.6Mb
      sub-directories of 1Mb or more:
        doc       5.9Mb
        extdata   2.5Mb
    ```

# evaluomeR

<details>

* Version: 1.6.0
* GitHub: https://github.com/neobernad/evaluomeR
* Source code: https://github.com/cran/evaluomeR
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 117

Run `revdep_details(, "evaluomeR")` for more info

</details>

## In both

*   checking top-level files ... NOTE
    ```
    File
      LICENSE
    is not mentioned in the DESCRIPTION file.
    ```

# EventPointer

<details>

* Version: 2.8.0
* GitHub: https://github.com/jpromeror/EventPointer
* Source code: https://github.com/cran/EventPointer
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 154

Run `revdep_details(, "EventPointer")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      ‘SGSeq:::addFeatureID’ ‘SGSeq:::addGeneID’ ‘SGSeq:::edges’
      ‘SGSeq:::exonGraph’ ‘SGSeq:::feature2name’ ‘SGSeq:::matchSGFeatures’
      ‘SGSeq:::nodes’ ‘SGSeq:::propagateAnnotation’
      ‘SGSeq:::splitCharacterList’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    annotate2: no visible global function definition for
      ‘addDummySpliceSites’
    annotate2: no visible global function definition for ‘annotatePaths’
    annotateFeatures2: no visible global function definition for
      ‘matchTxFeatures’
    Undefined global functions or variables:
      addDummySpliceSites annotatePaths matchTxFeatures
    ```

# ExCluster

<details>

* Version: 1.8.0
* GitHub: NA
* Source code: https://github.com/cran/ExCluster
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 32

Run `revdep_details(, "ExCluster")` for more info

</details>

## In both

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .BBSoptions
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

# ExpressionNormalizationWorkflow

<details>

* Version: 1.16.0
* GitHub: https://github.com/NA/NA
* Source code: https://github.com/cran/ExpressionNormalizationWorkflow
* Date/Publication: 2020-10-30
* Number of recursive dependencies: 113

Run `revdep_details(, "ExpressionNormalizationWorkflow")` for more info

</details>

## In both

*   checking whether package ‘ExpressionNormalizationWorkflow’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: /c4/home/henrik/repositories/matrixStats/revdep/checks/ExpressionNormalizationWorkflow/new/ExpressionNormalizationWorkflow.Rcheck/00_pkg_src/ExpressionNormalizationWorkflow/man/ExpressionNormalizationWorkflow-package.Rd:29: All text must be in a section
      Warning: /c4/home/henrik/repositories/matrixStats/revdep/checks/ExpressionNormalizationWorkflow/new/ExpressionNormalizationWorkflow.Rcheck/00_pkg_src/ExpressionNormalizationWorkflow/man/ExpressionNormalizationWorkflow-package.Rd:30: All text must be in a section
      Warning: replacing previous import ‘Biobase::anyMissing’ by ‘matrixStats::anyMissing’ when loading ‘ExpressionNormalizationWorkflow’
      Warning: replacing previous import ‘Biobase::rowMedians’ by ‘matrixStats::rowMedians’ when loading ‘ExpressionNormalizationWorkflow’
    See ‘/c4/home/henrik/repositories/matrixStats/revdep/checks/ExpressionNormalizationWorkflow/new/ExpressionNormalizationWorkflow.Rcheck/00install.out’ for details.
    ```

*   checking Rd files ... WARNING
    ```
    prepare_Rd: ExpressionNormalizationWorkflow-package.Rd:29: All text must be in a section
    prepare_Rd: ExpressionNormalizationWorkflow-package.Rd:30: All text must be in a section
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  7.2Mb
      sub-directories of 1Mb or more:
        doc       2.5Mb
        extdata   4.7Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    expSetobj: no visible global function definition for ‘new’
    pvcAnaly: no visible global function definition for ‘barplot’
    pvcAnaly: no visible global function definition for ‘axis’
    pvcAnaly: no visible global function definition for ‘text’
    pvcaBatchAssess: no visible global function definition for ‘cor’
    pvcaBatchAssess: no visible binding for global variable ‘na.omit’
    pvcaBatchAssess: no visible global function definition for ‘sigma’
    snmAnaly: no visible global function definition for ‘model.matrix’
    snmAnaly: no visible global function definition for ‘write.table’
    surVarAnaly: no visible global function definition for ‘model.matrix’
    Undefined global functions or variables:
      axis barplot cor model.matrix na.omit new sigma text write.table
    Consider adding
      importFrom("graphics", "axis", "barplot", "text")
      importFrom("methods", "new")
      importFrom("stats", "cor", "model.matrix", "na.omit", "sigma")
      importFrom("utils", "write.table")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

*   checking Rd cross-references ... NOTE
    ```
    Unknown package ‘<pkg>’ in Rd xrefs
    ```

*   checking for unstated dependencies in vignettes ... NOTE
    ```
    '::' or ':::' import not declared from: ‘BiocManager’
    ```

# FastHCS

<details>

* Version: 0.0.7
* GitHub: NA
* Source code: https://github.com/cran/FastHCS
* Date/Publication: 2020-05-10 12:50:03 UTC
* Number of recursive dependencies: 5

Run `revdep_details(, "FastHCS")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 19.6Mb
      sub-directories of 1Mb or more:
        data   1.3Mb
        libs  18.3Mb
    ```

# FastPCS

<details>

* Version: 0.1.3
* GitHub: NA
* Source code: https://github.com/cran/FastPCS
* Date/Publication: 2018-05-22 16:49:12 UTC
* Number of recursive dependencies: 6

Run `revdep_details(, "FastPCS")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 10.0Mb
      sub-directories of 1Mb or more:
        libs  10.0Mb
    ```

# FastRCS

<details>

* Version: 0.0.8
* GitHub: NA
* Source code: https://github.com/cran/FastRCS
* Date/Publication: 2018-05-13 19:59:51 UTC
* Number of recursive dependencies: 6

Run `revdep_details(, "FastRCS")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 21.6Mb
      sub-directories of 1Mb or more:
        libs  21.6Mb
    ```

# FDb.FANTOM4.promoters.hg19

<details>

* Version: 1.0.0
* GitHub: NA
* Source code: https://github.com/cran/FDb.FANTOM4.promoters.hg19
* Number of recursive dependencies: 80

Run `revdep_details(, "FDb.FANTOM4.promoters.hg19")` for more info

</details>

## In both

*   checking package subdirectories ... WARNING
    ```
    Found the following non-empty subdirectories of ‘inst’ also used by R:
      inst/build
    It is recommended not to interfere with package subdirectories used by
    R.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    Package listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘GenomicFeatures’
    A package should be listed in only one of these fields.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' calls in package code:
      ‘BSgenome.Hsapiens.UCSC.hg19’ ‘matrixStats’
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    Package in Depends field not imported from: ‘Biostrings’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ':::' call which should be '::': ‘matrixStats:::rowProds’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    oecg: no visible global function definition for ‘seqlengths’
    oecg: no visible binding for global variable ‘Hsapiens’
    oecg: no visible global function definition for ‘seqlevels’
    oecg: no visible global function definition for ‘seqnames’
    oecg: no visible global function definition for ‘resize’
    oecg: no visible global function definition for ‘start’
    oecg: no visible global function definition for ‘start<-’
    oecg: no visible global function definition for ‘end’
    oecg: no visible global function definition for ‘end<-’
    oecg: no visible global function definition for ‘getSeq’
    oecg: no visible global function definition for ‘width’
    oecg: no visible global function definition for ‘letterFrequency’
    oecg: no visible global function definition for ‘dinucleotideFrequency’
    Undefined global functions or variables:
      Hsapiens dinucleotideFrequency end end<- getSeq is letterFrequency
      resize seqlengths seqlevels seqnames start start<- width
    Consider adding
      importFrom("methods", "is")
      importFrom("stats", "end", "start")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# fishpond

<details>

* Version: 1.6.0
* GitHub: https://github.com/mikelove/fishpond
* Source code: https://github.com/cran/fishpond
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 169

Run `revdep_details(, "fishpond")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      ‘samr:::localfdr’ ‘samr:::predictlocalfdr’ ‘samr:::qvalue.func’
      ‘samr:::samr.compute.delta.table.seq’
      See the note in ?`:::` about the use of this operator.
    ```

# flowCore

<details>

* Version: 2.2.0
* GitHub: NA
* Source code: https://github.com/cran/flowCore
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 124

Run `revdep_details(, "flowCore")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 13.8Mb
      sub-directories of 1Mb or more:
        R         1.1Mb
        data      5.0Mb
        extdata   1.1Mb
        libs      5.4Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    eval,compensatedParameter-missing : <anonymous>: no visible binding for
      global variable ‘mat’
    eval,compensatedParameter-missing : <anonymous>: no visible binding for
      global variable ‘msv’
    show,flowFrame: no visible global function definition for
      ‘capture.output’
    Undefined global functions or variables:
      capture.output mat msv
    Consider adding
      importFrom("utils", "capture.output")
    to your NAMESPACE file.
    ```

*   checking compiled code ... NOTE
    ```
    File ‘flowCore/libs/flowCore.so’:
      Found ‘_ZSt4cerr’, possibly from ‘std::cerr’ (C++)
        Objects: ‘RcppExports.o’, ‘fcsTextParse.o’
    
    Compiled code should not call entry points which might terminate R nor
    write to stdout/stderr instead of to the console, nor use Fortran I/O
    nor system RNGs.
    
    See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
    ```

# flowSpy

<details>

* Version: 1.4.0
* GitHub: https://github.com/JhuangLab/flowSpy
* Source code: https://github.com/cran/flowSpy
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 240

Run `revdep_details(, "flowSpy")` for more info

</details>

## In both

*   checking whether package ‘flowSpy’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: Package 'flowSpy' is deprecated and will be removed from Bioconductor
    See ‘/c4/home/henrik/repositories/matrixStats/revdep/checks/flowSpy/new/flowSpy.Rcheck/00install.out’ for details.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.8Mb
      sub-directories of 1Mb or more:
        doc    3.2Mb
        libs   1.1Mb
    ```

# flowWorkspace

<details>

* Version: 4.2.0
* GitHub: NA
* Source code: https://github.com/cran/flowWorkspace
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 126

Run `revdep_details(, "flowWorkspace")` for more info

</details>

## In both

*   checking for missing documentation entries ... WARNING
    ```
    Undocumented code objects:
      ‘gs_clone’ ‘gs_copy_tree_only’
    Undocumented S4 methods:
      generic 'show' and siglist 'cytoframe'
      generic 'transform' and siglist 'cytoframe'
    All user-level objects in a package (including S4 classes and methods)
    should have documentation entries.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking Rd \usage sections ... WARNING
    ```
    ...
    
    Undocumented arguments in documentation object 'keyword-mutators'
      ‘cs’
    
    Undocumented arguments in documentation object 'load_cytoframe'
      ‘num_threads’
    
    Undocumented arguments in documentation object 'load_cytoframe_from_fcs'
      ‘backend’ ‘uri’
    
    Undocumented arguments in documentation object 'load_cytoset_from_fcs'
      ‘backend’ ‘backend_dir’
    
    Undocumented arguments in documentation object 'save_gs'
      ‘cdf’ ‘backend_readonly’
    
    Functions with \usage entries need to have the appropriate \alias
    entries, and all their arguments documented.
    The \usage entries must correspond to syntactically valid R code.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .github
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 92.5Mb
      sub-directories of 1Mb or more:
        doc    3.4Mb
        lib   55.0Mb
        libs  32.8Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Title field: should not end in a period.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘RBGL’ ‘RProtoBufLib’ ‘ggplot2’ ‘grDevices’ ‘graphics’ ‘utils’
      All declared Imports should be used.
    Unexported objects imported by ':::' calls:
      ‘flowCore:::.estimateLogicle’ ‘flowCore:::.spillover_pattern’
      ‘flowCore:::checkClass’ ‘flowCore:::guid’
      ‘flowCore:::logicle_transform’ ‘flowCore:::parse_pd_for_read_fs’
      ‘flowCore:::txt2spillmatrix’ ‘flowCore:::updateTransformKeywords’
      ‘flowCore:::validFilterResultList’ ‘graph:::.makeEdgeKeys’
      ‘ncdfFlow:::.isValidSamples’ ‘stats:::.splinefun’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    setNode,GatingSet-character-ANY: no visible global function definition
      for ‘is’
    show,cytoframe: no visible global function definition for
      ‘selectMethod’
    transform,GatingSet: no visible global function definition for ‘is’
    transform,GatingSet : <anonymous>: no visible global function
      definition for ‘is’
    transform,cytoframe: no visible global function definition for ‘is’
    transform,cytoset: no visible global function definition for ‘is’
    transform,cytoset : <anonymous>: no visible global function definition
      for ‘is’
    Undefined global functions or variables:
      . .hasSlot IQR Population as callNextMethod count desc extends is
      keys kwdError median na_idx new node obj old openCyto.count output
      parallel percent sampleName selectMethod validObject xml.count
    Consider adding
      importFrom("methods", ".hasSlot", "as", "callNextMethod", "extends",
                 "is", "new", "selectMethod", "validObject")
      importFrom("stats", "IQR", "median")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

*   checking for GNU extensions in Makefiles ... NOTE
    ```
    GNU make is a SystemRequirements.
    ```

*   checking compiled code ... NOTE
    ```
    File ‘flowWorkspace/libs/flowWorkspace.so’:
      Found ‘_ZSt4cerr’, possibly from ‘std::cerr’ (C++)
        Objects: ‘R_GatingSet.o’, ‘RcppExports.o’, ‘cytoframeAPI.o’,
          ‘cytosetAPI.o’
    
    Compiled code should not call entry points which might terminate R nor
    write to stdout/stderr instead of to the console, nor use Fortran I/O
    nor system RNGs.
    
    See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
    ```

# FRASER

<details>

* Version: 1.2.0
* GitHub: https://github.com/gagneurlab/FRASER
* Source code: https://github.com/cran/FRASER
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 183

Run `revdep_details(, "FRASER")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  8.3Mb
      sub-directories of 1Mb or more:
        R      1.3Mb
        doc    1.5Mb
        libs   4.8Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported object imported by a ':::' call: ‘S4Vectors:::selectSome’
      See the note in ?`:::` about the use of this operator.
    ```

# funtooNorm

<details>

* Version: 1.14.0
* GitHub: NA
* Source code: https://github.com/cran/funtooNorm
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 133

Run `revdep_details(, "funtooNorm")` for more info

</details>

## In both

*   checking Rd \usage sections ... WARNING
    ```
    Documented arguments not in \usage in documentation object 'show,SampleSet-method':
      ‘...’
    
    Functions with \usage entries need to have the appropriate \alias
    entries, and all their arguments documented.
    The \usage entries must correspond to syntactically valid R code.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

# gap

<details>

* Version: 1.2.2
* GitHub: https://github.com/jinghuazhao/R
* Source code: https://github.com/cran/gap
* Date/Publication: 2020-02-02 15:10:02 UTC
* Number of recursive dependencies: 147

Run `revdep_details(, "gap")` for more info

</details>

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘genetics’
    ```

# GenEst

<details>

* Version: 1.4.4
* GitHub: NA
* Source code: https://github.com/cran/GenEst
* Date/Publication: 2020-06-08 18:20:03 UTC
* Number of recursive dependencies: 86

Run `revdep_details(, "GenEst")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘htmlwidgets’
      All declared Imports should be used.
    ```

# GeneTonic

<details>

* Version: 1.2.0
* GitHub: https://github.com/federicomarini/GeneTonic
* Source code: https://github.com/cran/GeneTonic
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 176

Run `revdep_details(, "GeneTonic")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 16.5Mb
      sub-directories of 1Mb or more:
        data      1.2Mb
        doc      13.4Mb
        extdata   1.4Mb
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘pcaExplorer’
    ```

# genomation

<details>

* Version: 1.22.0
* GitHub: https://github.com/BIMSBbioinfo/genomation
* Source code: https://github.com/cran/genomation
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 105

Run `revdep_details(, "genomation")` for more info

</details>

## In both

*   checking examples ... WARNING
    ```
    Found the following significant warnings:
    
      Warning: 'GenomicRangesList' is deprecated.
    Deprecated functions may be defunct as soon as of the next release of
    R.
    See ?Deprecated.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  7.4Mb
      sub-directories of 1Mb or more:
        doc       3.5Mb
        extdata   1.2Mb
        libs      1.6Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘RUnit’
      All declared Imports should be used.
    Unexported object imported by a ':::' call: ‘BiocGenerics:::testPackage’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ScoreMatrixBin,RleList-GRangesList: no visible binding for global
      variable ‘id’
    ScoreMatrixBin,RleList-GRangesList: no visible global function
      definition for ‘:=’
    Undefined global functions or variables:
      := id
    ```

*   checking files in ‘vignettes’ ... NOTE
    ```
    The following directory looks like a leftover from 'knitr':
      ‘cache’
    Please remove from your package.
    ```

# ggdmc

<details>

* Version: 0.2.6.0
* GitHub: https://github.com/yxlin/ggdmc
* Source code: https://github.com/cran/ggdmc
* Date/Publication: 2019-04-29 05:10:03 UTC
* Number of recursive dependencies: 54

Run `revdep_details(, "ggdmc")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.3Mb
      sub-directories of 1Mb or more:
        libs   7.0Mb
    ```

# GGPA

<details>

* Version: 1.2.0
* GitHub: https://github.com/dongjunchung/GGPA
* Source code: https://github.com/cran/GGPA
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 77

Run `revdep_details(, "GGPA")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.5Mb
      sub-directories of 1Mb or more:
        doc    1.6Mb
        libs   3.7Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Packages in Depends field not imported from:
      ‘network’ ‘scales’ ‘sna’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking compiled code ... NOTE
    ```
    File ‘GGPA/libs/GGPA.so’:
      Found ‘_ZSt4cout’, possibly from ‘std::cout’ (C++)
        Object: ‘3_Param.o’
    
    Compiled code should not call entry points which might terminate R nor
    write to stdout/stderr instead of to the console, nor use Fortran I/O
    nor system RNGs.
    
    See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
    ```

# GJRM

<details>

* Version: 0.2-3
* GitHub: NA
* Source code: https://github.com/cran/GJRM
* Date/Publication: 2020-09-07 18:20:03 UTC
* Number of recursive dependencies: 85

Run `revdep_details(, "GJRM")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package which this enhances but not available for checking: ‘sp’
    ```

# glmGamPoi

<details>

* Version: 1.2.0
* GitHub: https://github.com/const-ae/glmGamPoi
* Source code: https://github.com/cran/glmGamPoi
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 158

Run `revdep_details(, "glmGamPoi")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  8.6Mb
      sub-directories of 1Mb or more:
        libs   7.6Mb
    ```

# GNET2

<details>

* Version: 1.6.0
* GitHub: https://github.com/chrischen1/GNET2
* Source code: https://github.com/cran/GNET2
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 99

Run `revdep_details(, "GNET2")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    ...
    > reg_names <- paste0('TF',1:5)
    > rownames(exp_data) <- c(reg_names,paste0('gene',1:(nrow(exp_data)-length(reg_names))))
    > colnames(exp_data) <- paste0('condition_',1:ncol(exp_data))
    > se <- SummarizedExperiment::SummarizedExperiment(assays=list(counts=exp_data))
    > gnet_result <- gnet(se,reg_names,init_method,init_group_num)
    Determining initial group number...
    Building module networks...
    Iteration 1
    Iteration 2
    Iteration 3
    Iteration 4
    Converged.
    Generating final network modules...
    Done.
    > save_gnet(gnet_result)
    Warning in tiff(paste0(save_path, "/module_", i, ".tiff"), compression = "lzw") :
      unable to open connection to X11 display ''
    Error in .External2(C_X11, paste0("tiff::", comp, ":", filename), g$width,  : 
      unable to start device TIFF
    Calls: save_gnet -> tiff
    Execution halted
    ```

# GPrank

<details>

* Version: 0.1.4
* GitHub: https://github.com/PROBIC/GPrank
* Source code: https://github.com/cran/GPrank
* Date/Publication: 2018-08-17 15:00:03 UTC
* Number of recursive dependencies: 40

Run `revdep_details(, "GPrank")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘GPrank-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: bitseq_setPlot
    > ### Title: Configuring the settings of the plots for the BitSeq example
    > ### Aliases: bitseq_setPlot
    > ### Keywords: internal plot
    > 
    > ### ** Examples
    > 
    > bitseq_setPlot("geneA_GPprofile.pdf")
    > bitseq_setPlot("geneA_GPprofile.png")
    Warning in png(filename = plotName) :
      unable to open connection to X11 display ''
    Error in .External2(C_X11, paste0("png::", filename), g$width, g$height,  : 
      unable to start device PNG
    Calls: bitseq_setPlot -> png
    Execution halted
    ```

# graper

<details>

* Version: 1.6.0
* GitHub: NA
* Source code: https://github.com/cran/graper
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 69

Run `revdep_details(, "graper")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.5Mb
      sub-directories of 1Mb or more:
        doc    1.8Mb
        libs   5.6Mb
    ```

# GUIDEseq

<details>

* Version: 1.20.0
* GitHub: NA
* Source code: https://github.com/cran/GUIDEseq
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 149

Run `revdep_details(, "GUIDEseq")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 11.0Mb
      sub-directories of 1Mb or more:
        extdata  10.5Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    getUniqueCleavageEvents: no visible binding for global variable
      'qwidth.first'
    getUniqueCleavageEvents: no visible binding for global variable
      'qwidth.last'
    getUniqueCleavageEvents: no visible binding for global variable
      'strand.last'
    getUniqueCleavageEvents: no visible binding for global variable
      'strand.first'
    getUniqueCleavageEvents: no visible binding for global variable
      'readName'
    getUniqueCleavageEvents: no visible binding for global variable 'UMI'
    offTargetAnalysisOfPeakRegions: no visible binding for global variable
      'thePeak'
    offTargetAnalysisOfPeakRegions: no visible binding for global variable
      'gRNAPlusPAM'
    offTargetAnalysisOfPeakRegions: no visible binding for global variable
      'offTarget'
    Undefined global functions or variables:
      SNratio UMI adjusted.p.value exons gRNAPlusPAM genes offTarget
      offTarget_Start qwidth.first qwidth.last readName seqlevelsStyle<-
      strand.first strand.last thePeak
    ```

*   checking Rd files ... NOTE
    ```
    prepare_Rd: annotateOffTargets.Rd:33-35: Dropping empty section \details
    prepare_Rd: annotateOffTargets.Rd:43-45: Dropping empty section \references
    prepare_Rd: createBarcodeFasta.Rd:42-43: Dropping empty section \value
    prepare_Rd: createBarcodeFasta.Rd:44-46: Dropping empty section \references
    prepare_Rd: getUsedBarcodes.Rd:39-41: Dropping empty section \references
    ```

# Gviz

<details>

* Version: 1.34.0
* GitHub: https://github.com/ivanek/Gviz
* Source code: https://github.com/cran/Gviz
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 144

Run `revdep_details(, "Gviz")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 10.4Mb
      sub-directories of 1Mb or more:
        R         1.7Mb
        doc       4.7Mb
        extdata   2.5Mb
    ```

# GWENA

<details>

* Version: 1.0.0
* GitHub: https://github.com/Kumquatum/GWENA
* Source code: https://github.com/cran/GWENA
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 143

Run `revdep_details(, "GWENA")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    ...
    > ### Aliases: plot_enrichment
    > 
    > ### ** Examples
    > 
    > custom_path <- system.file("extdata", "h.all.v6.2.symbols.gmt",
    +                            package = "GWENA", mustWork = TRUE)
    > multi_module <- list(mod1 = c("BIRC3", "PMAIP1", "CASP8", "JUN", "BCL2L11",
    +                               "MCL1", "IL1B", "SPTAN1", "DIABLO", "BAX",
    +                               "BIK", "IL1A", "BID", "CDKN1A", "GADD45A"),
    +                      mod2 = c("TAF1C", "TARBP2", "POLH", "CETN2", "POLD1",
    +                               "CANT1", "PDE4B", "DGCR8", "RAD51", "SURF1",
    +                               "PNP", "ADA", "NME3", "GTF3C5", "NT5C"))
    > multi_module_enriched <- bio_enrich(multi_module, custom_path)
    > plot_enrichment(multi_module_enriched)
    Error in .External2(C_X11, paste0("png::", filename), g$width, g$height,  : 
      unable to start device PNG
    Calls: plot_enrichment ... <Anonymous> -> ggplotly.ggplot -> gg2list -> dev_fun
    In addition: Warning message:
    In dev_fun(file = tempfile(), width = width %||% 640, height = height %||%  :
      unable to open connection to X11 display ''
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 10.3Mb
      sub-directories of 1Mb or more:
        data   5.6Mb
        doc    4.5Mb
    ```

# haploReconstruct

<details>

* Version: 0.1.2
* GitHub: NA
* Source code: https://github.com/cran/haploReconstruct
* Date/Publication: 2016-10-10 18:18:49
* Number of recursive dependencies: 19

Run `revdep_details(, "haploReconstruct")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  9.8Mb
      sub-directories of 1Mb or more:
        data   9.4Mb
    ```

# hipathia

<details>

* Version: 2.6.0
* GitHub: NA
* Source code: https://github.com/cran/hipathia
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 121

Run `revdep_details(, "hipathia")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 11.2Mb
      sub-directories of 1Mb or more:
        data      4.9Mb
        doc       1.7Mb
        extdata   4.3Mb
    ```

# InfiniumPurify

<details>

* Version: 1.3.1
* GitHub: NA
* Source code: https://github.com/cran/InfiniumPurify
* Date/Publication: 2017-01-14 12:12:25
* Number of recursive dependencies: 1

Run `revdep_details(, "InfiniumPurify")` for more info

</details>

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘minfi’
    ```

# kissDE

<details>

* Version: 1.10.0
* GitHub: NA
* Source code: https://github.com/cran/kissDE
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 139

Run `revdep_details(, "kissDE")` for more info

</details>

## In both

*   checking tests ...
    ```
    ...
      Warning (test_diffExpressedVariants_KS.R:7:3): diffExpressedVariants function works as expected on kissplice fasta file
      Warning (test_diffExpressedVariants_counts.R:7:3): diffExpressedVariants function works as expected on count file
      Warning (test_diffExpressedVariants_counts.R:7:3): diffExpressedVariants function works as expected on count file
      Warning (test_diffExpressedVariants_counts.R:27:5): diffExpressedVariants function works as expected on count file with 2 cores
      Warning (test_diffExpressedVariants_counts.R:27:5): diffExpressedVariants function works as expected on count file with 2 cores
      Warning (test_diffExpressedVariants_counts.R:45:3): pvalue parameter of diffExpressedVariants function works as expected
      Warning (test_diffExpressedVariants_counts.R:45:3): pvalue parameter of diffExpressedVariants function works as expected
      Warning (test_qualityControl.R:7:3): qualityControl work as expected
      Warning (test_qualityControl.R:7:3): qualityControl work as expected
      Warning (test_qualityControl.R:11:3): qualityControl work as expected
      Warning (test_qualityControl.R:11:3): qualityControl work as expected
      Warning (test_qualityControl.R:11:3): qualityControl work as expected
      ERROR (test_qualityControl.R:11:3): qualityControl work as expected
      Warning (test_writeOutputKissDE.R:7:3): writeOutputKissDE work as expected
      Warning (test_writeOutputKissDE.R:7:3): writeOutputKissDE work as expected
      Warning (test_writeOutputKissDE.R:16:3): writeOutputKissDE work as expected
      Warning (test_writeOutputKissDE.R:16:3): writeOutputKissDE work as expected
      
      [ FAIL 1 | WARN 19 | SKIP 0 | PASS 64 ]
      Error: Test failures
      Execution halted
    ```

# landmap

<details>

* Version: 0.0.6
* GitHub: https://github.com/envirometrix/landmap
* Source code: https://github.com/cran/landmap
* Date/Publication: 2020-11-18 23:20:14 UTC
* Number of recursive dependencies: 142

Run `revdep_details(, "landmap")` for more info

</details>

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 23 marked UTF-8 strings
    ```

# latrend

<details>

* Version: 1.0.1
* GitHub: https://github.com/philips-software/latrend
* Source code: https://github.com/cran/latrend
* Date/Publication: 2020-11-18 10:10:02 UTC
* Number of recursive dependencies: 180

Run `revdep_details(, "latrend")` for more info

</details>

## In both

*   checking whether package ‘latrend’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: no DISPLAY variable so Tk is not available
    See ‘/c4/home/henrik/repositories/matrixStats/revdep/checks/latrend/new/latrend.Rcheck/00install.out’ for details.
    ```

# liger

<details>

* Version: 1.1.2
* GitHub: https://github.com/JEFworks/liger
* Source code: https://github.com/cran/liger
* Date/Publication: 2020-07-16 15:10:07 UTC
* Number of recursive dependencies: 49

Run `revdep_details(, "liger")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.9Mb
      sub-directories of 1Mb or more:
        data   3.3Mb
        doc    2.1Mb
        libs   1.4Mb
    ```

# LSAmitR

<details>

* Version: 1.0-2
* GitHub: NA
* Source code: https://github.com/cran/LSAmitR
* Date/Publication: 2018-06-08 13:23:05 UTC
* Number of recursive dependencies: 167

Run `revdep_details(, "LSAmitR")` for more info

</details>

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 55 marked UTF-8 strings
    ```

# Luminescence

<details>

* Version: 0.9.8
* GitHub: https://github.com/R-Lum/Luminescence
* Source code: https://github.com/cran/Luminescence
* Date/Publication: 2020-11-16 13:00:06 UTC
* Number of recursive dependencies: 146

Run `revdep_details(, "Luminescence")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘RLumShiny’
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  7.1Mb
      sub-directories of 1Mb or more:
        R      1.6Mb
        help   1.1Mb
        libs   3.3Mb
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘RLumShiny’
    ```

# M3Drop

<details>

* Version: 1.16.0
* GitHub: https://github.com/tallulandrews/M3Drop
* Source code: https://github.com/cran/M3Drop
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 215

Run `revdep_details(, "M3Drop")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 12.9Mb
      sub-directories of 1Mb or more:
        doc  12.6Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Package in Depends field not imported from: ‘numDeriv’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    Missing or unexported object: ‘scater::exprs’
    ```

*   checking R code for possible problems ... NOTE
    ```
    M3DropConvertData: no visible global function definition for ‘assays’
    NBumiConvertData: no visible global function definition for ‘assays’
    bg__filter_cells: no visible binding for global variable ‘x’
    bg__get_extreme_residuals: no visible binding for global variable ‘x’
    Undefined global functions or variables:
      assays x
    ```

*   checking Rd files ... NOTE
    ```
    prepare_Rd: NBumi_FSOther.Rd:8-10: Dropping empty section \usage
    prepare_Rd: NBumi_TradDE.Rd:7-8: Dropping empty section \usage
    ```

# maEndToEnd

<details>

* Version: 2.9.0
* GitHub: NA
* Source code: https://github.com/cran/maEndToEnd
* Date/Publication: 2020-04-28
* Number of recursive dependencies: 224

Run `revdep_details(, "maEndToEnd")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      'Biobase', 'oligoClasses', 'ArrayExpress', 'pd.hugene.1.0.st.v1',
      'hugene10sttranscriptcluster.db', 'oligo', 'arrayQualityMetrics',
      'limma', 'topGO', 'ReactomePA', 'clusterProfiler', 'gplots',
      'ggplot2', 'geneplotter', 'pheatmap', 'RColorBrewer', 'dplyr',
      'tidyr', 'stringr', 'matrixStats', 'genefilter', 'openxlsx',
      'Rgraphviz'
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.0Mb
      sub-directories of 1Mb or more:
        doc   5.9Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    License components which are templates and need '+ file LICENSE':
      MIT
    ```

*   checking top-level files ... NOTE
    ```
    File
      LICENCE
    is not mentioned in the DESCRIPTION file.
    ```

*   checking for unstated dependencies in vignettes ... NOTE
    ```
    '::' or ':::' import not declared from: ‘AnnotationDbi’
    ```

# MatrixGenerics

<details>

* Version: 1.2.0
* GitHub: https://github.com/Bioconductor/MatrixGenerics
* Source code: https://github.com/cran/MatrixGenerics
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 64

Run `revdep_details(, "MatrixGenerics")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      ‘methods:::fromNextMethod’ ‘methods:::getGenericFromCall’
      See the note in ?`:::` about the use of this operator.
    There are ::: calls to the package's namespace in its code. A package
      almost never needs to use ::: for its own objects:
      ‘.load_next_suggested_package_to_search’
    ```

# mcmcsae

<details>

* Version: 0.5.0
* GitHub: NA
* Source code: https://github.com/cran/mcmcsae
* Date/Publication: 2020-09-01 16:00:03 UTC
* Number of recursive dependencies: 108

Run `revdep_details(, "mcmcsae")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 15.5Mb
      sub-directories of 1Mb or more:
        libs  14.3Mb
    ```

# MEAL

<details>

* Version: 1.20.0
* GitHub: NA
* Source code: https://github.com/cran/MEAL
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 212

Run `revdep_details(, "MEAL")` for more info

</details>

## In both

*   checking examples ... WARNING
    ```
    Found the following significant warnings:
    
      Warning: 'runRegionAnalysis' is deprecated.
    Deprecated functions may be defunct as soon as of the next release of
    R.
    See ?Deprecated.
    ```

*   checking top-level files ... NOTE
    ```
    File
      LICENSE
    is not mentioned in the DESCRIPTION file.
    ```

*   checking R code for possible problems ... NOTE
    ```
    plotRegion: no visible global function definition for
      ‘subsetByOverlaps’
    plotRegion: no visible global function definition for ‘mcols<-’
    runRDA: no visible global function definition for ‘rowData’
    runSVA: no visible global function definition for ‘resid’
    Undefined global functions or variables:
      mcols<- resid rowData subsetByOverlaps
    Consider adding
      importFrom("stats", "resid")
    to your NAMESPACE file.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘DMRcate’
    ```

# metagene

<details>

* Version: 2.22.0
* GitHub: https://github.com/CharlesJB/metagene
* Source code: https://github.com/cran/metagene
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 136

Run `revdep_details(, "metagene")` for more info

</details>

## In both

*   checking whether package ‘metagene’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/c4/home/henrik/repositories/matrixStats/revdep/checks/metagene/new/metagene.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘metagene’ ...
** using staged installation
** R
Warning: namespace ‘metagene’ is not available and has been replaced
by .GlobalEnv when processing object ‘demo_bh_multicore’
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘DESeq’
Calls: <Anonymous> ... loadNamespace -> withRestarts -> withOneRestart -> doWithOneRestart
Execution halted
ERROR: lazy loading failed for package ‘metagene’
* removing ‘/c4/home/henrik/repositories/matrixStats/revdep/checks/metagene/new/metagene.Rcheck/metagene’

```
### CRAN

```
* installing *source* package ‘metagene’ ...
** using staged installation
** R
Warning: namespace ‘metagene’ is not available and has been replaced
by .GlobalEnv when processing object ‘demo_bh_multicore’
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘DESeq’
Calls: <Anonymous> ... loadNamespace -> withRestarts -> withOneRestart -> doWithOneRestart
Execution halted
ERROR: lazy loading failed for package ‘metagene’
* removing ‘/c4/home/henrik/repositories/matrixStats/revdep/checks/metagene/old/metagene.Rcheck/metagene’

```
# metamicrobiomeR

<details>

* Version: 1.2
* GitHub: https://github.com/nhanhocu/metamicrobiomeR
* Source code: https://github.com/cran/metamicrobiomeR
* Date/Publication: 2020-11-09 11:20:05 UTC
* Number of recursive dependencies: 137

Run `revdep_details(, "metamicrobiomeR")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘lmerTest’
      All declared Imports should be used.
    ```

# MetaNeighbor

<details>

* Version: 1.10.0
* GitHub: NA
* Source code: https://github.com/cran/MetaNeighbor
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 93

Run `revdep_details(, "MetaNeighbor")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.7Mb
      sub-directories of 1Mb or more:
        data   7.1Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    MetaNeighbor : <anonymous>: no visible binding for global variable
      ‘cell_type’
    plotDotPlot: no visible binding for global variable ‘cluster’
    plotDotPlot: no visible binding for global variable ‘gene’
    plotDotPlot: no visible binding for global variable ‘cell_type’
    plotDotPlot: no visible binding for global variable
      ‘average_expression’
    plotDotPlot: no visible binding for global variable
      ‘percent_expressing’
    Undefined global functions or variables:
      average_expression cell_type cluster gene percent_expressing
    ```

# MethReg

<details>

* Version: 1.0.0
* GitHub: https://github.com/TransBioInfoLab/MethReg
* Source code: https://github.com/cran/MethReg
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 217

Run `revdep_details(, "MethReg")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    ...
    > 
    > ### ** Examples
    > 
    > regions.names <- c("chr3:189631389-189632889","chr4:43162098-43163498")
    > triplet <- create_triplet_distance_based(
    +    region = regions.names,
    +    motif.search.window.size = 500,
    +    target.method = "closest.gene"
    + )
    Finding target genes
    Removing regions overlapping promoter regions
    o Get promoter regions for hg38
    o Remove promoter regions
    Looking for TFBS
    Error in h(simpleError(msg, call)) : 
      error in evaluating the argument 'x' in selecting a method for function 'assay': BSgenome.Hsapiens.UCSC.hg38 package is not currently installed.
      You first need to install it, which you can do with:
          library(BiocManager)
          install("BSgenome.Hsapiens.UCSC.hg38")
    Calls: create_triplet_distance_based -> get_tf_in_region
    Execution halted
    ```

*   checking tests ...
    ```
    ...
          █
       1. └─MethReg::make_dnam_se(dna.met.chr21) test-utils.R:52:4
       2.   └─MethReg::get_met_probes_info(genome = genome, arrayType = arrayType)
       3.     └─MethReg:::check_package("sesame")
       4.       └─base::suppressMessages(...)
       5.         └─base::withCallingHandlers(...)
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      ERROR (test-create_triplet_distance_based.R:2:5): create_triplet_distance_based runs
      ERROR (test-filter_by_quantile.R:28:5): filter_dnam_by_quant_diff handles NAs for a SE
      FAILURE (test-get_promoter_avg.R:4:5): get_promoter_avg throws error if not promoter regions/probes
      ERROR (test-get_tf_in_region.R:5:5): get_tf_in_region is able to scan
      ERROR (test-get_tf_in_region.R:18:5): get_tf_in_region accepts granges as input
      Warning (test-plot_stratified_model.R:29:5): plot_interaction_model return a ggplot object
      Warning (test-stratified_model.R:26:5): stratified_model works
      ERROR (test-utils.R:19:5): map_probes_to_regions changes probes cpgs to regions
      ERROR (test-utils.R:52:5): make_dnam_se returns a SE with regions
      
      [ FAIL 7 | WARN 2 | SKIP 0 | PASS 147 ]
      Error: Test failures
      Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.2Mb
      sub-directories of 1Mb or more:
        R     1.6Mb
        doc   2.5Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    interaction_model: no visible binding for global variable ‘.’
    Undefined global functions or variables:
      .
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘TCGAbiolinks’
    ```

# methrix

<details>

* Version: 1.4.0
* GitHub: NA
* Source code: https://github.com/cran/methrix
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 153

Run `revdep_details(, "methrix")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    get_region_summary: no visible binding for global variable ‘..keep’
    write_bigwigs: no visible global function definition for ‘seqlengths<-’
    write_bigwigs: no visible global function definition for ‘seqlengths’
    Undefined global functions or variables:
      ..keep seqlengths seqlengths<-
    ```

# MethylAid

<details>

* Version: 1.24.0
* GitHub: NA
* Source code: https://github.com/cran/MethylAid
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 172

Run `revdep_details(, "MethylAid")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘MethylAidData’
    ```

# methylationArrayAnalysis

<details>

* Version: 1.14.0
* GitHub: NA
* Source code: https://github.com/cran/methylationArrayAnalysis
* Date/Publication: 2020-10-30
* Number of recursive dependencies: 218

Run `revdep_details(, "methylationArrayAnalysis")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      'knitr', 'rmarkdown', 'BiocStyle', 'limma', 'minfi',
      'IlluminaHumanMethylation450kanno.ilmn12.hg19',
      'IlluminaHumanMethylation450kmanifest', 'RColorBrewer', 'missMethyl',
      'matrixStats', 'minfiData', 'Gviz', 'DMRcate', 'stringr',
      'FlowSorted.Blood.450k'
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 322.7Mb
      sub-directories of 1Mb or more:
        doc        2.5Mb
        extdata  320.2Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Title field: should not end in a period.
    ```

# methylumi

<details>

* Version: 2.36.0
* GitHub: https://github.com/seandavi/methylumi
* Source code: https://github.com/cran/methylumi
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 199

Run `revdep_details(, "methylumi")` for more info

</details>

## In both

*   checking for missing documentation entries ... WARNING
    ```
    Undocumented code objects:
      ‘IDATsToMatrices’ ‘IDATtoMatrix’ ‘tcgaPipeline’
    Undocumented S4 methods:
      generic '[' and siglist 'MethyLumiM,ANY,ANY,ANY'
      generic '[' and siglist 'MethyLumiSet,ANY,ANY,ANY'
    All user-level objects in a package (including S4 classes and methods)
    should have documentation entries.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking contents of ‘data’ directory ... WARNING
    ```
    Files not of a type allowed in a ‘data’ directory:
      ‘5318317007_A_Grn.idat’ ‘5318317007_A_Red.idat’
      ‘5318317007_B_Grn.idat’ ‘5318317007_B_Red.idat’
      ‘5318317007_C_Grn.idat’ ‘5318317007_C_Red.idat’
    Please use e.g. ‘inst/extdata’ for non-R data files
    ```

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      'Biobase', 'scales', 'reshape2', 'ggplot2', 'matrixStats',
      'FDb.InfiniumMethylation.hg19', 'minfi'
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 11.8Mb
      sub-directories of 1Mb or more:
        R         1.3Mb
        data      6.8Mb
        doc       1.9Mb
        extdata   1.7Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Packages listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘Biobase’ ‘minfi’ ‘lattice’ ‘matrixStats’
    A package should be listed in only one of these fields.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' calls to packages already attached by Depends:
      ‘FDb.InfiniumMethylation.hg19’ ‘ggplot2’ ‘matrixStats’ ‘minfi’
      ‘reshape2’ ‘scales’
      Please remove these calls from your code.
    'library' or 'require' calls in package code:
      ‘Biostrings’ ‘MASS’ ‘lumi’ ‘parallel’ ‘rtracklayer’
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    Namespace in Imports field not imported from: ‘graphics’
      All declared Imports should be used.
    Packages in Depends field not imported from:
      ‘FDb.InfiniumMethylation.hg19’ ‘ggplot2’ ‘matrixStats’ ‘methods’
      ‘reshape2’ ‘scales’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ':::' call which should be '::': ‘lumi:::produceMethylationGEOSubmissionFile’
      See the note in ?`:::` about the use of this operator.
    Unexported objects imported by ':::' calls:
      ‘Biobase:::unsafeSetSlot’ ‘genefilter:::.findCentralMap’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      IlluminaHumanMethylation450kCOLORCHANNEL MethylSet RGChannelSet
      abline addColorChannelInfo aes allele as.dist axis box capture.output
      channel.probes colMedians colQuantiles colSds colorRampPalette
      coord_flip data dnorm dropouts drops ecdf facet_grid features
      gamma.integral gamma.mle gamma.mode geom_histogram ggplot hclust
      hm27.controls hm27.ordering hm450.controls hm450.ordering huber index
      intensity lines log_trans mclapply melt mu offset opts p.adjust
      packageDescription packageVersion par plot.density pnorm points
      position_identity read.csv read.delim read.table rect rowMins
      scale_colour_manual scale_fill_manual scale_shape_manual
      scale_x_continuous scale_y_continuous scale_y_discrete
      subsetCommonProbes text theme_bw title value variable weighted.mean
    Consider adding
      importFrom("grDevices", "colorRampPalette")
      importFrom("graphics", "abline", "axis", "box", "lines", "par",
                 "points", "rect", "text", "title")
      importFrom("stats", "as.dist", "dnorm", "ecdf", "hclust", "offset",
                 "p.adjust", "pnorm", "weighted.mean")
      importFrom("utils", "capture.output", "data", "packageDescription",
                 "packageVersion", "read.csv", "read.delim", "read.table")
    to your NAMESPACE file.
    ```

*   checking Rd files ... NOTE
    ```
    prepare_Rd: estimateM.Rd:34-36: Dropping empty section \seealso
    prepare_Rd: estimateM.Rd:37-39: Dropping empty section \examples
    ```

*   checking for unstated dependencies in vignettes ... NOTE
    ```
    'library' or 'require' call not declared from: ‘TxDb.Hsapiens.UCSC.hg19.knownGene’
    ```

# MFHD

<details>

* Version: 0.0.1
* GitHub: NA
* Source code: https://github.com/cran/MFHD
* Date/Publication: 2013-10-23 23:50:20
* Number of recursive dependencies: 58

Run `revdep_details(, "MFHD")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    ...
    plot.bagplot : cut.z.pg: no visible global function definition for
      ‘points’
    plot.bagplot : find.cut.z.pg: no visible global function definition for
      ‘points’
    plot.bagplot : find.cut.z.pg: no visible global function definition for
      ‘lines’
    plot.bagplot: no visible global function definition for ‘boxplot’
    plot.bagplot: no visible global function definition for ‘points’
    plot.bagplot: no visible global function definition for ‘segments’
    plot.bagplot: no visible global function definition for ‘text’
    plot.bagplot: no visible global function definition for ‘lines’
    plot.bagplot: no visible global function definition for ‘polygon’
    Undefined global functions or variables:
      boxplot chull complete.cases identify lines points polygon prcomp
      quantile sd segments text xy.coords
    Consider adding
      importFrom("grDevices", "chull", "xy.coords")
      importFrom("graphics", "boxplot", "identify", "lines", "points",
                 "polygon", "segments", "text")
      importFrom("stats", "complete.cases", "prcomp", "quantile", "sd")
    to your NAMESPACE file.
    ```

# mgcViz

<details>

* Version: 0.1.6
* GitHub: https://github.com/mfasiolo/mgcViz
* Source code: https://github.com/cran/mgcViz
* Date/Publication: 2020-03-04 15:10:02 UTC
* Number of recursive dependencies: 99

Run `revdep_details(, "mgcViz")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.1Mb
      sub-directories of 1Mb or more:
        doc   4.4Mb
    ```

# MHTcop

<details>

* Version: 0.1.1
* GitHub: NA
* Source code: https://github.com/cran/MHTcop
* Date/Publication: 2019-01-21 16:10:03 UTC
* Number of recursive dependencies: 41

Run `revdep_details(, "MHTcop")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘MCMCpack’ ‘copula’
      All declared Imports should be used.
    ```

# microbiomeExplorer

<details>

* Version: 1.0.1
* GitHub: NA
* Source code: https://github.com/cran/microbiomeExplorer
* Date/Publication: 2020-10-29
* Number of recursive dependencies: 196

Run `revdep_details(, "microbiomeExplorer")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 12.1Mb
      sub-directories of 1Mb or more:
        doc       4.6Mb
        extdata   4.1Mb
        shiny     2.8Mb
    ```

# MinimumDistance

<details>

* Version: 1.34.0
* GitHub: NA
* Source code: https://github.com/cran/MinimumDistance
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 73

Run `revdep_details(, "MinimumDistance")` for more info

</details>

## In both

*   checking Rd files ... WARNING
    ```
    checkRd: (5) MinimumDistance.Rd:0-7: Must have a \description
    ```

*   checking for code/documentation mismatches ... WARNING
    ```
    Functions or methods with usage in documentation object 'coerce' but not in code:
      ‘as’
    ```

*   checking Rd \usage sections ... WARNING
    ```
    Objects in \usage without \alias in documentation object 'coerce':
      ‘as’
    
    Functions with \usage entries need to have the appropriate \alias
    entries, and all their arguments documented.
    The \usage entries must correspond to syntactically valid R code.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking package dependencies ... NOTE
    ```
    Package which this enhances but not available for checking: ‘doSNOW’
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘MatrixGenerics’
      All declared Imports should be used.
    ```

*   checking R code for possible problems ... NOTE
    ```
    callDenovoSegments: no visible global function definition for
      ‘read.bsfiles’
    combineRanges: no visible global function definition for ‘RangedData’
    getRefGene: no visible global function definition for ‘RangedData’
    overlapsCentromere: no visible global function definition for
      ‘RangedData’
    pruneMD: no visible global function definition for ‘RangedDataCBS’
    pruneTrioSet: no visible global function definition for
      ‘RangedDataList’
    read.bsfiles2: no visible binding for global variable ‘read.bsfiles’
    GenomeAnnotatedDataFrameFrom,character: no visible global function
      definition for ‘read.bsfiles’
    calculateMindist,list: no visible binding for global variable ‘elt’
    Undefined global functions or variables:
      RangedData RangedDataCBS RangedDataList elt read.bsfiles
    ```

*   checking contents of ‘data’ directory ... NOTE
    ```
    Output for data("trioSetListExample", package = "MinimumDistance"):
      No methods found in package ‘oligoClasses’ for request: ‘mean’ when loading ‘crlmm’
    ```

# mixOmics

<details>

* Version: 6.14.0
* GitHub: https://github.com/mixOmicsTeam/mixOmics
* Source code: https://github.com/cran/mixOmics
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 99

Run `revdep_details(, "mixOmics")` for more info

</details>

## In both

*   checking tests ...
    ```
    ...
      
        |                                                                            
        |                                                                      |   0%
        |                                                                            
        |=======================                                               |  33%
        |                                                                            
        |===============================================                       |  67%
        |                                                                            
        |======================================================================| 100%
      Performing repeated cross-validation...
      
        |                                                                            
        |                                                                      |   0%══ testthat results  ═══════════════════════════════════════════════════════════
      Warning (test-network.R:13:3): network works for rcc
      ERROR (test-network.R:13:3): network works for rcc
      Warning (test-network.R:28:3): network works for spls
      ERROR (test-network.R:28:3): network works for spls
      
      [ FAIL 2 | WARN 2 | SKIP 0 | PASS 124 ]
      Error: Test failures
      Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 10.7Mb
      sub-directories of 1Mb or more:
        R      1.3Mb
        data   2.9Mb
        doc    5.9Mb
    ```

# mnem

<details>

* Version: 1.6.5
* GitHub: NA
* Source code: https://github.com/cran/mnem
* Date/Publication: 2020-11-16
* Number of recursive dependencies: 156

Run `revdep_details(, "mnem")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘epiNEM’
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  8.3Mb
      sub-directories of 1Mb or more:
        data   2.3Mb
        doc    3.8Mb
        libs   2.0Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    createApp: no visible binding for global variable ‘..tmp’
    Undefined global functions or variables:
      ..tmp
    ```

# moc.gapbk

<details>

* Version: 0.1.0
* GitHub: NA
* Source code: https://github.com/cran/moc.gapbk
* Date/Publication: 2019-03-07 17:20:03 UTC
* Number of recursive dependencies: 40

Run `revdep_details(, "moc.gapbk")` for more info

</details>

## In both

*   checking S3 generic/method consistency ... WARNING
    ```
    --------------------------------------------------------------------------
    [[30941,1],0]: A high-performance Open MPI point-to-point messaging module
    was unable to find any relevant network interfaces:
    
    Module: OpenFabrics (openib)
      Host: c4-dev2
    
    Another transport will be used instead, although this may result in
    lower performance.
    --------------------------------------------------------------------------
    See section ‘Generic functions and methods’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking replacement functions ... WARNING
    ```
    --------------------------------------------------------------------------
    [[30878,1],0]: A high-performance Open MPI point-to-point messaging module
    was unable to find any relevant network interfaces:
    
    Module: OpenFabrics (openib)
      Host: c4-dev2
    
    Another transport will be used instead, although this may result in
    lower performance.
    --------------------------------------------------------------------------
    The argument of a replacement function which corresponds to the right
    hand side must be named ‘value’.
    ```

*   checking for missing documentation entries ... WARNING
    ```
    --------------------------------------------------------------------------
    [[18386,1],0]: A high-performance Open MPI point-to-point messaging module
    was unable to find any relevant network interfaces:
    
    Module: OpenFabrics (openib)
      Host: c4-dev2
    
    Another transport will be used instead, although this may result in
    lower performance.
    --------------------------------------------------------------------------
    All user-level objects in a package should have documentation entries.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking for code/documentation mismatches ... WARNING
    ```
    ...
    --------------------------------------------------------------------------
    --------------------------------------------------------------------------
    [[18315,1],0]: A high-performance Open MPI point-to-point messaging module
    was unable to find any relevant network interfaces:
    
    Module: OpenFabrics (openib)
      Host: c4-dev2
    
    Another transport will be used instead, although this may result in
    lower performance.
    --------------------------------------------------------------------------
    --------------------------------------------------------------------------
    [[18321,1],0]: A high-performance Open MPI point-to-point messaging module
    was unable to find any relevant network interfaces:
    
    Module: OpenFabrics (openib)
      Host: c4-dev2
    
    Another transport will be used instead, although this may result in
    lower performance.
    --------------------------------------------------------------------------
    ```

*   checking dependencies in R code ... NOTE
    ```
    --------------------------------------------------------------------------
    [[30965,1],0]: A high-performance Open MPI point-to-point messaging module
    was unable to find any relevant network interfaces:
    
    Module: OpenFabrics (openib)
      Host: c4-dev2
    
    Another transport will be used instead, although this may result in
    lower performance.
    --------------------------------------------------------------------------
    Namespaces in Imports field not imported from:
      ‘Rmisc’ ‘amap’ ‘fields’ ‘matrixStats’ ‘mco’ ‘miscTools’ ‘plyr’
      ‘reshape2’
      All declared Imports should be used.
    ```

*   checking foreign function calls ... NOTE
    ```
    --------------------------------------------------------------------------
    [[30822,1],0]: A high-performance Open MPI point-to-point messaging module
    was unable to find any relevant network interfaces:
    
    Module: OpenFabrics (openib)
      Host: c4-dev2
    
    Another transport will be used instead, although this may result in
    lower performance.
    --------------------------------------------------------------------------
    See chapter ‘System and foreign language interfaces’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    --------------------------------------------------------------------------
    [[30728,1],0]: A high-performance Open MPI point-to-point messaging module
    was unable to find any relevant network interfaces:
    
    Module: OpenFabrics (openib)
      Host: c4-dev2
    
    Another transport will be used instead, although this may result in
    lower performance.
    --------------------------------------------------------------------------
    ```

*   checking Rd \usage sections ... NOTE
    ```
    --------------------------------------------------------------------------
    [[18291,1],0]: A high-performance Open MPI point-to-point messaging module
    was unable to find any relevant network interfaces:
    
    Module: OpenFabrics (openib)
      Host: c4-dev2
    
    Another transport will be used instead, although this may result in
    lower performance.
    --------------------------------------------------------------------------
    The \usage entries for S3 methods should use the \method markup and not
    their full name.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

# monocle

<details>

* Version: 2.18.0
* GitHub: NA
* Source code: https://github.com/cran/monocle
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 250

Run `revdep_details(, "monocle")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘Rcpp’ ‘biocViews’
      All declared Imports should be used.
    Missing or unexported object: ‘scater::newSCESet’
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    extract_good_ordering: no visible global function definition for ‘nei’
    fit_model_helper: no visible binding for global variable ‘Size_Factor’
    get_next_node_id: no visible binding for '<<-' assignment to
      ‘next_node’
    get_next_node_id: no visible binding for global variable ‘next_node’
    make_canonical: no visible global function definition for ‘nei’
    measure_diameter_path: no visible global function definition for ‘nei’
    orderCells: no visible binding for '<<-' assignment to ‘next_node’
    plot_multiple_branches_pseudotime: no visible binding for global
      variable ‘pseudocount’
    plot_multiple_branches_pseudotime: no visible binding for global
      variable ‘Branch’
    project2MST: no visible global function definition for ‘nei’
    reverseEmbeddingCDS : <anonymous>: no visible global function
      definition for ‘quantile’
    Undefined global functions or variables:
      Branch Size_Factor nei next_node pseudocount quantile
      use_for_ordering
    Consider adding
      importFrom("stats", "quantile")
    to your NAMESPACE file.
    ```

*   checking files in ‘vignettes’ ... NOTE
    ```
    The following directory looks like a leftover from 'knitr':
      ‘figure’
    Please remove from your package.
    ```

# motifbreakR

<details>

* Version: 2.4.0
* GitHub: https://github.com/Simon-Coetzee/motifbreakR
* Source code: https://github.com/cran/motifbreakR
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 156

Run `revdep_details(, "motifbreakR")` for more info

</details>

## In both

*   checking whether package ‘motifbreakR’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘S4Vectors::as.data.frame’ by ‘motifStack::as.data.frame’ when loading ‘motifbreakR’
    See ‘/c4/home/henrik/repositories/matrixStats/revdep/checks/motifbreakR/new/motifbreakR.Rcheck/00install.out’ for details.
    ```

*   checking for missing documentation entries ... WARNING
    ```
    Undocumented code objects:
      'rs1006140' 'variants.from.file'
    Undocumented data sets:
      'rs1006140'
    All user-level objects in a package should have documentation entries.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking Rd \usage sections ... WARNING
    ```
    Undocumented arguments in documentation object 'motifbreakR'
      ‘legacy.score’
    
    Undocumented arguments in documentation object 'scoreAllWindows'
      ‘snp.seq’ ‘snp.seq.rc’ ‘pwm’ ‘from’ ‘to’ ‘pwm.range’ ‘calcp’
    
    Undocumented arguments in documentation object 'snps.from.file'
      ‘indels’
    
    Functions with \usage entries need to have the appropriate \alias
    entries, and all their arguments documented.
    The \usage entries must correspond to syntactically valid R code.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      'compiler' 'grImport'
      All declared Imports should be used.
    Package in Depends field not imported from: 'MotifDb'
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    formatVcfOut: no visible binding for global variable 'search.genome'
    plotMotifLogoStack.2: no visible global function definition for 'par'
    preparePWM: no visible binding for global variable 'threshold'
    snps.from.file: no visible binding for global variable 'DNA_ALPHABET'
    wScore: no visible global function definition for 'scoreMotif'
    Undefined global functions or variables:
      DNA_ALPHABET par scoreMotif search.genome threshold
    Consider adding
      importFrom("graphics", "par")
    to your NAMESPACE file.
    ```

# mrfDepth

<details>

* Version: 1.0.13
* GitHub: https://github.com/PSegaert/mrfDepth
* Source code: https://github.com/cran/mrfDepth
* Date/Publication: 2020-08-26 16:10:33 UTC
* Number of recursive dependencies: 65

Run `revdep_details(, "mrfDepth")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 20.7Mb
      sub-directories of 1Mb or more:
        data   1.6Mb
        libs  18.8Mb
    ```

# MultiBD

<details>

* Version: 0.2.0
* GitHub: NA
* Source code: https://github.com/cran/MultiBD
* Date/Publication: 2016-12-05 18:28:46
* Number of recursive dependencies: 74

Run `revdep_details(, "MultiBD")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  9.4Mb
      sub-directories of 1Mb or more:
        libs   7.9Mb
    ```

# muscat

<details>

* Version: 1.4.0
* GitHub: https://github.com/HelenaLC/muscat
* Source code: https://github.com/cran/muscat
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 209

Run `revdep_details(, "muscat")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  9.0Mb
      sub-directories of 1Mb or more:
        data   2.3Mb
        doc    6.0Mb
    ```

# muscData

<details>

* Version: 1.4.0
* GitHub: https://github.com/HelenaLC/muscData
* Source code: https://github.com/cran/muscData
* Date/Publication: 2020-10-29
* Number of recursive dependencies: 248

Run `revdep_details(, "muscData")` for more info

</details>

## In both

*   checking for missing documentation entries ... WARNING
    ```
    Undocumented code objects:
      ‘Crowell19_4vs4’
    All user-level objects in a package should have documentation entries.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .BBSoptions
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Package in Depends field not imported from: ‘SingleCellExperiment’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

# NanoStringDiff

<details>

* Version: 1.20.0
* GitHub: NA
* Source code: https://github.com/cran/NanoStringDiff
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 53

Run `revdep_details(, "NanoStringDiff")` for more info

</details>

## In both

*   checking for missing documentation entries ... WARNING
    ```
    Undocumented code objects:
      ‘compute.baseSigma’ ‘est.dispersion’ ‘fun5’ ‘glmfit.OneGroup’
      ‘glmfit.full’ ‘glmfit.reduce’ ‘rnegbinom’
    All user-level objects in a package should have documentation entries.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    glmfit.full: no visible global function definition for ‘lm’
    glmfit.full: no visible global function definition for ‘median’
    glmfit.full: no visible global function definition for ‘IQR’
    glmfit.full : get.phi: no visible global function definition for
      ‘optimize’
    glmfit.full : get.beta.full: no visible global function definition for
      ‘optim’
    glmfit.reduce: no visible global function definition for ‘lm’
    glmfit.reduce : get.beta.reduce: no visible global function definition
      for ‘optim’
    rnegbinom: no visible global function definition for ‘rpois’
    rnegbinom: no visible global function definition for ‘rgamma’
    Undefined global functions or variables:
      IQR abline glm lm median optim optimize p.adjust par pchisq poisson
      read.table rgamma rowVars rpois textxy title
    Consider adding
      importFrom("graphics", "abline", "par", "title")
      importFrom("stats", "IQR", "glm", "lm", "median", "optim", "optimize",
                 "p.adjust", "pchisq", "poisson", "rgamma", "rpois")
      importFrom("utils", "read.table")
    to your NAMESPACE file.
    ```

*   checking installed files from ‘inst/doc’ ... NOTE
    ```
    The following files look like leftovers/mistakes:
      ‘NanoStringDiff.log’
    Please remove them from your package.
    ```

# nearBynding

<details>

* Version: 1.0.0
* GitHub: NA
* Source code: https://github.com/cran/nearBynding
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 124

Run `revdep_details(, "nearBynding")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    ...
    > ### Name: visualizeCapRStereogene
    > ### Title: visualizeCapRStereogene
    > ### Aliases: visualizeCapRStereogene
    > 
    > ### ** Examples
    > 
    > ## pull example files
    > get_outfiles()
    [1] TRUE
    > ## heatmap
    > visualizeCapRStereogene(CapR_prefix = "chr4and5_3UTR",
    +                         protein_file = "chr4and5_liftOver",
    +                         heatmap = TRUE,
    +                         out_file = "all_contexts_heatmap",
    +                         x_lim = c(-500, 500))
    Warning in jpeg(out_file, height = 4, width = 8, units = "in", res = 300) :
      unable to open connection to X11 display ''
    Error in .External2(C_X11, paste0("jpeg::", quality, ":", filename), g$width,  : 
      unable to start device JPEG
    Calls: visualizeCapRStereogene -> jpeg
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  8.9Mb
      sub-directories of 1Mb or more:
        extdata   8.1Mb
    ```

# NetLogoR

<details>

* Version: 0.3.7
* GitHub: https://github.com/PredictiveEcology/NetLogoR
* Source code: https://github.com/cran/NetLogoR
* Date/Publication: 2020-03-02 10:00:08 UTC
* Number of recursive dependencies: 156

Run `revdep_details(, "NetLogoR")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘fastshp’
    ```

# NormalyzerDE

<details>

* Version: 1.8.0
* GitHub: https://github.com/ComputationalProteomics/NormalyzerDE
* Source code: https://github.com/cran/NormalyzerDE
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 153

Run `revdep_details(, "NormalyzerDE")` for more info

</details>

## In both

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

# omicplotR

<details>

* Version: 1.10.0
* GitHub: NA
* Source code: https://github.com/cran/omicplotR
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 76

Run `revdep_details(, "omicplotR")` for more info

</details>

## In both

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .github
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Package in Depends field not imported from: ‘ALDEx2’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

# omicsPrint

<details>

* Version: 1.10.0
* GitHub: NA
* Source code: https://github.com/cran/omicsPrint
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 121

Run `revdep_details(, "omicsPrint")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  8.6Mb
      sub-directories of 1Mb or more:
        data   7.3Mb
        doc    1.2Mb
    ```

# OptimalDesign

<details>

* Version: 1.0.1
* GitHub: NA
* Source code: https://github.com/cran/OptimalDesign
* Date/Publication: 2019-12-02 08:50:07 UTC
* Number of recursive dependencies: 47

Run `revdep_details(, "OptimalDesign")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Packages which this enhances but not available for checking:
      'gurobi', 'slam'
    ```

# OUTRIDER

<details>

* Version: 1.8.0
* GitHub: https://github.com/gagneurlab/OUTRIDER
* Source code: https://github.com/cran/OUTRIDER
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 169

Run `revdep_details(, "OUTRIDER")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    ...
     92:        312.80 124021.44    FALSE                0              0       5.5
     93:        829.59 144898.81    FALSE                0              0       5.5
     94:         65.67 102654.99    FALSE                0              0       5.5
     95:         51.78  64681.51    FALSE                0              0       5.5
     96:        176.16 153477.93    FALSE                0              0       5.5
     97:       1416.38 107837.16    FALSE                0              0       5.5
     98:         79.79 176300.28    FALSE                0              0       5.5
     99:        643.74 201805.85    FALSE                0              0       5.5
    100:       1076.47 244265.44    FALSE                0              0       5.5
         meanCorrected     theta aberrant AberrantBySample AberrantByGene padj_rank
    > 
    > plotAberrantPerSample(ods)
    Warning: Transformation introduced infinite values in continuous y-axis
    Warning: Removed 10 rows containing missing values (geom_bar).
    > plotVolcano(ods, 1)
    Warning in dev_fun(file = tempfile(), width = width %||% 640, height = height %||%  :
      unable to open connection to X11 display ''
    Error in .External2(C_X11, paste0("png::", filename), g$width, g$height,  : 
      unable to start device PNG
    Calls: plotVolcano ... .local -> ggplotly -> ggplotly.ggplot -> gg2list -> dev_fun
    Execution halted
    ```

*   checking tests ...
    ```
    ...
      Backtrace:
           █
        1. ├─testthat::expect_equal(...) test_plotting.R:17:4
        2. │ └─testthat::quasi_label(enquo(object), label, arg = "object")
        3. │   └─rlang::eval_bare(expr, quo_get_env(quo))
        4. ├─OUTRIDER::plotVolcano(ods, 30)
        5. └─OUTRIDER::plotVolcano(ods, 30)
        6.   └─OUTRIDER:::.local(object, ...)
        7.     ├─plotly::ggplotly(p, tooltip = "text")
        8.     └─plotly:::ggplotly.ggplot(p, tooltip = "text")
        9.       └─plotly::gg2list(...)
       10.         └─grDevices:::dev_fun(...)
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      Warning (test_filtering.R:21:5): Expression filtering
      Warning (test_plotting.R:17:5): plotting
      ERROR (test_plotting.R:17:5): plotting
      
      [ FAIL 1 | WARN 2 | SKIP 0 | PASS 82 ]
      Error: Test failures
      Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.0Mb
      sub-directories of 1Mb or more:
        doc    2.3Mb
        libs   2.8Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘Rcpp’
      All declared Imports should be used.
    ```

# pandaR

<details>

* Version: 1.22.0
* GitHub: NA
* Source code: https://github.com/cran/pandaR
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 66

Run `revdep_details(, "pandaR")` for more info

</details>

## In both

*   checking package subdirectories ... WARNING
    ```
    Invalid citation information in ‘inst/CITATION’:
      Error in tools:::.parse_CITATION_file(file, meta$Encoding): non-ASCII input in a CITATION file without a declared encoding
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  9.3Mb
      sub-directories of 1Mb or more:
        data   9.0Mb
    ```

*   checking S3 generic/method consistency ... NOTE
    ```
    Found the following apparent S3 methods exported but not registered:
      summary.panda
    See section ‘Registering S3 methods’ in the ‘Writing R Extensions’
    manual.
    ```

*   checking Rd \usage sections ... NOTE
    ```
    S3 methods shown with full name in documentation object 'summary.panda':
      ‘summary.panda’
    
    The \usage entries for S3 methods should use the \method markup and not
    their full name.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

# PathoStat

<details>

* Version: 1.16.0
* GitHub: https://github.com/mani2012/PathoStat
* Source code: https://github.com/cran/PathoStat
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 192

Run `revdep_details(, "PathoStat")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘ComplexHeatmap’ ‘RColorBrewer’
      All declared Imports should be used.
    ```

# pcaMethods

<details>

* Version: 1.82.0
* GitHub: https://github.com/hredestig/pcamethods
* Source code: https://github.com/cran/pcaMethods
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 53

Run `revdep_details(, "pcaMethods")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    ...
    svdImpute: no visible global function definition for ‘prcomp’
    svdPca: no visible global function definition for ‘prcomp’
    plot,pcaRes: no visible global function definition for ‘gray’
    plot,pcaRes: no visible global function definition for ‘barplot’
    plot,pcaRes: no visible global function definition for ‘legend’
    slplot,pcaRes: no visible global function definition for ‘par’
    slplot,pcaRes: no visible global function definition for ‘layout’
    slplot,pcaRes: no visible global function definition for ‘abline’
    slplot,pcaRes: no visible global function definition for ‘lines’
    Undefined global functions or variables:
      abline barplot cor cov gray layout legend lines median na.omit pairs
      par points prcomp qf rnorm runif setTxtProgressBar text
      txtProgressBar
    Consider adding
      importFrom("grDevices", "gray")
      importFrom("graphics", "abline", "barplot", "layout", "legend",
                 "lines", "pairs", "par", "points", "text")
      importFrom("stats", "cor", "cov", "median", "na.omit", "prcomp", "qf",
                 "rnorm", "runif")
      importFrom("utils", "setTxtProgressBar", "txtProgressBar")
    to your NAMESPACE file.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘ade4’
    ```

# Pigengene

<details>

* Version: 1.16.0
* GitHub: NA
* Source code: https://github.com/cran/Pigengene
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 141

Run `revdep_details(, "Pigengene")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    ...
    > ### Title: Infers robust biological signatures from gene expression data
    > ### Aliases: Pigengene-package Pigengene
    > ### Keywords: package datasets documentation models
    > 
    > ### ** Examples
    > 
    > data(aml)
    > data(mds)
    > d1 <- rbind(aml,mds)
    > Labels <- c(rep("AML",nrow(aml)),rep("MDS",nrow(mds)))
    > names(Labels) <- rownames(d1)
    > p1 <- one.step.pigengene(Data=d1,saveDir='pigengene', bnNum=10, verbose=1,
    +    seed=1, Labels=Labels, toCompact=FALSE, doHeat=FALSE)
    Pigengene started analizing 366 samples using 1000 genes...
    Warning: executing %dopar% sequentially: no parallel backend registered
    Warning in png(filename = plotFile) :
      unable to open connection to X11 display ''
    Error in .External2(C_X11, paste0("png::", filename), g$width, g$height,  : 
      unable to start device PNG
    Calls: one.step.pigengene -> wgcna.one.step -> png
    Execution halted
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' calls in package code:
      ‘AnnotationDbi’ ‘biomaRt’ ‘energy’ ‘org.Hs.eg.db’ ‘org.Mm.eg.db’
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    Found the following possibly unsafe calls:
    File ‘Pigengene/R/bn.calculation.R’:
      assignInNamespace("supported.clusters", fixArgs, "bnlearn")
    
    combine.networks: no visible binding for global variable ‘netwok’
    Undefined global functions or variables:
      netwok
    ```

*   checking contents of ‘data’ directory ... NOTE
    ```
    Output for data("pigengene", package = "Pigengene"):
      
    ```

# pipeComp

<details>

* Version: 1.0.0
* GitHub: https://github.com/plger/pipeComp
* Source code: https://github.com/cran/pipeComp
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 203

Run `revdep_details(, "pipeComp")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.1Mb
      sub-directories of 1Mb or more:
        doc   4.5Mb
    ```

*   checking top-level files ... NOTE
    ```
    File
      LICENSE
    is not mentioned in the DESCRIPTION file.
    ```

*   checking dependencies in R code ... NOTE
    ```
    There are ::: calls to the package's namespace in its code. A package
      almost never needs to use ::: for its own objects:
      ‘.compileExcludedCells’ ‘.getMM’ ‘.homogenizeDEA’ ‘.runf’
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    scrna_describeDatasets: no visible binding for global variable ‘nb’
    scrna_describeDatasets : pf: no visible binding for global variable
      ‘.x’
    scrna_describeDatasets : rd: no visible binding for global variable ‘y’
    scrna_describeDatasets : rd: no visible binding for global variable
      ‘cluster’
    scrna_evalPlot_filtering: no visible binding for global variable
      ‘max.lost’
    scrna_evalPlot_filtering: no visible binding for global variable
      ‘mean_F1’
    scrna_evalPlot_filtering: no visible binding for global variable ‘filt’
    scrna_evalPlot_filtering: no visible binding for global variable
      ‘doubletmethod’
    scrna_evalPlot_filtering: no visible binding for global variable
      ‘method’
    scrna_evalPlot_overall: no visible binding for global variable
      ‘true.nbClusts’
    scrna_evalPlot_overall: no visible binding for global variable ‘n_clus’
    Undefined global functions or variables:
      .x cluster doubletmethod filt max.lost mean_F1 method n_clus nb
      true.nbClusts y
    ```

*   checking for unstated dependencies in vignettes ... NOTE
    ```
    'library' or 'require' calls not declared from:
      ‘edgeR’ ‘sva’
    ```

# pmp

<details>

* Version: 1.2.0
* GitHub: NA
* Source code: https://github.com/cran/pmp
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 100

Run `revdep_details(, "pmp")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.5Mb
      sub-directories of 1Mb or more:
        data   1.8Mb
        doc    5.5Mb
    ```

# PrecisionTrialDrawer

<details>

* Version: 1.6.0
* GitHub: NA
* Source code: https://github.com/cran/PrecisionTrialDrawer
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 133

Run `revdep_details(, "PrecisionTrialDrawer")` for more info

</details>

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘LowMACA’
    ```

# ProteoMM

<details>

* Version: 1.8.0
* GitHub: NA
* Source code: https://github.com/cran/ProteoMM
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 99

Run `revdep_details(, "ProteoMM")` for more info

</details>

## In both

*   checking DESCRIPTION meta-information ... NOTE
    ```
    License components which are templates and need '+ file LICENSE':
      MIT
    ```

# PSCBS

<details>

* Version: 0.65.0
* GitHub: https://github.com/HenrikBengtsson/PSCBS
* Source code: https://github.com/cran/PSCBS
* Date/Publication: 2019-05-05 22:40:09 UTC
* Number of recursive dependencies: 90

Run `revdep_details(, "PSCBS")` for more info

</details>

## In both

*   checking running R code from vignettes ...
    ```
    ...
    +     aspectRatio = 0.35, {
    +         plotTracks(fit)
    +     })
    Warning in png("figures/PairedPSCBS,exData,chr01,CBS,tracks.png", width = 840,  :
      unable to open connection to X11 display ''
    
      When sourcing ‘CBS.R’:
    Error: unable to start device PNG
    Execution halted
    when running code in ‘PairedPSCBS.tex.rsp’
      ...
    > toPNG(fullname, tags = c(class(fit)[1], "tracks"), 
    +     aspectRatio = 0.6, {
    +         plotTracks(fit)
    +     })
    Warning in png("figures/PairedPSCBS,exData,chr01,PairedPSCBS,tracks.png",  :
      unable to open connection to X11 display ''
    
      When sourcing ‘PairedPSCBS.R’:
    Error: unable to start device PNG
    Execution halted
    ```

# QDNAseq

<details>

* Version: 1.26.0
* GitHub: https://github.com/ccagc/QDNAseq
* Source code: https://github.com/cran/QDNAseq
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 72

Run `revdep_details(, "QDNAseq")` for more info

</details>

## In both

*   checking running R code from vignettes ...
    ```
      ‘QDNAseq.Rnw’... failed
     ERROR
    Errors in running code in vignettes:
    when running code in ‘QDNAseq.Rnw’
      ...
    experimentData: use 'experimentData(object)'
    Annotation:  
    
    > png("rawprofile.png")
    Warning in png("rawprofile.png") :
      unable to open connection to X11 display ''
    
      When sourcing ‘QDNAseq.R’:
    Error: unable to start device PNG
    Execution halted
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘future’
      All declared Imports should be used.
    ```

# QFeatures

<details>

* Version: 1.0.0
* GitHub: https://github.com/RforMassSpectrometry/QFeatures
* Source code: https://github.com/cran/QFeatures
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 124

Run `revdep_details(, "QFeatures")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Unexported object imported by a ':::' call: ‘MultiAssayExperiment:::.sampleMapFromData’
      See the note in ?`:::` about the use of this operator.
    ```

# r2dRue

<details>

* Version: 1.0.4
* GitHub: NA
* Source code: https://github.com/cran/r2dRue
* Date/Publication: 2013-06-28 12:22:19
* Number of recursive dependencies: 4

Run `revdep_details(, "r2dRue")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' call to ‘rgdal’ which was already attached by Depends.
      Please remove these calls from your code.
    Packages in Depends field not imported from:
      ‘matrixStats’ ‘rgdal’ ‘sp’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    rueObsMe: no visible global function definition for ‘readGDAL’
    rueObsMe: no visible global function definition for ‘setTxtProgressBar’
    showInfo: no visible global function definition for ‘head’
    solarRad: no visible global function definition for ‘is.projected’
    solarRad: no visible global function definition for ‘coordinates’
    solarRad12M: no visible global function definition for ‘writeGDAL’
    summarize: no visible global function definition for ‘writeGDAL’
    Undefined global functions or variables:
      GDAL.close GDAL.open GDALinfo abline axis.Date barplot boxplot
      coordinates cor density gdalDrivers head hist image is.projected
      layout layout.show lines na.omit pf pt read.table readGDAL rect
      rowCounts rowMedians rowRanges rowSds rowVars setTxtProgressBar
      spplot title txtProgressBar write.table writeGDAL
    Consider adding
      importFrom("graphics", "abline", "axis.Date", "barplot", "boxplot",
                 "hist", "image", "layout", "layout.show", "lines", "rect",
                 "title")
      importFrom("stats", "cor", "density", "na.omit", "pf", "pt")
      importFrom("utils", "head", "read.table", "setTxtProgressBar",
                 "txtProgressBar", "write.table")
    to your NAMESPACE file.
    ```

# RadioGx

<details>

* Version: 1.0.0
* GitHub: NA
* Source code: https://github.com/cran/RadioGx
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 134

Run `revdep_details(, "RadioGx")` for more info

</details>

## In both

*   checking whether package ‘RadioGx’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: no DISPLAY variable so Tk is not available
    See ‘/c4/home/henrik/repositories/matrixStats/revdep/checks/RadioGx/new/RadioGx.Rcheck/00install.out’ for details.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 14 marked UTF-8 strings
    ```

# randomizationInference

<details>

* Version: 1.0.3
* GitHub: NA
* Source code: https://github.com/cran/randomizationInference
* Date/Publication: 2015-01-09 07:58:52
* Number of recursive dependencies: 2

Run `revdep_details(, "randomizationInference")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    anovaF: no visible global function definition for ‘anova’
    anovaF: no visible global function definition for ‘lm’
    randInterval: no visible global function definition for ‘quantile’
    randInterval : <anonymous>: no visible global function definition for
      ‘quantile’
    randPlot: no visible global function definition for ‘par’
    randPlot: no visible global function definition for ‘hist’
    randPlot: no visible global function definition for ‘abline’
    Undefined global functions or variables:
      abline anova hist lm par quantile
    Consider adding
      importFrom("graphics", "abline", "hist", "par")
      importFrom("stats", "anova", "lm", "quantile")
    to your NAMESPACE file.
    ```

# regsem

<details>

* Version: 1.6.2
* GitHub: NA
* Source code: https://github.com/cran/regsem
* Date/Publication: 2020-10-01 17:00:02 UTC
* Number of recursive dependencies: 168

Run `revdep_details(, "regsem")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.4Mb
      sub-directories of 1Mb or more:
        libs   5.1Mb
    ```

# RJcluster

<details>

* Version: 0.1.0
* GitHub: NA
* Source code: https://github.com/cran/RJcluster
* Date/Publication: 2020-09-23 08:00:06 UTC
* Number of recursive dependencies: 57

Run `revdep_details(, "RJcluster")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘doParallel’ ‘parallel’
      All declared Imports should be used.
    ```

# RNAmodR

<details>

* Version: 1.4.0
* GitHub: https://github.com/FelixErnst/RNAmodR
* Source code: https://github.com/cran/RNAmodR
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 182

Run `revdep_details(, "RNAmodR")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      'BiocGenerics:::replaceSlots' 'Biostrings:::XString'
      'IRanges:::.valid.CompressedList' 'IRanges:::coerceToCompressedList'
      'S4Vectors:::.valid.DataFrame' 'S4Vectors:::labeledLine'
      'S4Vectors:::make_zero_col_DataFrame'
      'S4Vectors:::prepare_objects_to_bind'
      See the note in ?`:::` about the use of this operator.
    There are ::: calls to the package's namespace in its code. A package
      almost never needs to use ::: for its own objects:
      '.get_inosine_score'
    ```

# RnBeads

<details>

* Version: 2.8.0
* GitHub: NA
* Source code: https://github.com/cran/RnBeads
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 253

Run `revdep_details(, "RnBeads")` for more info

</details>

## In both

*   checking tests ...
    ```
    ...
      RUNIT TEST PROTOCOL -- Fri Nov 20 05:37:39 2020 
      *********************************************** 
      Number of test functions: 24 
      Number of errors: 1 
      Number of failures: 0 
      
       
      1 Test Suite : 
      RnBeads RUnit Tests - 24 test functions, 1 error, 0 failures
      ERROR in test_report_2: Error in .External2(C_X11, paste0("png::", filename), g$width, g$height,  : 
        unable to start device PNG
      
      Test files with failing tests
      
         test_report_2.R 
           test_report_2 
      
      
      Error in BiocGenerics:::testPackage("RnBeads") : 
        unit tests failed for package RnBeads
      Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      'BiocGenerics', 'S4Vectors', 'GenomicRanges', 'MASS', 'cluster',
      'ff', 'fields', 'ggplot2', 'gplots', 'gridExtra', 'limma',
      'matrixStats', 'illuminaio', 'methylumi', 'plyr'
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  9.0Mb
      sub-directories of 1Mb or more:
        R     3.1Mb
        doc   3.7Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Package listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘qvalue’
    A package should be listed in only one of these fields.
    ```

*   checking whether the namespace can be loaded with stated dependencies ... NOTE
    ```
    Warning: no function found corresponding to methods exports from ‘RnBeads’ for: ‘samples’
    
    A namespace must be able to be loaded with just the base namespace
    loaded: otherwise if the namespace gets loaded by a saved object, the
    session will be unable to start.
    
    Probably some imports need to be declared in the NAMESPACE file.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      'Gviz:::.getBMFeatureMap' 'doParallel:::.options'
      'grDevices:::.smoothScatterCalcDensity'
      'minfi:::.default.450k.annotation' 'minfi:::.extractFromRGSet450k'
      'minfi:::.normalizeFunnorm450k'
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      RGChannelSet Rd2HTML RefFreeEwasModel SNP Sample Segment Slide Target
      Term UcscTrack Value addSex as.profileCGH assayDataElement
      assayDataElementNames barcode browserSession bv calculateFDRs chrom
      clone combinedRank combinedRank.var comma covgMedian covgPercLow
      covgPercUp cv.glmnet daglad diffmeth diffmeth.p.adj.fdr
      diffmeth.p.val dinucleotideFrequency eps expectedCounts featureData
      featureData<- featureNames featureNames<- foreach geneCounts genome<-
      getCN getDoParWorkers getGreen getManifest getMeth getRed getSex
      getTable getUnmeth getVarCov glmnet grid.draw grid.newpage group
      group1 group2 i impute.knn install is.subsegmentation k
      letterFrequency lme loadRegionDB log10FDR log10P mapToGenome mclapply
      mean.diff mean.quot.log2 melt mergeRegionDBs muted n.sites num.sites
      numSites numeric.names oddsRatios p.vals.t.na.adj pData
      percent_format phenoData phenoData<- plotAlphaDistributionOneChr
      plotFinalSegmentation plotTracks preprocessSWAN pvalues qvalue
      readMethylome readSNPTable refText reg.type region.size
      registerDoParallel relative.coord removeSNPs report runLOLA samples
      segmentPMDs segmentUMRsLMRs seqlengths seqlevels<- sigCategories
      sites2ignore size solve.QP stopCluster sva target texthere tsne type
      types ucscTableQuery universeCounts useMart v var.diff varFit
      varLabels x x2 xmlValue y y2 yint
    ```

# scDataviz

<details>

* Version: 1.0.0
* GitHub: https://github.com/kevinblighe/scDataviz
* Source code: https://github.com/cran/scDataviz
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 185

Run `revdep_details(, "scDataviz")` for more info

</details>

## In both

*   checking tests ...
    ```
    ...
      Welcome to Bioconductor
      
          Vignettes contain introductory material; view with
          'browseVignettes()'. To cite Bioconductor, see
          'citation("Biobase")', and for packages 'citation("pkgname")'.
      
      
      Attaching package: 'Biobase'
      
      The following object is masked from 'package:MatrixGenerics':
      
          rowMedians
      
      The following objects are masked from 'package:matrixStats':
      
          anyMissing, rowMedians
      
      Error in library("RUnit", quietly = TRUE) : 
        there is no package called 'RUnit'
      Calls: <Anonymous> -> library
      Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.6Mb
      sub-directories of 1Mb or more:
        doc   5.7Mb
    ```

# scmap

<details>

* Version: 1.12.0
* GitHub: https://github.com/hemberg-lab/scmap
* Source code: https://github.com/cran/scmap
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 90

Run `revdep_details(, "scmap")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.8Mb
      sub-directories of 1Mb or more:
        data   2.6Mb
        doc    1.1Mb
        libs   1.9Mb
    ```

*   checking top-level files ... NOTE
    ```
    File
      LICENSE
    is not mentioned in the DESCRIPTION file.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘Biobase’
      All declared Imports should be used.
    ```

# scone

<details>

* Version: 1.14.0
* GitHub: https://github.com/YosefLab/scone
* Source code: https://github.com/cran/scone
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 224

Run `revdep_details(, "scone")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    ...
    sconeReport : server: no visible global function definition for ‘theme’
    sconeReport : server: no visible global function definition for
      ‘element_blank’
    sconeReport : server: no visible global function definition for
      ‘ggplotly’
    sconeReport : server: no visible global function definition for
      ‘geom_violin’
    sconeReport : server: no visible global function definition for
      ‘coord_cartesian’
    sconeReport : server: no visible global function definition for
      ‘scale_fill_manual’
    sconeReport : server: no visible global function definition for
      ‘geom_point’
    sconeReport : server: no visible global function definition for
      ‘guides’
    Undefined global functions or variables:
      %>% aes coord_cartesian element_blank geom_bar geom_point geom_violin
      ggplot ggplotly guides labs plot_ly plotlyOutput renderVisNetwork
      scale_fill_manual theme visEdges visGroups visHierarchicalLayout
      visLegend visNetwork visNetworkOutput visNetworkProxy visOptions
      visSelectNodes ylim
    ```

# scp

<details>

* Version: 1.0.0
* GitHub: https://github.com/UCLouvain-CBIO/scp
* Source code: https://github.com/cran/scp
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 151

Run `revdep_details(, "scp")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      ‘MultiAssayExperiment:::.sampleMapFromData’ ‘QFeatures:::.get_Hits’
      See the note in ?`:::` about the use of this operator.
    ```

# scPCA

<details>

* Version: 1.4.0
* GitHub: https://github.com/PhilBoileau/scPCA
* Source code: https://github.com/cran/scPCA
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 190

Run `revdep_details(, "scPCA")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘DelayedArray’
      All declared Imports should be used.
    ```

# sctransform

<details>

* Version: 0.3.1
* GitHub: https://github.com/ChristophH/sctransform
* Source code: https://github.com/cran/sctransform
* Date/Publication: 2020-10-08 12:00:02 UTC
* Number of recursive dependencies: 70

Run `revdep_details(, "sctransform")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package which this enhances but not available for checking: ‘glmGamPoi’
    ```

# seeds

<details>

* Version: 0.9.1
* GitHub: https://github.com/Newmi1988/seeds
* Source code: https://github.com/cran/seeds
* Date/Publication: 2020-07-14 00:00:02 UTC
* Number of recursive dependencies: 99

Run `revdep_details(, "seeds")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘rsbml’
    ```

# semtree

<details>

* Version: 0.9.14
* GitHub: NA
* Source code: https://github.com/cran/semtree
* Date/Publication: 2020-01-07 22:00:02 UTC
* Number of recursive dependencies: 89

Run `revdep_details(, "semtree")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘expm’ ‘matrixStats’ ‘matrixcalc’
      All declared Imports should be used.
    ```

# sesame

<details>

* Version: 1.8.2
* GitHub: https://github.com/zwdzwd/sesame
* Source code: https://github.com/cran/sesame
* Date/Publication: 2020-11-13
* Number of recursive dependencies: 176

Run `revdep_details(, "sesame")` for more info

</details>

## In both

*   checking tests ...
    ```
    ...
      > test_check("sesame")
      ── Warning (test_sesamize.R:16:5): RGChannelSetToSigSet gives correct results ──
      there is no package called 'IlluminaHumanMethylation450kmanifest'
      
      ── ERROR (test_sesamize.R:16:5): RGChannelSetToSigSet gives correct results ────
      Error: cannot load manifest package IlluminaHumanMethylation450kmanifest
      Backtrace:
          █
       1. └─minfi::preprocessRaw(rgSet) test_sesamize.R:16:4
       2.   └─minfi::getManifestInfo(rgSet, "locusNames")
       3.     └─minfi::getProbeInfo(object, type = "I")
       4.       ├─minfi::getManifest(object)
       5.       └─minfi::getManifest(object)
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      Warning (test_sesamize.R:16:5): RGChannelSetToSigSet gives correct results
      ERROR (test_sesamize.R:16:5): RGChannelSetToSigSet gives correct results
      
      [ FAIL 1 | WARN 1 | SKIP 0 | PASS 15 ]
      Error: Test failures
      Execution halted
    ```

# Seurat

<details>

* Version: 3.2.2
* GitHub: https://github.com/satijalab/seurat
* Source code: https://github.com/cran/Seurat
* Date/Publication: 2020-09-26 04:30:12 UTC
* Number of recursive dependencies: 228

Run `revdep_details(, "Seurat")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘loomR’
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 11.1Mb
      sub-directories of 1Mb or more:
        R      1.2Mb
        libs   9.1Mb
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘loomR’
    ```

# SGP

<details>

* Version: 1.9-5.0
* GitHub: https://github.com/CenterForAssessment/SGP
* Source code: https://github.com/cran/SGP
* Date/Publication: 2020-01-30 22:40:09 UTC
* Number of recursive dependencies: 112

Run `revdep_details(, "SGP")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.7Mb
      sub-directories of 1Mb or more:
        R     1.3Mb
        doc   3.4Mb
    ```

# shinyMethyl

<details>

* Version: 1.26.0
* GitHub: NA
* Source code: https://github.com/cran/shinyMethyl
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 143

Run `revdep_details(, "shinyMethyl")` for more info

</details>

## In both

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    server.shinyMethyl : <anonymous>: no visible global function definition
      for ‘lines’
    server.shinyMethyl : <anonymous>: no visible global function definition
      for ‘legend’
    ui.shinyMethyl: no visible binding for '<<-' assignment to
      ‘sampleColors’
    shinySummarize,GenomicRatioSet: no visible global function definition
      for ‘prcomp’
    shinySummarize,RGChannelSet: no visible global function definition for
      ‘prcomp’
    Undefined global functions or variables:
      abline complete.cases current.control.type current.density.type
      current.probe.type genderCutoff grid legend lines lm
      mouse.click.indices palette par points prcomp rect write.csv
    Consider adding
      importFrom("grDevices", "palette")
      importFrom("graphics", "abline", "grid", "legend", "lines", "par",
                 "points", "rect")
      importFrom("stats", "complete.cases", "lm", "prcomp")
      importFrom("utils", "write.csv")
    to your NAMESPACE file.
    ```

# SIAMCAT

<details>

* Version: 1.10.0
* GitHub: NA
* Source code: https://github.com/cran/SIAMCAT
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 116

Run `revdep_details(, "SIAMCAT")` for more info

</details>

## In both

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  8.6Mb
      sub-directories of 1Mb or more:
        doc       3.3Mb
        extdata   2.0Mb
        help      1.9Mb
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘randomForest’
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 10 marked UTF-8 strings
    ```

# SICtools

<details>

* Version: 1.20.0
* GitHub: NA
* Source code: https://github.com/cran/SICtools
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 40

Run `revdep_details(, "SICtools")` for more info

</details>

## In both

*   checking whether package ‘SICtools’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      errmod.c:79:8: warning: self-comparison always evaluates to false [-Wtautological-compare]
      md5.c:155:26: warning: argument to ‘sizeof’ in ‘memset’ call is the same expression as the destination; did you mean to dereference it? [-Wsizeof-pointer-memaccess]
      Warning: replacing previous import ‘plyr::count’ by ‘matrixStats::count’ when loading ‘SICtools’
    See ‘/c4/home/henrik/repositories/matrixStats/revdep/checks/SICtools/new/SICtools.Rcheck/00install.out’ for details.
    ```

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      'Rsamtools', 'doParallel', 'Biostrings', 'stringr', 'matrixStats',
      'plyr', 'GenomicRanges', 'IRanges'
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .BBSoptions
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    .indelDiffFunc: no visible global function definition for ‘fisher.test’
    .indelDiffFunc: no visible global function definition for ‘dist’
    indelDiff: no visible global function definition for ‘read.delim’
    snpDiff : calcInfoRange : <anonymous>: no visible global function
      definition for ‘fisher.test’
    snpDiff : calcInfoRange : <anonymous>: no visible global function
      definition for ‘dist’
    Undefined global functions or variables:
      dist fisher.test read.delim
    Consider adding
      importFrom("stats", "dist", "fisher.test")
      importFrom("utils", "read.delim")
    to your NAMESPACE file.
    ```

# sigminer

<details>

* Version: 1.1.0
* GitHub: https://github.com/ShixiangWang/sigminer
* Source code: https://github.com/cran/sigminer
* Date/Publication: 2020-11-11 07:40:06 UTC
* Number of recursive dependencies: 197

Run `revdep_details(, "sigminer")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.8Mb
      sub-directories of 1Mb or more:
        extdata   3.9Mb
    ```

# sindyr

<details>

* Version: 0.2.3
* GitHub: NA
* Source code: https://github.com/cran/sindyr
* Date/Publication: 2020-06-09 05:20:03 UTC
* Number of recursive dependencies: 87

Run `revdep_details(, "sindyr")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘crqa’ ‘plot3D’
      All declared Imports should be used.
    ```

# Single.mTEC.Transcriptomes

<details>

* Version: 1.18.0
* GitHub: NA
* Source code: https://github.com/cran/Single.mTEC.Transcriptomes
* Date/Publication: 2020-10-29
* Number of recursive dependencies: 167

Run `revdep_details(, "Single.mTEC.Transcriptomes")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 904.4Mb
      sub-directories of 1Mb or more:
        data  895.1Mb
        doc     9.3Mb
    ```

# singleCellTK

<details>

* Version: 2.0.0
* GitHub: https://github.com/compbiomed/singleCellTK
* Source code: https://github.com/cran/singleCellTK
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 332

Run `revdep_details(, "singleCellTK")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    ...
    The error most likely occurred in:
    
    > ### Name: importExampleData
    > ### Title: Retrieve example datasets
    > ### Aliases: importExampleData
    > 
    > ### ** Examples
    > 
    > sce <- importExampleData("pbmc3k")
    snapshotDate(): 2020-10-02
    see ?TENxPBMCData and browseVignettes('TENxPBMCData') for documentation
    Error: failed to load resource
      name: EH1607
      title: PBMC, 3k scRNA-seq gene (row) annotation
      reason: Corrupt Cache: resource path
      See vignette section on corrupt cache
      cache: /c4/home/henrik/.cache/ExperimentHub
      potential duplicate files: 
        b93f7382989a_3697
        b96e433cd1a3_3697
    Execution halted
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      'BiocGenerics' 'ggplotify' 'kableExtra' 'shinyBS' 'shinyFiles'
      'shinyWidgets' 'shinyjqui' 'shinythemes'
      All declared Imports should be used.
    ```

# singscore

<details>

* Version: 1.10.0
* GitHub: https://github.com/DavisLaboratory/singscore
* Source code: https://github.com/cran/singscore
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 123

Run `revdep_details(, "singscore")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘singscore-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: plotDispersion
    > ### Title: Plot the score v.s. despersion for all samples
    > ### Aliases: plotDispersion
    > 
    > ### ** Examples
    > 
    > ranked <- rankGenes(toy_expr_se)
    > scoredf <- simpleScore(ranked, upSet = toy_gs_up, downSet = toy_gs_dn)
    > plotDispersion(scoredf)
    > plotDispersion(scoredf, isInteractive = TRUE)
    Warning in dev_fun(file = tempfile(), width = width %||% 640, height = height %||%  :
      unable to open connection to X11 display ''
    Error in .External2(C_X11, paste0("png::", filename), g$width, g$height,  : 
      unable to start device PNG
    Calls: plotDispersion ... <Anonymous> -> ggplotly.ggplot -> gg2list -> dev_fun
    Execution halted
    ```

*   checking tests ...
    ```
    ...
      Warning (test-plotRankDens.R:6:3): plotRankDensity is plotting
      Warning (test-plotRankDens.R:18:3): Generic features for plotRankDensity
      Warning (test-plotRankDens.R:19:3): Generic features for plotRankDensity
      Warning (test-plotRankDens.R:20:3): Generic features for plotRankDensity
      Warning (test-plotRankDens.R:22:3): Generic features for plotRankDensity
      Warning (test-projectScoreLandscape.R:42:2): projectScoreLandscape colours work for different annotations
      Warning (test-projectScoreLandscape.R:45:2): projectScoreLandscape colours work for different annotations
      Warning (test-projectScoreLandscape.R:48:2): projectScoreLandscape colours work for different annotations
      Warning (test-projectScoreLandscape.R:51:2): projectScoreLandscape colours work for different annotations
      Warning (test-projectScoreLandscape.R:57:2): projectScoreLandscape colours work for different annotations
      Warning (test-projectScoreLandscape.R:70:2): projectScoreLandscape works for interactive plots
      Warning (test-projectScoreLandscape.R:70:2): projectScoreLandscape works for interactive plots
      ERROR (test-projectScoreLandscape.R:70:2): projectScoreLandscape works for interactive plots
      Warning (test-scoring.R:12:3): checkGenes works
      Warning (test-scoring.R:13:3): checkGenes works
      Warning (test-scoring.R:65:3): score calculation works well when direction is unknown
      Warning (test-scoring.R:110:3): scoring using stable genes
      
      [ FAIL 3 | WARN 25 | SKIP 0 | PASS 123 ]
      Error: Test failures
      Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.3Mb
      sub-directories of 1Mb or more:
        R      1.2Mb
        data   1.9Mb
        doc    1.8Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    plotDispersion: no visible binding for global variable ‘Score’
    plotDispersion: no visible binding for global variable ‘Dispersion’
    plotDispersion: no visible binding for global variable ‘Class’
    plotDispersion: no visible binding for global variable ‘SampleText’
    plotDispersion: no visible binding for global variable ‘SampleLabel’
    plotRankDensity_intl: no visible binding for global variable ‘Ranks’
    plotRankDensity_intl: no visible binding for global variable ‘upDown’
    plotRankDensity_intl: no visible binding for global variable
      ‘..density..’
    plotRankDensity_intl: no visible binding for global variable ‘EntrezID’
    plotScoreLandscape: no visible binding for global variable ‘sc1’
    plotScoreLandscape: no visible binding for global variable ‘sc2’
    projectScoreLandscape: no visible binding for global variable
      ‘SampleText’
    projectScoreLandscape: no visible binding for global variable ‘Class’
    projectScoreLandscape: no visible binding for global variable
      ‘SampleLabel’
    Undefined global functions or variables:
      ..density.. Class Dispersion EntrezID Ranks SampleLabel SampleText
      Score sc1 sc2 upDown
    ```

# slingshot

<details>

* Version: 1.8.0
* GitHub: https://github.com/kstreet13/slingshot
* Source code: https://github.com/cran/slingshot
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 188

Run `revdep_details(, "slingshot")` for more info

</details>

## In both

*   checking for unstated dependencies in vignettes ... NOTE
    ```
    '::' or ':::' import not declared from: ‘mgcv’
    'library' or 'require' call not declared from: ‘mgcv’
    ```

# spathial

<details>

* Version: 0.1.2
* GitHub: NA
* Source code: https://github.com/cran/spathial
* Date/Publication: 2020-04-10 18:20:02 UTC
* Number of recursive dependencies: 29

Run `revdep_details(, "spathial")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘irlba’ ‘knitr’ ‘rmarkdown’
      All declared Imports should be used.
    ```

# splatter

<details>

* Version: 1.14.0
* GitHub: https://github.com/Oshlack/splatter
* Source code: https://github.com/cran/splatter
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 241

Run `revdep_details(, "splatter")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  9.8Mb
      sub-directories of 1Mb or more:
        doc   8.9Mb
    ```

# spqn

<details>

* Version: 1.2.0
* GitHub: https://github.com/hansenlab/spqn
* Source code: https://github.com/cran/spqn
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 84

Run `revdep_details(, "spqn")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Package in Depends field not imported from: ‘BiocGenerics’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

# SRGnet

<details>

* Version: 1.16.0
* GitHub: NA
* Source code: https://github.com/cran/SRGnet
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 104

Run `revdep_details(, "SRGnet")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      'EBcoexpress', 'MASS', 'igraph', 'pvclust', 'gbm', 'limma', 'DMwR',
      'matrixStats', 'Hmisc'
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking R code for possible problems ... NOTE
    ```
    SRGnet: no visible binding for global variable
      ‘Differentially_expressed_genes’
    SRGnet: no visible binding for global variable ‘Transcriptomics’
    SRGnet: no visible binding for global variable ‘PLCRG’
    SRGnet: no visible global function definition for ‘ebPatterns’
    SRGnet: no visible global function definition for ‘par’
    SRGnet: no visible global function definition for ‘boxplot’
    SRGnet: no visible global function definition for ‘predict’
    SRGnet: no visible global function definition for ‘terrain.colors’
    SRGnet: no visible global function definition for ‘na.omit’
    SRGnet: no visible global function definition for ‘write.table’
    Undefined global functions or variables:
      Differentially_expressed_genes PLCRG Transcriptomics boxplot
      ebPatterns na.omit par predict terrain.colors write.table
    Consider adding
      importFrom("grDevices", "terrain.colors")
      importFrom("graphics", "boxplot", "par")
      importFrom("stats", "na.omit", "predict")
      importFrom("utils", "write.table")
    to your NAMESPACE file.
    ```

# stm

<details>

* Version: 1.3.6
* GitHub: https://github.com/bstewart/stm
* Source code: https://github.com/cran/stm
* Date/Publication: 2020-09-18 13:50:02 UTC
* Number of recursive dependencies: 124

Run `revdep_details(, "stm")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.4Mb
      sub-directories of 1Mb or more:
        data   1.7Mb
        libs   2.6Mb
    ```

# STROMA4

<details>

* Version: 1.14.0
* GitHub: NA
* Source code: https://github.com/cran/STROMA4
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 12

Run `revdep_details(, "STROMA4")` for more info

</details>

## In both

*   checking running R code from vignettes ...
    ```
      ‘STROMA4-vignette.Rnw’... failed
     ERROR
    Errors in running code in vignettes:
    when running code in ‘STROMA4-vignette.Rnw’
      ...
    
    > if (!requireNamespace("BiocManager", quietly = TRUE)) install.packages("BiocManager")
    Installing package into ‘/c4/home/henrik/repositories/matrixStats/revdep/checks/STROMA4/new/STROMA4.Rcheck’
    (as ‘lib’ is unspecified)
    
      When sourcing ‘STROMA4-vignette.R’:
    Error: trying to use CRAN without setting a mirror
    Execution halted
    ```

*   checking for unstated dependencies in vignettes ... NOTE
    ```
    'library' or 'require' call not declared from: ‘BiocManager’
    ```

# summarytools

<details>

* Version: 0.9.6
* GitHub: https://github.com/dcomtois/summarytools
* Source code: https://github.com/cran/summarytools
* Date/Publication: 2020-03-02 07:20:02 UTC
* Number of recursive dependencies: 70

Run `revdep_details(, "summarytools")` for more info

</details>

## In both

*   checking whether package ‘summarytools’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: no DISPLAY variable so Tk is not available
    See ‘/c4/home/henrik/repositories/matrixStats/revdep/checks/summarytools/new/summarytools.Rcheck/00install.out’ for details.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 78 marked UTF-8 strings
    ```

# SuperPCA

<details>

* Version: 0.3.0
* GitHub: NA
* Source code: https://github.com/cran/SuperPCA
* Date/Publication: 2020-03-04 11:10:02 UTC
* Number of recursive dependencies: 37

Run `revdep_details(, "SuperPCA")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘R.matlab’ ‘fBasics’ ‘spls’
      All declared Imports should be used.
    ```

# TCA

<details>

* Version: 1.1.0
* GitHub: https://github.com/cozygene/TCA
* Source code: https://github.com/cran/TCA
* Date/Publication: 2019-11-16 17:10:02 UTC
* Number of recursive dependencies: 71

Run `revdep_details(, "TCA")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘Matrix’ ‘glmnet’ ‘quadprog’
      All declared Imports should be used.
    ```

# tenXplore

<details>

* Version: 1.12.0
* GitHub: NA
* Source code: https://github.com/cran/tenXplore
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 121

Run `revdep_details(, "tenXplore")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    apd: warning in dir(system.file("app", package = "tenXplore"), full =
      TRUE): partial argument match of 'full' to 'full.names'
    ```

# topGO

<details>

* Version: 2.42.0
* GitHub: NA
* Source code: https://github.com/cran/topGO
* Date/Publication: 2020-10-30
* Number of recursive dependencies: 47

Run `revdep_details(, "topGO")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      'BiocGenerics', 'graph', 'Biobase', 'GO.db', 'AnnotationDbi',
      'SparseM'
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' calls in package code:
      ‘Rgraphviz’ ‘multtest’
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    Missing object imported by a ':::' call: ‘globaltest:::globaltest’
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    GOplot.counts : plotSigChart: no visible global function definition for
      ‘getY’
    GOplot.counts: no visible global function definition for
      ‘getDefaultAttrs’
    GOplot.counts: no visible global function definition for ‘agopen’
    getPvalues: no visible global function definition for ‘mt.teststat’
    getPvalues: no visible global function definition for ‘mt.rawp2adjp’
    printDOT: no visible global function definition for ‘getDefaultAttrs’
    printDOT: no visible global function definition for ‘toDot’
    GOSumTest,classicScore: no visible binding for global variable
      ‘.PERMSUM.MAT’
    GOSumTest,classicScore: no visible binding for global variable
      ‘.PERMSUM.LOOKUP’
    initialize,classicExpr: no visible global function definition for
      ‘error’
    scoresInTerm,topGOdata-missing: no visible global function definition
      for ‘scoreInNode’
    Undefined global functions or variables:
      .PERMSUM.LOOKUP .PERMSUM.MAT AgNode agopen drawTxtLabel error
      getDefaultAttrs getNodeCenter getNodeLW getNodeXY getX getY
      mt.rawp2adjp mt.teststat name pieGlyph scoreInNode toDot txtLabel
    ```

# tradeSeq

<details>

* Version: 1.4.0
* GitHub: https://github.com/statOmics/tradeSeq
* Source code: https://github.com/cran/tradeSeq
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 192

Run `revdep_details(, "tradeSeq")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.6Mb
      sub-directories of 1Mb or more:
        doc   6.6Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    .plotSmoothers: no visible binding for global variable ‘time’
    .plotSmoothers: no visible binding for global variable ‘gene_count’
    .plotSmoothers: no visible binding for global variable ‘lineage’
    .plotSmoothers_conditions: no visible binding for global variable
      ‘time’
    .plotSmoothers_conditions: no visible binding for global variable
      ‘gene_count’
    .plotSmoothers_conditions: no visible binding for global variable
      ‘lineage’
    .plotSmoothers_conditions: no visible binding for global variable
      ‘pCol’
    .plotSmoothers_sce: no visible binding for global variable ‘time’
    .plotSmoothers_sce: no visible binding for global variable ‘gene_count’
    .plotSmoothers_sce: no visible binding for global variable ‘lineage’
    .plotSmoothers_sce: no visible binding for global variable ‘pCol’
    extract_monocle_info: no visible binding for global variable ‘.’
    Undefined global functions or variables:
      . X1 X2 dim1 dim2 gene_count l1 lineage pCol t1 time
    Consider adding
      importFrom("stats", "time")
    to your NAMESPACE file.
    ```

# treeHMM

<details>

* Version: 0.1.1
* GitHub: NA
* Source code: https://github.com/cran/treeHMM
* Date/Publication: 2019-12-12 17:10:02 UTC
* Number of recursive dependencies: 11

Run `revdep_details(, "treeHMM")` for more info

</details>

## In both

*   checking whether package ‘treeHMM’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: Strategy 'multiprocess' is deprecated in future (>= 1.20.0). Instead, explicitly specify either 'multisession' or 'multicore'. In the current R session, 'multiprocess' equals 'multicore'.
    See ‘/c4/home/henrik/repositories/matrixStats/revdep/checks/treeHMM/new/treeHMM.Rcheck/00install.out’ for details.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘Matrix’
      All declared Imports should be used.
    ```

# VanillaICE

<details>

* Version: 1.52.0
* GitHub: NA
* Source code: https://github.com/cran/VanillaICE
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 71

Run `revdep_details(, "VanillaICE")` for more info

</details>

## In both

*   checking Rd files ... WARNING
    ```
    checkRd: (5) VanillaICE.Rd:0-7: Must have a \description
    ```

*   checking running R code from vignettes ...
    ```
      ‘VanillaICE.Rnw’ using ‘UTF-8’... OK
      ‘crlmmDownstream.Rnw’ using ‘UTF-8’... failed
     WARNING
    Errors in running code in vignettes:
    when running code in ‘crlmmDownstream.Rnw’
      ...
    > data(cnSetExample, package = "crlmm")
    
    > se <- as(cnSetExample, "SnpArrayExperiment")
    Warning: executing %dopar% sequentially: no parallel backend registered
    
    > library(ArrayTV)
    
      When sourcing ‘crlmmDownstream.R’:
    Error: there is no package called ‘ArrayTV’
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘ArrayTV’
    
    Packages which this enhances but not available for checking:
      'doMC', 'doMPI', 'doSNOW', 'doParallel', 'doRedis'
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘BSgenome.Hsapiens.UCSC.hg18’ ‘MatrixGenerics’
      All declared Imports should be used.
    ```

# vasp

<details>

* Version: 1.2.0
* GitHub: https://github.com/yuhuihui2011/vasp
* Source code: https://github.com/cran/vasp
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 102

Run `revdep_details(, "vasp")` for more info

</details>

## In both

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Authors@R field gives more than one person with maintainer role:
      Huihui Yu <yuhuihui2011@foxmail.com> [aut, cre]
      Qian Du [aut, cre]
    ```

# wateRmelon

<details>

* Version: 1.34.0
* GitHub: NA
* Source code: https://github.com/cran/wateRmelon
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 166

Run `revdep_details(, "wateRmelon")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      'Biobase', 'limma', 'matrixStats', 'methylumi', 'lumi', 'ROC',
      'IlluminaHumanMethylation450kanno.ilmn12.hg19', 'illuminaio'
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Package listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘Biobase’
    A package should be listed in only one of these fields.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Packages in Depends field not imported from:
      ‘IlluminaHumanMethylation450kanno.ilmn12.hg19’ ‘matrixStats’
      ‘methylumi’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    Unexported objects imported by ':::' calls:
      ‘minfi:::.isRGOrStop’ ‘minfi:::pickCompProbes’
      ‘minfi:::projectCellType’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    tost,MethyLumiSet: no visible global function definition for
      ‘unmethylated’
    tost,MethyLumiSet: no visible global function definition for ‘pvals’
    tost,RGChannelSet: no visible global function definition for
      ‘detectionP’
    tost,RGChannelSet: no visible global function definition for
      ‘preprocessRaw’
    tost,RGChannelSet: no visible global function definition for ‘getMeth’
    tost,RGChannelSet: no visible global function definition for
      ‘getUnmeth’
    Undefined global functions or variables:
      .mclapply IlluminaHumanMethylation450kanno.ilmn12.hg19 MethylSet
      PATH_RES QCdata QCdata<- RGChannelSetExtended betas betas<- colData
      colMedians detectionP epic.controls epic.ordering getAnnotation
      getAnnotationObject getBeta getControlAddress getGreen
      getManifestInfo getMeth getNBeads getProbeInfo getProbeType getRed
      getSnpBeta getUnmeth hm27.controls hm27.ordering hm450.controls
      hm450.ordering intensitiesByChannel metadata methylated methylated<-
      methylumiR n.sd preprocessRaw preprocessSWAN probeAnnotationsCategory
      projectName pval.detect<- pvals subsetByLoci unmethylated
      unmethylated<- warn
    ```

# XBSeq

<details>

* Version: 1.22.0
* GitHub: https://github.com/Liuy12/XBSeq
* Source code: https://github.com/cran/XBSeq
* Date/Publication: 2020-10-27
* Number of recursive dependencies: 121

Run `revdep_details(, "XBSeq")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘DESeq’
    ```

*   checking whether the namespace can be loaded with stated dependencies ... NOTE
    ```
    Warning: no function found corresponding to methods exports from ‘XBSeq’ for: ‘conditions’, ‘conditions<-’, ‘dispTable’
    
    A namespace must be able to be loaded with just the base namespace
    loaded: otherwise if the namespace gets loaded by a saved object, the
    session will be unable to start.
    
    Probably some imports need to be declared in the NAMESPACE file.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    estimateRealCount,XBSeqDataSet: no visible global function definition
      for ‘assay’
    estimateRealCount,XBSeqDataSet: no visible global function definition
      for ‘assay<-’
    estimateSCV,XBSeqDataSet: no visible global function definition for
      ‘conditions’
    estimateSCV,XBSeqDataSet: no visible global function definition for
      ‘dispTable<-’
    Undefined global functions or variables:
      ..count.. DataFrame Gamma Group Sample SummarizedExperiment assay
      assay<- assays baseMean coefficients complete.cases conditions cor
      data ddelap dispTable dispTable<- dnbinom dpois formula glm
      log2FoldChange median optim p.adjust pbeta predict qbeta quantile
      rnbinom scvBiasCorrectionFits
    Consider adding
      importFrom("stats", "Gamma", "coefficients", "complete.cases", "cor",
                 "dnbinom", "dpois", "formula", "glm", "median", "optim",
                 "p.adjust", "pbeta", "predict", "qbeta", "quantile",
                 "rnbinom")
      importFrom("utils", "data")
    to your NAMESPACE file.
    ```

