# abcrf

Version: 1.7

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Jean-Michel Marin <jean-michel.marin@umontpellier.fr>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.3Mb
      sub-directories of 1Mb or more:
        data   2.6Mb
        libs   2.5Mb
    ```

# ACNE

Version: 0.8.1

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Henrik Bengtsson <henrikb@braju.com>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# adjclust

Version: 0.5.2

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Pierre Neuvial <pierre.neuvial@math.univ-toulouse.fr>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘Matrix’
      All declared Imports should be used.
    ```

# amplican

Version: 1.0.0

## In both

*   checking Rd cross-references ... WARNING
    ```
    Unknown package ‘CrispRVariants’ in Rd xrefs
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Eivind Valen <eivind.valen@gmail.com>’
    
    The Title field should be in title case, current version then in title case:
    ‘Automated analysis of CRISPR experiments.’
    ‘Automated Analysis of CRISPR Experiments.’
    
    The Description field should start with a capital letter.
    
    This build time stamp is over a month old.
    
    Size of tarball: 7170712 bytes
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 14.3Mb
      sub-directories of 1Mb or more:
        doc  13.6Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Title field: should not end in a period.
    ```

# antiProfiles

Version: 1.18.0

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Hector Corrada Bravo <hcorrada@gmail.com>’
    
    The Title field should be in title case, current version then in title case:
    ‘Implementation of gene expression anti-profiles’
    ‘Implementation of Gene Expression Anti-Profiles’
    
    This build time stamp is over a month old.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ev: no visible global function definition for ‘predict’
    ev: no visible global function definition for ‘smoothScatter’
    ev : f1: no visible global function definition for ‘predict’
    ev: no visible global function definition for ‘curve’
    Undefined global functions or variables:
      curve predict smoothScatter
    Consider adding
      importFrom("graphics", "curve", "smoothScatter")
      importFrom("stats", "predict")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    Computing sds
    Computing ranges
    Computing tissue specificity
    Computing normal tissue outside ranges
    Making probe data frame
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'antiProfiles.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `fullpage.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.13 \usepackage
                    [numbers]{natbib}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# aroma.affymetrix

Version: 3.1.0

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Henrik Bengtsson <henrikb@braju.com>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# aroma.cn

Version: 1.6.1

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Henrik Bengtsson <henrikb@braju.com>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# aroma.core

Version: 3.1.1

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Henrik Bengtsson <henrikb@braju.com>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      ‘sfit’ ‘expectile’ ‘HaarSeg’ ‘mpcbs’
    ```

# aroma.light

Version: 3.8.0

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Henrik Bengtsson <henrikb@braju.com>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      inst/rsp/.rspPlugins
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    
    CRAN-pack does not know about
      inst/rsp/.rspPlugins
    ```

# aSPU

Version: 1.48

## In both

*   checking top-level files ... WARNING
    ```
    Conversion of ‘README.md’ failed:
    pandoc: Could not fetch http://www.r-pkg.org/badges/version/aSPU
    FailedConnectionException2 "www.r-pkg.org" 80 False getAddrInfo: does not exist (Name or service not known)
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Il-Youp Kwak <ikwak@umn.edu>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# BASiCS

Version: 1.0.0

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Catalina A. Vallejos <cnvallej@uc.cl>’
    
    The Title field should be in title case, current version then in title case:
    ‘Bayesian Analysis of Single-Cell Sequencing data’
    ‘Bayesian Analysis of Single-Cell Sequencing Data’
    
    The Date field is not in ISO 8601 yyyy-mm-dd format.
    
    This build time stamp is over a month old.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  7.3Mb
      sub-directories of 1Mb or more:
        data   1.8Mb
        libs   4.4Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Maintainer field differs from that derived from Authors@R
      Maintainer: ‘Catalina A. Vallejos <cnvallej@uc.cl>’
      Authors@R:  ‘Catalina Vallejos <cnvallej@uc.cl>’
    ```

# BatchQC

Version: 1.6.1

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Found3batches
    Adjusting for1covariate(s) or covariate level(s)
    Fitting L/S model and finding priors
    Finding parametric adjustments
    Adjusting the Data
    
    ! LaTeX Error: File `framed.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    Enter file name: 
    ! Emergency stop.
    <read *> 
             
    l.39 \definecolor
    
    pandoc: Error producing PDF
    Error: processing vignette 'BatchQC_usage_advanced.Rmd' failed with diagnostics:
    pandoc document conversion failed with error 43
    Execution halted
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Solaiappan Manimaran <manimaran_1975@hotmail.com>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking R code for possible problems ... NOTE
    ```
    gene_plot: no visible global function definition for ‘boxplot’
    gene_plot: no visible binding for global variable ‘batch’
    gls.series.C: no visible global function definition for ‘lm.fit’
    sample_plot: no visible global function definition for ‘boxplot’
    sample_plot: no visible binding for global variable ‘batch’
    Undefined global functions or variables:
      batch boxplot lm.fit
    Consider adding
      importFrom("graphics", "boxplot")
      importFrom("stats", "lm.fit")
    to your NAMESPACE file.
    ```

# BayesTwin

Version: 1.0

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘rjags’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Inga Schwabe <bayestwin@gmail.com>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# bdynsys

Version: 1.3

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Viktoria Spaiser <viktoria.sp@web.de>’
    
    The Description field should not start with the package name,
      'This package' or similar.
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    plot_data: no visible global function definition for ‘dev.set’
    plot_data: no visible global function definition for ‘postscript’
    plot_data: no visible global function definition for ‘plot’
    plot_data: no visible global function definition for ‘matplot’
    plot_data: no visible global function definition for ‘points’
    plot_data: no visible global function definition for ‘legend’
    plot_data: no visible global function definition for ‘dev.off’
    polyfitbayes: no visible global function definition for ‘runif’
    polyfitbayes: no visible global function definition for ‘var’
    polyfitreg: no visible global function definition for ‘lm’
    polyfitreg: no visible binding for global variable ‘na.exclude’
    polyfitreg: no visible global function definition for ‘var’
    Undefined global functions or variables:
      dev.off dev.set grid legend lm matplot na.exclude plot points
      postscript runif sd var write.table
    Consider adding
      importFrom("grDevices", "dev.off", "dev.set", "postscript")
      importFrom("graphics", "grid", "legend", "matplot", "plot", "points")
      importFrom("stats", "lm", "na.exclude", "runif", "sd", "var")
      importFrom("utils", "write.table")
    to your NAMESPACE file.
    ```

# bigstep

Version: 0.7.4

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Piotr Szulc <piotr.michal.szulc@gmail.com>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# bingat

Version: 1.3

## In both

*   checking Rd cross-references ... WARNING
    ```
    Unknown package ‘genalg’ in Rd xrefs
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Berkley Shands <rpackages@biorankings.com>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# bmrm

Version: 3.4

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Julien Prados <julien.prados@unige.ch>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# bnbc

Version: 1.0.0

## In both

*   checking whether package ‘bnbc’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/netapp/home/hb/repositories/matrixStats/revdep/checks/bnbc/new/bnbc.Rcheck/00install.out’ for details.
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Kipper Fletez-Brant <cafletezbrant@gmail.com>’
    
    The Title field should be in title case, current version then in title case:
    ‘Bandwise normalization and batch correction of Hi-C data’
    ‘Bandwise Normalization and Batch Correction of Hi-C Data’
    
    This build time stamp is over a month old.
    ```

## Installation

### Devel

```
* installing *source* package ‘bnbc’ ...
** libs
g++  -I/netapp/home/hb/shared/apps/manual/r/R-3.4.3-20180109/lib64/R/include -DNDEBUG  -I"/netapp/home/hb/repositories/matrixStats/revdep/library/bnbc/Rcpp/include" -I/usr/local/include   -fpic  -g -O2  -c RcppExports.cpp -o RcppExports.o
g++  -I/netapp/home/hb/shared/apps/manual/r/R-3.4.3-20180109/lib64/R/include -DNDEBUG  -I"/netapp/home/hb/repositories/matrixStats/revdep/library/bnbc/Rcpp/include" -I/usr/local/include   -fpic  -g -O2  -c update_bands.cpp -o update_bands.o
g++ -shared -L/usr/local/lib64 -o bnbc.so RcppExports.o update_bands.o
installing to /netapp/home/hb/repositories/matrixStats/revdep/checks/bnbc/new/bnbc.Rcheck/bnbc/libs
** R
** data
** inst
** preparing package for lazy loading
Error in loadNamespace(j <- i[[1L]], c(lib.loc, .libPaths()), versionCheck = vI[[j]]) : 
  there is no package called ‘tiff’
ERROR: lazy loading failed for package ‘bnbc’
* removing ‘/netapp/home/hb/repositories/matrixStats/revdep/checks/bnbc/new/bnbc.Rcheck/bnbc’

```
### CRAN

```
* installing *source* package ‘bnbc’ ...
** libs
g++  -I/netapp/home/hb/shared/apps/manual/r/R-3.4.3-20180109/lib64/R/include -DNDEBUG  -I"/netapp/home/hb/repositories/matrixStats/revdep/library/bnbc/Rcpp/include" -I/usr/local/include   -fpic  -g -O2  -c RcppExports.cpp -o RcppExports.o
g++  -I/netapp/home/hb/shared/apps/manual/r/R-3.4.3-20180109/lib64/R/include -DNDEBUG  -I"/netapp/home/hb/repositories/matrixStats/revdep/library/bnbc/Rcpp/include" -I/usr/local/include   -fpic  -g -O2  -c update_bands.cpp -o update_bands.o
g++ -shared -L/usr/local/lib64 -o bnbc.so RcppExports.o update_bands.o
installing to /netapp/home/hb/repositories/matrixStats/revdep/checks/bnbc/old/bnbc.Rcheck/bnbc/libs
** R
** data
** inst
** preparing package for lazy loading
Error in loadNamespace(j <- i[[1L]], c(lib.loc, .libPaths()), versionCheck = vI[[j]]) : 
  there is no package called ‘tiff’
ERROR: lazy loading failed for package ‘bnbc’
* removing ‘/netapp/home/hb/repositories/matrixStats/revdep/checks/bnbc/old/bnbc.Rcheck/bnbc’

```
# bnclassify

Version: 0.3.4

## In both

*   checking top-level files ... WARNING
    ```
    Conversion of ‘README.md’ failed:
    pandoc: Could not fetch https://travis-ci.org/bmihaljevic/bnclassify.svg?branch=master
    FailedConnectionException2 "travis-ci.org" 443 True getAddrInfo: does not exist (Name or service not known)
    ```

*   checking Rd cross-references ... WARNING
    ```
    Unknown package ‘memoise’ in Rd xrefs
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    ! LaTeX Error: File `framed.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    Enter file name: 
    ! Emergency stop.
    <read *> 
             
    l.39 \definecolor
    
    pandoc: Error producing PDF
    Error: processing vignette 'introduction.Rmd' failed with diagnostics:
    pandoc document conversion failed with error 43
    Execution halted
    ```

# brms

Version: 2.0.1

## In both

*   checking top-level files ... WARNING
    ```
    Conversion of ‘README.md’ failed:
    pandoc: Could not fetch https://travis-ci.org/paul-buerkner/brms.svg?branch=master
    FailedConnectionException2 "travis-ci.org" 443 True getAddrInfo: does not exist (Name or service not known)
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Paul-Christian Bürkner <paul.buerkner@gmail.com>’
    
    The Date field is over a month old.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.4Mb
      sub-directories of 1Mb or more:
        R     2.2Mb
        doc   2.4Mb
    ```

# bsseq

Version: 1.14.0

## In both

*   checking top-level files ... WARNING
    ```
    Conversion of ‘README.md’ failed:
    pandoc: Could not fetch http://bioconductor.org/shields/build/release/bioc/bsseq.svg
    FailedConnectionException2 "bioconductor.org" 80 False getAddrInfo: does not exist (Name or service not known)
    ```

*   checking Rd cross-references ... WARNING
    ```
    Unknown package ‘bsseqData’ in Rd xrefs
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Attaching package: 'matrixStats'
    
    The following objects are masked from 'package:Biobase':
    
        anyMissing, rowMedians
    
    
    Attaching package: 'DelayedArray'
    
    The following objects are masked from 'package:matrixStats':
    
        colMaxs, colMins, colRanges, rowMaxs, rowMins, rowRanges
    
    The following object is masked from 'package:base':
    
        apply
    
    Quitting from lines 29-31 (bsseq_analysis.Rmd) 
    Error: processing vignette 'bsseq_analysis.Rmd' failed with diagnostics:
    there is no package called 'bsseqData'
    Execution halted
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Kasper Daniel Hansen <kasperdanielhansen@gmail.com>’
    
    The Title field should be in title case, current version then in title case:
    ‘Analyze, manage and store bisulfite sequencing data’
    ‘Analyze, Manage and Store Bisulfite Sequencing Data’
    
    This build time stamp is over a month old.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘bsseqData’
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.5Mb
      sub-directories of 1Mb or more:
        data   2.7Mb
        doc    2.3Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘HDF5Array’
      All declared Imports should be used.
    ```

# bumphunter

Version: 1.20.0

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/test-all.R’ failed.
    Last 13 lines of output:
             }
             typeName = gsub("^CURLE_", "", typeName)
             fun = (if (asError) 
                 stop
             else warning)
             fun(structure(list(message = msg, call = sys.call()), class = c(typeName, "GenericCurlError", 
                 "error", "condition")))
         }(6L, "Could not resolve host: genome.ucsc.edu; Name or service not known", TRUE)
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 10 SKIPPED: 0 FAILED: 1
      1. Error: (unknown) (@test_annotation.R#31) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Rafael A. Irizarry <rafa@jimmy.harvard.edu>’
    
    This build time stamp is over a month old.
    ```

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      ‘S4Vectors’ ‘IRanges’ ‘GenomeInfoDb’ ‘GenomicRanges’ ‘foreach’
      ‘iterators’ ‘parallel’ ‘locfit’
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported object imported by a ':::' call: ‘doParallel:::.options’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    bumphunterEngine: no visible binding for global variable ‘bootstraps’
    smarterNearest: no visible global function definition for
      ‘PartitioningByEnd’
    Undefined global functions or variables:
      PartitioningByEnd bootstraps
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    The following object is masked from ‘package:base’:
    
        isNamespaceLoaded
    
    [bumphunterEngine] Estimating p-values and FWER.
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'bumphunter.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `fullpage.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.10 \usepackage
                    [numbers]{natbib}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# calmate

Version: 0.12.1

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Henrik Bengtsson <henrikb@braju.com>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# carx

Version: 0.7.1

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Chao Wang <chao-wang@uiowa.edu>’
    
    This build time stamp is over a month old.
    ```

# CATALYST

Version: 1.2.0

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Helena Lucia Crowell <crowellh@student.ethz.ch>’
    
    This build time stamp is over a month old.
    
    Size of tarball: 7971061 bytes
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 11.9Mb
      sub-directories of 1Mb or more:
        data   2.3Mb
        doc    9.2Mb
    ```

# cellWise

Version: 1.0.0

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Jakob Raymaekers <jakob.raymaekers@kuleuven.be>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# CEMiTool

Version: 1.0.1

## In both

*   checking examples ... ERROR
    ```
    ...
    List of 2
     $ echo : logi FALSE
     $ cache: logi FALSE
    
    output file: /netapp/home/hb/repositories/matrixStats/revdep/checks/CEMiTool/new/CEMiTool.Rcheck/Reports/Report/report.knit.md
    
    /netapp/home/hb/apps/local/bin/pandoc +RTS -K512m -RTS /netapp/home/hb/repositories/matrixStats/revdep/checks/CEMiTool/new/CEMiTool.Rcheck/Reports/Report/report.utf8.md --to latex --from markdown+autolink_bare_uris+ascii_identifiers+tex_math_single_backslash --output /netapp/home/hb/repositories/matrixStats/revdep/checks/CEMiTool/new/CEMiTool.Rcheck/Reports/Report/report.pdf --table-of-contents --toc-depth 2 --template /netapp/home/hb/repositories/matrixStats/revdep/library/CEMiTool/rmarkdown/rmd/latex/default-1.17.0.2.tex --number-sections --highlight-style tango --latex-engine pdflatex --variable graphics=yes --variable 'geometry:margin=1in' 
    ! LaTeX Error: File `titling.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    Enter file name: 
    ! Emergency stop.
    <read *> 
             
    l.66 ^^M
    
    pandoc: Error producing PDF
    Error: pandoc document conversion failed with error 43
    Execution halted
    ```

*   checking top-level files ... WARNING
    ```
    Conversion of ‘README.md’ failed:
    pandoc: Could not fetch https://travis-ci.org/csbl-usp/CEMiTool.svg?branch=master
    FailedConnectionException2 "travis-ci.org" 443 True getAddrInfo: does not exist (Name or service not known)
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Helder Nakaya <hnakaya@usp.br>’
    
    The Title field should be in title case, current version then in title case:
    ‘Co-expression Modules identification Tool’
    ‘Co-Expression Modules Identification Tool’
    
    This build time stamp is over a month old.
    
    Size of tarball: 5638196 bytes
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  8.8Mb
      sub-directories of 1Mb or more:
        data      3.1Mb
        doc       2.3Mb
        extdata   2.8Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    plot_qq,CEMiTool: no visible binding for global variable ‘data’
    plot_sample_tree,CEMiTool: no visible global function definition for
      ‘hclust’
    plot_sample_tree,CEMiTool: no visible global function definition for
      ‘dist’
    plot_sample_tree,CEMiTool: no visible global function definition for
      ‘png’
    plot_sample_tree,CEMiTool: no visible global function definition for
      ‘dev.off’
    save_plots,CEMiTool : <anonymous>: no visible global function
      definition for ‘dev.off’
    save_plots,CEMiTool: no visible global function definition for
      ‘dev.off’
    Undefined global functions or variables:
      ..eq.label.. ..rr.label.. Mean Variance as.dist cem data dev.off dist
      hclust head png tail var
    Consider adding
      importFrom("grDevices", "dev.off", "png")
      importFrom("stats", "as.dist", "dist", "hclust", "var")
      importFrom("utils", "data", "head", "tail")
    to your NAMESPACE file.
    ```

# Chicago

Version: 1.6.0

## In both

*   checking Rd cross-references ... WARNING
    ```
    Unknown package ‘PCHiCdata’ in Rd xrefs
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Loading required package: data.table
    
    Welcome to CHiCAGO - version 1.6.0
    If you are new to CHiCAGO, please consider reading the vignette through the command: vignette("Chicago").
    NOTE: Default values of tlb.minProxOEPerBin and tlb.minProxB2BPerBin changed as of Version 1.1.5. No action is required unless you specified non-default values, or wish to re-run the pipeline on old chicagoData objects. See news(package="Chicago")
    Quitting from lines 44-46 (Chicago.Rmd) 
    Error: processing vignette 'Chicago.Rmd' failed with diagnostics:
    there is no package called 'PCHiCdata'
    Execution halted
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Mikhail Spivakov <spivakov@babraham.ac.uk>’
    
    The Title field starts with the package name.
    
    This build time stamp is over a month old.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘PCHiCdata’
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .BBSoptions
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    
    CRAN-pack does not know about
      .BBSoptions
    ```

# childhoodmortality

Version: 0.1.0

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Casey Breen <cbreen@umn.edu>’
    
    This build time stamp is over a month old.
    ```

# chromswitch

Version: 1.0.0

## In both

*   checking top-level files ... WARNING
    ```
    Conversion of ‘README.md’ failed:
    pandoc: Could not fetch https://travis-ci.org/sjessa/chromswitch.svg?branch=master
    FailedConnectionException2 "travis-ci.org" 443 True getAddrInfo: does not exist (Name or service not known)
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Selin Jessa <selinjessa@gmail.com>’
    
    The Title field should be in title case, current version then in title case:
    ‘An R package to detect chromatin state switches from epigenomic data’
    ‘An R Package to Detect Chromatin State Switches from Epigenomic Data’
    
    The Description field should not start with the package name,
      'This package' or similar.
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# Clomial

Version: 1.14.0

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Habil Zare <zare@txstate.edu>’
    
    The Title field should be in title case, current version then in title case:
    ‘Infers clonal composition of a tumor’
    ‘Infers Clonal Composition of a Tumor’
    
    The Description field should not start with the package name,
      'This package' or similar.
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking R code for possible problems ... NOTE
    ```
    Clomial: no visible global function definition for ‘runif’
    Clomial.generate.data: no visible global function definition for
      ‘runif’
    Clomial.generate.data: no visible global function definition for
      ‘rbinom’
    Clomial.likelihood: no visible global function definition for ‘dbinom’
    Phi: no visible global function definition for ‘dbinom’
    choose.best: no visible global function definition for ‘tail’
    compute.P.reparam : update.Wj: no visible global function definition
      for ‘optim’
    compute.P.reparam : plot.obj: no visible global function definition for
      ‘plot’
    compute.q: no visible global function definition for ‘dbinom’
    Undefined global functions or variables:
      dbinom optim plot rbinom runif tail
    Consider adding
      importFrom("graphics", "plot")
      importFrom("stats", "dbinom", "optim", "rbinom", "runif")
      importFrom("utils", "tail")
    to your NAMESPACE file.
    ```

# clusterExperiment

Version: 1.4.0

## In both

*   checking Rd cross-references ... WARNING
    ```
    Unknown package ‘ConsensusClusterPlus’ in Rd xrefs
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Elizabeth Purdom <epurdom@stat.berkeley.edu>’
    
    No package encoding and non-ASCII characters in the following R files:
      R/dataCreation.R
        101: # [1] <e2><80><98>1.3.3.9001<e2><80><99>
      R/mergeClusters.R
        622: # If warn is zero (the default) warnings are stored until the top<e2><80><93>level function returns. If 10 or fewer warnings were signalled they will be printed otherwise a message saying how many were signalled. An object called last.warning is created and can be printed through the function warnings. If warn is one, warnings are printed as they occur. If warn is two or larger all warnings are turned into errors.
    
    This build time stamp is over a month old.
    
    Size of tarball: 8558636 bytes
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 12.3Mb
      sub-directories of 1Mb or more:
        data   1.3Mb
        doc   10.1Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    ':::' call which should be '::': ‘ape:::.PlotPhyloEnv’
      See the note in ?`:::` about the use of this operator.
    Unexported object imported by a ':::' call: ‘ape:::.matchDataPhylo’
      See the note in ?`:::` about the use of this operator.
    ```

# CNPBayes

Version: 1.8.0

## In both

*   checking top-level files ... WARNING
    ```
    Conversion of ‘README.md’ failed:
    pandoc: Could not fetch https://travis-ci.org/scristia/CNPBayes.svg?branch=master
    FailedConnectionException2 "travis-ci.org" 443 True getAddrInfo: does not exist (Name or service not known)
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Warning in has_utility("pdfcrop") :
      pdfcrop not installed or not in PATH
    This is pdfTeX, Version 3.1415926-2.5-1.40.14 (TeX Live 2013)
     restricted \write18 enabled.
    entering extended mode
    ! LaTeX Error: File `titling.sty' not found.
    
    ! Emergency stop.
    <read *> 
    
    Error: processing vignette 'Convergence.Rmd' failed with diagnostics:
    Failed to compile Convergence.tex. See Convergence.log for more info.
    Execution halted
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Jacob Carey <jcarey15@jhu.edu>’
    
    The Title field should be in title case, current version then in title case:
    ‘Bayesian mixture models for copy number polymorphisms’
    ‘Bayesian Mixture Models for Copy Number Polymorphisms’
    
    The Date field is not in ISO 8601 yyyy-mm-dd format.
    
    This build time stamp is over a month old.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 17.1Mb
      sub-directories of 1Mb or more:
        doc    2.9Mb
        libs  12.6Mb
    ```

# cointmonitoR

Version: 0.1.0

## In both

*   checking top-level files ... WARNING
    ```
    Conversion of ‘README.md’ failed:
    pandoc: Could not fetch https://travis-ci.org/aschersleben/cointmonitoR.svg?branch=master
    FailedConnectionException2 "travis-ci.org" 443 True getAddrInfo: does not exist (Name or service not known)
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Philipp Aschersleben <aschersleben@statistik.tu-dortmund.de>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# cointReg

Version: 0.2.0

## In both

*   checking top-level files ... WARNING
    ```
    Conversion of ‘README.md’ failed:
    pandoc: Could not fetch https://travis-ci.org/aschersleben/cointReg.svg?branch=master
    FailedConnectionException2 "travis-ci.org" 443 True getAddrInfo: does not exist (Name or service not known)
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Philipp Aschersleben <aschersleben@statistik.tu-dortmund.de>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# consensusOV

Version: 1.0.0

## In both

*   checking top-level files ... WARNING
    ```
    Conversion of ‘README.md’ failed:
    pandoc: Could not fetch https://travis-ci.org/bhklab/consensusOV.svg?branch=master
    FailedConnectionException2 "travis-ci.org" 443 True getAddrInfo: does not exist (Name or service not known)
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Benjamin Haibe-Kains <benjamin.haibe.kains@utoronto.ca>’
    
    The Title field should be in title case, current version then in title case:
    ‘Gene expression-based subtype classification for high-grade serous ovarian cancer’
    ‘Gene Expression-Based Subtype Classification for High-Grade Serous Ovarian Cancer’
    
    The Description field should not start with the package name,
      'This package' or similar.
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# CopywriteR

Version: 2.10.0

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Thomas Kuilman <t.kuilman@nki.nl>’
    
    Uses the superseded package: ‘snow’
    
    No package encoding and non-ASCII characters in the following R files:
      R/CopywriteR.R
        614:     ## <e2><80><98>Map<e2><80><99> applies a function to the corresponding elements of given vectors.
    
    The Title field should be in title case, current version then in title case:
    ‘Copy number information from targeted sequencing using off-target reads’
    ‘Copy Number Information from Targeted Sequencing using Off-Target Reads’
    
    The Description field should not start with the package name,
      'This package' or similar.
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      definition for ‘text’
    plotCNA : <anonymous> : <anonymous>: no visible global function
      definition for ‘axis’
    plotCNA : <anonymous> : <anonymous>: no visible global function
      definition for ‘abline’
    plotCNA : <anonymous> : <anonymous>: no visible global function
      definition for ‘dev.off’
    preCopywriteR: no visible global function definition for ‘as’
    Undefined global functions or variables:
      abline as axis dev.off dpois ecdf getClass lines loess packageVersion
      par pdf plot png points ppois predict read.table rgb segments text
      write.table
    Consider adding
      importFrom("grDevices", "dev.off", "pdf", "png", "rgb")
      importFrom("graphics", "abline", "axis", "lines", "par", "plot",
                 "points", "segments", "text")
      importFrom("methods", "as", "getClass")
      importFrom("stats", "dpois", "ecdf", "loess", "ppois", "predict")
      importFrom("utils", "packageVersion", "read.table", "write.table")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
      neighborhood radius 0.001
    Warning in simpleLoess(y, x, w, span, degree = degree, parametric = parametric,  :
      reciprocal condition number  0
    Warning in simpleLoess(y, x, w, span, degree = degree, parametric = parametric,  :
      There are other near singularities as well. 1e-06
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'CopywriteR.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `titlesec.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.95 \makeatletter
                      ^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# cosinor2

Version: 0.1.0

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Augustin Mutak <aumutak@ffzg.hr>’
    
    This build time stamp is over a month old.
    ```

# crossmeta

Version: 1.4.0

## In both

*   checking examples ... ERROR
    ```
    ...
    > ### Name: symbol_annot
    > ### Title: Add hgnc symbol to expression set.
    > ### Aliases: symbol_annot
    > 
    > ### ** Examples
    > 
    > library(lydata)
    > 
    > # location of raw data
    > data_dir <- system.file("extdata", package = "lydata")
    > 
    > # load eset
    > eset <- load_raw("GSE9601", data_dir)[[1]]
    > 
    > # annotate eset (need if load_raw failed to annotate)
    > eset <- symbol_annot(eset)
    AnnotatingWarning in read.dcf(con) :
      URL 'http://bioconductor.org/BiocInstaller.dcf': status was 'Couldn't resolve host name'
    Error: 'no packages in repository (no internet connection?)' while trying
      http://bioconductor.org/packages/3.6/bioc
    Execution halted
    ```

*   checking Rd cross-references ... WARNING
    ```
    Unknown packages ‘PADOG’, ‘GeneMeta’ in Rd xrefs
    ```

*   checking for code/documentation mismatches ... WARNING
    ```
    Data with usage in documentation object 'gs.names' but not in code:
      gs.names
    
    Data with usage in documentation object 'gslist' but not in code:
      gslist
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Alex Pickering <alexvpickering@gmail.com>’
    
    This build time stamp is over a month old.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Missing or unexported objects:
      ‘crossmeta::gs.names’ ‘crossmeta::gslist’
    ```

*   checking R code for possible problems ... NOTE
    ```
    explore_paths : server: no visible binding for global variable ‘gslist’
    explore_paths : server: no visible binding for global variable
      ‘gs.names’
    Undefined global functions or variables:
      gs.names gslist
    ```

# DAMOCLES

Version: 1.1

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Rampal S. Etienne <r.s.etienne@rug.nl>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      ‘cophenetic’
    DAMOCLES_bootstrap: no visible global function definition for ‘sd’
    DAMOCLES_bootstrap: no visible global function definition for
      ‘quantile’
    DAMOCLES_sim: no visible global function definition for ‘na.omit’
    DAMOCLES_sim: no visible global function definition for ‘runif’
    DAMOCLES_sim: no visible global function definition for ‘lines’
    DAMOCLES_sim: no visible global function definition for ‘rnorm’
    DAMOCLES_sim: no visible global function definition for ‘dexp’
    DAMOCLES_sim: no visible global function definition for ‘dist’
    DAMOCLES_simplex: no visible global function definition for
      ‘flush.console’
    Undefined global functions or variables:
      cophenetic dexp dist flush.console lines na.omit quantile rnorm runif
      sd
    Consider adding
      importFrom("graphics", "lines")
      importFrom("stats", "cophenetic", "dexp", "dist", "na.omit",
                 "quantile", "rnorm", "runif", "sd")
      importFrom("utils", "flush.console")
    to your NAMESPACE file.
    ```

# DatabionicSwarm

Version: 0.9.8

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Michael Thrun <m.thrun@gmx.net>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘rgl’
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.8Mb
      sub-directories of 1Mb or more:
        libs   5.6Mb
    ```

# DeepBlueR

Version: 1.4.1

## In both

*   checking whether package ‘DeepBlueR’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/netapp/home/hb/repositories/matrixStats/revdep/checks/DeepBlueR/new/DeepBlueR.Rcheck/00install.out’ for details.
    ```

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Felipe Albrecht <felipe.albrecht@mpi-inf.mpg.de>, Markus List <markus.list@mpi-inf.mpg.de>’
    
    The maintainer field is invalid or specifies more than one person
    
    The Title field is just the package name: provide a real title.
    
    This build time stamp is over a month old.
    ```

## Installation

### Devel

```
* installing *source* package ‘DeepBlueR’ ...
** R
** demo
** inst
** preparing package for lazy loading
Error in function (type, msg, asError = TRUE)  : 
  Could not resolve host: deepblue.mpi-inf.mpg.de; Name or service not known
Error : unable to load R code in package ‘DeepBlueR’
ERROR: lazy loading failed for package ‘DeepBlueR’
* removing ‘/netapp/home/hb/repositories/matrixStats/revdep/checks/DeepBlueR/new/DeepBlueR.Rcheck/DeepBlueR’

```
### CRAN

```
* installing *source* package ‘DeepBlueR’ ...
** R
** demo
** inst
** preparing package for lazy loading
Error in function (type, msg, asError = TRUE)  : 
  Could not resolve host: deepblue.mpi-inf.mpg.de; Name or service not known
Error : unable to load R code in package ‘DeepBlueR’
ERROR: lazy loading failed for package ‘DeepBlueR’
* removing ‘/netapp/home/hb/repositories/matrixStats/revdep/checks/DeepBlueR/old/DeepBlueR.Rcheck/DeepBlueR’

```
# DelayedArray

Version: 0.4.1

## Newly fixed

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Warning: running command 'kpsewhich framed.sty' had status 1
    Warning in test_latex_pkg("framed", system.file("misc", "framed.sty", package = "knitr")) :
      unable to find LaTeX package 'framed'; will use a copy from knitr
    Quitting from lines 513-516 (Working_with_large_arrays.Rnw) 
    Error: processing vignette 'Working_with_large_arrays.Rnw' failed with diagnostics:
    HDF5. File accessability. Unable to open file.
    Execution halted
    ```

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/run_unitTests.R’ failed.
    Last 13 lines of output:
      
          colMaxs, colMins, colRanges, rowMaxs, rowMins, rowRanges
      
      The following object is masked from 'package:base':
      
          apply
      
      [1] TRUE
      > DelayedArray:::.test()
      ------------------------------------------------------------------
      Running tests with realization backend set to "RleArray" ...
      Error in library("RUnit", quietly = TRUE) : 
        there is no package called 'RUnit'
      Calls: <Anonymous> -> <Anonymous> -> library
      Execution halted
    ```

*   checking for missing documentation entries ... WARNING
    ```
    Undocumented S4 methods:
      generic '[<-' and siglist 'DelayedArray'
      generic 'dbinom' and siglist 'DelayedArray'
      generic 'dlogis' and siglist 'DelayedArray'
      generic 'dpois' and siglist 'DelayedArray'
      generic 'pbinom' and siglist 'DelayedArray'
      generic 'plogis' and siglist 'DelayedArray'
      generic 'ppois' and siglist 'DelayedArray'
      generic 'qbinom' and siglist 'DelayedArray'
      generic 'qlogis' and siglist 'DelayedArray'
      generic 'qpois' and siglist 'DelayedArray'
    All user-level objects in a package (including S4 classes and methods)
    should have documentation entries.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Hervé Pagès <hpages@fredhutch.org>’
    
    The Title field should be in title case, current version then in title case:
    ‘Delayed operations on array-like objects’
    ‘Delayed Operations on Array-Like Objects’
    
    This build time stamp is over a month old.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      ‘BiocGenerics:::testPackage’ ‘S4Vectors:::anyMissing’
      ‘S4Vectors:::anyMissingOrOutside’ ‘S4Vectors:::diffWithInitialZero’
      ‘S4Vectors:::extract_positions_from_Rle’ ‘S4Vectors:::sapply_isNULL’
      See the note in ?`:::` about the use of this operator.
    ```

# DelayedMatrixStats

Version: 1.0.1

## In both

*   checking top-level files ... WARNING
    ```
    Conversion of ‘README.md’ failed:
    pandoc: Could not fetch https://travis-ci.org/PeteHaitch/DelayedMatrixStats.svg?branch=master
    FailedConnectionException2 "travis-ci.org" 443 True getAddrInfo: does not exist (Name or service not known)
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Author field differs from that derived from Authors@R
      Author:    ‘Peter Hickey <peter.hickey@gmail.com>’
      Authors@R: ‘Peter Hickey [aut, cre]’
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      ‘DelayedArray:::.as_array_or_matrix’
      ‘DelayedArray:::.execute_delayed_ops’
      ‘DelayedArray:::.get_DelayedArray_dim_before_transpose’
      ‘DelayedArray:::.get_DelayedArray_dimnames_before_transpose’
      ‘DelayedArray:::.get_ans_type’
      ‘DelayedArray:::.reduce_array_dimensions’
      ‘DelayedArray:::RleArraySeed’ ‘DelayedArray:::colblock_APPLY’
      ‘DelayedArray:::extract_block’ ‘DelayedArray:::get_Nindex_lengths’
      ‘DelayedArray:::get_Nindex_names_along’
      ‘DelayedArray:::get_max_block_length’
      ‘DelayedArray:::get_max_spacings_for_linear_blocks’
      ‘DelayedArray:::get_verbose_block_processing’
      ‘DelayedArray:::subset_by_Nindex’ ‘DelayedArray:::to_linear_index’
      See the note in ?`:::` about the use of this operator.
    ```

# DGCA

Version: 1.0.1

## In both

*   checking top-level files ... WARNING
    ```
    Conversion of ‘README.md’ failed:
    pandoc: Could not fetch https://travis-ci.org/andymckenzie/DGCA.svg?branch=master
    FailedConnectionException2 "travis-ci.org" 443 True getAddrInfo: does not exist (Name or service not known)
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Andrew McKenzie <amckenz@gmail.com>’
    
    This build time stamp is over a month old.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘MEGENA’
    ```

# diffloop

Version: 1.6.0

## In both

*   checking top-level files ... WARNING
    ```
    Conversion of ‘README.md’ failed:
    pandoc: Could not fetch https://travis-ci.org/aryeelab/diffloop.svg?branch=master
    FailedConnectionException2 "travis-ci.org" 443 True getAddrInfo: does not exist (Name or service not known)
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Caleb Lareau <caleblareau@g.harvard.edu>’
    
    The Title field should be in title case, current version then in title case:
    ‘Identifying differential DNA loops from chromatin topology data’
    ‘Identifying Differential DNA Loops from Chromatin Topology Data’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    
    CRAN-pack does not know about
      .travis.yml
    ```

# Doscheda

Version: 1.0.0

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘Doscheda-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: corrPlot
    > ### Title: Plot showing correlation between all channels across replicates
    > ### Aliases: corrPlot corrPlot,ChemoProtSet-method corrPlot,ANY,ANY-method
    > 
    > ### ** Examples
    > 
    > 
    > ex <- processedExample
    > ex <- runNormalisation(ex)
    Error in curl::curl_fetch_memory(url, handle = handle) : 
      Could not resolve host: www.humanmine.org; Name or service not known
    Calls: runNormalisation ... request_fetch -> request_fetch.write_memory -> <Anonymous> -> .Call
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
             modelType = "linear", accessionID = "Accession", uniquePeptides = "UniquePeps") at testthat/test-normal.R:8
      2: uniprotGene(organism)
      3: httr::POST("http://www.humanmine.org/humanmine/service/query/results", body = list(query = query, 
             format = "json"), encode = "form")
      4: request_perform(req, hu$handle$handle)
      5: request_fetch(req$output, req$url, handle)
      6: request_fetch.write_memory(req$output, req$url, handle)
      7: curl::curl_fetch_memory(url, handle = handle)
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 0 SKIPPED: 0 FAILED: 1
      1. Error: normalize_data med works (@test-normal.R#8) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Warning: 'html_document2' is deprecated.
    Use 'html_document' instead.
    See help("Deprecated")
    Quitting from lines 148-150 (Doscheda.Rmd) 
    Error: processing vignette 'Doscheda.Rmd' failed with diagnostics:
    Could not resolve host: www.humanmine.org; Name or service not known
    Execution halted
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Bruno Contrino <br1contrino@yahoo.co.uk>’
    
    The Description field should not start with the package name,
      'This package' or similar.
    
    This build time stamp is over a month old.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.1Mb
      sub-directories of 1Mb or more:
        data             2.1Mb
        doc              1.7Mb
        shiny-examples   2.0Mb
    ```

# dplR

Version: 1.6.6

## In both

*   checking Rd cross-references ... WARNING
    ```
    Unknown package ‘gmp’ in Rd xrefs
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Andy Bunn <andy.bunn@wwu.edu>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘gmp’
    ```

# EasyqpcR

Version: 1.20.0

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘gWidgetsRGtk2’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Le Pape Sylvain <sylvain.le.pape@univ-poitiers.fr>’
    
    The Title field starts with the package name.
    The Title field should be in title case, current version then in title case:
    ‘EasyqpcR for low-throughput real-time quantitative PCR data analysis’
    ‘EasyqpcR for Low-Throughput Real-Time Quantitative PCR Data Analysis’
    
    The Description field should not start with the package name,
      'This package' or similar.
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking package namespace information ... NOTE
    ```
      Namespace with empty importFrom: ‘gWidgetsRGtk2’
    ```

# EMDomics

Version: 2.8.0

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Sadhika Malladi <contact@sadhikamalladi.com> and Daniel Schmolze <emd@schmolze.com>’
    
    The maintainer field is invalid or specifies more than one person
    
    This build time stamp is over a month old.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Authors@R field gives more than one person with maintainer role:
      Sadhika Malladi <contact@sadhikamalladi.com> [aut, cre]
      Daniel Schmolze <emd@schmolze.com> [aut, cre]
    Maintainer field differs from that derived from Authors@R
      Maintainer: ‘Sadhika Malladi <contact@sadhikamalladi.com> and Daniel Schmolze <emd@schmolze.com>’
      Authors@R:  ‘’
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    calculate_cvm : <anonymous>: no visible global function definition for
      ‘median’
    calculate_cvm_gene: no visible global function definition for ‘combn’
    calculate_emd: no visible global function definition for ‘combn’
    calculate_emd : <anonymous>: no visible global function definition for
      ‘median’
    calculate_emd_gene: no visible global function definition for ‘combn’
    calculate_ks: no visible global function definition for ‘combn’
    calculate_ks : <anonymous>: no visible global function definition for
      ‘p.adjust’
    calculate_ks : <anonymous>: no visible global function definition for
      ‘median’
    calculate_ks_gene: no visible global function definition for ‘combn’
    calculate_ks_gene: no visible global function definition for ‘ks.test’
    Undefined global functions or variables:
      combn hist ks.test median p.adjust
    Consider adding
      importFrom("graphics", "hist")
      importFrom("stats", "ks.test", "median", "p.adjust")
      importFrom("utils", "combn")
    to your NAMESPACE file.
    ```

# EnrichedHeatmap

Version: 1.9.2

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    trying URL 'https://jokergoo.github.io/supplementary/EnrichedHeatmap-supplementary/roadmap_normalized_matrices.RData'
    Warning in download.file("https://jokergoo.github.io/supplementary/EnrichedHeatmap-supplementary/roadmap_normalized_matrices.RData",  :
      URL 'https://jokergoo.github.io/supplementary/EnrichedHeatmap-supplementary/roadmap_normalized_matrices.RData': status was 'Couldn't resolve host name'
    Quitting from lines 41-50 (roadmap.Rmd) 
    Error: processing vignette 'roadmap.Rmd' failed with diagnostics:
    cannot open URL 'https://jokergoo.github.io/supplementary/EnrichedHeatmap-supplementary/roadmap_normalized_matrices.RData'
    Execution halted
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Zuguang Gu <z.gu@dkfz.de>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    
    Size of tarball: 9463257 bytes
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 10.5Mb
      sub-directories of 1Mb or more:
        doc       7.0Mb
        extdata   2.5Mb
    ```

*   checking Rd \usage sections ... NOTE
    ```
    S3 methods shown with full name in documentation object 'rbind.normalizedMatrix':
      ‘rbind.normalizedMatrix’
    
    The \usage entries for S3 methods should use the \method markup and not
    their full name.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

# EventPointer

Version: 1.2.0

## In both

*   checking top-level files ... WARNING
    ```
    Conversion of ‘README.md’ failed:
    pandoc: Could not fetch https://github.com/jpromeror/EventPointer/blob/master/vignettes/Figure1.png
    FailedConnectionException2 "github.com" 443 True getAddrInfo: does not exist (Name or service not known)
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Juan Pablo Romero <jpromero@ceit.es>’
    
    Unknown, possibly mis-spelled, fields in DESCRIPTION:
      ‘Url’
    
    The Title field should be in title case, current version then in title case:
    ‘An effective identification of alternative splicing events using junction arrays and RNA-Seq data’
    ‘An Effective Identification of Alternative Splicing Events using Junction Arrays and RNA-Seq Data’
    
    The Description field should not start with the package name,
      'This package' or similar.
    
    This build time stamp is over a month old.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      ‘SGSeq:::edges’ ‘SGSeq:::exonGraph’ ‘SGSeq:::nodes’
      See the note in ?`:::` about the use of this operator.
    ```

# expss

Version: 0.8.4

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Gregory Demin <gdemin@gmail.com>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# FADA

Version: 1.3.2

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘David Causeur <david.causeur@agrocampus-ouest.fr>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# FastHCS

Version: 0.0.5

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Kaveh Vakili <vakili.kaveh.email@gmail.com>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 16.3Mb
      sub-directories of 1Mb or more:
        data   1.3Mb
        libs  14.9Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    FHCSpsdo: no visible global function definition for ‘is’
    FastHCS: no visible global function definition for ‘complete.cases’
    compPcaParams: no visible global function definition for ‘quantile’
    compPcaParams: no visible global function definition for ‘qchisq’
    compPcaParams: no visible global function definition for ‘qnorm’
    compPcaParams: no visible global function definition for ‘sd’
    plot.FastHCS: no visible global function definition for ‘plot’
    plot.FastHCS: no visible global function definition for ‘abline’
    Undefined global functions or variables:
      abline complete.cases is plot qchisq qnorm quantile sd
    Consider adding
      importFrom("graphics", "abline", "plot")
      importFrom("methods", "is")
      importFrom("stats", "complete.cases", "qchisq", "qnorm", "quantile",
                 "sd")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# FastPCS

Version: 0.1.2

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Kaveh Vakili <vakili.kaveh.email@gmail.com>’
    
    The Title field starts with the package name.
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  8.8Mb
      sub-directories of 1Mb or more:
        libs   8.7Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    FastPCS: no visible global function definition for ‘complete.cases’
    FastPCS: no visible global function definition for ‘cov’
    FastPCS: no visible global function definition for ‘mahalanobis’
    FastPCS: no visible global function definition for ‘qchisq’
    FastPCS: no visible global function definition for ‘quantile’
    plot.FastPCS: no visible global function definition for ‘plot’
    plot.FastPCS: no visible global function definition for ‘abline’
    plot.FastPCS: no visible global function definition for ‘qchisq’
    Undefined global functions or variables:
      abline complete.cases cov mahalanobis plot qchisq quantile
    Consider adding
      importFrom("graphics", "abline", "plot")
      importFrom("stats", "complete.cases", "cov", "mahalanobis", "qchisq",
                 "quantile")
    to your NAMESPACE file.
    ```

# FastRCS

Version: 0.0.7

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Kaveh Vakili <vakili.kaveh.email@gmail.com>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 16.6Mb
      sub-directories of 1Mb or more:
        libs  16.5Mb
    ```

# fergm

Version: 0.2.1

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Benjamin W. Campbell <campbell.1721@osu.edu>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# filesstrings

Version: 2.0.2

## In both

*   checking top-level files ... WARNING
    ```
    Conversion of ‘README.md’ failed:
    pandoc: Could not fetch https://travis-ci.org/rorynolan/filesstrings.svg?branch=master
    FailedConnectionException2 "travis-ci.org" 443 True getAddrInfo: does not exist (Name or service not known)
    ```

*   checking Rd cross-references ... WARNING
    ```
    Unknown package ‘dplyr’ in Rd xrefs
    ```

# flowCore

Version: 1.44.1

## In both

*   checking whether package ‘flowCore’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/netapp/home/hb/repositories/matrixStats/revdep/checks/flowCore/new/flowCore.Rcheck/00install.out’ for details.
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘M.Jiang <wjiang2@fhcrc.org>’
    
    The Title field starts with the package name.
    The Title field should be in title case, current version then in title case:
    ‘flowCore: Basic structures for flow cytometry data’
    ‘flowCore: Basic Structures for Flow Cytometry Data’
    
    Size of tarball: 8554328 bytes
    ```

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      ‘flowViz’ ‘flowStats’ ‘flowWorkspace’ ‘openCyto’ ‘ggcyto’
    ```

## Installation

### Devel

```
* installing *source* package ‘flowCore’ ...
** libs
g++ -std=gnu++11 -I/netapp/home/hb/shared/apps/manual/r/R-3.4.3-20180109/lib64/R/include -DNDEBUG  -I"/netapp/home/hb/repositories/matrixStats/revdep/library/flowCore/Rcpp/include" -I"/netapp/home/hb/repositories/matrixStats/revdep/library/flowCore/BH/include" -I/usr/local/include   -fpic  -g -O2 -c FastLogicle.cpp -o FastLogicle.o
g++ -std=gnu++11 -I/netapp/home/hb/shared/apps/manual/r/R-3.4.3-20180109/lib64/R/include -DNDEBUG  -I"/netapp/home/hb/repositories/matrixStats/revdep/library/flowCore/Rcpp/include" -I"/netapp/home/hb/repositories/matrixStats/revdep/library/flowCore/BH/include" -I/usr/local/include   -fpic  -g -O2 -c Hyperlog.cpp -o Hyperlog.o
g++ -std=gnu++11 -I/netapp/home/hb/shared/apps/manual/r/R-3.4.3-20180109/lib64/R/include -DNDEBUG  -I"/netapp/home/hb/repositories/matrixStats/revdep/library/flowCore/Rcpp/include" -I"/netapp/home/hb/repositories/matrixStats/revdep/library/flowCore/BH/include" -I/usr/local/include   -fpic  -g -O2 -c Logicle.cpp -o Logicle.o
g++ -std=gnu++11 -I/netapp/home/hb/shared/apps/manual/r/R-3.4.3-20180109/lib64/R/include -DNDEBUG  -I"/netapp/home/hb/repositories/matrixStats/revdep/library/flowCore/Rcpp/include" -I"/netapp/home/hb/repositories/matrixStats/revdep/library/flowCore/BH/include" -I/usr/local/include   -fpic  -g -O2 -c RcppExports.cpp -o RcppExports.o
g++ -std=gnu++11 -I/netapp/home/hb/shared/apps/manual/r/R-3.4.3-20180109/lib64/R/include -DNDEBUG  -I"/netapp/home/hb/repositories/matrixStats/revdep/library/flowCore/Rcpp/include" -I"/netapp/home/hb/repositories/matrixStats/revdep/library/flowCore/BH/include" -I/usr/local/include   -fpic  -g -O2 -c biexponential.cpp -o biexponential.o
g++ -std=gnu++11 -I/netapp/home/hb/shared/apps/manual/r/R-3.4.3-20180109/lib64/R/include -DNDEBUG  -I"/netapp/home/hb/repositories/matrixStats/revdep/library/flowCore/Rcpp/include" -I"/netapp/home/hb/repositories/matrixStats/revdep/library/flowCore/BH/include" -I/usr/local/include   -fpic  -g -O2 -c convertRawBytes.cpp -o convertRawBytes.o
g++ -std=gnu++11 -I/netapp/home/hb/shared/apps/manual/r/R-3.4.3-20180109/lib64/R/include -DNDEBUG  -I"/netapp/home/hb/repositories/matrixStats/revdep/library/flowCore/Rcpp/include" -I"/netapp/home/hb/repositories/matrixStats/revdep/library/flowCore/BH/include" -I/usr/local/include   -fpic  -g -O2 -c fcsTextParse.cpp -o fcsTextParse.o
In file included from /netapp/home/hb/repositories/matrixStats/revdep/library/flowCore/BH/include/boost/algorithm/string/std/string_traits.hpp:16:0,
                 from /netapp/home/hb/repositories/matrixStats/revdep/library/flowCore/BH/include/boost/algorithm/string/std_containers_traits.hpp:19,
                 from /netapp/home/hb/repositories/matrixStats/revdep/library/flowCore/BH/include/boost/algorithm/string.hpp:18,
                 from fcsTextParse.cpp:8:
/netapp/home/hb/repositories/matrixStats/revdep/library/flowCore/BH/include/boost/algorithm/string/sequence_traits.hpp:15:30: fatal error: boost/mpl/bool.hpp: No such file or directory
 #include <boost/mpl/bool.hpp>
                              ^
compilation terminated.
make: *** [fcsTextParse.o] Error 1
ERROR: compilation failed for package ‘flowCore’
* removing ‘/netapp/home/hb/repositories/matrixStats/revdep/checks/flowCore/new/flowCore.Rcheck/flowCore’

```
### CRAN

```
* installing *source* package ‘flowCore’ ...
** libs
g++ -std=gnu++11 -I/netapp/home/hb/shared/apps/manual/r/R-3.4.3-20180109/lib64/R/include -DNDEBUG  -I"/netapp/home/hb/repositories/matrixStats/revdep/library/flowCore/Rcpp/include" -I"/netapp/home/hb/repositories/matrixStats/revdep/library/flowCore/BH/include" -I/usr/local/include   -fpic  -g -O2 -c FastLogicle.cpp -o FastLogicle.o
g++ -std=gnu++11 -I/netapp/home/hb/shared/apps/manual/r/R-3.4.3-20180109/lib64/R/include -DNDEBUG  -I"/netapp/home/hb/repositories/matrixStats/revdep/library/flowCore/Rcpp/include" -I"/netapp/home/hb/repositories/matrixStats/revdep/library/flowCore/BH/include" -I/usr/local/include   -fpic  -g -O2 -c Hyperlog.cpp -o Hyperlog.o
g++ -std=gnu++11 -I/netapp/home/hb/shared/apps/manual/r/R-3.4.3-20180109/lib64/R/include -DNDEBUG  -I"/netapp/home/hb/repositories/matrixStats/revdep/library/flowCore/Rcpp/include" -I"/netapp/home/hb/repositories/matrixStats/revdep/library/flowCore/BH/include" -I/usr/local/include   -fpic  -g -O2 -c Logicle.cpp -o Logicle.o
g++ -std=gnu++11 -I/netapp/home/hb/shared/apps/manual/r/R-3.4.3-20180109/lib64/R/include -DNDEBUG  -I"/netapp/home/hb/repositories/matrixStats/revdep/library/flowCore/Rcpp/include" -I"/netapp/home/hb/repositories/matrixStats/revdep/library/flowCore/BH/include" -I/usr/local/include   -fpic  -g -O2 -c RcppExports.cpp -o RcppExports.o
g++ -std=gnu++11 -I/netapp/home/hb/shared/apps/manual/r/R-3.4.3-20180109/lib64/R/include -DNDEBUG  -I"/netapp/home/hb/repositories/matrixStats/revdep/library/flowCore/Rcpp/include" -I"/netapp/home/hb/repositories/matrixStats/revdep/library/flowCore/BH/include" -I/usr/local/include   -fpic  -g -O2 -c biexponential.cpp -o biexponential.o
g++ -std=gnu++11 -I/netapp/home/hb/shared/apps/manual/r/R-3.4.3-20180109/lib64/R/include -DNDEBUG  -I"/netapp/home/hb/repositories/matrixStats/revdep/library/flowCore/Rcpp/include" -I"/netapp/home/hb/repositories/matrixStats/revdep/library/flowCore/BH/include" -I/usr/local/include   -fpic  -g -O2 -c convertRawBytes.cpp -o convertRawBytes.o
g++ -std=gnu++11 -I/netapp/home/hb/shared/apps/manual/r/R-3.4.3-20180109/lib64/R/include -DNDEBUG  -I"/netapp/home/hb/repositories/matrixStats/revdep/library/flowCore/Rcpp/include" -I"/netapp/home/hb/repositories/matrixStats/revdep/library/flowCore/BH/include" -I/usr/local/include   -fpic  -g -O2 -c fcsTextParse.cpp -o fcsTextParse.o
In file included from /netapp/home/hb/repositories/matrixStats/revdep/library/flowCore/BH/include/boost/algorithm/string/std/string_traits.hpp:16:0,
                 from /netapp/home/hb/repositories/matrixStats/revdep/library/flowCore/BH/include/boost/algorithm/string/std_containers_traits.hpp:19,
                 from /netapp/home/hb/repositories/matrixStats/revdep/library/flowCore/BH/include/boost/algorithm/string.hpp:18,
                 from fcsTextParse.cpp:8:
/netapp/home/hb/repositories/matrixStats/revdep/library/flowCore/BH/include/boost/algorithm/string/sequence_traits.hpp:15:30: fatal error: boost/mpl/bool.hpp: No such file or directory
 #include <boost/mpl/bool.hpp>
                              ^
compilation terminated.
make: *** [fcsTextParse.o] Error 1
ERROR: compilation failed for package ‘flowCore’
* removing ‘/netapp/home/hb/repositories/matrixStats/revdep/checks/flowCore/old/flowCore.Rcheck/flowCore’

```
# fslr

Version: 2.12

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: ‘oro.nifti’ ‘neurobase’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘John Muschelli <muschellij2@gmail.com>’
    
    This build time stamp is over a month old.
    ```

# funtooNorm

Version: 1.2.0

## In both

*   checking top-level files ... WARNING
    ```
    Conversion of ‘README.md’ failed:
    pandoc: Could not fetch https://travis-ci.org/GreenwoodLab/funtooNorm.svg?branch=master
    FailedConnectionException2 "travis-ci.org" 443 True getAddrInfo: does not exist (Name or service not known)
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Kathleen Klein <kathleen.klein@mail.mcgill.ca>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# GAD

Version: 1.1.1

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Leonardo Sandrini-Neto <leonardosandrini@gmail.com>’
    
    The Title field starts with the package name.
    The Title field should be in title case, current version then in title case:
    ‘GAD: Analysis of variance from general principles’
    ‘GAD: Analysis of Variance from General Principles’
    
    The Description field should not start with the package name,
      'This package' or similar.
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# gcapc

Version: 1.2.0

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Mingxiang Teng <tengmx@gmail.com>’
    
    This build time stamp is over a month old.
    ```

# GeneralizedUmatrix

Version: 0.9.5

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Michael Thrun <mthrun@mathematik.uni-marburg.de>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘rgl’
    ```

# genomation

Version: 1.10.0

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Altuna Akalin <aakalin@gmail.com>, Vedran Franke <vedran.franke@gmail.com>, Katarzyna Wreczycka <katwre@gmail.com>’
    
    The maintainer field is invalid or specifies more than one person
    
    The Title field should be in title case, current version then in title case:
    ‘Summary, annotation and visualization of genomic data’
    ‘Summary, Annotation and Visualization of Genomic Data’
    
    The Description field should not start with the package name,
      'This package' or similar.
    
    This build time stamp is over a month old.
    
    Size of tarball: 6334364 bytes
    ```

*   checking top-level files ... WARNING
    ```
    Conversion of ‘README.md’ failed:
    pandoc: Could not fetch https://github.com/BIMSBbioinfo/genomation/blob/master/inst/genomation_logo.png
    FailedConnectionException2 "github.com" 443 True getAddrInfo: does not exist (Name or service not known)
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Warning in block_exec(params) :
      failed to tidy R code in chunk <eadAllPeaks>
    reason: Error in loadNamespace(name) : there is no package called 'formatR'
    
    Warning in block_exec(params) :
      failed to tidy R code in chunk <findFeatureComb>
    reason: Error in loadNamespace(name) : there is no package called 'formatR'
    
    Warning in block_exec(params) :
      failed to tidy R code in chunk <visualizeFeatureComb>
    reason: Error in loadNamespace(name) : there is no package called 'formatR'
    
    Warning in block_exec(params) :
      failed to tidy R code in chunk <annotationHubExample1>
    reason: Error in loadNamespace(name) : there is no package called 'formatR'
    
    pandoc: Could not fetch https://raw.githubusercontent.com/BIMSBbioinfo/genomation/master/vignettes/Figures/genomationFlowChart1.png
    FailedConnectionException2 "raw.githubusercontent.com" 443 True getAddrInfo: does not exist (Name or service not known)
    Error: processing vignette 'GenomationManual.Rmd' failed with diagnostics:
    pandoc document conversion failed with error 67
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.9Mb
      sub-directories of 1Mb or more:
        doc       3.7Mb
        extdata   1.2Mb
        libs      1.2Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘RUnit’
      All declared Imports should be used.
    Unexported object imported by a ':::' call: ‘BiocGenerics:::testPackage’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ScoreMatrixBin,RleList-GRangesList: no visible binding for global
      variable ‘cvg’
    ScoreMatrixBin,RleList-GRangesList: no visible binding for global
      variable ‘id’
    ScoreMatrixBin,RleList-GRangesList: no visible global function
      definition for ‘:=’
    Undefined global functions or variables:
      := cvg id
    ```

*   checking files in ‘vignettes’ ... NOTE
    ```
    The following directory looks like a leftover from 'knitr':
      ‘cache’
    Please remove from your package.
    ```

# GenRank

Version: 1.6.0

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Chakravarthi Kanduri <chakra.kanduri@gmail.com>’
    
    The Title field should be in title case, current version then in title case:
    ‘Candidate gene prioritization based on convergent evidence’
    ‘Candidate Gene Prioritization Based on Convergent Evidence’
    
    This build time stamp is over a month old.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Author field differs from that derived from Authors@R
      Author:    ‘Chakravarthi Kanduri’
      Authors@R: ‘Chakravarthi Kanduri [aut, cre]’
    ```

*   checking R code for possible problems ... NOTE
    ```
    CombP: no visible global function definition for ‘read.table’
    ComputeCE: no visible global function definition for ‘read.table’
    ComputeCE: no visible global function definition for ‘complete.cases’
    ComputeRP: no visible global function definition for ‘read.table’
    ComputeRP: no visible global function definition for ‘complete.cases’
    Undefined global functions or variables:
      complete.cases read.table
    Consider adding
      importFrom("stats", "complete.cases")
      importFrom("utils", "read.table")
    to your NAMESPACE file.
    ```

# GJRM

Version: 0.1-4

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: ‘VineCopula’ ‘Rmpfr’ ‘copula’
    
    Package which this enhances but not available for checking: ‘sp’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Giampiero Marra <giampiero.marra@ucl.ac.uk>’
    
    This build time stamp is over a month old.
    ```

# GUIDEseq

Version: 1.8.0

## In both

*   checking whether package ‘GUIDEseq’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/netapp/home/hb/repositories/matrixStats/revdep/checks/GUIDEseq/new/GUIDEseq.Rcheck/00install.out’ for details.
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Lihua Julie Zhu <julie.zhu@umassmed.edu>’
    
    The Title field should be in title case, current version then in title case:
    ‘GUIDE-seq analysis pipeline’
    ‘GUIDE-Seq Analysis Pipeline’
    
    The Description field should not start with the package name,
      'This package' or similar.
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

## Installation

### Devel

```
* installing *source* package ‘GUIDEseq’ ...
** R
** data
** inst
** preparing package for lazy loading
Warning in read.dcf(con) :
  URL 'http://bioconductor.org/BiocInstaller.dcf': status was 'Couldn't resolve host name'
Error in library.dynam(lib, package, package.lib) : 
  shared object ‘httpuv.so’ not found
ERROR: lazy loading failed for package ‘GUIDEseq’
* removing ‘/netapp/home/hb/repositories/matrixStats/revdep/checks/GUIDEseq/new/GUIDEseq.Rcheck/GUIDEseq’

```
### CRAN

```
* installing *source* package ‘GUIDEseq’ ...
** R
** data
** inst
** preparing package for lazy loading
Warning in read.dcf(con) :
  URL 'http://bioconductor.org/BiocInstaller.dcf': status was 'Couldn't resolve host name'
Error in library.dynam(lib, package, package.lib) : 
  shared object ‘httpuv.so’ not found
ERROR: lazy loading failed for package ‘GUIDEseq’
* removing ‘/netapp/home/hb/repositories/matrixStats/revdep/checks/GUIDEseq/old/GUIDEseq.Rcheck/GUIDEseq’

```
# Gviz

Version: 1.22.2

## In both

*   checking whether package ‘Gviz’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/netapp/home/hb/repositories/matrixStats/revdep/checks/Gviz/new/Gviz.Rcheck/00install.out’ for details.
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Florian Hahne <florian.hahne@novartis.com>’
    
    The Title field should be in title case, current version then in title case:
    ‘Plotting data and annotation information along genomic coordinates’
    ‘Plotting Data and Annotation Information along Genomic Coordinates’
    
    This build time stamp is over a month old.
    ```

## Installation

### Devel

```
* installing *source* package ‘Gviz’ ...
** R
** data
** inst
** preparing package for lazy loading
Error in library.dynam(lib, package, package.lib) : 
  shared object ‘Biobase.so’ not found
ERROR: lazy loading failed for package ‘Gviz’
* removing ‘/netapp/home/hb/repositories/matrixStats/revdep/checks/Gviz/new/Gviz.Rcheck/Gviz’

```
### CRAN

```
* installing *source* package ‘Gviz’ ...
** R
** data
** inst
** preparing package for lazy loading
Error in library.dynam(lib, package, package.lib) : 
  shared object ‘Biobase.so’ not found
ERROR: lazy loading failed for package ‘Gviz’
* removing ‘/netapp/home/hb/repositories/matrixStats/revdep/checks/Gviz/old/Gviz.Rcheck/Gviz’

```
# haploReconstruct

Version: 0.1.2

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Susanne U. Franssen <susefranssen@googlemail.com>’
    
    This build time stamp is over a month old.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  9.8Mb
      sub-directories of 1Mb or more:
        data   9.5Mb
    ```

# IMIFA

Version: 1.3.1

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘Rfast’
    
    Packages suggested but not available for checking: ‘Rmpfr’ ‘gmp’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Keefe Murphy <keefe.murphy@ucd.ie>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# jointseg

Version: 1.0.1

## In both

*   checking top-level files ... WARNING
    ```
    Conversion of ‘README.md’ failed:
    pandoc: Could not fetch https://travis-ci.org/mpierrejean/jointseg.svg
    FailedConnectionException2 "travis-ci.org" 443 True getAddrInfo: does not exist (Name or service not known)
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Morgane Pierre-Jean <mpierrejean.pro@gmail.com>’
    
    This build time stamp is over a month old.
    ```

# kernDeepStackNet

Version: 2.0.2

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Thomas Welchowski <welchow@imbie.meb.uni-bonn.de>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.8Mb
      sub-directories of 1Mb or more:
        libs   5.5Mb
    ```

# kgschart

Version: 1.3.5

## In both

*   checking top-level files ... WARNING
    ```
    Conversion of ‘README.md’ failed:
    pandoc: Could not fetch https://travis-ci.org/kota7/kgschart-r.svg?branch=master
    FailedConnectionException2 "travis-ci.org" 443 True getAddrInfo: does not exist (Name or service not known)
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Kota Mori <kmori05@gmail.com>’
    
    This build time stamp is over a month old.
    ```

# LFDREmpiricalBayes

Version: 1.0

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Ali Karimnezhad <ali_karimnezhad@yahoo.com>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'LFDREmpiricalBayes.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `changepage.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.10 \usepackage
                    {url}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# localgauss

Version: 0.40

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Tore Selland Kleppe <tore.kleppe@uis.no>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# loo

Version: 1.1.0

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Jonah Gabry <jsg2201@columbia.edu>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘rstanarm’
    ```

# LS2Wstat

Version: 2.0-3

## In both

*   checking whether package ‘LS2Wstat’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: no DISPLAY variable so Tk is not available
    See ‘/netapp/home/hb/repositories/matrixStats/revdep/checks/LS2Wstat/new/LS2Wstat.Rcheck/00install.out’ for details.
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Matt Nunes <m.nunes@lancaster.ac.uk>’
    
    The Title field should be in title case, current version then in title case:
    ‘A Multiscale Test of Spatial Stationarity for LS2W processes’
    ‘A Multiscale Test of Spatial Stationarity for LS2W Processes’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

*   checking R code for possible problems ... NOTE
    ```
    TOS2D: no visible global function definition for ‘medpolish’
    countTextures: no visible global function definition for ‘medpolish’
    plot.imageQT: no visible global function definition for ‘segments’
    simTexture: no visible global function definition for ‘rnorm’
    Undefined global functions or variables:
      medpolish rnorm segments
    Consider adding
      importFrom("graphics", "segments")
      importFrom("stats", "medpolish", "rnorm")
    to your NAMESPACE file.
    ```

# LSAmitR

Version: 1.0-1

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 55 marked UTF-8 strings
    ```

# ltmle

Version: 1.0-1

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Joshua Schwab <jschwab77@berkeley.edu>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# Luminescence

Version: 0.7.5

## In both

*   checking Rd cross-references ... WARNING
    ```
    Unknown packages ‘rjags’, ‘RLumShiny’, ‘RLumModel’ in Rd xrefs
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Sebastian Kreutzer <sebastian.kreutzer@u-bordeaux-montaigne.fr>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      ‘RLumShiny’ ‘RLumModel’ ‘rjags’
    ```

# metagene

Version: 2.10.1

## In both

*   checking top-level files ... WARNING
    ```
    Conversion of ‘README.md’ failed:
    pandoc: Could not fetch http://bioconductor.org/shields/years-in-bioc/metagene.svg
    FailedConnectionException2 "bioconductor.org" 80 False getAddrInfo: does not exist (Name or service not known)
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Charles Joly Beauparlant <charles.joly-beauparlant@crchul.ulaval.ca>’
    
    Unknown, possibly mis-spelled, fields in DESCRIPTION:
      ‘Author@R’
    
    The Title field should be in title case, current version then in title case:
    ‘A package to produce metagene plots’
    ‘A Package to Produce Metagene Plots’
    
    The Description field should not start with the package name,
      'This package' or similar.
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    
    Size of tarball: 6908124 bytes
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 10.3Mb
      sub-directories of 1Mb or more:
        doc       3.4Mb
        extdata   5.6Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    avoid_gaps_update: no visible binding for global variable 'value'
    avoid_gaps_update: no visible binding for global variable 'bam'
    avoid_gaps_update: no visible binding for global variable '.N'
    avoid_gaps_update: no visible binding for global variable 'tab'
    avoid_gaps_update: no visible binding for global variable 'nuc'
    permutation_test: no visible global function definition for '.'
    permutation_test: no visible binding for global variable 'value'
    plot_metagene: no visible binding for global variable 'bin'
    plot_metagene: no visible binding for global variable 'value'
    plot_metagene: no visible binding for global variable 'qinf'
    plot_metagene: no visible binding for global variable 'qsup'
    plot_metagene: no visible binding for global variable 'group'
    plot_metagene: no visible binding for global variable 'nuc'
    plot_metagene: no visible binding for global variable 'design'
    plot_metagene: no visible binding for global variable 'nuctot'
    Undefined global functions or variables:
      . .N bam bin design group nuc nuctot qinf qsup tab value
    ```

# metagenomeSeq

Version: 1.20.1

## In both

*   checking top-level files ... WARNING
    ```
    Conversion of ‘README.md’ failed:
    pandoc: Could not fetch https://travis-ci.org/HCBravoLab/metagenomeSeq.svg?branch=master
    FailedConnectionException2 "travis-ci.org" 443 True getAddrInfo: does not exist (Name or service not known)
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Joseph N. Paulson <jpaulson@jimmy.harvard.edu>’
    
    Reading CITATION file fails with
      package ‘metagenomeSeq’ not found
    when package is not installed.
    
    The Title field should be in title case, current version then in title case:
    ‘Statistical analysis for sparse high-throughput sequencing’
    ‘Statistical Analysis for Sparse High-Throughput Sequencing’
    
    The Description field should not start with the package name,
      'This package' or similar.
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘interactiveDisplay’
    ```

# MetaQC

Version: 0.1.13

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Don Kang <donkang75@gmail.com>’
    
    Uses the superseded package: ‘doSNOW’
    
    The Title field starts with the package name.
    
    The Description field should not start with the package name,
      'This package' or similar.
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' call to ‘survival’ in package code.
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    Packages in Depends field not imported from:
      ‘foreach’ ‘iterators’ ‘proto’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      ‘as.dist’
    requireAll: no visible global function definition for
      ‘installed.packages’
    requireAll: no visible binding for global variable ‘biocLite’
    requireAll: no visible global function definition for ‘biocLite’
    Undefined global functions or variables:
      %do% %dopar% abline arrows as.dist axis b biocLite box cor cor.test
      coxph d dat download.file fisher.test foreach g gray i ii
      installed.packages iter j jj ks.test lm makeCluster median na.omit
      p.adjust pchisq plot pnorm points prcomp proto pt pv qnorm
      registerDoMC registerDoSNOW sd t.test text w wilcox.test x
    Consider adding
      importFrom("grDevices", "gray")
      importFrom("graphics", "abline", "arrows", "axis", "box", "plot",
                 "points", "text")
      importFrom("stats", "as.dist", "cor", "cor.test", "fisher.test",
                 "ks.test", "lm", "median", "na.omit", "p.adjust", "pchisq",
                 "pnorm", "prcomp", "pt", "qnorm", "sd", "t.test",
                 "wilcox.test")
      importFrom("utils", "download.file", "installed.packages")
    to your NAMESPACE file.
    ```

# metavizr

Version: 1.2.0

## In both

*   checking whether package ‘metavizr’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/netapp/home/hb/repositories/matrixStats/revdep/checks/metavizr/new/metavizr.Rcheck/00install.out’ for details.
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Hector Corrada Bravo <hcorrada@gmail.com>’
    
    Found the following (possibly) invalid URLs:
      URL: "https://github.com/epiviz/epiviz.git"
        From: man/setMetavizStandalone.Rd
        Message: Invalid URI scheme
    
    The Title field should be in title case, current version then in title case:
    ‘R Interface to the metaviz web app for interactive metagenomics data analysis and visualization’
    ‘R Interface to the Metaviz Web App for Interactive Metagenomics Data Analysis and Visualization’
    
    The Description field should not start with the package name,
      'This package' or similar.
    
    This build time stamp is over a month old.
    ```

## Installation

### Devel

```
* installing *source* package ‘metavizr’ ...
** R
** inst
** preparing package for lazy loading
Warning in read.dcf(con) :
  URL 'http://bioconductor.org/BiocInstaller.dcf': status was 'Couldn't resolve host name'
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called 'git2r'
ERROR: lazy loading failed for package 'metavizr'
* removing '/netapp/home/hb/repositories/matrixStats/revdep/checks/metavizr/new/metavizr.Rcheck/metavizr'

```
### CRAN

```
* installing *source* package ‘metavizr’ ...
** R
** inst
** preparing package for lazy loading
Warning in read.dcf(con) :
  URL 'http://bioconductor.org/BiocInstaller.dcf': status was 'Couldn't resolve host name'
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called 'git2r'
ERROR: lazy loading failed for package 'metavizr'
* removing '/netapp/home/hb/repositories/matrixStats/revdep/checks/metavizr/old/metavizr.Rcheck/metavizr'

```
# MethylAid

Version: 1.12.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Loading required package: locfit
    locfit 1.5-9.1 	 2013-03-22
    Loading required package: IlluminaHumanMethylation450kmanifest
    Loading required package: IlluminaHumanMethylation450kanno.ilmn12.hg19
    [read.metharray.sheet] Found the following CSV files:
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'MethylAid.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `titlesec.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.95 \makeatletter
                      ^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘M. van Iterson <mviterson@gmail.com>’
    
    The Title field should be in title case, current version then in title case:
    ‘Visual and interactive quality control of large Illumina DNA Methylation array data sets’
    ‘Visual and Interactive Quality Control of Large Illumina DNA Methylation Array Data Sets’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘MethylAidData’
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Authors@R field gives no person with name and author role
    Author field differs from that derived from Authors@R
      Author:    ‘Maarten van Iterson [aut, cre], Elmar Tobi[ctb], Roderick Slieker[ctb], Wouter den Hollander[ctb], Rene Luijk[ctb] and Bas Heijmans[ctb]’
      Authors@R: ‘Maarten van Iterson [cre], Elmar Tobi [ctb], Roderick Slieker [ctb], Wouter den Hollander [ctb], Rene Luijk [ctb], Bas Heijmans [ctb]’
    
    Maintainer field differs from that derived from Authors@R
      Maintainer: ‘M. van Iterson <mviterson@gmail.com>’
      Authors@R:  ‘Maarten van Iterson <mviterson@gmail.com>’
    ```

# methylumi

Version: 2.24.1

## In both

*   checking for missing documentation entries ... WARNING
    ```
    Undocumented code objects:
      ‘IDATsToMatrices’ ‘IDATtoMatrix’ ‘tcgaPipeline’
    Undocumented S4 methods:
      generic '[' and siglist 'MethyLumiM,ANY,ANY,ANY'
      generic '[' and siglist 'MethyLumiSet,ANY,ANY,ANY'
    All user-level objects in a package (including S4 classes and methods)
    should have documentation entries.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking contents of ‘data’ directory ... WARNING
    ```
    Files not of a type allowed in a ‘data’ directory:
      ‘5318317007_A_Grn.idat’ ‘5318317007_A_Red.idat’
      ‘5318317007_B_Grn.idat’ ‘5318317007_B_Red.idat’
      ‘5318317007_C_Grn.idat’ ‘5318317007_C_Red.idat’
    Please use e.g. ‘inst/extdata’ for non-R data files
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Error in re-building vignettes:
      ...
    Warning: running command 'kpsewhich framed.sty' had status 1
    Warning in test_latex_pkg("framed", system.file("misc", "framed.sty", package = "knitr")) :
      unable to find LaTeX package 'framed'; will use a copy from knitr
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'methylumi.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `fullpage.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.58 \usepackage
                    {hyperref}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Sean Davis <sdavis2@mail.nih.gov>’
    
    The Title field should be in title case, current version then in title case:
    ‘Handle Illumina methylation data’
    ‘Handle Illumina Methylation Data’
    
    The Description field should not start with the package name,
      'This package' or similar.
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    
    Size of tarball: 6178909 bytes
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘methyAnalysis’
    
    Depends: includes the non-default packages:
      ‘Biobase’ ‘scales’ ‘reshape2’ ‘ggplot2’ ‘matrixStats’
      ‘FDb.InfiniumMethylation.hg19’ ‘minfi’
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    
    CRAN-pack does not know about
      .travis.yml
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  9.8Mb
      sub-directories of 1Mb or more:
        data      6.8Mb
        extdata   1.7Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Packages listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘Biobase’ ‘minfi’ ‘lattice’ ‘matrixStats’
    A package should be listed in only one of these fields.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' calls to packages already attached by Depends:
      ‘FDb.InfiniumMethylation.hg19’ ‘ggplot2’ ‘matrixStats’ ‘minfi’
      ‘reshape2’ ‘scales’
      Please remove these calls from your code.
    'library' or 'require' calls in package code:
      ‘Biostrings’ ‘MASS’ ‘lumi’ ‘parallel’ ‘rtracklayer’
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    Namespace in Imports field not imported from: ‘graphics’
      All declared Imports should be used.
    Packages in Depends field not imported from:
      ‘FDb.InfiniumMethylation.hg19’ ‘ggplot2’ ‘matrixStats’ ‘methods’
      ‘reshape2’ ‘scales’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ':::' call which should be '::': ‘lumi:::produceMethylationGEOSubmissionFile’
      See the note in ?`:::` about the use of this operator.
    Unexported objects imported by ':::' calls:
      ‘Biobase:::unsafeSetSlot’ ‘genefilter:::.findCentralMap’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      IlluminaHumanMethylation450kCOLORCHANNEL MethylSet RGChannelSet
      abline addColorChannelInfo aes allele as.dist axis box capture.output
      channel.probes colMedians colQuantiles colSds colorRampPalette
      coord_flip data dnorm dropouts drops ecdf facet_grid features
      gamma.integral gamma.mle gamma.mode geom_histogram ggplot hclust
      hm27.controls hm27.ordering hm450.controls hm450.ordering huber index
      intensity lines log_trans mclapply melt mu offset opts p.adjust
      packageDescription packageVersion par plot.density pnorm points
      position_identity read.csv read.delim read.table rect rowMins
      scale_colour_manual scale_fill_manual scale_shape_manual
      scale_x_continuous scale_y_continuous scale_y_discrete
      subsetCommonProbes text theme_bw title value variable weighted.mean
    Consider adding
      importFrom("grDevices", "colorRampPalette")
      importFrom("graphics", "abline", "axis", "box", "lines", "par",
                 "points", "rect", "text", "title")
      importFrom("stats", "as.dist", "dnorm", "ecdf", "hclust", "offset",
                 "p.adjust", "pnorm", "weighted.mean")
      importFrom("utils", "capture.output", "data", "packageDescription",
                 "packageVersion", "read.csv", "read.delim", "read.table")
    to your NAMESPACE file.
    ```

*   checking Rd files ... NOTE
    ```
    prepare_Rd: estimateM.Rd:34-36: Dropping empty section \seealso
    prepare_Rd: estimateM.Rd:37-39: Dropping empty section \examples
    ```

*   checking for unstated dependencies in vignettes ... NOTE
    ```
    'library' or 'require' call not declared from: ‘TxDb.Hsapiens.UCSC.hg19.knownGene’
    ```

# MFHD

Version: 0.0.1

## In both

*   checking whether package ‘MFHD’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/netapp/home/hb/repositories/matrixStats/revdep/checks/MFHD/new/MFHD.Rcheck/00install.out’ for details.
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘K. Vakili <kaveh.vakili@wis.kuleuven.be>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

## Installation

### Devel

```
* installing *source* package ‘MFHD’ ...
** package ‘MFHD’ successfully unpacked and MD5 sums checked
** R
** preparing package for lazy loading
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘rgl’
ERROR: lazy loading failed for package ‘MFHD’
* removing ‘/netapp/home/hb/repositories/matrixStats/revdep/checks/MFHD/new/MFHD.Rcheck/MFHD’

```
### CRAN

```
* installing *source* package ‘MFHD’ ...
** package ‘MFHD’ successfully unpacked and MD5 sums checked
** R
** preparing package for lazy loading
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘rgl’
ERROR: lazy loading failed for package ‘MFHD’
* removing ‘/netapp/home/hb/repositories/matrixStats/revdep/checks/MFHD/old/MFHD.Rcheck/MFHD’

```
# MIGSA

Version: 1.2.0

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘MIGSA-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: Genesets-enrichr
    > ### Title: List and download gene sets from enrichr database
    > ### Aliases: Genesets-enrichr enrichrGeneSets
    > ###   enrichrGeneSets,character-method enrichrGeneSets,ANY-method
    > ###   downloadEnrichrGeneSets downloadEnrichrGeneSets,character-method
    > ###   downloadEnrichrGeneSets,ANY-method
    > 
    > ### ** Examples
    > 
    > ## Lets list all the gene sets that can be downloaded from Enichr website.
    > enrichrGeneSets();
    Warning in readLines(biocURL) :
      URL 'http://www.bioconductor.org/main.html': status was 'Couldn't resolve host name'
    Error in enrichrGeneSets() : You must have internet connection.
    Calls: enrichrGeneSets -> enrichrGeneSets
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/runTests.R’ failed.
    Last 13 lines of output:
      
          Filter, Find, Map, Position, Reduce, anyDuplicated, append,
          as.data.frame, cbind, colMeans, colSums, colnames, do.call,
          duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
          lapply, lengths, mapply, match, mget, order, paste, pmax, pmax.int,
          pmin, pmin.int, rank, rbind, rowMeans, rowSums, rownames, sapply,
          setdiff, sort, table, tapply, union, unique, unsplit, which,
          which.max, which.min
      
      
      
      Error in library("RUnit", quietly = TRUE) : 
        there is no package called 'RUnit'
      Calls: <Anonymous> -> library
      Execution halted
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Juan C. Rodriguez <jcrodriguez@bdmg.com.ar>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    Scale for 'x' is already present. Adding another scale for 'x', which will
    replace the existing scale.
    Scale for 'y' is already present. Adding another scale for 'y', which will
    replace the existing scale.
    Loading required package: GO.db
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'MIGSA.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `multirow.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.7 \DeclareMathOperator
                            *{\argmax}{arg\,max}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# mixOmics

Version: 6.3.1

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘rgl’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Kim-Anh Le Cao <kimanh.lecao@unimelb.edu.au>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# mmtfa

Version: 0.1

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Jeffrey L. Andrews <jeffrey.andrews@macewan.ca>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    plot.mmtfa: no visible global function definition for ‘plot’
    plot.mmtfa: no visible global function definition for ‘title’
    plot.mmtfa: no visible global function definition for ‘contour’
    plot.mmtfa: no visible global function definition for ‘rgb’
    plot.mmtfa : dunivt: no visible global function definition for
      ‘mahalanobis’
    plot.mmtfa : dunivt: no visible global function definition for ‘dnorm’
    plot.mmtfa: no visible global function definition for ‘density’
    plot.mmtfa: no visible global function definition for ‘curve’
    sginit: no visible global function definition for ‘cov.wt’
    sgupdate: no visible global function definition for ‘cov.wt’
    winit: no visible global function definition for ‘mahalanobis’
    Undefined global functions or variables:
      contour cov.wt curve density dnorm kmeans mahalanobis par plot
      rainbow rgb runif title uniroot
    Consider adding
      importFrom("grDevices", "rainbow", "rgb")
      importFrom("graphics", "contour", "curve", "par", "plot", "title")
      importFrom("stats", "cov.wt", "density", "dnorm", "kmeans",
                 "mahalanobis", "runif", "uniroot")
    to your NAMESPACE file.
    ```

# MoEClust

Version: 1.0.0

## In both

*   checking top-level files ... WARNING
    ```
    Conversion of ‘README.md’ failed:
    pandoc: Could not fetch https://www.r-pkg.org/badges/version/MoEClust
    FailedConnectionException2 "www.r-pkg.org" 443 True getAddrInfo: does not exist (Name or service not known)
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Keefe Murphy <keefe.murphy@ucd.ie>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# monocle

Version: 2.6.1

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'monocle-vignette.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `sectsty.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.36 \sectionfont
                     {\sffamily\bfseries\color{RoyalBlue}\sectionrule{0pt}{0pt}{...
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    ...
    Maintainer: ‘Cole Trapnell <coletrap@uw.edu>’
    
    Unknown, possibly mis-spelled, fields in DESCRIPTION:
      ‘Remotes’
    
    No package encoding and non-ASCII characters in the following R files:
      R/order_cells.R
        1418:       # Here<e2><9c><ac>s how to take the top PCA loading genes, but using
    
    The Title field should be in title case, current version then in title case:
    ‘Clustering, differential expression, and trajectory analysis for single- cell RNA-Seq’
    ‘Clustering, Differential Expression, and Trajectory Analysis for Single- Cell RNA-Seq’
    
    The Description field should not start with the package name,
      'This package' or similar.
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    
    Size of tarball: 9497684 bytes
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘Rcpp’ ‘biocViews’
      All declared Imports should be used.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    exportCDS: no visible binding for global variable ‘use_for_ordering’
    extract_good_ordering: no visible global function definition for ‘nei’
    fit_model_helper: no visible binding for global variable ‘Size_Factor’
    get_next_node_id: no visible binding for '<<-' assignment to
      ‘next_node’
    get_next_node_id: no visible binding for global variable ‘next_node’
    make_canonical: no visible global function definition for ‘nei’
    measure_diameter_path: no visible global function definition for ‘nei’
    orderCells: no visible binding for '<<-' assignment to ‘next_node’
    plot_multiple_branches_pseudotime: no visible global function
      definition for ‘exprs<-’
    plot_multiple_branches_pseudotime: no visible binding for global
      variable ‘pseudocount’
    plot_multiple_branches_pseudotime: no visible binding for global
      variable ‘Branch’
    plot_multiple_branches_pseudotime: no visible binding for global
      variable ‘mutate’
    project2MST: no visible global function definition for ‘nei’
    Undefined global functions or variables:
      Branch Size_Factor exprs<- jaccard_coeff mutate nei next_node
      pseudocount use_for_ordering
    ```

*   checking files in ‘vignettes’ ... NOTE
    ```
    The following directory looks like a leftover from 'knitr':
      ‘figure’
    Please remove from your package.
    ```

# motifbreakR

Version: 1.8.0

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘motifStack’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Simon Gert Coetzee <Simon.Coetzee@cshs.org>’
    
    The Title field should be in title case, current version then in title case:
    ‘A Package For Predicting The Disruptiveness Of Single Nucleotide Polymorphisms On Transcription Factor Binding Sites’
    ‘A Package for Predicting the Disruptiveness of Single Nucleotide Polymorphisms on Transcription Factor Binding Sites’
    
    This build time stamp is over a month old.
    ```

# mrfDepth

Version: 1.0.6

## In both

*   checking top-level files ... WARNING
    ```
    Conversion of ‘README.md’ failed:
    pandoc: Could not fetch https://www.r-pkg.org/badges/version/mrfDepth
    FailedConnectionException2 "www.r-pkg.org" 443 True getAddrInfo: does not exist (Name or service not known)
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Pieter Segaert <Pieter.Segaert@kuleuven.be>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 15.4Mb
      sub-directories of 1Mb or more:
        libs  14.6Mb
    ```

# MultiBD

Version: 0.2.0

## In both

*   checking top-level files ... WARNING
    ```
    Conversion of ‘README.md’ failed:
    pandoc: Could not fetch https://travis-ci.org/msuchard/MultiBD.svg?branch=master
    FailedConnectionException2 "travis-ci.org" 443 True getAddrInfo: does not exist (Name or service not known)
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    ! LaTeX Error: File `framed.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    Enter file name: 
    ! Emergency stop.
    <read *> 
             
    l.39 \definecolor
    
    pandoc: Error producing PDF
    Error: processing vignette 'SIR-MCMC.Rmd' failed with diagnostics:
    pandoc document conversion failed with error 43
    Execution halted
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Marc A. Suchard <msuchard@ucla.edu>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 10.5Mb
      sub-directories of 1Mb or more:
        libs   8.9Mb
    ```

# NanoStringDiff

Version: 1.8.0

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘hong wang <hong.wang@uky.edu>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      ‘optimize’
    glmfit.full: no visible global function definition for ‘lm’
    glmfit.full: no visible global function definition for ‘median’
    glmfit.full: no visible global function definition for ‘IQR’
    glmfit.full : get.phi: no visible global function definition for
      ‘optimize’
    glmfit.full : get.beta.full: no visible global function definition for
      ‘optim’
    glmfit.reduce: no visible global function definition for ‘lm’
    glmfit.reduce : get.beta.reduce: no visible global function definition
      for ‘optim’
    rnegbinom: no visible global function definition for ‘rpois’
    rnegbinom: no visible global function definition for ‘rgamma’
    Undefined global functions or variables:
      IQR glm lm median optim optimize p.adjust pchisq poisson read.table
      rgamma rpois
    Consider adding
      importFrom("stats", "IQR", "glm", "lm", "median", "optim", "optimize",
                 "p.adjust", "pchisq", "poisson", "rgamma", "rpois")
      importFrom("utils", "read.table")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    
        Vignettes contain introductory material; view with
        'browseVignettes()'. To cite Bioconductor, see
        'citation("Biobase")', and for packages 'citation("pkgname")'.
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'NanoStringDiff.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `sectsty.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.37 \sectionfont
                     {\sffamily\bfseries\color{BiocBlue}\sectionrule{0pt}{0pt}{-...
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# neurobase

Version: 1.13.2

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 189-219 (nifti_basics.Rmd) 
    Error: processing vignette 'nifti_basics.Rmd' failed with diagnostics:
    Could not resolve host: raw.githubusercontent.com; Name or service not known
    Execution halted
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘John Muschelli <muschellij2@gmail.com>’
    
    This build time stamp is over a month old.
    ```

# NSA

Version: 0.0.32

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Maria Ortiz-Estevez <mortizest@gmail.com>’
    
    The Title field should be in title case, current version then in title case:
    ‘Post-normalization of total copy numbers’
    ‘Post-Normalization of Total Copy Numbers’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      ‘R.methodsS3’ ‘MASS’ ‘matrixStats’ ‘R.oo’ ‘R.utils’ ‘aroma.core’
      ‘aroma.affymetrix’ ‘DNAcopy’
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Packages in Depends field not imported from:
      ‘DNAcopy’ ‘MASS’ ‘R.methodsS3’ ‘R.oo’ ‘aroma.affymetrix’ ‘aroma.core’
      ‘matrixStats’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    setTags.SampleNormalization: no visible global function definition for
      ‘trim’
    snpsNByTotalAndFracB.matrix: no visible global function definition for
      ‘throw’
    snpsNByTotalAndFracB.matrix: no visible global function definition for
      ‘str’
    snpsNByTotalAndFracB.matrix: no visible global function definition for
      ‘rowAlls’
    Undefined global functions or variables:
      AffymetrixCdfFile CNA Object approxfun aromaSettings byPath extend
      extractMatrix findUnitsTodo getAsteriskTags getChipType getFile
      getFullName getFullNames getGenomeInformation getName getNames
      getPath getPathname getPathnames getPositions getRam getRootPath
      getTags getUnitsOnChromosome hist median nbrOfFiles newInstance
      packageDescription rowAlls rowMedians segment setTags str throw trim
      verbose
    Consider adding
      importFrom("graphics", "hist")
      importFrom("stats", "approxfun", "median")
      importFrom("utils", "packageDescription", "str")
    to your NAMESPACE file.
    ```

# pandaR

Version: 1.10.0

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Joseph N. Paulson <jpaulson@jimmy.harvard.edu>, Dan Schlauch <dschlauch@fas.harvard.edu>’
    
    The maintainer field is invalid or specifies more than one person
    
    This build time stamp is over a month old.
    
    Size of tarball: 9093987 bytes
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  9.4Mb
      sub-directories of 1Mb or more:
        data   9.1Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    dFunction: no visible global function definition for ‘cor’
    importPandaMatlab: no visible global function definition for
      ‘read.delim’
    panda: no visible global function definition for ‘cor’
    panda: no visible global function definition for ‘aggregate’
    plot.panda: no visible global function definition for ‘hist’
    plotCommunityDetection: no visible global function definition for
      ‘title’
    plotZbyTF: no visible global function definition for ‘aggregate’
    prepResult: no visible global function definition for ‘pnorm’
    Undefined global functions or variables:
      aggregate cor hist pnorm read.delim title
    Consider adding
      importFrom("graphics", "hist", "title")
      importFrom("stats", "aggregate", "cor", "pnorm")
      importFrom("utils", "read.delim")
    to your NAMESPACE file.
    ```

*   checking Rd \usage sections ... NOTE
    ```
    S3 methods shown with full name in documentation object 'plot.panda':
      ‘plot.panda’
    
    The \usage entries for S3 methods should use the \method markup and not
    their full name.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

# pcaMethods

Version: 1.70.0

## In both

*   checking Rd cross-references ... WARNING
    ```
    Unknown package ‘ade4’ in Rd xrefs
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Henning Redestig <henning.red@gmail.com>’
    
    The Title field should be in title case, current version then in title case:
    ‘A collection of PCA methods’
    ‘A Collection of PCA Methods’
    
    This build time stamp is over a month old.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    svdImpute: no visible global function definition for ‘prcomp’
    svdPca: no visible global function definition for ‘prcomp’
    plot,pcaRes: no visible global function definition for ‘gray’
    plot,pcaRes: no visible global function definition for ‘barplot’
    plot,pcaRes: no visible global function definition for ‘legend’
    slplot,pcaRes: no visible global function definition for ‘par’
    slplot,pcaRes: no visible global function definition for ‘layout’
    slplot,pcaRes: no visible global function definition for ‘abline’
    slplot,pcaRes: no visible global function definition for ‘lines’
    Undefined global functions or variables:
      abline barplot cor cov gray layout legend lines median na.omit pairs
      par points prcomp qf rnorm runif setTxtProgressBar text
      txtProgressBar
    Consider adding
      importFrom("grDevices", "gray")
      importFrom("graphics", "abline", "barplot", "layout", "legend",
                 "lines", "pairs", "par", "points", "text")
      importFrom("stats", "cor", "cov", "median", "na.omit", "prcomp", "qf",
                 "rnorm", "runif")
      importFrom("utils", "setTxtProgressBar", "txtProgressBar")
    to your NAMESPACE file.
    ```

# peakPick

Version: 0.11

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Paal O. Westermark <pal-olof.westermark@charite.de>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# phosphonormalizer

Version: 1.2.0

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Sohrab Saraei <sohrab.saraei@utu.fi>’
    
    The Title field should be in title case, current version then in title case:
    ‘Compensates for the bias introduced by median normalization in phosphoproteomics’
    ‘Compensates for the Bias Introduced by Median Normalization in Phosphoproteomics’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking package dependencies ... NOTE
    ```
    Package which this enhances but not available for checking: ‘MSnbase’
    ```

# Pigengene

Version: 1.4.0

## In both

*   checking examples ... ERROR
    ```
    ...
    Pigengene started analizing 366 samples using 1000 genes...
    Warning: executing %dopar% sequentially: no parallel backend registered
    Pigengenes...
    Pigengene plots in:
    /netapp/home/hb/repositories/matrixStats/revdep/checks/Pigengene/new/Pigengene.Rcheck/pigengene/plots
    learn.bn() with bnNum= 10 started at:
    2018-01-22 15:13:50
    Warning in check.threshold(threshold, strength) :
      the threshold is outside the range of the strength values.
    learn.bn() took:
    2.416554 secs
    Making decision trees...
    minPerLeaf: 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37
    costs:
        AML MDS
    AML   0   1
    MDS   1   0
    Error in UseMethod("as.party") : 
      no applicable method for 'as.party' applied to an object of class "C5.0"
    Calls: one.step.pigengene ... unlist -> get.fitted.leaf -> eval -> parse -> <Anonymous>
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Warning: running command 'kpsewhich framed.sty' had status 1
    Warning in test_latex_pkg("framed", system.file("misc", "framed.sty", package = "knitr")) :
      unable to find LaTeX package 'framed'; will use a copy from knitr
    Warning in has_utility("pdfcrop") :
      pdfcrop not installed or not in PATH
    Quitting from lines 114-116 (Pigengene_inference.Rnw) 
    Error: processing vignette 'Pigengene_inference.Rnw' failed with diagnostics:
    no applicable method for 'as.party' applied to an object of class "C5.0"
    Execution halted
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Habil Zare <zare@txstate.edu>’
    
    The Title field should be in title case, current version then in title case:
    ‘Infers biological signatures from gene expression data’
    ‘Infers Biological Signatures from Gene Expression Data’
    
    The Description field should not start with the package name,
      'This package' or similar.
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' calls in package code:
      ‘AnnotationDbi’ ‘biomaRt’ ‘energy’ ‘org.Hs.eg.db’ ‘org.Mm.eg.db’
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    Found the following possibly unsafe calls:
    File ‘Pigengene/R/bn.calculation.R’:
      assignInNamespace("supported.clusters", fixArgs, "bnlearn")
    
    check.pigengene.input: no visible global function definition for
      ‘setNames’
    welch.pvalue: no visible global function definition for ‘as.formula’
    welch.pvalue: no visible global function definition for ‘oneway.test’
    Undefined global functions or variables:
      as.formula oneway.test setNames
    Consider adding
      importFrom("stats", "as.formula", "oneway.test", "setNames")
    to your NAMESPACE file.
    ```

# POMaSPU

Version: 1.0.0

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Junghi Kim <junghikim0@gmail.com>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# PSCBS

Version: 0.63.0

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Henrik Bengtsson <henrikb@braju.com>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# QDNAseq

Version: 1.14.0

## In both

*   checking for missing documentation entries ... WARNING
    ```
    Undocumented code objects:
      ‘exportVCF’
    All user-level objects in a package should have documentation entries.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Daoud Sie <d.sie@vumc.nl>’
    
    The Title field should be in title case, current version then in title case:
    ‘Quantitative DNA sequencing for chromosomal aberrations’
    ‘Quantitative DNA Sequencing for Chromosomal Aberrations’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    25323814808468135.336.738865427951103207.660.738865426552879207.650
    FINISHED!
    
    Total time:0minutes
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'QDNAseq.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `titlesec.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.95 \makeatletter
                      ^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# r2dRue

Version: 1.0.4

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘rgdal’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Alberto Ruiz Moreno <aruiz@eeza.csic.es>’
    
    No package encoding and non-ASCII characters in the following R files:
      R/r2dRue.r
        976: 	#tama<f1>o del ultimo bloque
        1072: 	#as<ed> que hay que forzar a ser matriz
      R/rgf.R
        90: #          1<aa> modificacion: si inFl 
        120: # CHANGES: 20/04/2010 - a<f1>adir comprobacion de parametros 
    
    The Title field should be in title case, current version then in title case:
    ‘2d Rain Use Efficience model’
    ‘2d Rain Use Efficience Model’
    
    The Description field should start with a capital letter.
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# RAC

Version: 1.1.1

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Baldan D. <damiano.baldan91@gmail.com>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# randomizationInference

Version: 1.0.3

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Joseph J. Lee <joseph.j.lee@post.harvard.edu>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking R code for possible problems ... NOTE
    ```
    anovaF: no visible global function definition for ‘anova’
    anovaF: no visible global function definition for ‘lm’
    randInterval: no visible global function definition for ‘quantile’
    randInterval : <anonymous>: no visible global function definition for
      ‘quantile’
    randPlot: no visible global function definition for ‘par’
    randPlot: no visible global function definition for ‘hist’
    randPlot: no visible global function definition for ‘abline’
    Undefined global functions or variables:
      abline anova hist lm par quantile
    Consider adding
      importFrom("graphics", "abline", "hist", "par")
      importFrom("stats", "anova", "lm", "quantile")
    to your NAMESPACE file.
    ```

# RnBeads

Version: 1.10.5

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Fabian Mueller <rnbeads@mpi-inf.mpg.de>’
    
    The Title field is just the package name: provide a real title.
    
    The Description field should not start with the package name,
      'This package' or similar.
    
    Size of tarball: 6475192 bytes
    ```

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      ‘BiocGenerics’ ‘S4Vectors’ ‘GenomicRanges’ ‘MASS’ ‘cluster’ ‘ff’
      ‘fields’ ‘ggplot2’ ‘gplots’ ‘gridExtra’ ‘limma’ ‘matrixStats’
      ‘illuminaio’ ‘methylumi’ ‘plyr’
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    
    CRAN-pack does not know about
      .travis.yml
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.6Mb
      sub-directories of 1Mb or more:
        R     1.1Mb
        bin   1.0Mb
        doc   3.2Mb
    ```

*   checking whether the namespace can be loaded with stated dependencies ... NOTE
    ```
    Warning: no function found corresponding to methods exports from ‘RnBeads’ for: ‘samples’
    
    A namespace must be able to be loaded with just the base namespace
    loaded: otherwise if the namespace gets loaded by a saved object, the
    session will be unable to start.
    
    Probably some imports need to be declared in the NAMESPACE file.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Error in gzfile(file, "rb") : cannot open the connection
    Error in gzfile(file, "rb") : cannot open the connection
    Unexported objects imported by ':::' calls:
      ‘doParallel:::.options’ ‘grDevices:::.smoothScatterCalcDensity’
      ‘minfi:::.default.450k.annotation’ ‘minfi:::.extractFromRGSet450k’
      ‘minfi:::.normalizeFunnorm450k’
      See the note in ?`:::` about the use of this operator.
    Unavailable namespace imported from by a ':::' call: ‘Gviz’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      IdeogramTrack IlluminaHumanMethylation450kmanifest
      IlluminaHumanMethylationEPICmanifest Index Intensity Measure
      PairsBootRefFreeEwasModel Probe RGChannelSet Rd2HTML RefFreeEwasModel
      SNP Sample Slide Target Term UcscTrack Value addSex as.profileCGH
      assayDataElement assayDataElementNames barcode bv chrom combinedRank
      combinedRank.var comma covgMedian covgPercLow covgPercUp cv.glmnet
      daglad diffmeth diffmeth.p.adj.fdr diffmeth.p.val
      dinucleotideFrequency eps expectedCounts featureData featureData<-
      featureNames featureNames<- foreach geneCounts genome<- getCN
      getDoParWorkers getGreen getManifest getMeth getRed getSex getUnmeth
      getVarCov glmnet grid.draw grid.newpage group group1 group2 i
      impute.knn intensities is.subsegmentation k letterFrequency lme
      loadRegionDB log10FDR log10P mapToGenome mean.diff mean.quot.log2
      melt mergeRegionDBs muted n.sites ng.2 num.sites numSites
      numeric.names oddsRatios p.vals.t.na.adj p.valsvar..na.adj pData
      percent_format phenoData phenoData<- plotTracks preprocessSWAN
      pvalues qvalue refText reg.type region.size registerDoParallel
      relative.coord report rnb.logger.info rnb.logger.status runLOLA
      samples seqlengths seqlevels<- sigCategories sites2ignore size
      solve.QP stopCluster sva target tsne type types universeCounts
      useMart v var.diff varFit varLabels x xmlValue y yint
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'RnBeads.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `vmargin.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.10 \usepackage
                    [american]{babel}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# RTransProb

Version: 0.1.0

## In both

*   checking examples ... ERROR
    ```
    ...
    The error most likely occurred in:
    
    > ### Name: TransitionProb
    > ### Title: Estimation of credit transition probabilities
    > ### Aliases: TransitionProb
    > 
    > ### ** Examples
    > 
    > 
    > #Example 1:
    > #When start date and end date are not specified, the entire dataset is used and the package
    > #performs TTC calculations. Equally when snapshots and interval are not specified the defaults
    > #are 1.
    > snapshots <- 0
    > interval <- 0
    > startDate  <- 0
    > endDate    <- 0
    > Example1<-TransitionProb(data,startDate,endDate,'cohort', snapshots, interval)
    Error in if (is.nan(rating)) { : the condition has length > 1
    Calls: TransitionProb -> getidTotCntCohort
    Execution halted
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Ab NDiaye <pabdndiaye@gmail.com>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# samr

Version: 2.0

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Rob Tibshirani <tibs@stanford.edu>’
    
    This build time stamp is over a month old.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' calls to packages already attached by Depends:
      ‘impute’ ‘matrixStats’
      Please remove these calls from your code.
    Packages in Depends field not imported from:
      ‘impute’ ‘matrixStats’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    samr.plot: no visible global function definition for ‘abline’
    samr.tail.strength: no visible global function definition for ‘var’
    samr.xl.compute.plot.xy: no visible global function definition for
      ‘points’
    samr.xl.compute.plot.xy: no visible global function definition for
      ‘abline’
    samr.xl.impute.data: no visible global function definition for
      ‘impute.knn’
    samr.xl.impute.data: no visible binding for global variable
      ‘samr.xl.var.knn.neighbors’
    Undefined global functions or variables:
      abline dev.off impute.knn lines mad median mtext par plot points
      predict quantile rowMedians rpois runif samr.xl.var.knn.neighbors
      smooth.spline title var
    Consider adding
      importFrom("grDevices", "dev.off")
      importFrom("graphics", "abline", "lines", "mtext", "par", "plot",
                 "points", "title")
      importFrom("stats", "mad", "median", "predict", "quantile", "rpois",
                 "runif", "smooth.spline", "var")
    to your NAMESPACE file.
    ```

*   checking Rd \usage sections ... NOTE
    ```
    S3 methods shown with full name in documentation object 'samr-internal':
      ‘plot.SAMoutput’ ‘print.SAMoutput’
    
    The \usage entries for S3 methods should use the \method markup and not
    their full name.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

# scmap

Version: 1.1.5

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error: processing vignette 'scmap.Rmd' failed with diagnostics:
    there is no package called ‘BiocStyle’
    Execution halted
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Vladimir Kiselev <vladimir.yu.kiselev@gmail.com>’
    
    The Title field should be in title case, current version then in title case:
    ‘A tool for unsupervised projection of single cell RNA-seq data’
    ‘A Tool for Unsupervised Projection of Single Cell RNA-Seq Data’
    
    This build time stamp is over a month old.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.7Mb
      sub-directories of 1Mb or more:
        data   2.7Mb
        doc    1.3Mb
        libs   1.4Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Author field differs from that derived from Authors@R
      Author:    ‘Vladimir Kiselev’
      Authors@R: ‘Vladimir Kiselev [cre, aut], Andrew Yiu [ctb], Martin Hemberg [aut]’
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘Biobase’
      All declared Imports should be used.
    ```

# scone

Version: 1.2.0

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘scone-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: SCRAN_FN
    > ### Title: Simple deconvolution normalization wrapper
    > ### Aliases: SCRAN_FN
    > 
    > ### ** Examples
    > 
    > ei <- matrix(0:76,nrow = 7)
    > eo <- SCRAN_FN(ei)
    Error in SCRAN_FN(ei) : scran package needed for SCRAN_FN()
    Execution halted
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Michael Cole <mbeloc@gmail.com>’
    
    The Title field should be in title case, current version then in title case:
    ‘Single Cell Overview of Normalized Expression data’
    ‘Single Cell Overview of Normalized Expression Data’
    
    The Description field should not start with the package name,
      'This package' or similar.
    
    This build time stamp is over a month old.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    sconeReport : server: no visible global function definition for ‘theme’
    sconeReport : server: no visible global function definition for
      ‘element_blank’
    sconeReport : server: no visible global function definition for
      ‘ggplotly’
    sconeReport : server: no visible global function definition for
      ‘geom_violin’
    sconeReport : server: no visible global function definition for
      ‘coord_cartesian’
    sconeReport : server: no visible global function definition for
      ‘scale_fill_manual’
    sconeReport : server: no visible global function definition for
      ‘geom_point’
    sconeReport : server: no visible global function definition for
      ‘guides’
    Undefined global functions or variables:
      %>% aes coord_cartesian element_blank geom_bar geom_point geom_violin
      ggplot ggplotly guides labs plot_ly plotlyOutput renderVisNetwork
      scale_fill_manual theme visEdges visGroups visHierarchicalLayout
      visLegend visNetwork visNetworkOutput visNetworkProxy visOptions
      visSelectNodes ylim
    ```

# SemiParSampleSel

Version: 1.5

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘copula’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Giampiero Marra <giampiero.marra@ucl.ac.uk>’
    
    This build time stamp is over a month old.
    ```

# SGP

Version: 1.7-0.0

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Damian W. Betebenner <dbetebenner@nciea.org>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# shinyMethyl

Version: 1.14.0

## In both

*   checking top-level files ... WARNING
    ```
    Conversion of ‘README.md’ failed:
    pandoc: Could not fetch https://travis-ci.org/Jfortin1/shinyMethyl.svg?branch=master
    FailedConnectionException2 "travis-ci.org" 443 True getAddrInfo: does not exist (Name or service not known)
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Setting options('download.file.method.GEOquery'='auto')
    Setting options('GEOquery.inmemory.gpl'=FALSE)
    Loading required package: IlluminaHumanMethylation450kmanifest
    Loading required package: IlluminaHumanMethylation450kanno.ilmn12.hg19
    Loading required package: shiny
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'shinyMethyl.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `titlesec.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.95 \makeatletter
                      ^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Jean-Philippe Fortin <jfortin@jhsph.edu>’
    
    Unknown, possibly mis-spelled, fields in DESCRIPTION:
      ‘Url’
    
    The Title field should be in title case, current version then in title case:
    ‘Interactive visualization for Illumina methylation arrays’
    ‘Interactive Visualization for Illumina Methylation Arrays’
    
    This build time stamp is over a month old.
    
    Size of tarball: 9379921 bytes
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    
    CRAN-pack does not know about
      .travis.yml
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    server.shinyMethyl : <anonymous>: no visible global function definition
      for ‘lines’
    server.shinyMethyl : <anonymous>: no visible global function definition
      for ‘legend’
    ui.shinyMethyl: no visible binding for '<<-' assignment to
      ‘sampleColors’
    shinySummarize,GenomicRatioSet: no visible global function definition
      for ‘prcomp’
    shinySummarize,RGChannelSet: no visible global function definition for
      ‘prcomp’
    Undefined global functions or variables:
      abline complete.cases current.control.type current.density.type
      current.probe.type genderCutoff grid legend lines lm
      mouse.click.indices palette par plot points prcomp rect write.csv
    Consider adding
      importFrom("grDevices", "palette")
      importFrom("graphics", "abline", "grid", "legend", "lines", "par",
                 "plot", "points", "rect")
      importFrom("stats", "complete.cases", "lm", "prcomp")
      importFrom("utils", "write.csv")
    to your NAMESPACE file.
    ```

# SICtools

Version: 1.8.0

## In both

*   checking whether package ‘SICtools’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/netapp/home/hb/repositories/matrixStats/revdep/checks/SICtools/new/SICtools.Rcheck/00install.out’ for details.
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Xiaobin Xing <xiaobinxing0316@gmail.com>’
    
    The Title field should be in title case, current version then in title case:
    ‘Find SNV/Indel differences between two bam files with near relationship’
    ‘Find SNV/Indel Differences Between Two Bam Files with Near Relationship’
    
    The Description field should not start with the package name,
      'This package' or similar.
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      ‘Rsamtools’ ‘doParallel’ ‘Biostrings’ ‘stringr’ ‘matrixStats’ ‘plyr’
      ‘GenomicRanges’ ‘IRanges’
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .BBSoptions
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    
    CRAN-pack does not know about
      .BBSoptions
    ```

## Installation

### Devel

```
* installing *source* package ‘SICtools’ ...
** libs
** arch - 
Makefile:24: warning: overriding recipe for target `.c.o'
/netapp/home/hb/shared/apps/manual/r/R-3.4.3-20180109/lib64/R/etc/Makeconf:159: warning: ignoring old recipe for target `.c.o'
make[1]: Entering directory `/netapp/home/hb/repositories/matrixStats/revdep/checks/SICtools/new/SICtools.Rcheck/00_pkg_src/SICtools/src'
make[2]: Entering directory `/netapp/home/hb/repositories/matrixStats/revdep/checks/SICtools/new/SICtools.Rcheck/00_pkg_src/SICtools/src'
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -DBGZF_CACHE -I. bgzf.c -o bgzf.o
bgzf.c: In function ‘worker_aux’:
bgzf.c:392:9: warning: variable ‘tmp’ set but not used [-Wunused-but-set-variable]
  int i, tmp, stop = 0;
         ^
bgzf.c: In function ‘bgzf_close’:
bgzf.c:572:11: warning: variable ‘count’ set but not used [-Wunused-but-set-variable]
  int ret, count, block_length;
           ^
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. kstring.c -o kstring.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. bam_aux.c -o bam_aux.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. bam.c -o bam.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. bam_import.c -o bam_import.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. sam.c -o sam.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. bam_index.c -o bam_index.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. bam_pileup.c -o bam_pileup.o
bam_pileup.c: In function ‘resolve_cigar2’:
bam_pileup.c:75:9: warning: variable ‘is_head’ set but not used [-Wunused-but-set-variable]
  int k, is_head = 0;
         ^
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. bam_lpileup.c -o bam_lpileup.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. bam_md.c -o bam_md.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. razf.c -o razf.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. faidx.c -o faidx.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. bedidx.c -o bedidx.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. knetfile.c -o knetfile.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. bam_sort.c -o bam_sort.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. sam_header.c -o sam_header.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. bam_reheader.c -o bam_reheader.o
bam_reheader.c: In function ‘bam_reheader’:
bam_reheader.c:12:16: warning: variable ‘old’ set but not used [-Wunused-but-set-variable]
  bam_header_t *old;
                ^
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. kprobaln.c -o kprobaln.o
kprobaln.c: In function ‘kpa_glocal’:
kprobaln.c:78:21: warning: variable ‘is_diff’ set but not used [-Wunused-but-set-variable]
  int bw, bw2, i, k, is_diff = 0, is_backward = 1, Pr;
                     ^
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. bam_cat.c -o bam_cat.o
ar -csru libbam.a bgzf.o kstring.o bam_aux.o bam.o bam_import.o sam.o bam_index.o bam_pileup.o bam_lpileup.o bam_md.o razf.o faidx.o bedidx.o knetfile.o bam_sort.o sam_header.o bam_reheader.o kprobaln.o bam_cat.o
make[2]: Leaving directory `/netapp/home/hb/repositories/matrixStats/revdep/checks/SICtools/new/SICtools.Rcheck/00_pkg_src/SICtools/src'
make[2]: Entering directory `/netapp/home/hb/repositories/matrixStats/revdep/checks/SICtools/new/SICtools.Rcheck/00_pkg_src/SICtools/src/bcftools'
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I.. -I. bcf.c -o bcf.o
bcf.c: In function ‘bcf_fmt_core’:
bcf.c:233:9: warning: variable ‘iPL’ set but not used [-Wunused-but-set-variable]
     int iPL = -1;
         ^
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I.. -I. vcf.c -o vcf.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I.. -I. bcfutils.c -o bcfutils.o
bcfutils.c: In function ‘bcf_fit_alt’:
bcfutils.c:119:9: warning: variable ‘ipl’ set but not used [-Wunused-but-set-variable]
     int ipl=-1, igt=-1;
         ^
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I.. -I. prob1.c -o prob1.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I.. -I. em.c -o em.o
em.c: In function ‘bcf_em1’:
em.c:174:12: warning: variable ‘n2’ set but not used [-Wunused-but-set-variable]
  int i, n, n2;
            ^
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I.. -I. kfunc.c -o kfunc.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I.. -I. kmin.c -o kmin.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I.. -I. index.c -o index.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I.. -I. fet.c -o fet.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I.. -I. mut.c -o mut.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I.. -I. bcf2qcall.c -o bcf2qcall.o
ar -csru libbcf.a bcf.o vcf.o bcfutils.o prob1.o em.o kfunc.o kmin.o index.o fet.o mut.o bcf2qcall.o
make[2]: Leaving directory `/netapp/home/hb/repositories/matrixStats/revdep/checks/SICtools/new/SICtools.Rcheck/00_pkg_src/SICtools/src/bcftools'
make[2]: Entering directory `/netapp/home/hb/repositories/matrixStats/revdep/checks/SICtools/new/SICtools.Rcheck/00_pkg_src/SICtools/src/misc'
make[2]: Nothing to be done for `lib'.
make[2]: Leaving directory `/netapp/home/hb/repositories/matrixStats/revdep/checks/SICtools/new/SICtools.Rcheck/00_pkg_src/SICtools/src/misc'
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. bam_tview.c -o bam_tview.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. bam_plcmd.c -o bam_plcmd.o
bam_plcmd.c: In function ‘mpileup’:
bam_plcmd.c:333:28: warning: variable ‘ref16’ set but not used [-Wunused-but-set-variable]
    int total_depth, _ref0, ref16;
                            ^
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. sam_view.c -o sam_view.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. bam_rmdup.c -o bam_rmdup.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. bam_rmdupse.c -o bam_rmdupse.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. bam_mate.c -o bam_mate.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. bam_stat.c -o bam_stat.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. bam_color.c -o bam_color.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. bamtk.c -o bamtk.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. kaln.c -o kaln.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. bam2bcf.c -o bam2bcf.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. bam2bcf_indel.c -o bam2bcf_indel.o
bam2bcf_indel.c: In function ‘bcf_call_gap_prep’:
bam2bcf_indel.c:121:28: warning: variable ‘query’ set but not used [-Wunused-but-set-variable]
  char *inscns = 0, *ref2, *query, **ref_sample;
                            ^
bam2bcf_indel.c:121:21: warning: variable ‘ref2’ set but not used [-Wunused-but-set-variable]
  char *inscns = 0, *ref2, *query, **ref_sample;
                     ^
bam2bcf_indel.c:120:16: warning: variable ‘ref_type’ set but not used [-Wunused-but-set-variable]
  int N, l_run, ref_type;
                ^
bam2bcf_indel.c:119:82: warning: variable ‘score2’ set but not used [-Wunused-but-set-variable]
  int i, s, j, k, t, n_types, *types, max_rd_len, left, right, max_ins, *score1, *score2, max_ref2;
                                                                                  ^
bam2bcf_indel.c:119:73: warning: variable ‘score1’ set but not used [-Wunused-but-set-variable]
  int i, s, j, k, t, n_types, *types, max_rd_len, left, right, max_ins, *score1, *score2, max_ref2;
                                                                         ^
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. errmod.c -o errmod.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. sample.c -o sample.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. cut_target.c -o cut_target.o
cut_target.c: In function ‘main_cut_target’:
cut_target.c:137:36: warning: variable ‘lastpos’ set but not used [-Wunused-but-set-variable]
  int c, tid, pos, n, lasttid = -1, lastpos = -1, l, max_l;
                                    ^
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. phase.c -o phase.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. bam2depth.c -o bam2depth.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. padding.c -o padding.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. bedcov.c -o bedcov.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. bamshuf.c -o bamshuf.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. bam_tview_curses.c -o bam_tview_curses.o
bam_tview_curses.c:5:20: fatal error: curses.h: No such file or directory
 #include <curses.h>
                    ^
compilation terminated.
make[1]: *** [bam_tview_curses.o] Error 1
make[1]: Leaving directory `/netapp/home/hb/repositories/matrixStats/revdep/checks/SICtools/new/SICtools.Rcheck/00_pkg_src/SICtools/src'
make: *** [all-recur] Error 1
ERROR: compilation failed for package ‘SICtools’
* removing ‘/netapp/home/hb/repositories/matrixStats/revdep/checks/SICtools/new/SICtools.Rcheck/SICtools’

```
### CRAN

```
* installing *source* package ‘SICtools’ ...
** libs
** arch - 
Makefile:24: warning: overriding recipe for target `.c.o'
/netapp/home/hb/shared/apps/manual/r/R-3.4.3-20180109/lib64/R/etc/Makeconf:159: warning: ignoring old recipe for target `.c.o'
make[1]: Entering directory `/netapp/home/hb/repositories/matrixStats/revdep/checks/SICtools/old/SICtools.Rcheck/00_pkg_src/SICtools/src'
make[2]: Entering directory `/netapp/home/hb/repositories/matrixStats/revdep/checks/SICtools/old/SICtools.Rcheck/00_pkg_src/SICtools/src'
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -DBGZF_CACHE -I. bgzf.c -o bgzf.o
bgzf.c: In function ‘worker_aux’:
bgzf.c:392:9: warning: variable ‘tmp’ set but not used [-Wunused-but-set-variable]
  int i, tmp, stop = 0;
         ^
bgzf.c: In function ‘bgzf_close’:
bgzf.c:572:11: warning: variable ‘count’ set but not used [-Wunused-but-set-variable]
  int ret, count, block_length;
           ^
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. kstring.c -o kstring.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. bam_aux.c -o bam_aux.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. bam.c -o bam.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. bam_import.c -o bam_import.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. sam.c -o sam.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. bam_index.c -o bam_index.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. bam_pileup.c -o bam_pileup.o
bam_pileup.c: In function ‘resolve_cigar2’:
bam_pileup.c:75:9: warning: variable ‘is_head’ set but not used [-Wunused-but-set-variable]
  int k, is_head = 0;
         ^
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. bam_lpileup.c -o bam_lpileup.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. bam_md.c -o bam_md.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. razf.c -o razf.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. faidx.c -o faidx.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. bedidx.c -o bedidx.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. knetfile.c -o knetfile.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. bam_sort.c -o bam_sort.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. sam_header.c -o sam_header.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. bam_reheader.c -o bam_reheader.o
bam_reheader.c: In function ‘bam_reheader’:
bam_reheader.c:12:16: warning: variable ‘old’ set but not used [-Wunused-but-set-variable]
  bam_header_t *old;
                ^
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. kprobaln.c -o kprobaln.o
kprobaln.c: In function ‘kpa_glocal’:
kprobaln.c:78:21: warning: variable ‘is_diff’ set but not used [-Wunused-but-set-variable]
  int bw, bw2, i, k, is_diff = 0, is_backward = 1, Pr;
                     ^
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. bam_cat.c -o bam_cat.o
ar -csru libbam.a bgzf.o kstring.o bam_aux.o bam.o bam_import.o sam.o bam_index.o bam_pileup.o bam_lpileup.o bam_md.o razf.o faidx.o bedidx.o knetfile.o bam_sort.o sam_header.o bam_reheader.o kprobaln.o bam_cat.o
make[2]: Leaving directory `/netapp/home/hb/repositories/matrixStats/revdep/checks/SICtools/old/SICtools.Rcheck/00_pkg_src/SICtools/src'
make[2]: Entering directory `/netapp/home/hb/repositories/matrixStats/revdep/checks/SICtools/old/SICtools.Rcheck/00_pkg_src/SICtools/src/bcftools'
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I.. -I. bcf.c -o bcf.o
bcf.c: In function ‘bcf_fmt_core’:
bcf.c:233:9: warning: variable ‘iPL’ set but not used [-Wunused-but-set-variable]
     int iPL = -1;
         ^
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I.. -I. vcf.c -o vcf.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I.. -I. bcfutils.c -o bcfutils.o
bcfutils.c: In function ‘bcf_fit_alt’:
bcfutils.c:119:9: warning: variable ‘ipl’ set but not used [-Wunused-but-set-variable]
     int ipl=-1, igt=-1;
         ^
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I.. -I. prob1.c -o prob1.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I.. -I. em.c -o em.o
em.c: In function ‘bcf_em1’:
em.c:174:12: warning: variable ‘n2’ set but not used [-Wunused-but-set-variable]
  int i, n, n2;
            ^
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I.. -I. kfunc.c -o kfunc.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I.. -I. kmin.c -o kmin.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I.. -I. index.c -o index.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I.. -I. fet.c -o fet.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I.. -I. mut.c -o mut.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I.. -I. bcf2qcall.c -o bcf2qcall.o
ar -csru libbcf.a bcf.o vcf.o bcfutils.o prob1.o em.o kfunc.o kmin.o index.o fet.o mut.o bcf2qcall.o
make[2]: Leaving directory `/netapp/home/hb/repositories/matrixStats/revdep/checks/SICtools/old/SICtools.Rcheck/00_pkg_src/SICtools/src/bcftools'
make[2]: Entering directory `/netapp/home/hb/repositories/matrixStats/revdep/checks/SICtools/old/SICtools.Rcheck/00_pkg_src/SICtools/src/misc'
make[2]: Nothing to be done for `lib'.
make[2]: Leaving directory `/netapp/home/hb/repositories/matrixStats/revdep/checks/SICtools/old/SICtools.Rcheck/00_pkg_src/SICtools/src/misc'
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. bam_tview.c -o bam_tview.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. bam_plcmd.c -o bam_plcmd.o
bam_plcmd.c: In function ‘mpileup’:
bam_plcmd.c:333:28: warning: variable ‘ref16’ set but not used [-Wunused-but-set-variable]
    int total_depth, _ref0, ref16;
                            ^
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. sam_view.c -o sam_view.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. bam_rmdup.c -o bam_rmdup.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. bam_rmdupse.c -o bam_rmdupse.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. bam_mate.c -o bam_mate.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. bam_stat.c -o bam_stat.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. bam_color.c -o bam_color.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. bamtk.c -o bamtk.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. kaln.c -o kaln.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. bam2bcf.c -o bam2bcf.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. bam2bcf_indel.c -o bam2bcf_indel.o
bam2bcf_indel.c: In function ‘bcf_call_gap_prep’:
bam2bcf_indel.c:121:28: warning: variable ‘query’ set but not used [-Wunused-but-set-variable]
  char *inscns = 0, *ref2, *query, **ref_sample;
                            ^
bam2bcf_indel.c:121:21: warning: variable ‘ref2’ set but not used [-Wunused-but-set-variable]
  char *inscns = 0, *ref2, *query, **ref_sample;
                     ^
bam2bcf_indel.c:120:16: warning: variable ‘ref_type’ set but not used [-Wunused-but-set-variable]
  int N, l_run, ref_type;
                ^
bam2bcf_indel.c:119:82: warning: variable ‘score2’ set but not used [-Wunused-but-set-variable]
  int i, s, j, k, t, n_types, *types, max_rd_len, left, right, max_ins, *score1, *score2, max_ref2;
                                                                                  ^
bam2bcf_indel.c:119:73: warning: variable ‘score1’ set but not used [-Wunused-but-set-variable]
  int i, s, j, k, t, n_types, *types, max_rd_len, left, right, max_ins, *score1, *score2, max_ref2;
                                                                         ^
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. errmod.c -o errmod.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. sample.c -o sample.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. cut_target.c -o cut_target.o
cut_target.c: In function ‘main_cut_target’:
cut_target.c:137:36: warning: variable ‘lastpos’ set but not used [-Wunused-but-set-variable]
  int c, tid, pos, n, lasttid = -1, lastpos = -1, l, max_l;
                                    ^
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. phase.c -o phase.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. bam2depth.c -o bam2depth.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. padding.c -o padding.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. bedcov.c -o bedcov.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. bamshuf.c -o bamshuf.o
gcc -c -g -Wall -O2 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -D_CURSES_LIB=1 -I. bam_tview_curses.c -o bam_tview_curses.o
bam_tview_curses.c:5:20: fatal error: curses.h: No such file or directory
 #include <curses.h>
                    ^
compilation terminated.
make[1]: *** [bam_tview_curses.o] Error 1
make[1]: Leaving directory `/netapp/home/hb/repositories/matrixStats/revdep/checks/SICtools/old/SICtools.Rcheck/00_pkg_src/SICtools/src'
make: *** [all-recur] Error 1
ERROR: compilation failed for package ‘SICtools’
* removing ‘/netapp/home/hb/repositories/matrixStats/revdep/checks/SICtools/old/SICtools.Rcheck/SICtools’

```
# Single.mTEC.Transcriptomes

Version: 1.6.0

## In both

*   checking data for ASCII and uncompressed saves ... WARNING
    ```
      Warning: package needs dependence on R (>= 2.10)
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    final dispersion estimates
    Warning: Removed 1634 rows containing non-finite values (stat_ydensity).
    Warning: Removed 1634 rows containing non-finite values (stat_boxplot).
    Warning: Removed 1454 rows containing non-finite values (stat_ydensity).
    Warning: Removed 1454 rows containing non-finite values (stat_boxplot).
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'mTECs.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `titlesec.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.95 \makeatletter
                      ^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Alejandro Reyes <alejandro.reyes@embl.de>’
    
    The Title field should be in title case, current version then in title case:
    ‘Single Cell Transcriptome Data and Analysis of Mouse mTEC cells’
    ‘Single Cell Transcriptome Data and Analysis of Mouse mTEC Cells’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    
    Size of tarball: 947852922 bytes
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 908.2Mb
      sub-directories of 1Mb or more:
        data  898.7Mb
        doc     9.4Mb
    ```

# sizeMat

Version: 0.3.0

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Josymar Torrejon-Magallanes <ejosymart@gmail.com>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘mcmc’
      All declared Imports should be used.
    ```

# SRGnet

Version: 1.4.0

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Isar Nassiri <isar_nassiri@urmc.rochester.edu>’
    
    The Title field starts with the package name.
    The Title field should be in title case, current version then in title case:
    ‘SRGnet: An R package for studying synergistic response to gene mutations from transcriptomics data from transcriptomics data’
    ‘SRGnet: An R Package for Studying Synergistic Response to Gene Mutations from Transcriptomics Data from Transcriptomics Data’
    
    This build time stamp is over a month old.
    ```

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      ‘EBcoexpress’ ‘MASS’ ‘igraph’ ‘pvclust’ ‘gbm’ ‘limma’ ‘DMwR’
      ‘matrixStats’ ‘Hmisc’
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Author field differs from that derived from Authors@R
      Author:    ‘Isar Nassiri [aut, cre], Matthew McCall [aut, cre]’
      Authors@R: ‘Isar Nassiri [aut, cre]’
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    SRGnet: no visible binding for global variable
      ‘Differentially_expressed_genes’
    SRGnet: no visible binding for global variable ‘Transcriptomics’
    SRGnet: no visible binding for global variable ‘PLCRG’
    SRGnet: no visible global function definition for ‘ebPatterns’
    SRGnet: no visible global function definition for ‘par’
    SRGnet: no visible global function definition for ‘boxplot’
    SRGnet: no visible global function definition for ‘plot’
    SRGnet: no visible global function definition for ‘predict’
    SRGnet: no visible global function definition for ‘terrain.colors’
    SRGnet: no visible global function definition for ‘na.omit’
    SRGnet: no visible global function definition for ‘write.table’
    Undefined global functions or variables:
      Differentially_expressed_genes PLCRG Transcriptomics boxplot
      ebPatterns na.omit par plot predict terrain.colors write.table
    Consider adding
      importFrom("grDevices", "terrain.colors")
      importFrom("graphics", "boxplot", "par", "plot")
      importFrom("stats", "na.omit", "predict")
      importFrom("utils", "write.table")
    to your NAMESPACE file.
    ```

# stapler

Version: 0.6.3

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘John Muschelli <muschellij2@gmail.com>’
    
    This build time stamp is over a month old.
    ```

# stm

Version: 1.3.0

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      ....................................................................................................
      Completed E-Step (1 seconds). 
      .....................................................................................................
      Completed M-Step (11 seconds). 
      Model Terminated Before Convergence Reached 
      ── 1. Error: plot.STM doesn't throw error  (@test-visualize.R#4)  ──────────────
      cannot open the connection to 'http://goo.gl/VPdxlS'
      1: load(url("http://goo.gl/VPdxlS")) at testthat/test-visualize.R:4
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 9 SKIPPED: 0 FAILED: 1
      1. Error: plot.STM doesn't throw error  (@test-visualize.R#4) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Brandon Stewart <bms4@princeton.edu>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    stm v1.3.0 (2017-09-08) successfully loaded. See ?stm for help.
    Warning in load(url("http://goo.gl/VPdxlS")) :
      URL 'http://goo.gl/VPdxlS': status was 'Couldn't resolve host name'
    
    Error: processing vignette 'stmVignette.Rnw' failed with diagnostics:
     chunk 7 
    Error in load(url("http://goo.gl/VPdxlS")) : 
      cannot open the connection to 'http://goo.gl/VPdxlS'
    Execution halted
    ```

# STROMA4

Version: 1.2.0

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Sadiq Saleh <sadiq.mehdiismailsaleh@mail.mcgill.ca>’
    
    The Description field should not start with the package name,
      'This package' or similar.
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Warning in file(filename, "r", encoding = encoding) :
      URL 'https://bioconductor.org/biocLite.R': status was 'Couldn't resolve host name'
    
    Error: processing vignette 'STROMA4-vignette.Rnw' failed with diagnostics:
     chunk 1 
    Error in file(filename, "r", encoding = encoding) : 
      cannot open the connection to 'https://bioconductor.org/biocLite.R'
    Execution halted
    ```

# StructFDR

Version: 1.2

## In both

*   checking examples ... ERROR
    ```
    ...
    + 	obj <- apply(X, 1, function(x) {
    + 				ttest.obj <- t.test(x ~ Y)
    + 				c(ttest.obj$p.value, sign(ttest.obj$statistic))
    + 			})
    +     return(list(p.value=obj[1, ], e.sign=obj[2, ]))
    + }
    > 
    > perm.func <- function (X, Y) {
    + 	return(list(X=X, Y=sample(Y)))
    + }
    > 
    > # Call StructFDR
    > tree.fdr.obj <- StructFDR(X, Y, D, test.func, perm.func)
    Warning in StructFDR(X, Y, D, test.func, perm.func) :
      Both the data matrix and the distance matrix should have labels (rownames) to avoid potential errors!
    
    Test on original data sets  ...
    Test on permuted data sets  ...
    Error in if (alt.FDR == "Permutation") { : the condition has length > 1
    Calls: StructFDR
    Execution halted
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Jun Chen <chen.jun2@mayo.edu>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# summarytools

Version: 0.8.1

## In both

*   checking Rd cross-references ... WARNING
    ```
    Unknown package ‘Hmisc’ in Rd xrefs
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 78 marked Latin-1 strings
    ```

# support

Version: 0.1.0

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘support-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: sp
    > ### Title: Generating support points using difference-of-convex programming
    > ### Aliases: sp
    > 
    > ### ** Examples
    > 
    >   ## Don't show: 
    >     #Generate 25 support points for the 2-d i.i.d. N(0,1) distribution
    >     n <- 25 #number of points
    >     p <- 2 #dimension
    >     des <- sp(n,p,dist.str=rep("normal",p))
    Error in if (is.na(bd)) { : the condition has length > 1
    Calls: sp
    Execution halted
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Simon Mak <smak6@gatech.edu>’
    
    The Title field starts with the package name.
    
    This build time stamp is over a month old.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘doParallel’ ‘matrixStats’ ‘nloptr’
      All declared Imports should be used.
    ```

# sva

Version: 3.26.0

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Jeffrey T. Leek <jtleek@gmail.com>, John D. Storey <jstorey@princeton.edu>, W. Evan Johnson <wej@bu.edu>’
    
    The maintainer field is invalid or specifies more than one person
    
    This build time stamp is over a month old.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    Adjusting for0covariate(s) or covariate level(s)
    Fitting L/S model and finding priors
    Finding parametric adjustments
    Adjusting the Data
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'sva.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `titlesec.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.95 \makeatletter
                      ^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# tenXplore

Version: 1.0.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Warning in engine$weave(file, quiet = quiet, encoding = enc) :
      The vignette engine knitr::rmarkdown is not available, because the rmarkdown package is not installed. Please install it.
    Quitting from lines 30-61 (tenXplore.Rmd) 
    Error: processing vignette 'tenXplore.Rmd' failed with diagnostics:
    could not find function "Biocpkg"
    Execution halted
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘VJ Carey <stvjc@channing.harvard.edu>’
    
    The Title field should be in title case, current version then in title case:
    ‘ontological exploration of scRNA-seq of 1.3 million mouse neurons from 10x genomics’
    ‘Ontological Exploration of scRNA-Seq of 1.3 Million Mouse Neurons from 10x Genomics’
    
    This build time stamp is over a month old.
    ```

*   checking R code for possible problems ... NOTE
    ```
    tenXplore: no visible binding for global variable ‘CellTypes’
    Undefined global functions or variables:
      CellTypes
    ```

# tmlenet

Version: 0.1.0

## In both

*   checking top-level files ... WARNING
    ```
    Conversion of ‘README.md’ failed:
    pandoc: Could not fetch https://travis-ci.org/osofr/tmlenet.svg?branch=master
    FailedConnectionException2 "travis-ci.org" 443 True getAddrInfo: does not exist (Name or service not known)
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Oleg Sofrygin <oleg.sofrygin@gmail.com>’
    
    This build time stamp is over a month old.
    ```

# topGO

Version: 2.30.0

## In both

*   checking examples ... ERROR
    ```
    ...
    > ## GenTable
    > ########################################
    > 
    > ## load two topGOresult sample objects: resultFisher and resultKS
    > data(results.tGO)
    > 
    > ## generate the result of Fisher's exact test
    > sig.tab <- GenTable(GOdata, Fis = resultFisher, topNodes = 20)
    > 
    > ## results of both test
    > sig.tab <- GenTable(GOdata, resultFisher, resultKS, topNodes = 20)
    > 
    > ## results of both test with specified names
    > sig.tab <- GenTable(GOdata, Fis = resultFisher, KS = resultKS, topNodes = 20)
    > 
    > ## results of both test with specified names and specified ordering
    > sig.tab <- GenTable(GOdata, Fis = resultFisher, KS = resultKS, orderBy = "KS", ranksOf = "Fis", topNodes = 20)
    Error in data.frame(`GO ID` = whichTerms, Term = shortNames, stringsAsFactors = FALSE) : 
      row names contain missing values
    Calls: GenTable -> GenTable -> .local -> data.frame
    Execution halted
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Adrian Alexa <adrian.alexa@gmail.com>’
    
    The Description field should not start with the package name,
      'This package' or similar.
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      ‘BiocGenerics’ ‘graph’ ‘Biobase’ ‘GO.db’ ‘AnnotationDbi’ ‘SparseM’
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' calls in package code:
      ‘Rgraphviz’ ‘multtest’
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    Missing object imported by a ':::' call: ‘globaltest:::globaltest’
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    GOplot.counts : plotSigChart: no visible global function definition for
      ‘getY’
    GOplot.counts: no visible global function definition for
      ‘getDefaultAttrs’
    GOplot.counts: no visible global function definition for ‘agopen’
    getPvalues: no visible global function definition for ‘mt.teststat’
    getPvalues: no visible global function definition for ‘mt.rawp2adjp’
    printDOT: no visible global function definition for ‘getDefaultAttrs’
    printDOT: no visible global function definition for ‘toDot’
    GOSumTest,classicScore: no visible binding for global variable
      ‘.PERMSUM.MAT’
    GOSumTest,classicScore: no visible binding for global variable
      ‘.PERMSUM.LOOKUP’
    initialize,classicExpr: no visible global function definition for
      ‘error’
    scoresInTerm,topGOdata-missing: no visible global function definition
      for ‘scoreInNode’
    Undefined global functions or variables:
      .PERMSUM.LOOKUP .PERMSUM.MAT AgNode agopen drawTxtLabel error
      getDefaultAttrs getNodeCenter getNodeLW getNodeXY getX getY
      mt.rawp2adjp mt.teststat name pieGlyph scoreInNode toDot txtLabel
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    			 -- Classic Algorithm -- 
    
    		 the algorithm is scoring 4040 nontrivial nodes
    		 parameters: 
    			 test statistic: fisher
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'topGO.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `sectsty.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.11 \usepackage
                    {caption}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# ttScreening

Version: 1.5

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Meredith Ray <mere2110@yahoo.com>’
    
    The Title field should be in title case, current version then in title case:
    ‘Genome-wide DNA methylation sites screening by use of training and testing samples.’
    ‘Genome-Wide DNA Methylation Sites Screening by Use of Training and Testing Samples.’
    
    The Description field should not start with the package name,
      'This package' or similar.
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Title field: should not end in a period.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    ttScreening: no visible global function definition for ‘gaussian’
    ttScreening: no visible global function definition for ‘glm.control’
    ttScreening: no visible global function definition for ‘model.response’
    ttScreening: no visible global function definition for ‘is.empty.model’
    ttScreening: no visible global function definition for ‘model.matrix’
    ttScreening: no visible global function definition for ‘na.omit’
    ttScreening: no visible global function definition for ‘terms’
    ttScreening: no visible global function definition for ‘capture.output’
    ttScreening: no visible global function definition for ‘glm’
    ttScreening: no visible binding for global variable ‘gaussian’
    ttScreening: no visible global function definition for ‘pt’
    ttScreening: no visible global function definition for ‘p.adjust’
    Undefined global functions or variables:
      capture.output gaussian glm glm.control is.empty.model model.matrix
      model.response na.omit p.adjust pt terms
    Consider adding
      importFrom("stats", "gaussian", "glm", "glm.control", "is.empty.model",
                 "model.matrix", "model.response", "na.omit", "p.adjust",
                 "pt", "terms")
      importFrom("utils", "capture.output")
    to your NAMESPACE file.
    ```

# VanillaICE

Version: 1.40.0

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Robert Scharpf <rscharpf@jhsph.edu>’
    
    The Title field should be in title case, current version then in title case:
    ‘A Hidden Markov Model for high throughput genotyping arrays’
    ‘A Hidden Markov Model for High Throughput Genotyping Arrays’
    
    The Date field is not in ISO 8601 yyyy-mm-dd format.
    
    This build time stamp is over a month old.
    ```

*   checking package dependencies ... NOTE
    ```
    Packages which this enhances but not available for checking:
      ‘doMC’ ‘doMPI’ ‘doSNOW’ ‘doParallel’ ‘doRedis’
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘BSgenome.Hsapiens.UCSC.hg18’
      All declared Imports should be used.
    Unexported object imported by a ':::' call: ‘SummarizedExperiment:::.ShallowSimpleListAssays0’
      See the note in ?`:::` about the use of this operator.
    ```

# visualFields

Version: 0.4.3

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Ivan Marin-Franch <ivan.marin@uv.es>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# wateRmelon

Version: 1.22.0

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Leo <lschal@essex.ac.uk>’
    
    Unknown, possibly mis-spelled, fields in DESCRIPTION:
      ‘Tue Mar 22 11’
    
    The Title field should be in title case, current version then in title case:
    ‘Illumina 450 methylation array normalization and metrics’
    ‘Illumina 450 Methylation Array Normalization and Metrics’
    
    The Description field should start with a capital letter.
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      ‘Biobase’ ‘limma’ ‘matrixStats’ ‘methylumi’ ‘lumi’ ‘ROC’
      ‘IlluminaHumanMethylation450kanno.ilmn12.hg19’ ‘illuminaio’
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Package listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘Biobase’
    A package should be listed in only one of these fields.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Packages in Depends field not imported from:
      ‘IlluminaHumanMethylation450kanno.ilmn12.hg19’ ‘ROC’ ‘illuminaio’
      ‘limma’ ‘lumi’ ‘matrixStats’ ‘methods’ ‘methylumi’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    Unexported object imported by a ':::' call: ‘minfi:::.isRGOrStop’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      .mclapply IlluminaHumanMethylation450kanno.ilmn12.hg19 MethylSet
      PATH_RES QCdata QCdata<- RGChannelSetExtended addAnnotationInfo
      addControlData2methyLumiM as betas betas<- colData colMedians
      controlData<- data density detectionP epic.controls epic.ordering
      getAnnotation getBeta getControlAddress getGreen getManifestInfo
      getMeth getNBeads getProbeInfo getRed getSnpBeta getUnmeth
      hm27.controls hm27.ordering hm450.controls hm450.ordering
      intensitiesByChannel is kmeans lm lumiMethyB mad median metadata
      methylated methylated<- methylumiR n.sd na.exclude new
      normalizeQuantiles pAUC packageVersion preprocessRaw preprocessSWAN
      probeAnnotationsCategory projectName pval.detect<- pvals qchisq
      quantile read.csv read.table readIDAT rocdemo.sca sd t.test
      unmethylated unmethylated<- warn write.csv write.table
    Consider adding
      importFrom("methods", "as", "is", "new")
      importFrom("stats", "density", "kmeans", "lm", "mad", "median",
                 "na.exclude", "qchisq", "quantile", "sd", "t.test")
      importFrom("utils", "data", "packageVersion", "read.csv", "read.table",
                 "write.csv", "write.table")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# WGCNA

Version: 1.61

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Peter Langfelder <Peter.Langfelder@gmail.com>’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# XBSeq

Version: 1.8.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    The following objects are masked from 'package:Biobase':
    
        anyMissing, rowMedians
    
    
    Attaching package: 'DelayedArray'
    
    The following objects are masked from 'package:matrixStats':
    
        colMaxs, colMins, colRanges, rowMaxs, rowMins, rowRanges
    
    The following object is masked from 'package:base':
    
        apply
    
        Welcome to 'XBSeq'.
    Warning: The 'python' engine in knitr requires the reticulate package. If you do not want to use the reticulate package, set the chunk option python.reticulate = FALSE.
    Quitting from lines 58-60 (XBSeq.Rmd) 
    Error: processing vignette 'XBSeq.Rmd' failed with diagnostics:
    there is no package called 'reticulate'
    Execution halted
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Yuanhang Liu <liuy12@uthscsa.edu>’
    
    The Title field should be in title case, current version then in title case:
    ‘Test for differential expression for RNA-seq data’
    ‘Test for Differential Expression for RNA-Seq Data’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking whether the namespace can be loaded with stated dependencies ... NOTE
    ```
    Warning: no function found corresponding to methods exports from ‘XBSeq’ for: ‘conditions’, ‘conditions<-’, ‘dispTable’
    
    A namespace must be able to be loaded with just the base namespace
    loaded: otherwise if the namespace gets loaded by a saved object, the
    session will be unable to start.
    
    Probably some imports need to be declared in the NAMESPACE file.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    estimateRealCount,XBSeqDataSet: no visible global function definition
      for ‘assay’
    estimateRealCount,XBSeqDataSet: no visible global function definition
      for ‘assay<-’
    estimateSCV,XBSeqDataSet: no visible global function definition for
      ‘conditions’
    estimateSCV,XBSeqDataSet: no visible global function definition for
      ‘dispTable<-’
    Undefined global functions or variables:
      ..count.. DataFrame Gamma Group Sample SummarizedExperiment assay
      assay<- assays baseMean coefficients complete.cases conditions cor
      data ddelap dispTable dispTable<- dnbinom dpois formula glm
      log2FoldChange median optim p.adjust pbeta predict qbeta quantile
      rnbinom scvBiasCorrectionFits
    Consider adding
      importFrom("stats", "Gamma", "coefficients", "complete.cases", "cor",
                 "dnbinom", "dpois", "formula", "glm", "median", "optim",
                 "p.adjust", "pbeta", "predict", "qbeta", "quantile",
                 "rnbinom")
      importFrom("utils", "data")
    to your NAMESPACE file.
    ```

# yarn

Version: 1.4.0

## In both

*   checking examples ... ERROR
    ```
    ...
    > ### Title: Annotate your Expression Set with biomaRt
    > ### Aliases: annotateFromBiomart
    > 
    > ### ** Examples
    > 
    > 
    > data(skin)
    > # subsetting and changing column name just for a silly example
    > skin <- skin[1:10,]
    > colnames(fData(skin)) = paste("names",1:6)
    > host<-"dec2013.archive.ensembl.org"; biomart<-"ENSEMBL_MART_ENSEMBL";
    > genes <- sapply(strsplit(rownames(skin),split="\\."),function(i)i[1])
    > newskin <-annotateFromBiomart(skin,genes=genes,biomar=biomart,host=host)
    Request to BioMart web service failed.
    The BioMart web service you're accessing may be down.
    Check the following URL and see if this website is available:
    http://dec2013.archive.ensembl.org:80/biomart/martservice?type=registry&requestid=biomaRt
    Error in if (!grepl(x = registry, pattern = "^\n*<MartRegistry>")) { : 
      argument is of length zero
    Calls: annotateFromBiomart -> useMart -> listMarts
    Execution halted
    ```

*   checking top-level files ... WARNING
    ```
    Conversion of ‘README.md’ failed:
    pandoc: Could not fetch https://travis-ci.org/jnpaulson/yarn.svg?branch=master
    FailedConnectionException2 "travis-ci.org" 443 True getAddrInfo: does not exist (Name or service not known)
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    ! LaTeX Error: File `framed.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    Enter file name: 
    ! Emergency stop.
    <read *> 
             
    l.39 \definecolor
    
    pandoc: Error producing PDF
    Error: processing vignette 'yarn.Rmd' failed with diagnostics:
    pandoc document conversion failed with error 43
    Execution halted
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Joseph N Paulson <jpaulson@jimmy.harvard.edu>’
    
    The Title field starts with the package name.
    
    This build time stamp is over a month old.
    ```

# zinbwave

Version: 1.0.0

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘copula’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Davide Risso <risso.davide@gmail.com>’
    
    This build time stamp is over a month old.
    ```

