# Setup

## Platform

|setting  |value                        |
|:--------|:----------------------------|
|version  |R version 3.3.3 (2017-03-06) |
|system   |x86_64, linux-gnu            |
|ui       |X11                          |
|language |en                           |
|collate  |en_US.UTF-8                  |
|tz       |America/Los_Angeles          |
|date     |2017-04-11                   |

## Packages

|package        |*  |version |date       |source          |
|:--------------|:--|:-------|:----------|:---------------|
|base64enc      |   |0.1-3   |2015-07-28 |cran (@0.1-3)   |
|ggplot2        |   |2.2.1   |2016-12-30 |cran (@2.2.1)   |
|knitr          |   |1.15.1  |2016-11-22 |cran (@1.15.1)  |
|matrixStats    |   |0.52.1  |2017-04-04 |cran (@0.52.1)  |
|microbenchmark |   |1.4-2.1 |2015-11-25 |cran (@1.4-2.1) |
|R.devices      |   |2.15.1  |2016-11-10 |cran (@2.15.1)  |
|R.rsp          |   |0.40.0  |2016-12-06 |cran (@0.40.0)  |

# Check results

111 packages

|package                |version | errors| warnings| notes|
|:----------------------|:-------|------:|--------:|-----:|
|ACNE                   |0.8.1   |      0|        0|     0|
|anomalyDetection       |0.1.1   |      0|        0|     0|
|antiProfiles           |1.14.0  |      0|        0|     1|
|aroma.affymetrix       |3.1.0   |      0|        0|     0|
|aroma.cn               |1.6.1   |      0|        0|     0|
|aroma.core             |3.1.0   |      0|        0|     0|
|aroma.light            |3.4.0   |      0|        0|     1|
|aSPU                   |1.46    |      0|        0|     0|
|BatchQC                |1.2.1   |      0|        0|     0|
|bdynsys                |1.3     |      0|        0|     1|
|bigstep                |0.7.4   |      0|        0|     0|
|bingat                 |1.2.2   |      0|        0|     0|
|brms                   |1.6.0   |      0|        0|     0|
|bsseq                  |1.10.0  |      0|        0|     0|
|bumphunter             |1.14.0  |      0|        0|     5|
|calmate                |0.12.1  |      0|        0|     0|
|carx                   |0.6.2   |      0|        0|     0|
|cellWise               |1.0.0   |      0|        0|     0|
|ChAMP                  |2.6.2   |      0|        1|     3|
|Chicago                |1.2.0   |      0|        0|     3|
|ChIPpeakAnno           |3.8.9   |      0|        0|     2|
|Clomial                |1.10.0  |      0|        0|     1|
|clusterExperiment      |1.0.0   |      0|        1|     0|
|CNPBayes               |1.4.0   |      0|        0|     2|
|cointmonitoR           |0.1.0   |      0|        0|     0|
|cointReg               |0.2.0   |      0|        0|     0|
|CopywriteR             |2.6.0   |      0|        0|     1|
|crlmm                  |1.32.0  |      0|        1|     4|
|crossmeta              |1.0.1   |      0|        0|     1|
|DAMOCLES               |1.1     |      0|        0|     1|
|DeepBlueR              |1.0.11  |      0|        1|     1|
|DGCA                   |1.0.1   |      0|        0|     0|
|diffloop               |1.2.2   |      0|        0|     0|
|DisHet                 |0.1.0   |      0|        0|     0|
|dplR                   |1.6.5   |      0|        0|     0|
|EasyqpcR               |1.16.0  |      1|        0|     1|
|EMDomics               |2.4.0   |      0|        0|     3|
|EnrichedHeatmap        |1.4.0   |      0|        0|     4|
|expss                  |0.7.1   |      0|        0|     0|
|FADA                   |1.3.2   |      0|        0|     0|
|FastHCS                |0.0.5   |      0|        0|     2|
|FastPCS                |0.1.2   |      0|        0|     3|
|FastRCS                |0.0.7   |      0|        0|     1|
|filesstrings           |0.3.2   |      0|        0|     0|
|flowCore               |1.40.6  |      0|        1|     9|
|fslr                   |2.12    |      0|        0|     0|
|GAD                    |1.1.1   |      0|        0|     0|
|genomation             |1.6.0   |      0|        1|     3|
|GenRank                |1.2.0   |      0|        0|     1|
|GPrank                 |0.1.2   |      0|        0|     0|
|GUIDEseq               |1.4.1   |      1|        1|     3|
|Gviz                   |1.18.2  |      0|        2|     4|
|haploReconstruct       |0.1.2   |      0|        0|     1|
|IMIFA                  |1.1.0   |      0|        0|     0|
|InfiniumPurify         |1.3.1   |      0|        0|     0|
|kernDeepStackNet       |2.0.1   |      0|        0|     1|
|localgauss             |0.40    |      0|        0|     0|
|loo                    |1.1.0   |      0|        0|     0|
|LS2Wstat               |2.0-3   |      0|        1|     2|
|ltmle                  |0.9-9   |      0|        0|     0|
|Luminescence           |0.7.4   |      0|        0|     0|
|metagene               |2.6.1   |      0|        0|     1|
|metagenomeSeq          |1.16.0  |      0|        1|     0|
|MetaQC                 |0.1.13  |      0|        0|     3|
|MethylAid              |1.8.0   |      0|        0|     1|
|methylumi              |2.20.0  |      0|        1|     8|
|MFHD                   |0.0.1   |      0|        0|     1|
|minfi                  |1.20.2  |      0|        0|     4|
|MinimumDistance        |1.18.0  |      0|        0|     1|
|mmtfa                  |0.1     |      0|        0|     1|
|monocle                |2.2.0   |      0|        0|     2|
|motifbreakR            |1.4.0   |      0|        0|     2|
|MPAgenomics            |1.1.2   |      0|        0|     2|
|mrfDepth               |1.0.3   |      0|        0|     1|
|MultiBD                |0.2.0   |      0|        0|     1|
|NanoStringDiff         |1.4.0   |      0|        0|     1|
|neurobase              |1.13.2  |      0|        0|     0|
|NSA                    |0.0.32  |      0|        0|     6|
|pandaR                 |1.6.0   |      0|        0|     4|
|PathoStat              |1.0.0   |      0|        1|     0|
|pcaMethods             |1.66.0  |      0|        0|     2|
|peakPick               |0.11    |      0|        0|     0|
|Pigengene              |1.0.0   |      0|        0|     2|
|PSCBS                  |0.62.0  |      0|        0|     0|
|QDNAseq                |1.10.0  |      0|        0|     0|
|r2dRue                 |1.0.4   |      0|        0|     2|
|RAC                    |1.1.1   |      0|        0|     0|
|randomizationInference |1.0.3   |      0|        0|     1|
|RnBeads                |1.6.1   |      0|        7|     6|
|samr                   |2.0     |      0|        0|     5|
|scater                 |1.2.0   |      0|        0|     0|
|SemiParBIVProbit       |3.8-1   |      0|        0|     0|
|SemiParSampleSel       |1.4     |      0|        0|     0|
|SGP                    |1.6-0.0 |      0|        0|     0|
|shinyMethyl            |1.10.0  |      0|        0|     2|
|SICtools               |1.4.0   |      0|        1|     4|
|sizeMat                |0.3.0   |      0|        0|     0|
|SRGnet                 |1.0.0   |      0|        0|     2|
|statar                 |0.6.3   |      0|        0|     0|
|stm                    |1.2.2   |      0|        0|     0|
|StructFDR              |1.1     |      0|        0|     0|
|summarytools           |0.6.5   |      0|        0|     0|
|tmlenet                |0.1.0   |      0|        0|     0|
|topGO                  |2.26.0  |      0|        0|     4|
|ttScreening            |1.5     |      0|        0|     2|
|VanillaICE             |1.36.0  |      0|        0|     3|
|visualFields           |0.4.3   |      0|        0|     0|
|wateRmelon             |1.18.0  |      0|        0|     6|
|WGCNA                  |1.51    |      0|        0|     0|
|XBSeq                  |1.4.0   |      0|        0|     3|
|yarn                   |1.0.1   |      0|        0|     0|

## ACNE (0.8.1)
Maintainer: Henrik Bengtsson <henrikb@braju.com>  
Bug reports: https://github.com/HenrikBengtsson/ACNE/issues

0 errors | 0 warnings | 0 notes

## anomalyDetection (0.1.1)
Maintainer: Bradley Boehmke <bradleyboehmke@gmail.com>  
Bug reports: https://github.com/bradleyboehmke/anomalyDetection/issues

0 errors | 0 warnings | 0 notes

## antiProfiles (1.14.0)
Maintainer: Hector Corrada Bravo <hcorrada@gmail.com>

0 errors | 0 warnings | 1 note 

```
checking R code for possible problems ... NOTE
ev: no visible global function definition for ‘predict’
ev: no visible global function definition for ‘smoothScatter’
ev : f1: no visible global function definition for ‘predict’
ev: no visible global function definition for ‘curve’
Undefined global functions or variables:
  curve predict smoothScatter
Consider adding
  importFrom("graphics", "curve", "smoothScatter")
  importFrom("stats", "predict")
to your NAMESPACE file.
```

## aroma.affymetrix (3.1.0)
Maintainer: Henrik Bengtsson <henrikb@braju.com>  
Bug reports: https://github.com/HenrikBengtsson/aroma.affymetrix/issues

0 errors | 0 warnings | 0 notes

## aroma.cn (1.6.1)
Maintainer: Henrik Bengtsson <henrikb@braju.com>  
Bug reports: https://github.com/HenrikBengtsson/aroma.cn/issues

0 errors | 0 warnings | 0 notes

## aroma.core (3.1.0)
Maintainer: Henrik Bengtsson <henrikb@braju.com>  
Bug reports: https://github.com/HenrikBengtsson/aroma.core/issues

0 errors | 0 warnings | 0 notes

## aroma.light (3.4.0)
Maintainer: Henrik Bengtsson <henrikb@braju.com>  
Bug reports: https://github.com/HenrikBengtsson/aroma.light/issues

0 errors | 0 warnings | 1 note 

```
checking for hidden files and directories ... NOTE
Found the following hidden files and directories:
  inst/rsp/.rspPlugins
These were most likely included in error. See section ‘Package
structure’ in the ‘Writing R Extensions’ manual.
```

## aSPU (1.46)
Maintainer: Il-Youp Kwak <ikwak@umn.edu>

0 errors | 0 warnings | 0 notes

## BatchQC (1.2.1)
Maintainer: Solaiappan Manimaran <manimaran_1975@hotmail.com>  
Bug reports: https://github.com/mani2012/BatchQC/issues

0 errors | 0 warnings | 0 notes

## bdynsys (1.3)
Maintainer: Viktoria Spaiser <viktoria.sp@web.de>

0 errors | 0 warnings | 1 note 

```
checking R code for possible problems ... NOTE
bayfacfig: no visible global function definition for ‘dev.set’
bayfacfig: no visible global function definition for ‘postscript’
bayfacfig: no visible global function definition for ‘points’
bayfacfig: no visible global function definition for ‘legend’
bayfacfig: no visible global function definition for ‘dev.off’
dysymod: no visible global function definition for ‘write.table’
errorcorr: no visible global function definition for ‘sd’
phaseportdat: no visible global function definition for ‘dev.set’
phaseportdat: no visible global function definition for ‘postscript’
... 24 lines ...
polyfitreg: no visible binding for global variable ‘na.exclude’
polyfitreg: no visible global function definition for ‘var’
Undefined global functions or variables:
  dev.off dev.set grid legend lm matplot na.exclude plot points
  postscript runif sd var write.table
Consider adding
  importFrom("grDevices", "dev.off", "dev.set", "postscript")
  importFrom("graphics", "grid", "legend", "matplot", "plot", "points")
  importFrom("stats", "lm", "na.exclude", "runif", "sd", "var")
  importFrom("utils", "write.table")
to your NAMESPACE file.
```

## bigstep (0.7.4)
Maintainer: Piotr Szulc <piotr.michal.szulc@gmail.com>

0 errors | 0 warnings | 0 notes

## bingat (1.2.2)
Maintainer: Berkley Shands <rpackages@biorankings.com>

0 errors | 0 warnings | 0 notes

## brms (1.6.0)
Maintainer: Paul-Christian Bürkner <paul.buerkner@gmail.com>  
Bug reports: https://github.com/paul-buerkner/brms/issues

0 errors | 0 warnings | 0 notes

## bsseq (1.10.0)
Maintainer: Kasper Daniel Hansen <kasperdanielhansen@gmail.com>  
Bug reports: https://github.com/kasperdanielhansen/bsseq/issues

0 errors | 0 warnings | 0 notes

## bumphunter (1.14.0)
Maintainer: Rafael A. Irizarry <rafa@jimmy.harvard.edu>

0 errors | 0 warnings | 5 notes

```
checking package dependencies ... NOTE
Depends: includes the non-default packages:
  ‘S4Vectors’ ‘IRanges’ ‘GenomeInfoDb’ ‘GenomicRanges’ ‘foreach’
  ‘iterators’ ‘parallel’ ‘locfit’
Adding so many packages to the search path is excessive and importing
selectively is preferable.

checking DESCRIPTION meta-information ... NOTE
Malformed Description field: should contain one or more complete sentences.

checking dependencies in R code ... NOTE
Unexported object imported by a ':::' call: ‘doParallel:::.options’
  See the note in ?`:::` about the use of this operator.

checking R code for possible problems ... NOTE
bumphunterEngine: no visible binding for global variable ‘bootstraps’
Undefined global functions or variables:
  bootstraps

checking Rd line widths ... NOTE
Rd file 'annotateTranscripts.Rd':
  \usage lines wider than 90 characters:
     annotateTranscripts(txdb, annotationPackage = NULL, by = c("tx","gene"), codingOnly=FALSE, verbose = TRUE, requireAnnotation = FALSE)

Rd file 'bumphunter.Rd':
  \usage lines wider than 90 characters:
     bumphunter(object, design, chr=NULL, pos, cluster=NULL,coef=2, cutoff=NULL, pickCutoff = FALSE, pickCutoffQ = 0.99, maxGap=500, nullMet ... [TRUNCATED]
     bumphunterEngine(mat, design, chr = NULL, pos, cluster = NULL, coef = 2, cutoff = NULL, pickCutoff = FALSE, pickCutoffQ = 0.99, maxGap  ... [TRUNCATED]

Rd file 'matchGenes.Rd':
  \usage lines wider than 90 characters:
     matchGenes(x, subject, type = c("any", "fiveprime"), promoterDist = 2500, skipExons = FALSE, verbose = TRUE)
  \examples lines wider than 100 characters:
         islands=makeGRangesFromDataFrame(read.delim("http://rafalab.jhsph.edu/CGI/model-based-cpg-islands-hg19.txt")[1:100,])

These lines will be truncated in the PDF manual.
```

## calmate (0.12.1)
Maintainer: Henrik Bengtsson <henrikb@braju.com>  
Bug reports: https://github.com/HenrikBengtsson/calmate/issues

0 errors | 0 warnings | 0 notes

## carx (0.6.2)
Maintainer: Chao Wang <chao-wang@uiowa.edu>

0 errors | 0 warnings | 0 notes

## cellWise (1.0.0)
Maintainer: Jakob Raymaekers <jakob.raymaekers@kuleuven.be>

0 errors | 0 warnings | 0 notes

## ChAMP (2.6.2)
Maintainer: Yuan Tian  <champ450k@gmail.com>

0 errors | 1 warning  | 3 notes

```
checking package subdirectories ... WARNING
Problems with news in ‘inst/NEWS.Rd’:
  inst/NEWS.Rd:13: unexpected END_OF_INPUT '
  '

checking package dependencies ... NOTE
Depends: includes the non-default packages:
  ‘minfi’ ‘ChAMPdata’ ‘FEM’ ‘DMRcate’ ‘Illumina450ProbeVariants.db’
  ‘IlluminaHumanMethylationEPICmanifest’
Adding so many packages to the search path is excessive and importing
selectively is preferable.

checking installed package size ... NOTE
  installed size is 11.1Mb
  sub-directories of 1Mb or more:
    doc       8.9Mb
    extdata   1.9Mb

checking R code for possible problems ... NOTE
Found the following calls to data() loading into the global environment:
File ‘ChAMP/R/Block.GUI.R’:
  data(probe.features.epic)
  data(probe.features)
File ‘ChAMP/R/CpG.GUI.R’:
  data(probe.features.epic)
  data(probe.features)
File ‘ChAMP/R/DMP.GUI.R’:
  data(MatchGeneName)
... 43 lines ...
  data(probe.features.epic)
  data(probe.features)
File ‘ChAMP/R/champ.norm.R’:
  data(probeInfoALL.epic.lv)
  data(probeInfoALL.lv)
  data(probeInfoALL.epic.lv)
  data(probeInfoALL.lv)
File ‘ChAMP/R/champ.refbase.R’:
  data(CellTypeMeans27K)
  data(CellTypeMeans450K)
See section ‘Good practice’ in ‘?data’.
```

## Chicago (1.2.0)
Maintainer: Mikhail Spivakov <spivakov@babraham.ac.uk>

0 errors | 0 warnings | 3 notes

```
checking for hidden files and directories ... NOTE
Found the following hidden files and directories:
  .BBSoptions
These were most likely included in error. See section ‘Package
structure’ in the ‘Writing R Extensions’ manual.

checking top-level files ... NOTE
Non-standard file/directory found at top level:
  ‘session.txt’

checking Rd line widths ... NOTE
Rd file 'mergeSamples.Rd':
  \usage lines wider than 90 characters:
           NcolNormPrefix = "NNorm", mergeMethod = c("weightedMean", "mean")[1], repNormCounts = (mergeMethod=="mean"))

Rd file 'peakEnrichment4Features.Rd':
  \examples lines wider than 100 characters:
     peakEnrichment4Features(cdUnitTest, folder=ChIPdir, list_frag = featuresList, no_bins = 500, sample_number = 100)

These lines will be truncated in the PDF manual.
```

## ChIPpeakAnno (3.8.9)
Maintainer: Lihua Julie Zhu <julie.zhu@umassmed.edu>,
 Jianhong Ou <Jianhong.ou@umassmed.edu>

0 errors | 0 warnings | 2 notes

```
checking package dependencies ... NOTE
Depends: includes the non-default packages:
  ‘grid’ ‘IRanges’ ‘Biostrings’ ‘GenomicRanges’ ‘S4Vectors’
  ‘VennDiagram’
Adding so many packages to the search path is excessive and importing
selectively is preferable.

checking installed package size ... NOTE
  installed size is 19.5Mb
  sub-directories of 1Mb or more:
    data     12.7Mb
    extdata   5.2Mb
```

## Clomial (1.10.0)
Maintainer: Habil Zare <zare@txstate.edu>

0 errors | 0 warnings | 1 note 

```
checking R code for possible problems ... NOTE
Clomial: no visible global function definition for ‘runif’
Clomial.generate.data: no visible global function definition for
  ‘runif’
Clomial.generate.data: no visible global function definition for
  ‘rbinom’
Clomial.likelihood: no visible global function definition for ‘dbinom’
Phi: no visible global function definition for ‘dbinom’
choose.best: no visible global function definition for ‘tail’
compute.P.reparam : update.Wj: no visible global function definition
  for ‘optim’
compute.P.reparam : plot.obj: no visible global function definition for
  ‘plot’
compute.q: no visible global function definition for ‘dbinom’
Undefined global functions or variables:
  dbinom optim plot rbinom runif tail
Consider adding
  importFrom("graphics", "plot")
  importFrom("stats", "dbinom", "optim", "rbinom", "runif")
  importFrom("utils", "tail")
to your NAMESPACE file.
```

## clusterExperiment (1.0.0)
Maintainer: Elizabeth Purdom <epurdom@stat.berkeley.edu>  
Bug reports: https://github.com/epurdom/clusterExperiment/issues

0 errors | 1 warning  | 0 notes

```
checking Rd cross-references ... WARNING
Package unavailable to check Rd xrefs: ‘ConsensusClusterPlus’
Missing link or links in documentation object 'seqCluster.Rd':
  ‘tightClust’

See section 'Cross-references' in the 'Writing R Extensions' manual.

```

## CNPBayes (1.4.0)
Maintainer: Jacob Carey <jcarey15@jhu.edu>  
Bug reports: https://github.com/scristia/CNPBayes/issues

0 errors | 0 warnings | 2 notes

```
checking installed package size ... NOTE
  installed size is 11.6Mb
  sub-directories of 1Mb or more:
    libs   9.6Mb

checking DESCRIPTION meta-information ... NOTE
Package listed in more than one of Depends, Imports, Suggests, Enhances:
  ‘GenomicRanges’
A package should be listed in only one of these fields.
```

## cointmonitoR (0.1.0)
Maintainer: Philipp Aschersleben <aschersleben@statistik.tu-dortmund.de>  
Bug reports: https://github.com/aschersleben/cointmonitoR/issues

0 errors | 0 warnings | 0 notes

## cointReg (0.2.0)
Maintainer: Philipp Aschersleben <aschersleben@statistik.tu-dortmund.de>  
Bug reports: https://github.com/aschersleben/cointReg/issues

0 errors | 0 warnings | 0 notes

## CopywriteR (2.6.0)
Maintainer: Thomas Kuilman <t.kuilman@nki.nl>

0 errors | 0 warnings | 1 note 

```
checking R code for possible problems ... NOTE
.peakCutoff: no visible global function definition for ‘dpois’
.peakCutoff: no visible global function definition for ‘ppois’
.tng: no visible global function definition for ‘png’
.tng: no visible global function definition for ‘par’
.tng: no visible global function definition for ‘dev.off’
.tng: no visible global function definition for ‘loess’
.tng: no visible global function definition for ‘predict’
.tng: no visible global function definition for ‘points’
.tng: no visible global function definition for ‘rgb’
... 41 lines ...
  par pdf plot png points ppois predict read.table rgb segments text
  write.table
Consider adding
  importFrom("grDevices", "dev.off", "pdf", "png", "rgb")
  importFrom("graphics", "abline", "axis", "lines", "par", "plot",
             "points", "segments", "text")
  importFrom("methods", "as", "getClass")
  importFrom("stats", "dpois", "ecdf", "loess", "ppois", "predict")
  importFrom("utils", "packageVersion", "read.table", "write.table")
to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
contains 'methods').
```

## crlmm (1.32.0)
Maintainer: Benilton S Carvalho <benilton@unicamp.br>,
 Robert Scharpf <rscharpf@jhsph.edu>, Matt Ritchie
 <mritchie@wehi.EDU.AU>

0 errors | 1 warning  | 4 notes

```
checking dependencies in R code ... WARNING
'library' or 'require' call not declared from: ‘limma’
'library' or 'require' calls in package code:
  ‘RUnit’ ‘limma’
  Please use :: or requireNamespace() instead.
  See section 'Suggested packages' in the 'Writing R Extensions' manual.
Unexported object imported by a ':::' call: ‘Biobase:::assayDataEnvLock’
  See the note in ?`:::` about the use of this operator.

checking installed package size ... NOTE
  installed size is  5.1Mb
  sub-directories of 1Mb or more:
    data   2.7Mb

checking DESCRIPTION meta-information ... NOTE
Versioned 'LinkingTo' value for ‘preprocessCore’ is only usable in R >= 3.0.2

checking R code for possible problems ... NOTE
.test: no visible global function definition for ‘defineTestSuite’
.test: no visible global function definition for ‘runTestSuite’
.test: no visible global function definition for ‘printTextProtocol’
calculateRBafCNSet : processByChromosome: no visible global function
  definition for ‘position’
cnrmaAffy: no visible global function definition for ‘celfileName’
computeBR: no visible global function definition for ‘genomeBuild’
constructBafLrrSetListFrom: no visible global function definition for
  ‘genomeBuild’
constructOligoSetListFrom: no visible global function definition for
  ‘genomeBuild’
genotypeInf: no visible binding for global variable ‘anno’
plotSamples: no visible global function definition for ‘delete’
quantilenormalization: no visible global function definition for
  ‘delete’
snprmaAffy: no visible global function definition for ‘celfileName’
Undefined global functions or variables:
  anno celfileName defineTestSuite delete genomeBuild position
  printTextProtocol runTestSuite

checking Rd line widths ... NOTE
Rd file 'constructInf.Rd':
  \usage lines wider than 90 characters:
            arrayInfoColNames = list(barcode="SentrixBarcode_A",position="SentrixPosition_A"), highDensity = FALSE, sep = "_",
            fileExt = list(green = "Grn.idat", red = "Red.idat"), XY, cdfName, anno, genome, verbose = FALSE, batch=NULL, saveDate = TRUE)

Rd file 'genotype.Illumina.Rd':
  \usage lines wider than 90 characters:
           highDensity=FALSE, sep="_", fileExt=list(green="Grn.idat", red="Red.idat"), XY=NULL, anno, genome, 
           call.method="crlmm", trueCalls=NULL, cdfName, copynumber=TRUE, batch=NULL, saveDate=FALSE, stripNorm=TRUE, 
... 20 lines ...
Rd file 'readGenCallOutput.Rd':
  \usage lines wider than 90 characters:
     readGenCallOutput(filenames, path=".", cdfName, colnames=list("SampleID"="Sample ID", "SNPID"="SNP Name", "XRaw"="X Raw", "YRaw"="Y Raw ... [TRUNCATED]
                       type=list("SampleID"="character", "SNPID"="character", "XRaw"="integer", "YRaw"="integer"), verbose=FALSE)

Rd file 'snprma.Rd':
  \usage lines wider than 90 characters:
     snprma(filenames, mixtureSampleSize = 10^5, fitMixture = FALSE, eps = 0.1, verbose = TRUE, seed = 1, cdfName, sns)
     snprma2(filenames, mixtureSampleSize = 10^5, fitMixture = FALSE, eps = 0.1, verbose = TRUE, seed = 1, cdfName, sns)

These lines will be truncated in the PDF manual.
```

## crossmeta (1.0.1)
Maintainer: Alex Pickering <alexvpickering@gmail.com>

0 errors | 0 warnings | 1 note 

```
checking Rd cross-references ... NOTE
Package unavailable to check Rd xrefs: ‘GeneMeta’
```

## DAMOCLES (1.1)
Maintainer: Rampal S. Etienne <r.s.etienne@rug.nl>

0 errors | 0 warnings | 1 note 

```
checking R code for possible problems ... NOTE
DAMOCLES_ML: no visible global function definition for ‘flush.console’
DAMOCLES_bootstrap: no visible global function definition for
  ‘cophenetic’
DAMOCLES_bootstrap: no visible global function definition for ‘sd’
DAMOCLES_bootstrap: no visible global function definition for
  ‘quantile’
DAMOCLES_sim: no visible global function definition for ‘na.omit’
DAMOCLES_sim: no visible global function definition for ‘runif’
DAMOCLES_sim: no visible global function definition for ‘lines’
DAMOCLES_sim: no visible global function definition for ‘rnorm’
DAMOCLES_sim: no visible global function definition for ‘dexp’
DAMOCLES_sim: no visible global function definition for ‘dist’
DAMOCLES_simplex: no visible global function definition for
  ‘flush.console’
Undefined global functions or variables:
  cophenetic dexp dist flush.console lines na.omit quantile rnorm runif
  sd
Consider adding
  importFrom("graphics", "lines")
  importFrom("stats", "cophenetic", "dexp", "dist", "na.omit",
             "quantile", "rnorm", "runif", "sd")
  importFrom("utils", "flush.console")
to your NAMESPACE file.
```

## DeepBlueR (1.0.11)
Maintainer: Felipe Albrecht <felipe.albrecht@mpi-inf.mpg.de>, Markus List <markus.list@mpi-inf.mpg.de>

0 errors | 1 warning  | 1 note 

```
checking package subdirectories ... WARNING
Invalid citation information in ‘inst/CITATION’:
  Error in parse(file = cfile): /home/hb/repositories/matrixStats/revdep/checks/DeepBlueR.Rcheck/00_pkg_src/DeepBlueR/inst/CITATION:13:17: unexpected symbol
  12:          transforming, aggregating and downloading data from several epigenomic consortia.
  13:          URL = "https
                      ^

checking top-level files ... NOTE
Non-standard file/directory found at top level:
  ‘logo.png’
```

## DGCA (1.0.1)
Maintainer: Andrew McKenzie <amckenz@gmail.com>

0 errors | 0 warnings | 0 notes

## diffloop (1.2.2)
Maintainer: Caleb Lareau <caleblareau@g.harvard.edu>  
Bug reports: https://github.com/aryeelab/diffloop/issues

0 errors | 0 warnings | 0 notes

## DisHet (0.1.0)
Maintainer: Tao Wang <tao.wang@utsouthwestern.edu>

0 errors | 0 warnings | 0 notes

## dplR (1.6.5)
Maintainer: Andy Bunn <andy.bunn@wwu.edu>

0 errors | 0 warnings | 0 notes

## EasyqpcR (1.16.0)
Maintainer: Le Pape Sylvain <sylvain.le.pape@univ-poitiers.fr>

1 error  | 0 warnings | 1 note 

```
checking whether package ‘EasyqpcR’ can be installed ... ERROR
Installation failed.
See ‘/home/hb/repositories/matrixStats/revdep/checks/EasyqpcR.Rcheck/00install.out’ for details.

checking package namespace information ... NOTE
  Namespace with empty importFrom: ‘gWidgetsRGtk2’
```

## EMDomics (2.4.0)
Maintainer: Sadhika Malladi <contact@sadhikamalladi.com> and Daniel Schmolze <emd@schmolze.com>

0 errors | 0 warnings | 3 notes

```
checking DESCRIPTION meta-information ... NOTE
Authors@R field gives more than one person with maintainer role:
  Sadhika Malladi <contact@sadhikamalladi.com> [aut, cre]
  Daniel Schmolze <emd@schmolze.com> [aut, cre]

checking R code for possible problems ... NOTE
.cvm_pairwise_q: no visible global function definition for ‘combn’
.cvm_pairwise_q : <anonymous>: no visible global function definition
  for ‘median’
.emd_gene_pairwise: no visible global function definition for ‘hist’
.emd_pairwise_q: no visible global function definition for ‘combn’
.emd_pairwise_q : <anonymous>: no visible global function definition
  for ‘median’
.ks_pairwise_table: no visible global function definition for ‘ks.test’
calculate_cvm: no visible global function definition for ‘combn’
... 11 lines ...
calculate_ks : <anonymous>: no visible global function definition for
  ‘median’
calculate_ks_gene: no visible global function definition for ‘combn’
calculate_ks_gene: no visible global function definition for ‘ks.test’
Undefined global functions or variables:
  combn hist ks.test median p.adjust
Consider adding
  importFrom("graphics", "hist")
  importFrom("stats", "ks.test", "median", "p.adjust")
  importFrom("utils", "combn")
to your NAMESPACE file.

checking files in ‘vignettes’ ... NOTE
The following directory looks like a leftover from 'knitr':
  ‘figure’
Please remove from your package.
```

## EnrichedHeatmap (1.4.0)
Maintainer: Zuguang Gu <z.gu@dkfz.de>

0 errors | 0 warnings | 4 notes

```
checking top-level files ... NOTE
Non-standard file/directory found at top level:
  ‘test’

checking S3 generic/method consistency ... NOTE
Found the following apparent S3 methods exported but not registered:
  +.AdditiveUnit print.normalizedMatrix
See section ‘Registering S3 methods’ in the ‘Writing R Extensions’
manual.

checking R code for possible problems ... NOTE
+.AdditiveUnit: no visible global function definition for ‘new’
EnrichedHeatmapList: no visible global function definition for ‘new’
anno_enriched : <anonymous>: no visible global function definition for
  ‘col2rgb’
anno_enriched : <anonymous>: no visible global function definition for
  ‘rgb’
changeClassName: no visible global function definition for ‘new’
changeClassName: no visible global function definition for ‘slotNames’
changeClassName: no visible global function definition for ‘slot<-’
... 7 lines ...
  ‘selectMethod’
Undefined global functions or variables:
  col2rgb loess.control new predict rgb selectMethod slot slot<-
  slotNames
Consider adding
  importFrom("grDevices", "col2rgb", "rgb")
  importFrom("methods", "new", "selectMethod", "slot", "slot<-",
             "slotNames")
  importFrom("stats", "loess.control", "predict")
to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
contains 'methods').

checking Rd line widths ... NOTE
Rd file 'anno_enriched.rd':
  \usage lines wider than 90 characters:
         yaxis = TRUE, ylim = NULL, value = c("mean", "sum", "abs_mean", "abs_sum"), yaxis_side = "right",

Rd file 'normalizeToMatrix.rd':
  \usage lines wider than 90 characters:
         mean_mode = c("absolute", "weighted", "w0", "coverage"), include_target = any(width(target) > 1),

These lines will be truncated in the PDF manual.
```

## expss (0.7.1)
Maintainer: Gregory Demin <gdemin@gmail.com>  
Bug reports: https://github.com/gdemin/expss/issues

0 errors | 0 warnings | 0 notes

## FADA (1.3.2)
Maintainer: David Causeur <david.causeur@agrocampus-ouest.fr>

0 errors | 0 warnings | 0 notes

## FastHCS (0.0.5)
Maintainer: Kaveh Vakili <vakili.kaveh.email@gmail.com>

0 errors | 0 warnings | 2 notes

```
checking installed package size ... NOTE
  installed size is 13.8Mb
  sub-directories of 1Mb or more:
    data   1.3Mb
    libs  12.4Mb

checking R code for possible problems ... NOTE
FHCSpsdo: no visible global function definition for ‘is’
FastHCS: no visible global function definition for ‘complete.cases’
compPcaParams: no visible global function definition for ‘quantile’
compPcaParams: no visible global function definition for ‘qchisq’
compPcaParams: no visible global function definition for ‘qnorm’
compPcaParams: no visible global function definition for ‘sd’
plot.FastHCS: no visible global function definition for ‘plot’
plot.FastHCS: no visible global function definition for ‘abline’
Undefined global functions or variables:
  abline complete.cases is plot qchisq qnorm quantile sd
Consider adding
  importFrom("graphics", "abline", "plot")
  importFrom("methods", "is")
  importFrom("stats", "complete.cases", "qchisq", "qnorm", "quantile",
             "sd")
to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
contains 'methods').
```

## FastPCS (0.1.2)
Maintainer: Kaveh Vakili <vakili.kaveh.email@gmail.com>

0 errors | 0 warnings | 3 notes

```
checking installed package size ... NOTE
  installed size is  8.2Mb
  sub-directories of 1Mb or more:
    libs   8.1Mb

checking S3 generic/method consistency ... NOTE
Found the following apparent S3 methods exported but not registered:
  plot.FastPCS
See section ‘Registering S3 methods’ in the ‘Writing R Extensions’
manual.

checking R code for possible problems ... NOTE
FastPCS: no visible global function definition for ‘complete.cases’
FastPCS: no visible global function definition for ‘cov’
FastPCS: no visible global function definition for ‘mahalanobis’
FastPCS: no visible global function definition for ‘qchisq’
FastPCS: no visible global function definition for ‘quantile’
plot.FastPCS: no visible global function definition for ‘plot’
plot.FastPCS: no visible global function definition for ‘abline’
plot.FastPCS: no visible global function definition for ‘qchisq’
Undefined global functions or variables:
  abline complete.cases cov mahalanobis plot qchisq quantile
Consider adding
  importFrom("graphics", "abline", "plot")
  importFrom("stats", "complete.cases", "cov", "mahalanobis", "qchisq",
             "quantile")
to your NAMESPACE file.
```

## FastRCS (0.0.7)
Maintainer: Kaveh Vakili <vakili.kaveh.email@gmail.com>

0 errors | 0 warnings | 1 note 

```
checking installed package size ... NOTE
  installed size is 14.3Mb
  sub-directories of 1Mb or more:
    libs  14.2Mb
```

## filesstrings (0.3.2)
Maintainer: Rory Nolan <rorynoolan@gmail.com>  
Bug reports: https://www.github.com/rorynolan/filesstrings/issues

0 errors | 0 warnings | 0 notes

## flowCore (1.40.6)
Maintainer: M.Jiang <wjiang2@fhcrc.org>

0 errors | 1 warning  | 9 notes

```
checking Rd \usage sections ... WARNING
Documented arguments not in \usage in documentation object 'read.flowSet':
  ‘truncate_max_range’

Functions with \usage entries need to have the appropriate \alias
entries, and all their arguments documented.
The \usage entries must correspond to syntactically valid R code.
See chapter ‘Writing R documentation files’ in the ‘Writing R
Extensions’ manual.

checking if this is a source package ... NOTE
Found the following apparent object files/libraries:
  src/boost_regex/c_regex_traits.o src/boost_regex/cpp_regex_traits.o
  src/boost_regex/cregex.o src/boost_regex/fileiter.o
  src/boost_regex/icu.o src/boost_regex/instances.o
  src/boost_regex/posix_api.o src/boost_regex/regex.o
  src/boost_regex/regex_debug.o src/boost_regex/regex_raw_buffer.o
  src/boost_regex/regex_traits_defaults.o
  src/boost_regex/static_mutex.o src/boost_regex/usinstances.o
  src/boost_regex/w32_regex_traits.o src/boost_regex/wc_regex_traits.o
  src/boost_regex/wide_posix_api.o src/boost_regex/winstances.o
Object files/libraries should not be included in a source package.

checking installed package size ... NOTE
  installed size is 33.8Mb
  sub-directories of 1Mb or more:
    data      5.0Mb
    extdata   1.1Mb
    lib      20.5Mb
    libs      5.3Mb

checking DESCRIPTION meta-information ... NOTE
Versioned 'LinkingTo' value for ‘BH’ is only usable in R >= 3.0.2

checking top-level files ... NOTE
Non-standard files/directories found at top level:
  ‘examples’ ‘hyperlog.notice.html’ ‘paper’ ‘unitTests’

checking dependencies in R code ... NOTE
'library' or 'require' call to ‘Rgraphviz’ in package code.
  Please use :: or requireNamespace() instead.
  See section 'Suggested packages' in the 'Writing R Extensions' manual.

checking S3 generic/method consistency ... NOTE
Found the following apparent S3 methods exported but not registered:
  as.data.frame.manyFilterResult
See section ‘Registering S3 methods’ in the ‘Writing R Extensions’
manual.

checking R code for possible problems ... NOTE
rbind2,flowSet-flowSet: warning in assign(i, x[[i]], env = env):
  partial argument match of 'env' to 'envir'
rbind2,flowSet-flowSet: warning in assign(i, y[[i]], env = env):
  partial argument match of 'env' to 'envir'
%in%,flowFrame-norm2Filter: no visible global function definition for
  ‘cov.rob’
spillover,flowSet: no visible global function definition for
  ‘devAskNewPage’
spillover,flowSet : <anonymous>: no visible global function definition
... 6 lines ...
spillover,flowSet : <anonymous> : <anonymous>: no visible global
  function definition for ‘density’
split,flowFrame-factor: no visible binding for global variable ‘i’
summary,workFlow: no visible binding for global variable ‘wf’
Undefined global functions or variables:
  abline cov.rob density devAskNewPage i rangeGate wf
Consider adding
  importFrom("grDevices", "devAskNewPage")
  importFrom("graphics", "abline")
  importFrom("stats", "density")
to your NAMESPACE file.

checking Rd line widths ... NOTE
Rd file 'FCSTransTransform.Rd':
  \usage lines wider than 90 characters:
     FCSTransTransform(transformationId = "defaultFCSTransTransform", channelrange = 2^18, channeldecade = 4.5, range = 4096, cutoff = -111, ... [TRUNCATED]

Rd file 'biexponentialTransform.Rd':
  \usage lines wider than 90 characters:
     biexponentialTransform(transformationId="defaultBiexponentialTransform", a = 0.5, b = 1, c = 0.5, d = 1, f = 0, w = 0, tol = .Machine$d ... [TRUNCATED]

Rd file 'compensation-class.Rd':
... 10 lines ...
                 decades=0, ncdf = FALSE, min.limit=NULL, truncate_max_range = TRUE, dataset=NULL, emptyValue=TRUE, ...)

Rd file 'spillover-flowSet.Rd':
  \usage lines wider than 90 characters:
             ssc = "SSC-A", method = "median", stain_match = c("intensity", "ordered", "regexpr"),

Rd file 'splitScaleTransform.Rd':
  \usage lines wider than 90 characters:
     splitScaleTransform(transformationId="defaultSplitscaleTransform", maxValue=1023, transitionChannel=64, r=192)

These lines will be truncated in the PDF manual.

checking for GNU extensions in Makefiles ... NOTE
GNU make is a SystemRequirements.
```

## fslr (2.12)
Maintainer: John Muschelli <muschellij2@gmail.com>  
Bug reports: https://github.com/muschellij2/fslr/issues

0 errors | 0 warnings | 0 notes

## GAD (1.1.1)
Maintainer: Leonardo Sandrini-Neto <leonardosandrini@gmail.com>

0 errors | 0 warnings | 0 notes

## genomation (1.6.0)
Maintainer: Altuna Akalin <aakalin@gmail.com>, Vedran Franke
 <vedran.franke@gmail.com>  
Bug reports: https://github.com/BIMSBbioinfo/genomation/issues

0 errors | 1 warning  | 3 notes

```
checking for missing documentation entries ... WARNING
Undocumented S4 methods:
  generic '[' and siglist 'ScoreMatrix,ANY,ANY,ANY'
  generic '[' and siglist 'ScoreMatrixList,ANY,ANY,ANY'
All user-level objects in a package (including S4 classes and methods)
should have documentation entries.
See chapter ‘Writing R documentation files’ in the ‘Writing R
Extensions’ manual.

checking installed package size ... NOTE
  installed size is  6.3Mb
  sub-directories of 1Mb or more:
    doc       3.6Mb
    extdata   1.2Mb

checking dependencies in R code ... NOTE
Unexported object imported by a ':::' call: ‘BiocGenerics:::testPackage’
  See the note in ?`:::` about the use of this operator.

checking files in ‘vignettes’ ... NOTE
The following directory looks like a leftover from 'knitr':
  ‘cache’
Please remove from your package.
```

## GenRank (1.2.0)
Maintainer: Chakravarthi Kanduri <chakra.kanduri@gmail.com>  
Bug reports: https://github.com/chakri9/GenRank/issues

0 errors | 0 warnings | 1 note 

```
checking R code for possible problems ... NOTE
CombP: no visible global function definition for ‘read.table’
ComputeCE: no visible global function definition for ‘read.table’
ComputeCE: no visible global function definition for ‘complete.cases’
ComputeRP: no visible global function definition for ‘read.table’
ComputeRP: no visible global function definition for ‘complete.cases’
Undefined global functions or variables:
  complete.cases read.table
Consider adding
  importFrom("stats", "complete.cases")
  importFrom("utils", "read.table")
to your NAMESPACE file.
```

## GPrank (0.1.2)
Maintainer: Hande Topa <hande.topa@helsinki.fi>  
Bug reports: https://github.com/PROBIC/GPrank/issues

0 errors | 0 warnings | 0 notes

## GUIDEseq (1.4.1)
Maintainer: Lihua Julie Zhu <julie.zhu@umassmed.edu>

1 error  | 1 warning  | 3 notes

```
checking examples ... ERROR
Running examples in ‘GUIDEseq-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: GUIDEseqAnalysis
> ### Title: Analysis pipeline for GUIDE-seq dataset
> ### Aliases: GUIDEseqAnalysis
> ### Keywords: misc
> 
... 45 lines ...
>>> DONE searching
finish off-target search in sequence 2
finish off-target search in sequence 1
finish feature vector building
finish score calculation
[1] "Done!"
Warning in if (!is.na(TS2)) { :
  the condition has length > 1 and only the first element will be used
Error in .check_ncores(length(names)) : 3 simultaneous processes spawned
Calls: GUIDEseqAnalysis ... offTargetAnalysisOfPeakRegions -> makeCluster -> makePSOCKcluster -> .check_ncores
Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
            sure you are using the right genome. You can also alter your 
            search criteria such as increasing max.mismatch!
Warning in searchHits(gRNAs = gRNAs1, PAM = PAM, PAM.pattern = PAM.pattern,  :
  No matching found, please check your input sequence, and make
            sure you are using the right genome. You can also alter your 
            search criteria such as increasing max.mismatch!
Warning in searchHits(gRNAs = gRNAs1, PAM = PAM, PAM.pattern = PAM.pattern,  :
... 8 lines ...
Warning in searchHits(gRNAs = gRNAs1, PAM = PAM, PAM.pattern = PAM.pattern,  :
  No matching found, please check your input sequence, and make
            sure you are using the right genome. You can also alter your 
            search criteria such as increasing max.mismatch!
Warning in if (!is.na(TS2)) { :
  the condition has length > 1 and only the first element will be used

Error: processing vignette 'GUIDEseq.Rnw' failed with diagnostics:
 chunk 7 
Error in .check_ncores(length(names)) : 3 simultaneous processes spawned
Execution halted

checking installed package size ... NOTE
  installed size is 11.1Mb
  sub-directories of 1Mb or more:
    extdata  10.6Mb

checking R code for possible problems ... NOTE
annotateOffTargets: no visible binding for global variable
  'offTarget_Start'
annotateOffTargets: no visible global function definition for 'exons'
annotateOffTargets: no visible global function definition for
  'seqlevelsStyle<-'
annotateOffTargets: no visible global function definition for 'genes'
annotateOffTargets: no visible global function definition for 'toTable'
getPeaks: no visible binding for global variable 'adjusted.p.value'
getPeaks: no visible binding for global variable 'SNratio'
... 8 lines ...
  'strand.first'
getUniqueCleavageEvents: no visible binding for global variable
  'readName'
offTargetAnalysisOfPeakRegions: no visible binding for global variable
  'gRNAPlusPAM'
offTargetAnalysisOfPeakRegions: no visible binding for global variable
  'offTarget'
Undefined global functions or variables:
  SNratio adjusted.p.value exons gRNAPlusPAM genes offTarget
  offTarget_Start qwidth.first qwidth.last readName seqlevelsStyle<-
  strand.first strand.last toTable

checking Rd files ... NOTE
prepare_Rd: annotateOffTargets.Rd:33-35: Dropping empty section \details
prepare_Rd: annotateOffTargets.Rd:43-45: Dropping empty section \references
```

## Gviz (1.18.2)
Maintainer: Florian Hahne <florian.hahne@novartis.com>

0 errors | 2 warnings | 4 notes

```
checking for missing documentation entries ... WARNING
Undocumented S4 methods:
  generic '[' and siglist 'AlignedReadTrack,ANY,ANY,ANY'
  generic '[' and siglist 'DataTrack,ANY,ANY,ANY'
  generic '[' and siglist 'GenomeAxisTrack,ANY,ANY,ANY'
  generic '[' and siglist 'IdeogramTrack,ANY,ANY,ANY'
  generic '[' and siglist 'RangeTrack,ANY,ANY,ANY'
  generic '[' and siglist 'StackedTrack,ANY,ANY,ANY'
All user-level objects in a package (including S4 classes and methods)
should have documentation entries.
See chapter ‘Writing R documentation files’ in the ‘Writing R
Extensions’ manual.

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
    colMeans, colSums, expand.grid, rowMeans, rowSums

Loading required package: IRanges
Loading required package: GenomicRanges
Loading required package: GenomeInfoDb
Loading required package: grid
Loading required package: BSgenome
... 8 lines ...
    Vignettes contain introductory material; view with
    'browseVignettes()'. To cite Bioconductor, see
    'citation("Biobase")', and for packages
    'citation("pkgname")'.


Error: processing vignette 'Gviz.Rnw' failed with diagnostics:
 chunk 94 (label = BiomartGeneRegionTrackDo) 
Error in base::serialize(object, connection = NULL, ascii = ascii) : 
  trying to get slot "host" from an object of a basic class ("NULL") with no slots
Execution halted

checking installed package size ... NOTE
  installed size is  5.8Mb
  sub-directories of 1Mb or more:
    doc       1.6Mb
    extdata   2.1Mb

checking top-level files ... NOTE
Non-standard file/directory found at top level:
  ‘Rplots.pdf’

checking R code for possible problems ... NOTE
.buildRange,IRanges-ANY-ANY-ANY: warning in GRanges(seqnames =
  .chrName(chromosome), range = range, strand = if
  (!is.null(args$strand)) args$strand else "*"): partial argument match
  of 'range' to 'ranges'
collapseTrack,DataTrack: warning in GRanges(seqnames = seqnames(r)[1],
  range = rtmp): partial argument match of 'range' to 'ranges'
collapseTrack,DataTrack: warning in GRanges(seqnames =
  chromosome(GdObject), range = ir, strand =
  unique(as.character(strand(GdObject)))): partial argument match of
... 39 lines ...
  'setNames'
drawGD,AlignmentsTrack : <anonymous>: no visible global function
  definition for 'setNames'
Undefined global functions or variables:
  cdsBy dev.cur dev.off fiveUTRsByTranscript setNames
  threeUTRsByTranscript type.convert
Consider adding
  importFrom("grDevices", "dev.cur", "dev.off")
  importFrom("stats", "setNames")
  importFrom("utils", "type.convert")
to your NAMESPACE file.

checking Rd line widths ... NOTE
Rd file 'AlignmentsTrack-class.Rd':
  \usage lines wider than 90 characters:
                     stacking="squish", id, cigar, mapq, flag, isize, groupid, status, md, seqs,
                     name="AlignmentsTrack", isPaired=TRUE, importFunction, referenceSequence, ...)

Rd file 'AnnotationTrack-class.Rd':
  \examples lines wider than 100 characters:
     annTrack <- AnnotationTrack(start=st, end=ed, strand=str, chromosome=7, genome="hg19", feature="test",
                                 group=gr, id=paste("annTrack item", 1:4), name="generic annotation", stacking="squish")

Rd file 'CustomTrack-class.Rd':
  \usage lines wider than 90 characters:
     CustomTrack(plottingFunction=function(GdObject, prepare=FALSE, ...){}, variables=list(), name="CustomTrack",  ...)

Rd file 'HighlightTrack-class.Rd':
  \usage lines wider than 90 characters:
     HighlightTrack(trackList=list(), range=NULL, start=NULL, end=NULL, width=NULL, chromosome, genome,

These lines will be truncated in the PDF manual.
```

## haploReconstruct (0.1.2)
Maintainer: Susanne U. Franssen <susefranssen@googlemail.com>

0 errors | 0 warnings | 1 note 

```
checking installed package size ... NOTE
  installed size is  9.7Mb
  sub-directories of 1Mb or more:
    data   9.4Mb
```

## IMIFA (1.1.0)
Maintainer: Keefe Murphy <keefe.murphy@ucd.ie>

0 errors | 0 warnings | 0 notes

## InfiniumPurify (1.3.1)
Maintainer: Yufang Qin <yfqin@shou.edu.cn>

0 errors | 0 warnings | 0 notes

## kernDeepStackNet (2.0.1)
Maintainer: Thomas Welchowski <welchow@imbie.meb.uni-bonn.de>

0 errors | 0 warnings | 1 note 

```
checking installed package size ... NOTE
  installed size is  6.8Mb
  sub-directories of 1Mb or more:
    libs   6.5Mb
```

## localgauss (0.40)
Maintainer: Tore Selland Kleppe <tore.kleppe@uis.no>

0 errors | 0 warnings | 0 notes

## loo (1.1.0)
Maintainer: Jonah Gabry <jsg2201@columbia.edu>  
Bug reports: https://github.com/stan-dev/loo/issues

0 errors | 0 warnings | 0 notes

## LS2Wstat (2.0-3)
Maintainer: Matt Nunes <m.nunes@lancaster.ac.uk>

0 errors | 1 warning  | 2 notes

```
checking whether package ‘LS2Wstat’ can be installed ... WARNING
Found the following significant warnings:
  Warning: no DISPLAY variable so Tk is not available
See ‘/home/hb/repositories/matrixStats/revdep/checks/LS2Wstat.Rcheck/00install.out’ for details.

checking DESCRIPTION meta-information ... NOTE
Malformed Description field: should contain one or more complete sentences.

checking R code for possible problems ... NOTE
TOS2D: no visible global function definition for ‘medpolish’
countTextures: no visible global function definition for ‘medpolish’
plot.imageQT: no visible global function definition for ‘segments’
simTexture: no visible global function definition for ‘rnorm’
Undefined global functions or variables:
  medpolish rnorm segments
Consider adding
  importFrom("graphics", "segments")
  importFrom("stats", "medpolish", "rnorm")
to your NAMESPACE file.
```

## ltmle (0.9-9)
Maintainer: Joshua Schwab <joshuaschwab@yahoo.com>  
Bug reports: https://github.com/joshuaschwab/ltmle/issues

0 errors | 0 warnings | 0 notes

## Luminescence (0.7.4)
Maintainer: Sebastian Kreutzer <sebastian.kreutzer@u-bordeaux-montaigne.fr>  
Bug reports: https://github.com/R-Lum/Luminescence/issues

0 errors | 0 warnings | 0 notes

## metagene (2.6.1)
Maintainer: Charles Joly Beauparlant <charles.joly-beauparlant@crchul.ulaval.ca>  
Bug reports: https://github.com/CharlesJB/metagene/issues

0 errors | 0 warnings | 1 note 

```
checking R code for possible problems ... NOTE
get_demo_design: no visible global function definition for ‘read.table’
get_promoters_txdb: no visible global function definition for ‘is’
plot_metagene: no visible binding for global variable ‘position’
plot_metagene: no visible binding for global variable ‘value’
plot_metagene: no visible binding for global variable ‘qinf’
plot_metagene: no visible binding for global variable ‘qsup’
plot_metagene: no visible binding for global variable ‘group’
Undefined global functions or variables:
  group is position qinf qsup read.table value
Consider adding
  importFrom("methods", "is")
  importFrom("utils", "read.table")
to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
contains 'methods').
```

## metagenomeSeq (1.16.0)
Maintainer: Joseph N. Paulson <jpaulson@jimmy.harvard.edu>  
Bug reports: https://github.com/nosson/metagenomeSeq/issues

0 errors | 1 warning  | 0 notes

```
checking sizes of PDF files under ‘inst/doc’ ... WARNING
  ‘gs+qpdf’ made some significant size reductions:
     compacted ‘metagenomeSeq.pdf’ from 975Kb to 673Kb
  consider running tools::compactPDF(gs_quality = "ebook") on these files
```

## MetaQC (0.1.13)
Maintainer: Don Kang <donkang75@gmail.com>

0 errors | 0 warnings | 3 notes

```
checking dependencies in R code ... NOTE
'library' or 'require' call to ‘survival’ in package code.
  Please use :: or requireNamespace() instead.
  See section 'Suggested packages' in the 'Writing R Extensions' manual.
Packages in Depends field not imported from:
  ‘foreach’ ‘iterators’ ‘proto’
  These packages need to be imported from (in the NAMESPACE file)
  for when this namespace is loaded but not attached.

checking S3 generic/method consistency ... NOTE
Found the following apparent S3 methods exported but not registered:
  plot.proto print.proto
See section ‘Registering S3 methods’ in the ‘Writing R Extensions’
manual.

checking R code for possible problems ... NOTE
Download: no visible global function definition for ‘download.file’
GMT2List: no visible global function definition for ‘%dopar%’
GMT2List: no visible global function definition for ‘foreach’
GMT2List: no visible global function definition for ‘iter’
GMT2List: no visible binding for global variable ‘x’
GetEWPval: no visible global function definition for ‘pchisq’
GetPVal : t.test2: no visible global function definition for ‘pt’
GetPVal : <anonymous>: no visible global function definition for
  ‘t.test’
... 137 lines ...
  registerDoMC registerDoSNOW sd t.test text w wilcox.test x
Consider adding
  importFrom("grDevices", "gray")
  importFrom("graphics", "abline", "arrows", "axis", "box", "plot",
             "points", "text")
  importFrom("stats", "as.dist", "cor", "cor.test", "fisher.test",
             "ks.test", "lm", "median", "na.omit", "p.adjust", "pchisq",
             "pnorm", "prcomp", "pt", "qnorm", "sd", "t.test",
             "wilcox.test")
  importFrom("utils", "download.file", "installed.packages")
to your NAMESPACE file.
```

## MethylAid (1.8.0)
Maintainer: M. van Iterson <mviterson@gmail.com>  
Bug reports: https://github.com/mvaniterson/methylaid/issues

0 errors | 0 warnings | 1 note 

```
checking R code for possible problems ... NOTE
Found the following assignments to the global environment:
File ‘MethylAid/R/server.R’:
  assign("outliers", outliers, envir = globalenv())
  assign("background", as.background(background), envir = globalenv())
  assign("location", location, envir = globalenv())
  assign("location", location, envir = globalenv())
  assign("location", location, envir = globalenv())
  assign("location", location, envir = globalenv())
  assign("location", location, envir = globalenv())
File ‘MethylAid/R/util.R’:
  assign("highlight", highlight, envir = globalenv())
  assign("outliers", out, envir = globalenv())
```

## methylumi (2.20.0)
Maintainer: Sean Davis <sdavis2@mail.nih.gov>  
Bug reports: https://github.com/seandavi/methylumi/issues/new

0 errors | 1 warning  | 8 notes

```
checking for missing documentation entries ... WARNING
Undocumented code objects:
  ‘IDATsToMatrices’ ‘IDATtoMatrix’ ‘tcgaPipeline’
Undocumented data sets:
  ‘mset’
Undocumented S4 methods:
  generic '[' and siglist 'MethyLumiM,ANY,ANY,ANY'
  generic '[' and siglist 'MethyLumiSet,ANY,ANY,ANY'
All user-level objects in a package (including S4 classes and methods)
should have documentation entries.
See chapter ‘Writing R documentation files’ in the ‘Writing R
Extensions’ manual.

checking package dependencies ... NOTE
Depends: includes the non-default packages:
  ‘Biobase’ ‘scales’ ‘reshape2’ ‘ggplot2’ ‘matrixStats’
  ‘FDb.InfiniumMethylation.hg19’ ‘minfi’
Adding so many packages to the search path is excessive and importing
selectively is preferable.

checking installed package size ... NOTE
  installed size is  9.7Mb
  sub-directories of 1Mb or more:
    data      6.8Mb
    extdata   1.7Mb

checking DESCRIPTION meta-information ... NOTE
Packages listed in more than one of Depends, Imports, Suggests, Enhances:
  ‘Biobase’ ‘minfi’ ‘lattice’ ‘matrixStats’
A package should be listed in only one of these fields.

checking dependencies in R code ... NOTE
'library' or 'require' calls to packages already attached by Depends:
  ‘FDb.InfiniumMethylation.hg19’ ‘ggplot2’ ‘matrixStats’ ‘minfi’
  ‘reshape2’ ‘scales’
  Please remove these calls from your code.
'library' or 'require' calls in package code:
  ‘Biostrings’ ‘MASS’ ‘lumi’ ‘parallel’ ‘rtracklayer’
  Please use :: or requireNamespace() instead.
  See section 'Suggested packages' in the 'Writing R Extensions' manual.
Packages in Depends field not imported from:
  ‘FDb.InfiniumMethylation.hg19’ ‘ggplot2’ ‘matrixStats’ ‘methods’
  ‘reshape2’ ‘scales’
  These packages need to be imported from (in the NAMESPACE file)
  for when this namespace is loaded but not attached.
':::' call which should be '::': ‘lumi:::produceMethylationGEOSubmissionFile’
  See the note in ?`:::` about the use of this operator.
Unexported objects imported by ':::' calls:
  ‘Biobase:::unsafeSetSlot’ ‘genefilter:::.findCentralMap’
  See the note in ?`:::` about the use of this operator.

checking R code for possible problems ... NOTE
getBarcodes: warning in list.files(path = path, patt = "idat"): partial
  argument match of 'patt' to 'pattern'
.getFinalReportBlock: no visible global function definition for
  ‘read.table’
.mclapply: no visible global function definition for ‘mclapply’
.parallel : <anonymous>: no visible global function definition for
  ‘ecdf’
.readOldMethylationFile: no visible global function definition for
  ‘read.delim’
... 213 lines ...

Found the following calls to data() loading into the global environment:
File ‘methylumi/R/coercions.R’:
  data(hm450.controls)
  data(hm27.controls)
File ‘methylumi/R/methylumIDAT.R’:
  data(hm27.controls)
  data(hm450.controls)
  data(hm27.ordering)
  data(hm450.ordering)
See section ‘Good practice’ in ‘?data’.

checking Rd files ... NOTE
prepare_Rd: estimateM.Rd:34-36: Dropping empty section \seealso
prepare_Rd: estimateM.Rd:37-39: Dropping empty section \examples

checking Rd line widths ... NOTE
Rd file 'methylumIDAT.Rd':
  \examples lines wider than 100 characters:
       lumi450k <- methylumIDAT(barcodes,idatPath=system.file('extdata/idat',package='methylumi')) # no normalization done

These lines will be truncated in the PDF manual.

checking for unstated dependencies in vignettes ... NOTE
'library' or 'require' call not declared from: ‘TxDb.Hsapiens.UCSC.hg19.knownGene’
```

## MFHD (0.0.1)
Maintainer: K. Vakili <kaveh.vakili@wis.kuleuven.be>

0 errors | 0 warnings | 1 note 

```
checking R code for possible problems ... NOTE
MFHD: no visible global function definition for ‘complete.cases’
MFHD: no visible global function definition for ‘quantile’
PIT: no visible global function definition for ‘xy.coords’
PIT: no visible global function definition for ‘identify’
PIT: no visible global function definition for ‘points’
mfhd_compute.bagplot : cut.z.pg: no visible global function definition
  for ‘segments’
mfhd_compute.bagplot : cut.z.pg: no visible global function definition
  for ‘points’
... 35 lines ...
plot.bagplot: no visible global function definition for ‘lines’
plot.bagplot: no visible global function definition for ‘polygon’
Undefined global functions or variables:
  boxplot chull complete.cases identify lines plot points polygon
  prcomp quantile sd segments text xy.coords
Consider adding
  importFrom("grDevices", "chull", "xy.coords")
  importFrom("graphics", "boxplot", "identify", "lines", "plot",
             "points", "polygon", "segments", "text")
  importFrom("stats", "complete.cases", "prcomp", "quantile", "sd")
to your NAMESPACE file.
```

## minfi (1.20.2)
Maintainer: Kasper Daniel Hansen <kasperdanielhansen@gmail.com>  
Bug reports: https://github.com/kasperdanielhansen/minfi/issues

0 errors | 0 warnings | 4 notes

```
checking package dependencies ... NOTE
Depends: includes the non-default packages:
  ‘BiocGenerics’ ‘Biobase’ ‘GenomicRanges’ ‘SummarizedExperiment’
  ‘Biostrings’ ‘bumphunter’
Adding so many packages to the search path is excessive and importing
selectively is preferable.

checking dependencies in R code ... NOTE
Unexported objects imported by ':::' calls:
  ‘BiocGenerics:::replaceSlots’
  ‘SummarizedExperiment:::.valid.SummarizedExperiment.assays_ncol’
  ‘bumphunter:::.getEstimate’
  See the note in ?`:::` about the use of this operator.

checking R code for possible problems ... NOTE
Found the following calls to data() loading into the global environment:
File ‘minfi/R/estimateCellCounts.R’:
  data(list = referencePkg)
See section ‘Good practice’ in ‘?data’.

checking Rd line widths ... NOTE
Rd file 'estimateCellCounts.Rd':
  \usage lines wider than 90 characters:
                        referencePlatform = c("IlluminaHumanMethylation450k", "IlluminaHumanMethylationEPIC", "IlluminaHumanMethylation27k") ... [TRUNCATED]

These lines will be truncated in the PDF manual.
```

## MinimumDistance (1.18.0)
Maintainer: Robert B Scharpf <rscharpf@jhu.edu>

0 errors | 0 warnings | 1 note 

```
checking R code for possible problems ... NOTE
callDenovoSegments: no visible global function definition for
  ‘read.bsfiles’
read.bsfiles2: no visible binding for global variable ‘read.bsfiles’
GenomeAnnotatedDataFrameFrom,character: no visible global function
  definition for ‘read.bsfiles’
calculateMindist,list: no visible binding for global variable ‘elt’
Undefined global functions or variables:
  elt read.bsfiles
```

## mmtfa (0.1)
Maintainer: Jeffrey L. Andrews <jeffrey.andrews@macewan.ca>

0 errors | 0 warnings | 1 note 

```
checking R code for possible problems ... NOTE
contrandz: no visible global function definition for ‘runif’
dfupdatefun: no visible global function definition for ‘uniroot’
kmeansz: no visible global function definition for ‘kmeans’
plot.mmtfa: no visible global function definition for ‘rainbow’
plot.mmtfa: no visible global function definition for ‘par’
plot.mmtfa: no visible global function definition for ‘plot’
plot.mmtfa: no visible global function definition for ‘title’
plot.mmtfa: no visible global function definition for ‘contour’
plot.mmtfa: no visible global function definition for ‘rgb’
... 7 lines ...
sgupdate: no visible global function definition for ‘cov.wt’
winit: no visible global function definition for ‘mahalanobis’
Undefined global functions or variables:
  contour cov.wt curve density dnorm kmeans mahalanobis par plot
  rainbow rgb runif title uniroot
Consider adding
  importFrom("grDevices", "rainbow", "rgb")
  importFrom("graphics", "contour", "curve", "par", "plot", "title")
  importFrom("stats", "cov.wt", "density", "dnorm", "kmeans",
             "mahalanobis", "runif", "uniroot")
to your NAMESPACE file.
```

## monocle (2.2.0)
Maintainer: Cole Trapnell <coletrap@uw.edu>

0 errors | 0 warnings | 2 notes

```
checking top-level files ... NOTE
Non-standard file/directory found at top level:
  ‘examples’

checking R code for possible problems ... NOTE
assign_cell_lineage: no visible global function definition for ‘nei’
buildBranchCellDataSet: no visible global function definition for ‘nei’
count_leaf_descendents: no visible global function definition for ‘nei’
cth_classifier_cds: no visible global function definition for ‘nei’
cth_classifier_cell: no visible global function definition for ‘nei’
diff_test_helper: no visible binding for global variable ‘Size_Factor’
extract_good_branched_ordering: no visible global function definition
  for ‘nei’
extract_good_branched_ordering : extract_branched_ordering_helper: no
  visible global function definition for ‘nei’
extract_good_branched_ordering : assign_cell_state_helper: no visible
  global function definition for ‘nei’
extract_good_branched_ordering : assign_pseudotime_helper: no visible
  global function definition for ‘nei’
extract_good_ordering: no visible global function definition for ‘nei’
fit_model_helper: no visible binding for global variable ‘Size_Factor’
make_canonical: no visible global function definition for ‘nei’
measure_diameter_path: no visible global function definition for ‘nei’
project2MST: no visible global function definition for ‘nei’
relative2abs : <anonymous>: no visible binding for global variable
  ‘FPKM’
Undefined global functions or variables:
  FPKM Size_Factor nei
```

## motifbreakR (1.4.0)
Maintainer: Simon Gert Coetzee <Simon.Coetzee@cshs.org>  
Bug reports: https://github.com/Simon-Coetzee/motifbreakR/issues

0 errors | 0 warnings | 2 notes

```
checking dependencies in R code ... NOTE
Packages in Depends field not imported from:
  'MotifDb' 'grid'
  These packages need to be imported from (in the NAMESPACE file)
  for when this namespace is loaded but not attached.

checking R code for possible problems ... NOTE
plotMotifLogoStack.2: no visible global function definition for 'par'
snps.from.file: no visible global function definition for 'info'
snps.from.file: no visible global function definition for 'rowRanges'
snps.from.rsid: no visible global function definition for 'snpsById'
Undefined global functions or variables:
  info par rowRanges snpsById
Consider adding
  importFrom("graphics", "par")
to your NAMESPACE file.

Found the following assignments to the global environment:
File ‘motifbreakR/R/scoreMotif.R’:
  assign("tmp_motifStack_symbolsCache", list(), pos = ".GlobalEnv")
```

## MPAgenomics (1.1.2)
Maintainer: Samuel Blanck <samuel.blanck@inria.fr>

0 errors | 0 warnings | 2 notes

```
checking dependencies in R code ... NOTE
'library' or 'require' calls in package code:
  ‘R.devices’ ‘R.filesets’ ‘R.methodsS3’ ‘R.oo’ ‘aroma.affymetrix’
  ‘aroma.cn’ ‘aroma.core’ ‘aroma.light’ ‘matrixStats’ ‘snowfall’
  Please use :: or requireNamespace() instead.
  See section 'Suggested packages' in the 'Writing R Extensions' manual.
Unexported object imported by a ':::' call: ‘cghseg:::segmeanCO’
  See the note in ?`:::` about the use of this operator.

checking R code for possible problems ... NOTE
.varregtimescount: no visible global function definition for ‘var’
CGHSEGaroma: no visible global function definition for ‘read.csv’
CGHSEGaroma : <anonymous>: no visible global function definition for
  ‘points’
CGHSEGaroma : <anonymous>: no visible global function definition for
  ‘lines’
CGHSEGaroma : <anonymous>: no visible global function definition for
  ‘write.table’
CGHcall: no visible global function definition for ‘mad’
... 43 lines ...
tumorboostPlot: no visible global function definition for ‘par’
tumorboostPlot: no visible global function definition for ‘axis’
tumorboostPlot: no visible global function definition for ‘points’
Undefined global functions or variables:
  axis head lines lm mad median optim par points read.csv sd var
  write.table
Consider adding
  importFrom("graphics", "axis", "lines", "par", "points")
  importFrom("stats", "lm", "mad", "median", "optim", "sd", "var")
  importFrom("utils", "head", "read.csv", "write.table")
to your NAMESPACE file.
```

## mrfDepth (1.0.3)
Maintainer: Pieter Segaert <Pieter.Segaert@kuleuven.be>

0 errors | 0 warnings | 1 note 

```
checking installed package size ... NOTE
  installed size is 11.2Mb
  sub-directories of 1Mb or more:
    libs  10.4Mb
```

## MultiBD (0.2.0)
Maintainer: Marc A. Suchard <msuchard@ucla.edu>

0 errors | 0 warnings | 1 note 

```
checking installed package size ... NOTE
  installed size is  9.4Mb
  sub-directories of 1Mb or more:
    libs   7.7Mb
```

## NanoStringDiff (1.4.0)
Maintainer: hong wang <hong.wang@uky.edu>

0 errors | 0 warnings | 1 note 

```
checking R code for possible problems ... NOTE
compute.baseSigma: no visible global function definition for ‘IQR’
createNanoStringSetFromCsv: no visible global function definition for
  ‘read.table’
est.dispersion : get.phihat: no visible global function definition for
  ‘optimize’
estNormalizationFactors: no visible global function definition for
  ‘glm’
estNormalizationFactors: no visible global function definition for
  ‘poisson’
... 16 lines ...
  for ‘optim’
rnegbinom: no visible global function definition for ‘rpois’
rnegbinom: no visible global function definition for ‘rgamma’
Undefined global functions or variables:
  IQR glm lm median optim optimize p.adjust pchisq poisson read.table
  rgamma rpois
Consider adding
  importFrom("stats", "IQR", "glm", "lm", "median", "optim", "optimize",
             "p.adjust", "pchisq", "poisson", "rgamma", "rpois")
  importFrom("utils", "read.table")
to your NAMESPACE file.
```

## neurobase (1.13.2)
Maintainer: John Muschelli <muschellij2@gmail.com>  
Bug reports: https://github.com/muschellij2/nifti/issues

0 errors | 0 warnings | 0 notes

## NSA (0.0.32)
Maintainer: Maria Ortiz-Estevez <mortizest@gmail.com>

0 errors | 0 warnings | 6 notes

```
checking package dependencies ... NOTE
Depends: includes the non-default packages:
  ‘R.methodsS3’ ‘MASS’ ‘matrixStats’ ‘R.oo’ ‘R.utils’ ‘aroma.core’
  ‘aroma.affymetrix’ ‘DNAcopy’
Adding so many packages to the search path is excessive and importing
selectively is preferable.

checking top-level files ... NOTE
Non-standard file/directory found at top level:
  ‘incl’

checking dependencies in R code ... NOTE
Packages in Depends field not imported from:
  ‘DNAcopy’ ‘MASS’ ‘R.methodsS3’ ‘R.oo’ ‘aroma.affymetrix’ ‘aroma.core’
  ‘matrixStats’
  These packages need to be imported from (in the NAMESPACE file)
  for when this namespace is loaded but not attached.

checking S3 generic/method consistency ... NOTE
Found the following apparent S3 methods exported but not registered:
  NSAByTotalAndFracB.matrix allocateOutputDataSets.NSANormalization
  allocateOutputDataSets.SNPsNormalization
  allocateOutputDataSets.SampleNormalization
  as.character.NSANormalization as.character.SNPsNormalization
  as.character.SampleNormalization findArraysTodo.NSANormalization
  findArraysTodo.SampleNormalization findUnitsTodo.SNPsNormalization
  fitNSA.matrix fitNSAcnPs.matrix getDataSets.NSANormalization
  getDataSets.SNPsNormalization getDataSets.SampleNormalization
  getName.NSANormalization getName.SNPsNormalization
  getName.SampleNormalization getOutputDataSets.NSANormalization
  getOutputDataSets.SNPsNormalization
  getOutputDataSets.SampleNormalization getPath.NSANormalization
  getPath.SNPsNormalization getPath.SampleNormalization
  getRootPath.NSANormalization getRootPath.SNPsNormalization
  getRootPath.SampleNormalization process.NSANormalization
  process.SNPsNormalization process.SampleNormalization
  sampleNByTotalAndFracB.numeric snpsNByTotalAndFracB.matrix
See section ‘Registering S3 methods’ in the ‘Writing R Extensions’
manual.

checking R code for possible problems ... NOTE
NB: .First.lib is obsolete and will not be used in R >= 3.0.0

.First.lib: no visible global function definition for
  ‘packageDescription’
NSAByTotalAndFracB.matrix: no visible global function definition for
  ‘throw’
NSAByTotalAndFracB.matrix: no visible global function definition for
  ‘str’
NSANormalization: no visible global function definition for ‘throw’
... 279 lines ...
  extractMatrix findUnitsTodo getAsteriskTags getChipType getFile
  getFullName getFullNames getGenomeInformation getName getNames
  getPath getPathname getPathnames getPositions getRam getRootPath
  getTags getUnitsOnChromosome hist median nbrOfFiles newInstance
  packageDescription rowAlls rowMedians segment setTags str throw trim
  verbose
Consider adding
  importFrom("graphics", "hist")
  importFrom("stats", "approxfun", "median")
  importFrom("utils", "packageDescription", "str")
to your NAMESPACE file.

checking Rd line widths ... NOTE
Rd file 'NSANormalization.Rd':
  \examples lines wider than 100 characters:
     by <- 50e3; # 50kb bins; you may want to try with other amounts of smoothing xOut <- seq(from=xRange[1], to=xRange[2], by=by);
     plot(getSignals(cnCNPS), getSignals(cnSNPS), xlim=Clim, ylim=Clim); abline(a=0, b=1, col="red", lwd=2);

These lines will be truncated in the PDF manual.
```

## pandaR (1.6.0)
Maintainer: Joseph N. Paulson <jpaulson@jimmy.harvard.edu>, Dan Schlauch <dschlauch@fas.harvard.edu>

0 errors | 0 warnings | 4 notes

```
checking installed package size ... NOTE
  installed size is  9.2Mb
  sub-directories of 1Mb or more:
    data   9.0Mb

checking S3 generic/method consistency ... NOTE
Found the following apparent S3 methods exported but not registered:
  plot.panda
See section ‘Registering S3 methods’ in the ‘Writing R Extensions’
manual.

checking R code for possible problems ... NOTE
dFunction: no visible global function definition for ‘cor’
importPandaMatlab: no visible global function definition for
  ‘read.delim’
normalizeNetwork: no visible global function definition for ‘sd’
panda: no visible global function definition for ‘cor’
panda: no visible global function definition for ‘aggregate’
plot.panda: no visible global function definition for ‘hist’
plotCommunityDetection: no visible global function definition for
  ‘title’
plotZbyTF: no visible global function definition for ‘aggregate’
prepResult: no visible global function definition for ‘pnorm’
Undefined global functions or variables:
  aggregate cor hist pnorm read.delim sd title
Consider adding
  importFrom("graphics", "hist", "title")
  importFrom("stats", "aggregate", "cor", "pnorm", "sd")
  importFrom("utils", "read.delim")
to your NAMESPACE file.

checking Rd \usage sections ... NOTE
S3 methods shown with full name in documentation object 'plot.panda':
  ‘plot.panda’

The \usage entries for S3 methods should use the \method markup and not
their full name.
See chapter ‘Writing R documentation files’ in the ‘Writing R
Extensions’ manual.
```

## PathoStat (1.0.0)
Maintainer: Solaiappan Manimaran <manimaran_1975@hotmail.com>  
Bug reports: https://github.com/mani2012/PathoStat/issues

0 errors | 1 warning  | 0 notes

```
checking sizes of PDF files under ‘inst/doc’ ... WARNING
  ‘gs+qpdf’ made some significant size reductions:
     compacted ‘PathoStatUserManual.pdf’ from 1.9Mb to 1.0Mb
  consider running tools::compactPDF(gs_quality = "ebook") on these files
```

## pcaMethods (1.66.0)
Maintainer: Henning Redestig <henning.red@gmail.com>  
Bug reports: https://github.com/hredestig/pcamethods/issues

0 errors | 0 warnings | 2 notes

```
checking top-level files ... NOTE
Non-standard file/directory found at top level:
  ‘extra’

checking R code for possible problems ... NOTE
BPCA_initmodel: no visible global function definition for ‘cov’
Q2: no visible global function definition for ‘txtProgressBar’
Q2: no visible global function definition for ‘setTxtProgressBar’
Q2: no visible global function definition for ‘cor’
RnipalsPca: no visible global function definition for ‘na.omit’
cvseg : <anonymous>: no visible global function definition for
  ‘na.omit’
kEstimate: no visible global function definition for ‘var’
kEstimateFast: no visible global function definition for ‘var’
... 30 lines ...
  abline barplot cor cov gray layout legend lines median na.omit pairs
  par points prcomp qf rnorm runif sd setTxtProgressBar text
  txtProgressBar var
Consider adding
  importFrom("grDevices", "gray")
  importFrom("graphics", "abline", "barplot", "layout", "legend",
             "lines", "pairs", "par", "points", "text")
  importFrom("stats", "cor", "cov", "median", "na.omit", "prcomp", "qf",
             "rnorm", "runif", "sd", "var")
  importFrom("utils", "setTxtProgressBar", "txtProgressBar")
to your NAMESPACE file.
```

## peakPick (0.11)
Maintainer: Paal O. Westermark <pal-olof.westermark@charite.de>

0 errors | 0 warnings | 0 notes

## Pigengene (1.0.0)
Maintainer: Habil Zare <zare@txstate.edu>

0 errors | 0 warnings | 2 notes

```
checking dependencies in R code ... NOTE
'library' or 'require' calls in package code:
  ‘AnnotationDbi’ ‘biomaRt’ ‘energy’ ‘org.Hs.eg.db’ ‘org.Mm.eg.db’
  Please use :: or requireNamespace() instead.
  See section 'Suggested packages' in the 'Writing R Extensions' manual.

checking R code for possible problems ... NOTE
Found the following possibly unsafe calls:
File ‘Pigengene/R/bn.calculation.R’:
  assignInNamespace("supported.clusters", fixArgs, "bnlearn")

welch.pvalue: no visible global function definition for ‘as.formula’
welch.pvalue: no visible global function definition for ‘oneway.test’
Undefined global functions or variables:
  as.formula oneway.test
Consider adding
  importFrom("stats", "as.formula", "oneway.test")
to your NAMESPACE file.
```

## PSCBS (0.62.0)
Maintainer: Henrik Bengtsson <henrikb@braju.com>  
Bug reports: https://github.com/HenrikBengtsson/PSCBS/issues

0 errors | 0 warnings | 0 notes

## QDNAseq (1.10.0)
Maintainer: Daoud Sie <d.sie@vumc.nl>  
Bug reports: https://github.com/ccagc/QDNAseq/issues

0 errors | 0 warnings | 0 notes

## r2dRue (1.0.4)
Maintainer: Alberto Ruiz Moreno <aruiz@eeza.csic.es>

0 errors | 0 warnings | 2 notes

```
checking dependencies in R code ... NOTE
'library' or 'require' call to ‘rgdal’ which was already attached by Depends.
  Please remove these calls from your code.
Packages in Depends field not imported from:
  ‘matrixStats’ ‘rgdal’ ‘sp’
  These packages need to be imported from (in the NAMESPACE file)
  for when this namespace is loaded but not attached.

checking R code for possible problems ... NOTE
.gdalWriteFormats: no visible global function definition for
  ‘gdalDrivers’
aiObsEx: no visible global function definition for ‘readGDAL’
aiObsEx: no visible global function definition for ‘txtProgressBar’
aiObsEx: no visible global function definition for ‘setTxtProgressBar’
aiObsMe: no visible global function definition for ‘txtProgressBar’
aiObsMe: no visible global function definition for ‘readGDAL’
aiObsMe: no visible global function definition for ‘setTxtProgressBar’
assessment: no visible global function definition for ‘writeGDAL’
... 86 lines ...
  layout layout.show lines na.omit pf plot pt read.table readGDAL rect
  rowCounts rowMedians rowRanges rowSds rowVars setTxtProgressBar
  spplot title txtProgressBar write.table writeGDAL
Consider adding
  importFrom("graphics", "abline", "axis.Date", "barplot", "boxplot",
             "hist", "image", "layout", "layout.show", "lines", "plot",
             "rect", "title")
  importFrom("stats", "cor", "density", "na.omit", "pf", "pt")
  importFrom("utils", "head", "read.table", "setTxtProgressBar",
             "txtProgressBar", "write.table")
to your NAMESPACE file.
```

## RAC (1.1.1)
Maintainer: Baldan D. <damiano.baldan91@gmail.com>

0 errors | 0 warnings | 0 notes

## randomizationInference (1.0.3)
Maintainer: Joseph J. Lee <joseph.j.lee@post.harvard.edu>

0 errors | 0 warnings | 1 note 

```
checking R code for possible problems ... NOTE
anovaF: no visible global function definition for ‘anova’
anovaF: no visible global function definition for ‘lm’
randInterval: no visible global function definition for ‘quantile’
randInterval : <anonymous>: no visible global function definition for
  ‘quantile’
randPlot: no visible global function definition for ‘par’
randPlot: no visible global function definition for ‘hist’
randPlot: no visible global function definition for ‘abline’
Undefined global functions or variables:
  abline anova hist lm par quantile
Consider adding
  importFrom("graphics", "abline", "hist", "par")
  importFrom("stats", "anova", "lm", "quantile")
to your NAMESPACE file.
```

## RnBeads (1.6.1)
Maintainer: Fabian Mueller <rnbeads@mpi-inf.mpg.de>

0 errors | 7 warnings | 6 notes

```
checking for executable files ... WARNING
Found the following executable files:
  inst/bin/linux_x86.64/bedGraphToBigWig
  inst/bin/linux_x86.64/bedToBigBed
  inst/bin/macOSX.i386/bedGraphToBigWig
  inst/bin/macOSX.i386/bedToBigBed
Source packages should not contain undeclared executable files.
See section ‘Package structure’ in the ‘Writing R Extensions’ manual.

checking whether package ‘RnBeads’ can be installed ... WARNING
Found the following significant warnings:
  Warning: replacing previous import ‘gridExtra::combine’ by ‘methylumi::combine’ when loading ‘RnBeads’
See ‘/home/hb/repositories/matrixStats/revdep/checks/RnBeads.Rcheck/00install.out’ for details.

checking package subdirectories ... WARNING
Subdirectory ‘data’ contains no data sets.

checking dependencies in R code ... WARNING
'::' or ':::' import not declared from: 'GLAD'
'library' or 'require' calls not declared from:
  'GLAD' 'IlluminaHumanMethylation450kanno.ilmn12.hg19' 'scales'
'library' or 'require' calls in package code:
  'Category' 'GLAD' 'GOstats' 'Gviz'
  'IlluminaHumanMethylation450kanno.ilmn12.hg19'
  'IlluminaHumanMethylation450kmanifest' 'RefFreeEWAS' 'annotate'
  'biomaRt' 'doParallel' 'foreach' 'glmnet' 'impute' 'isva' 'minfi'
  'nlme' 'quadprog' 'scales' 'sva' 'wateRmelon'
  Please use :: or requireNamespace() instead.
  See section 'Suggested packages' in the 'Writing R Extensions' manual.
Missing or unexported object: 'GenomicRanges::unlist'
Unexported objects imported by ':::' calls:
  'Gviz:::.getBMFeatureMap' 'doParallel:::.options'
  'grDevices:::.smoothScatterCalcDensity'
  'minfi:::.default.450k.annotation' 'minfi:::.extractFromRGSet450k'
  'minfi:::.normalizeFunnorm450k'
  See the note in ?`:::` about the use of this operator.
  Including base/recommended package(s):
  'grDevices'
There are ::: calls to the package's namespace in its code. A package
  almost never needs to use ::: for its own objects:
  'CONTROL.TARGETS.SAMPLE.INDEPENDENT'

checking for missing documentation entries ... WARNING
Undocumented S4 methods:
  generic '[' and siglist 'BigFfMat,ANY,ANY,ANY'
  generic '[<-' and siglist 'BigFfMat,ANY,ANY,ANY'
  generic 'dim' and siglist 'BigFfMat'
All user-level objects in a package (including S4 classes and methods)
should have documentation entries.
See chapter ‘Writing R documentation files’ in the ‘Writing R
Extensions’ manual.

checking Rd \usage sections ... WARNING
Undocumented arguments in documentation object 'read.single.bed'
  ‘chr.col’ ‘start.col’ ‘end.col’ ‘strand.col’ ‘c.col’ ‘t.col’
  ‘is.epp.style’ ‘coord.shift’ ‘ffread’ ‘context’ ‘...’

Undocumented arguments in documentation object 'sampleCovgApply,RnBSet-method'
  ‘object’ ‘type’ ‘...’

Undocumented arguments in documentation object 'sampleMethApply,RnBSet-method'
  ‘object’ ‘type’ ‘...’

Functions with \usage entries need to have the appropriate \alias
entries, and all their arguments documented.
The \usage entries must correspond to syntactically valid R code.
See chapter ‘Writing R documentation files’ in the ‘Writing R
Extensions’ manual.

checking sizes of PDF files under ‘inst/doc’ ... WARNING
  ‘gs+qpdf’ made some significant size reductions:
     compacted ‘RnBeads.pdf’ from 2.8Mb to 1.5Mb
  consider running tools::compactPDF(gs_quality = "ebook") on these files

checking package dependencies ... NOTE
Depends: includes the non-default packages:
  ‘BiocGenerics’ ‘S4Vectors’ ‘GenomicRanges’ ‘MASS’ ‘RColorBrewer’
  ‘cluster’ ‘ff’ ‘fields’ ‘ggplot2’ ‘gplots’ ‘gridExtra’ ‘limma’
  ‘matrixStats’ ‘illuminaio’ ‘methylumi’ ‘plyr’
Adding so many packages to the search path is excessive and importing
selectively is preferable.

checking installed package size ... NOTE
  installed size is  6.1Mb
  sub-directories of 1Mb or more:
    R     1.0Mb
    bin   1.0Mb
    doc   3.1Mb

checking top-level files ... NOTE
Non-standard file/directory found at top level:
  ‘BinaryFiles’

checking whether the namespace can be loaded with stated dependencies ... NOTE
Warning: no function found corresponding to methods exports from ‘RnBeads’ for: ‘samples’

A namespace must be able to be loaded with just the base namespace
loaded: otherwise if the namespace gets loaded by a saved object, the
session will be unable to start.

Probably some imports need to be declared in the NAMESPACE file.

checking R code for possible problems ... NOTE
add.combination.plot: warning in facet_grid(Set ~ ., scale = "free",
  drop = TRUE): partial argument match of 'scale' to 'scales'
.stopImplicitCluster: no visible global function definition for
  ‘stopCluster’
BMIQ: no visible global function definition for ‘density’
BMIQ: no visible global function definition for ‘pbeta’
BMIQ: no visible global function definition for ‘qbeta’
MethyLumiSet2RnBeadSet: no visible global function definition for
  ‘phenoData’
... 730 lines ...
             "cutree", "dbeta", "density", "dist", "ecdf", "fisher.test",
             "hclust", "knots", "kruskal.test", "lm", "model.matrix",
             "optim", "order.dendrogram", "p.adjust", "pbeta", "pchisq",
             "pf", "pnorm", "prcomp", "predict", "pt", "qbeta",
             "reorder", "residuals", "rnorm", "rt", "t.test", "vcov",
             "wilcox.test")
  importFrom("utils", "browseURL", "capture.output", "combn", "data",
             "download.file", "installed.packages", "memory.size",
             "packageVersion", "read.csv", "read.delim", "read.table",
             "untar", "unzip", "write.table", "zip")
to your NAMESPACE file.

checking Rd line widths ... NOTE
Rd file 'addDiffMethTable-RnBDiffMeth-methods.Rd':
  \examples lines wider than 100 characters:
     dm <- rnb.execute.computeDiffMeth(rnb.set.example,pheno.cols="Sample_Group",region.types=c("genes","tiling"))
     dmt.sites <- computeDiffTab.extended.site(meth(rnb.set.example),sample.groups[[1]],sample.groups[[2]])

Rd file 'is.valid-RnBDiffMeth-methods.Rd':
  \examples lines wider than 100 characters:
     dm1 <- rnb.execute.computeDiffMeth(rnb.set.example,pheno.cols=c("Sample_Group"),region.types=c("genes","tiling"))
     dm2 <- rnb.execute.computeDiffMeth(rnb.set.example,pheno.cols=c("Sample_Group","Treatment"),region.types=c("promoters"))
... 17 lines ...
     txt <- c("A pessimist is a person who has had to listen to too many optimists. ", "<i>Don Marquis</i>")

Rd file 'rnb.runs.Rd':
  \examples lines wider than 100 characters:
     result <- rnb.run.import(data.source = data.source, data.type = "idat.dir", dir.reports = report.dir)

Rd file 'save.tables-RnBDiffMeth-methods.Rd':
  \examples lines wider than 100 characters:
     dm <- rnb.execute.computeDiffMeth(rnb.set.example,pheno.cols=c("Sample_Group","Treatment"),disk.dump=TRUE,disk.dump.dir=tempfile())

These lines will be truncated in the PDF manual.
```

## samr (2.0)
Maintainer: Rob Tibshirani <tibs@stanford.edu>

0 errors | 0 warnings | 5 notes

```
checking DESCRIPTION meta-information ... NOTE
Malformed Description field: should contain one or more complete sentences.

checking dependencies in R code ... NOTE
'library' or 'require' calls to packages already attached by Depends:
  ‘impute’ ‘matrixStats’
  Please remove these calls from your code.
Packages in Depends field not imported from:
  ‘impute’ ‘matrixStats’
  These packages need to be imported from (in the NAMESPACE file)
  for when this namespace is loaded but not attached.

checking S3 generic/method consistency ... NOTE
Found the following apparent S3 methods exported but not registered:
  plot.SAMoutput print.SAMoutput
See section ‘Registering S3 methods’ in the ‘Writing R Extensions’
manual.

checking R code for possible problems ... NOTE
Found an obsolete/platform-specific call in the following function:
  ‘samr.assess.samplesize.plot’
Found the platform-specific device:
  ‘win.metafile’
dev.new() is the preferred way to open a new device, in the unlikely
event one is needed.
est.s0: no visible global function definition for ‘quantile’
est.s0: no visible global function definition for ‘mad’
est.s0: no visible global function definition for ‘var’
... 62 lines ...
Undefined global functions or variables:
  abline dev.off impute.knn lines mad median mtext par plot points
  predict quantile rowMedians rpois runif samr.xl.var.knn.neighbors
  smooth.spline title var
Consider adding
  importFrom("grDevices", "dev.off")
  importFrom("graphics", "abline", "lines", "mtext", "par", "plot",
             "points", "title")
  importFrom("stats", "mad", "median", "predict", "quantile", "rpois",
             "runif", "smooth.spline", "var")
to your NAMESPACE file.

checking Rd \usage sections ... NOTE
S3 methods shown with full name in documentation object 'samr-internal':
  ‘plot.SAMoutput’ ‘print.SAMoutput’

The \usage entries for S3 methods should use the \method markup and not
their full name.
See chapter ‘Writing R documentation files’ in the ‘Writing R
Extensions’ manual.
```

## scater (1.2.0)
Maintainer: Davis McCarthy <davis@ebi.ac.uk>  
Bug reports: https://github.com/davismcc/scater/issues

0 errors | 0 warnings | 0 notes

## SemiParBIVProbit (3.8-1)
Maintainer: Giampiero Marra <giampiero.marra@ucl.ac.uk>

0 errors | 0 warnings | 0 notes

## SemiParSampleSel (1.4)
Maintainer: Giampiero Marra <giampiero.marra@ucl.ac.uk>

0 errors | 0 warnings | 0 notes

## SGP (1.6-0.0)
Maintainer: Damian W. Betebenner <dbetebenner@nciea.org>  
Bug reports: https://github.com/CenterForAssessment/SGP/issues

0 errors | 0 warnings | 0 notes

## shinyMethyl (1.10.0)
Maintainer: Jean-Philippe Fortin <jfortin@jhsph.edu>

0 errors | 0 warnings | 2 notes

```
checking for hidden files and directories ... NOTE
Found the following hidden files and directories:
  .travis.yml
These were most likely included in error. See section ‘Package
structure’ in the ‘Writing R Extensions’ manual.

checking R code for possible problems ... NOTE
addHoverDensity: no visible global function definition for ‘lines’
addHoverPoints: no visible global function definition for ‘points’
addHoverQC: no visible global function definition for ‘points’
densitiesPlot: no visible global function definition for ‘plot’
densitiesPlot: no visible global function definition for ‘lines’
plotDesign450k: no visible global function definition for ‘par’
plotDesign450k: no visible global function definition for ‘plot’
plotDiscrepancyGenders: no visible global function definition for
  ‘legend’
... 60 lines ...
Undefined global functions or variables:
  abline complete.cases current.control.type current.density.type
  current.probe.type genderCutoff grid legend lines lm
  mouse.click.indices palette par plot points prcomp rect write.csv
Consider adding
  importFrom("grDevices", "palette")
  importFrom("graphics", "abline", "grid", "legend", "lines", "par",
             "plot", "points", "rect")
  importFrom("stats", "complete.cases", "lm", "prcomp")
  importFrom("utils", "write.csv")
to your NAMESPACE file.
```

## SICtools (1.4.0)
Maintainer: Xiaobin Xing <xiaobinxing0316@gmail.com>

0 errors | 1 warning  | 4 notes

```
checking whether package ‘SICtools’ can be installed ... WARNING
Found the following significant warnings:
  Warning: replacing previous import ‘plyr::count’ by ‘matrixStats::count’ when loading ‘SICtools’
See ‘/home/hb/repositories/matrixStats/revdep/checks/SICtools.Rcheck/00install.out’ for details.

checking package dependencies ... NOTE
Depends: includes the non-default packages:
  ‘Rsamtools’ ‘doParallel’ ‘Biostrings’ ‘stringr’ ‘matrixStats’ ‘plyr’
  ‘GenomicRanges’ ‘IRanges’
Adding so many packages to the search path is excessive and importing
selectively is preferable.

checking for hidden files and directories ... NOTE
Found the following hidden files and directories:
  .BBSoptions
These were most likely included in error. See section ‘Package
structure’ in the ‘Writing R Extensions’ manual.

checking R code for possible problems ... NOTE
.indelDiffFunc: no visible global function definition for ‘fisher.test’
.indelDiffFunc: no visible global function definition for ‘dist’
indelDiff: no visible global function definition for ‘read.delim’
snpDiff : calcInfoRange : <anonymous>: no visible global function
  definition for ‘fisher.test’
snpDiff : calcInfoRange : <anonymous>: no visible global function
  definition for ‘dist’
Undefined global functions or variables:
  dist fisher.test read.delim
Consider adding
  importFrom("stats", "dist", "fisher.test")
  importFrom("utils", "read.delim")
to your NAMESPACE file.

checking Rd line widths ... NOTE
Rd file 'indelDiff.Rd':
  \usage lines wider than 90 characters:
     indelDiff(bam1, bam2, refFsa, regChr, regStart, regEnd, minBaseQuality = 13, minMapQuality = 0, nCores = 1, pValueCutOff= 0.05,gtDistCu ... [TRUNCATED]

Rd file 'snpDiff.Rd':
  \usage lines wider than 90 characters:
     snpDiff(bam1, bam2, refFsa, regChr, regStart, regEnd, minBaseQuality = 13, minMapQuality = 0, nCores = 1, pValueCutOff = 0.05, baseDist ... [TRUNCATED]

These lines will be truncated in the PDF manual.
```

## sizeMat (0.3.0)
Maintainer: Josymar Torrejon-Magallanes <ejosymart@gmail.com>

0 errors | 0 warnings | 0 notes

## SRGnet (1.0.0)
Maintainer: Isar Nassiri <isar_nassiri@urmc.rochester.edu>

0 errors | 0 warnings | 2 notes

```
checking package dependencies ... NOTE
Depends: includes the non-default packages:
  ‘EBcoexpress’ ‘MASS’ ‘igraph’ ‘pvclust’ ‘RedeR’ ‘gRain’ ‘gbm’ ‘limma’
  ‘DMwR’ ‘matrixStats’
Adding so many packages to the search path is excessive and importing
selectively is preferable.

checking R code for possible problems ... NOTE
SRG: no visible binding for global variable ‘Transcriptomics’
SRG: no visible global function definition for ‘model.matrix’
SRG: no visible global function definition for ‘write.table’
SRGnet: no visible binding for global variable
  ‘Differentially_expressed_genes’
SRGnet: no visible binding for global variable ‘Transcriptomics’
SRGnet: no visible binding for global variable ‘PLCRG’
SRGnet: no visible global function definition for ‘ebPatterns’
SRGnet: no visible global function definition for ‘type.convert’
... 10 lines ...
Undefined global functions or variables:
  Differentially_expressed_genes PLCRG Transcriptomics boxplot
  ebPatterns model.matrix na.omit par plot predict qt t.test
  terrain.colors type.convert write.table
Consider adding
  importFrom("grDevices", "terrain.colors")
  importFrom("graphics", "boxplot", "par", "plot")
  importFrom("stats", "model.matrix", "na.omit", "predict", "qt",
             "t.test")
  importFrom("utils", "type.convert", "write.table")
to your NAMESPACE file.
```

## statar (0.6.3)
Maintainer: Matthieu Gomez <mattg@princeton.edu>  
Bug reports: https://github.com/matthieugomez/statar/issues

0 errors | 0 warnings | 0 notes

## stm (1.2.2)
Maintainer: Brandon Stewart <bms4@princeton.edu>  
Bug reports: https://github.com/bstewart/stm/issues

0 errors | 0 warnings | 0 notes

## StructFDR (1.1)
Maintainer: Jun Chen <chen.jun2@mayo.edu>

0 errors | 0 warnings | 0 notes

## summarytools (0.6.5)
Maintainer: Dominic Comtois <dominic.comtois@gmail.com>  
Bug reports: https://github.com/dcomtois/summarytools/issues

0 errors | 0 warnings | 0 notes

## tmlenet (0.1.0)
Maintainer: Oleg Sofrygin <oleg.sofrygin@gmail.com>  
Bug reports: https://github.com/osofr/tmlenet/issues

0 errors | 0 warnings | 0 notes

## topGO (2.26.0)
Maintainer: Adrian Alexa <adrian.alexa@gmail.com>

0 errors | 0 warnings | 4 notes

```
checking package dependencies ... NOTE
Depends: includes the non-default packages:
  ‘BiocGenerics’ ‘graph’ ‘Biobase’ ‘GO.db’ ‘AnnotationDbi’ ‘SparseM’
Adding so many packages to the search path is excessive and importing
selectively is preferable.

checking dependencies in R code ... NOTE
'library' or 'require' calls in package code:
  ‘Rgraphviz’ ‘multtest’
  Please use :: or requireNamespace() instead.
  See section 'Suggested packages' in the 'Writing R Extensions' manual.
Missing object imported by a ':::' call: ‘globaltest:::globaltest’

checking R code for possible problems ... NOTE
GOplot: no visible global function definition for ‘getDefaultAttrs’
GOplot: no visible global function definition for ‘agopen’
GOplot.counts : plotSigChart : buildDrawing : <anonymous>: no visible
  global function definition for ‘getNodeCenter’
GOplot.counts : plotSigChart : buildDrawing : <anonymous>: no visible
  global function definition for ‘pieGlyph’
GOplot.counts : plotSigChart : buildDrawing : <anonymous>: no visible
  global function definition for ‘getX’
GOplot.counts : plotSigChart : buildDrawing : <anonymous>: no visible
... 35 lines ...
  getDefaultAttrs getNodeCenter getNodeLW getNodeXY getX getY
  mt.rawp2adjp mt.teststat name pieGlyph scoreInNode toDot txtLabel

Found the following assignments to the global environment:
File ‘topGO/R/topGOtests.R’:
  assign(".PERMSUM.LOOKUP", sizeLookUp, .GlobalEnv)
  assign(".PERMSUM.MAT", sapply(1:N, function(x) cumsum(sample(scoreVec, 
    maxSample))[goSize]), .GlobalEnv)
  assign(".PERMSUM.LOOKUP", sizeLookUp, .GlobalEnv)
  assign(".PERMSUM.MAT", sapply(1:N, function(x) cumsum(sample(scoreVec))[goSize]), 
    .GlobalEnv)

checking Rd line widths ... NOTE
Rd file 'diagnosticMethods.Rd':
  \examples lines wider than 100 characters:
     sig.tab <- GenTable(GOdata, Fis = resultFisher, KS = resultKS, orderBy = "KS", ranksOf = "Fis", topNodes = 20)

Rd file 'getPvalues.Rd':
  \usage lines wider than 90 characters:
        getPvalues(edata, classlabel, test = "t", alternative = c("greater", "two.sided", "less")[1],
        genesID = NULL, correction = c("none", "Bonferroni", "Holm", "Hochberg", "SidakSS", "SidakSD",

Rd file 'printGraph-methods.Rd':
  \usage lines wider than 90 characters:
                    useFullNames = TRUE, oldSigNodes = NULL, useInfo = c("none", "pval", "counts", "def", "np", "all")[1],
  \examples lines wider than 100 characters:
     printGraph(GOdata, resultFisher, firstSigNodes = 5, fn.prefix = "sampleFile", useInfo = "all", pdfSW = TRUE)

Rd file 'topGOdata-class.Rd':
  \examples lines wider than 100 characters:
                   description = "GO analysis of ALL data: Differential Expression between B-cell and T-cell",

These lines will be truncated in the PDF manual.
```

## ttScreening (1.5)
Maintainer: Meredith Ray <mere2110@yahoo.com>

0 errors | 0 warnings | 2 notes

```
checking DESCRIPTION meta-information ... NOTE
Malformed Title field: should not end in a period.

checking R code for possible problems ... NOTE
ttScreening: no visible global function definition for ‘gaussian’
ttScreening: no visible global function definition for ‘glm.control’
ttScreening: no visible global function definition for ‘model.response’
ttScreening: no visible global function definition for ‘is.empty.model’
ttScreening: no visible global function definition for ‘model.matrix’
ttScreening: no visible global function definition for ‘na.omit’
ttScreening: no visible global function definition for ‘terms’
ttScreening: no visible global function definition for ‘capture.output’
ttScreening: no visible global function definition for ‘glm’
ttScreening: no visible binding for global variable ‘gaussian’
ttScreening: no visible global function definition for ‘pt’
ttScreening: no visible global function definition for ‘p.adjust’
Undefined global functions or variables:
  capture.output gaussian glm glm.control is.empty.model model.matrix
  model.response na.omit p.adjust pt terms
Consider adding
  importFrom("stats", "gaussian", "glm", "glm.control", "is.empty.model",
             "model.matrix", "model.response", "na.omit", "p.adjust",
             "pt", "terms")
  importFrom("utils", "capture.output")
to your NAMESPACE file.
```

## VanillaICE (1.36.0)
Maintainer: Robert Scharpf <rscharpf@jhsph.edu>

0 errors | 0 warnings | 3 notes

```
checking package dependencies ... NOTE
Packages which this enhances but not available for checking:
  ‘doMPI’ ‘doRedis’

checking top-level files ... NOTE
Non-standard file/directory found at top level:
  ‘changelog’

checking dependencies in R code ... NOTE
Unexported object imported by a ':::' call: ‘SummarizedExperiment:::.ShallowSimpleListAssays0’
  See the note in ?`:::` about the use of this operator.
```

## visualFields (0.4.3)
Maintainer: Ivan Marin-Franch <ivan.marin@uv.es>

0 errors | 0 warnings | 0 notes

## wateRmelon (1.18.0)
Maintainer: Leo <lschal@essex.ac.uk>

0 errors | 0 warnings | 6 notes

```
checking package dependencies ... NOTE
Depends: includes the non-default packages:
  ‘Biobase’ ‘limma’ ‘matrixStats’ ‘methylumi’ ‘lumi’ ‘ROC’
  ‘IlluminaHumanMethylation450kanno.ilmn12.hg19’ ‘illuminaio’
Adding so many packages to the search path is excessive and importing
selectively is preferable.

checking installed package size ... NOTE
  installed size is 14.6Mb
  sub-directories of 1Mb or more:
    data  14.0Mb

checking DESCRIPTION meta-information ... NOTE
Package listed in more than one of Depends, Imports, Suggests, Enhances:
  ‘Biobase’
A package should be listed in only one of these fields.

checking dependencies in R code ... NOTE
Packages in Depends field not imported from:
  ‘IlluminaHumanMethylation450kanno.ilmn12.hg19’ ‘ROC’ ‘illuminaio’
  ‘limma’ ‘lumi’ ‘matrixStats’ ‘methods’ ‘methylumi’
  These packages need to be imported from (in the NAMESPACE file)
  for when this namespace is loaded but not attached.
Missing object imported by a ':::' call: ‘minfi:::.isRG’

checking R code for possible problems ... NOTE
DataToNChannelSet2: no visible global function definition for ‘is’
DataToNChannelSet2: no visible global function definition for ‘new’
IDATsToMatrices2: no visible global function definition for ‘.mclapply’
IDATtoMatrix2 : <anonymous>: no visible global function definition for
  ‘readIDAT’
NChannelSetToMethyLumiSet2: no visible global function definition for
  ‘new’
NChannelSetToMethyLumiSet2: no visible global function definition for
  ‘betas’
... 368 lines ...
Found the following calls to data() loading into the global environment:
File ‘wateRmelon/R/horv.R’:
  data(coef)
File ‘wateRmelon/R/readEPIC.R’:
  data(hm27.ordering)
  data(hm450.ordering)
  data(epic.ordering)
  data(hm27.controls)
  data(hm450.controls)
  data(epic.controls)
See section ‘Good practice’ in ‘?data’.

checking Rd line widths ... NOTE
Rd file 'BMIQ.Rd':
  \usage lines wider than 90 characters:
     BMIQ(beta.v, design.v, nL = 3, doH = TRUE, nfit = 50000, th1.v = c(0.2, 0.75), th2.v = NULL, niter = 5, tol = 0.001, plots = TRUE, samp ... [TRUNCATED]
     BMIQ(beta.v, nL=3, doH=TRUE, nfit=5000, th1.v=c(0.2,0.75), th2.v=NULL, niter=5, tol=0.001, plots=FALSE,  pri=FALSE ) 

Rd file 'wm_internal.Rd':
  \usage lines wider than 90 characters:
     DataToNChannelSet2(mats, chans = c(Cy3 = "GRN", Cy5 = "RED"), parallel = F, protocol.data = F, IDAT = TRUE, force=F)

These lines will be truncated in the PDF manual.
```

## WGCNA (1.51)
Maintainer: Peter Langfelder <Peter.Langfelder@gmail.com>

0 errors | 0 warnings | 0 notes

## XBSeq (1.4.0)
Maintainer: Yuanhang Liu <liuy12@uthscsa.edu>

0 errors | 0 warnings | 3 notes

```
checking whether the namespace can be loaded with stated dependencies ... NOTE
Warning: no function found corresponding to methods exports from ‘XBSeq’ for: ‘conditions’, ‘conditions<-’, ‘dispTable’

A namespace must be able to be loaded with just the base namespace
loaded: otherwise if the namespace gets loaded by a saved object, the
session will be unable to start.

Probably some imports need to be declared in the NAMESPACE file.

checking R code for possible problems ... NOTE
Loglikhood : <anonymous>: no visible global function definition for
  ‘dpois’
MAplot: no visible binding for global variable ‘baseMean’
MAplot: no visible global function definition for ‘quantile’
MAplot: no visible binding for global variable ‘log2FoldChange’
XBSeqDataSet: no visible global function definition for ‘DataFrame’
XBSeqDataSet: no visible global function definition for
  ‘SummarizedExperiment’
XBSeqDataSet: no visible global function definition for ‘formula’
... 45 lines ...
  importFrom("stats", "Gamma", "coefficients", "complete.cases", "cor",
             "dnbinom", "dpois", "formula", "glm", "median", "optim",
             "p.adjust", "pbeta", "predict", "qbeta", "quantile",
             "rnbinom")
  importFrom("utils", "data")
to your NAMESPACE file.

Found the following calls to data() loading into the global environment:
File ‘XBSeq/R/methods.R’:
  data("scvBiasCorrectionFits")
See section ‘Good practice’ in ‘?data’.

checking Rd line widths ... NOTE
Rd file 'XBSeq.Rd':
  \usage lines wider than 90 characters:
        sharingMode = "maximum", fitType = "local", pvals_only = FALSE, paraMethod='NP', big_count = 900)

Rd file 'XBplot.Rd':
  \usage lines wider than 90 characters:
     XBplot(XB, Samplenum = NULL, unit = c('counts', 'LogTPM'), Libsize = NULL, Genelength = NULL, xlab = 'log2 TPM', ylab = 'Frequencies',  ... [TRUNCATED]

Rd file 'estimateSCV.Rd':
  \usage lines wider than 90 characters:
     estimateSCV( object, method = c( "pooled", "per-condition", "blind" ), sharingMode = c( "maximum", "fit-only", "gene-est-only" ),

These lines will be truncated in the PDF manual.
```

## yarn (1.0.1)
Maintainer: Joseph N Paulson <jpaulson@jimmy.harvard.edu>

0 errors | 0 warnings | 0 notes

