# Setup

## Platform

|setting  |value                        |
|:--------|:----------------------------|
|version  |R version 3.4.1 (2017-06-30) |
|system   |x86_64, linux-gnu            |
|ui       |X11                          |
|language |en                           |
|collate  |en_US.UTF-8                  |
|tz       |America/Los_Angeles          |
|date     |2017-08-31                   |

## Packages

|package        |*  |version |date       |source          |
|:--------------|:--|:-------|:----------|:---------------|
|base64enc      |   |0.1-3   |2015-07-28 |cran (@0.1-3)   |
|ggplot2        |   |2.2.1   |2016-12-30 |cran (@2.2.1)   |
|knitr          |   |1.17    |2017-08-10 |cran (@1.17)    |
|matrixStats    |   |0.52.2  |2017-04-14 |cran (@0.52.2)  |
|microbenchmark |   |1.4-2.1 |2015-11-25 |cran (@1.4-2.1) |
|R.devices      |   |2.15.1  |2016-11-10 |cran (@2.15.1)  |
|R.rsp          |   |0.41.0  |2017-04-16 |cran (@0.41.0)  |

# Check results

142 packages

|package                |version | errors| warnings| notes|
|:----------------------|:-------|------:|--------:|-----:|
|abcrf                  |1.6     |      0|        0|     1|
|ACNE                   |0.8.1   |      0|        0|     0|
|anomalyDetection       |0.1.2   |      1|        0|     0|
|antiProfiles           |1.17.0  |      0|        0|     1|
|aroma.affymetrix       |3.1.0   |      0|        0|     0|
|aroma.cn               |1.6.1   |      0|        0|     0|
|aroma.core             |3.1.0   |      0|        0|     1|
|aroma.light            |3.7.0   |      0|        0|     2|
|aSPU                   |1.48    |      0|        0|     0|
|BatchQC                |1.5.0   |      0|        1|     1|
|BayesTwin              |1.0     |      0|        0|     0|
|bdynsys                |1.3     |      0|        0|     1|
|bigstep                |0.7.4   |      0|        0|     0|
|bingat                 |1.3     |      0|        0|     1|
|bmrm                   |3.4     |      0|        0|     0|
|brms                   |1.9.0   |      0|        0|     0|
|bsseq                  |1.13.6  |      0|        0|     0|
|bumphunter             |1.17.2  |      0|        0|     5|
|calmate                |0.12.1  |      0|        0|     0|
|carx                   |0.6.2   |      0|        0|     0|
|CATALYST               |1.1.5   |      0|        0|     1|
|cellWise               |1.0.0   |      0|        0|     0|
|CEMiTool               |0.99.9  |      0|        0|     2|
|ChAMP                  |2.9.9   |      0|        1|     5|
|Chicago                |1.5.0   |      1|        1|     5|
|ChIPpeakAnno           |3.11.4  |      0|        0|     2|
|Clomial                |1.13.0  |      0|        0|     1|
|clusterExperiment      |1.3.2   |      2|        1|     2|
|CNPBayes               |1.7.1   |      0|        0|     4|
|cointmonitoR           |0.1.0   |      0|        0|     0|
|cointReg               |0.2.0   |      0|        0|     0|
|consensusOV            |0.99.3  |      0|        1|     1|
|CopywriteR             |2.9.0   |      0|        0|     1|
|cosinor2               |0.1.0   |      0|        0|     0|
|CpGFilter              |1.1     |      0|        0|     0|
|crlmm                  |1.35.0  |      0|        0|     6|
|crossmeta              |1.3.0   |      0|        0|     3|
|DAMOCLES               |1.1     |      0|        0|     1|
|DatabionicSwarm        |0.9.7   |      0|        0|     0|
|DeepBlueR              |1.3.10  |      0|        0|     1|
|DelayedArray           |0.3.19  |      1|        0|     0|
|DGCA                   |1.0.1   |      0|        0|     0|
|diffloop               |1.5.0   |      0|        0|     1|
|DisHet                 |0.1.0   |      0|        0|     0|
|dplR                   |1.6.6   |      0|        0|     0|
|DPWeibull              |1.0     |      0|        0|     0|
|dtree                  |0.2.3   |      0|        0|     0|
|EasyqpcR               |1.19.0  |      1|        0|     1|
|EMDomics               |2.7.0   |      0|        0|     3|
|EnrichedHeatmap        |1.7.0   |      0|        0|     4|
|EventPointer           |1.1.0   |      0|        0|     3|
|expss                  |0.8.1   |      0|        0|     0|
|FADA                   |1.3.2   |      0|        0|     0|
|FastHCS                |0.0.5   |      0|        0|     3|
|FastPCS                |0.1.2   |      0|        0|     4|
|FastRCS                |0.0.7   |      0|        0|     2|
|fergm                  |0.2.0   |      0|        0|     0|
|filesstrings           |1.1.0   |      0|        0|     0|
|flowCore               |1.43.5  |      0|        2|    10|
|fslr                   |2.12    |      0|        0|     0|
|funtooNorm             |1.1.0   |      0|        1|     0|
|GAD                    |1.1.1   |      0|        0|     0|
|gcapc                  |1.1.9   |      0|        0|     0|
|GeneralizedUmatrix     |0.9.5   |      0|        0|     0|
|genomation             |1.9.3   |      0|        0|     4|
|GenRank                |1.5.0   |      0|        0|     1|
|GJRM                   |0.1-1   |      0|        0|     0|
|GPrank                 |0.1.2   |      0|        0|     0|
|GUIDEseq               |1.7.3   |      1|        1|     3|
|Gviz                   |1.21.2  |      0|        2|     4|
|haploReconstruct       |0.1.2   |      0|        0|     1|
|IMIFA                  |1.3.1   |      0|        0|     0|
|InfiniumPurify         |1.3.1   |      0|        0|     0|
|kernDeepStackNet       |2.0.2   |      0|        0|     1|
|kgschart               |1.3.5   |      0|        0|     0|
|kpmt                   |0.1.0   |      0|        0|     0|
|localgauss             |0.40    |      0|        0|     1|
|loo                    |1.1.0   |      0|        0|     0|
|LS2Wstat               |2.0-3   |      0|        1|     2|
|ltmle                  |0.9-10  |      0|        0|     0|
|Luminescence           |0.7.5   |      0|        0|     0|
|metagene               |2.9.1   |      0|        0|     1|
|metagenomeSeq          |1.19.0  |      0|        1|     0|
|MetaQC                 |0.1.13  |      0|        0|     3|
|metavizr               |1.1.2   |      1|        0|     0|
|MethylAid              |1.11.1  |      0|        0|     1|
|methylumi              |2.23.0  |      0|        2|     9|
|MFHD                   |0.0.1   |      0|        0|     1|
|MIGSA                  |1.1.1   |      1|        1|     1|
|minfi                  |1.23.3  |      0|        0|     2|
|MinimumDistance        |1.21.2  |      0|        0|     1|
|mmtfa                  |0.1     |      0|        0|     1|
|monocle                |2.5.4   |      0|        5|     3|
|motifbreakR            |1.7.0   |      0|        0|     2|
|MPAgenomics            |1.1.2   |      0|        0|     2|
|mrfDepth               |1.0.4   |      0|        0|     1|
|MultiBD                |0.2.0   |      0|        0|     2|
|nandb                  |0.2.1   |      0|        0|     0|
|NanoStringDiff         |1.7.0   |      0|        0|     1|
|neurobase              |1.13.2  |      0|        0|     0|
|NSA                    |0.0.32  |      0|        0|     6|
|pandaR                 |1.9.0   |      0|        0|     4|
|PathoStat              |1.3.0   |      0|        1|     0|
|pcaMethods             |1.69.0  |      0|        0|     3|
|peakPick               |0.11    |      0|        0|     0|
|phosphonormalizer      |1.1.7   |      0|        0|     1|
|Pigengene              |1.3.6   |      0|        0|     2|
|POMaSPU                |1.0.0   |      0|        0|     0|
|PSCBS                  |0.63.0  |      0|        0|     0|
|QDNAseq                |1.13.0  |      0|        1|     0|
|r2dRue                 |1.0.4   |      0|        0|     2|
|RAC                    |1.1.1   |      0|        0|     0|
|randomizationInference |1.0.3   |      0|        0|     1|
|RnBeads                |1.9.0   |      1|        3|     5|
|RTransProb             |0.1.0   |      1|        0|     0|
|samr                   |2.0     |      0|        0|     6|
|scater                 |1.5.0   |      0|        0|     3|
|scmap                  |0.99.2  |      0|        0|     1|
|scone                  |1.1.2   |      2|        0|     4|
|SemiParSampleSel       |1.5     |      0|        0|     0|
|SGP                    |1.7-0.0 |      0|        0|     0|
|shinyMethyl            |1.13.0  |      0|        0|     2|
|SICtools               |1.7.0   |      0|        1|     4|
|sizeMat                |0.3.0   |      0|        0|     0|
|splatter               |1.1.4   |      1|        0|     1|
|SRGnet                 |1.3.0   |      0|        0|     2|
|statar                 |0.6.5   |      0|        0|     0|
|stm                    |1.2.2   |      0|        0|     0|
|STROMA4                |1.1.0   |      0|        0|     1|
|StructFDR              |1.2     |      1|        0|     0|
|summarytools           |0.6.5   |      0|        0|     0|
|sva                    |3.25.4  |      0|        0|     1|
|tmlenet                |0.1.0   |      0|        0|     1|
|topGO                  |2.29.0  |      0|        0|     4|
|ttScreening            |1.5     |      0|        0|     2|
|VanillaICE             |1.39.2  |      0|        0|     4|
|visualFields           |0.4.3   |      0|        0|     0|
|wateRmelon             |1.21.2  |      0|        0|     5|
|WGCNA                  |1.61    |      0|        0|     0|
|XBSeq                  |1.7.0   |      1|        0|     3|
|yarn                   |1.3.0   |      0|        0|     0|
|zinbwave               |0.99.6  |      0|        0|     0|

## abcrf (1.6)
Maintainer: Jean-Michel Marin <jean-michel.marin@umontpellier.fr>

0 errors | 0 warnings | 1 note 

```
checking installed package size ... NOTE
  installed size is  5.2Mb
  sub-directories of 1Mb or more:
    data   2.6Mb
    libs   2.4Mb
```

## ACNE (0.8.1)
Maintainer: Henrik Bengtsson <henrikb@braju.com>  
Bug reports: https://github.com/HenrikBengtsson/ACNE/issues

0 errors | 0 warnings | 0 notes

## anomalyDetection (0.1.2)
Maintainer: Bradley Boehmke <bradleyboehmke@gmail.com>  
Bug reports: https://github.com/AFIT-R/anomalyDetection/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  6: doTryCatch(return(expr), name, parentenv, handler)
  7: bd_row(m1, 1:5)
  
  var1 & var2 (r = -0.753)
  var3 & var4 (r = 0.576)
  var4 & var6 (r = -0.539)
  var4 & var7 (r = -0.507)
  var2 & var9 (r = -0.513)
  
  testthat results ================================================================
  OK: 120 SKIPPED: 0 FAILED: 1
  1. Error: bd_row provides proper messages and warnings (@test_bd_row.R#12) 
  
  Error: testthat unit tests failed
  Execution halted
```

## antiProfiles (1.17.0)
Maintainer: Hector Corrada Bravo <hcorrada@gmail.com>

0 errors | 0 warnings | 1 note 

```
checking R code for possible problems ... NOTE
ev: no visible global function definition for ‘predict’
ev: no visible global function definition for ‘smoothScatter’
ev : f1: no visible global function definition for ‘predict’
ev: no visible global function definition for ‘curve’
Undefined global functions or variables:
  curve predict smoothScatter
Consider adding
  importFrom("graphics", "curve", "smoothScatter")
  importFrom("stats", "predict")
to your NAMESPACE file.
```

## aroma.affymetrix (3.1.0)
Maintainer: Henrik Bengtsson <henrikb@braju.com>  
Bug reports: https://github.com/HenrikBengtsson/aroma.affymetrix/issues

0 errors | 0 warnings | 0 notes

## aroma.cn (1.6.1)
Maintainer: Henrik Bengtsson <henrikb@braju.com>  
Bug reports: https://github.com/HenrikBengtsson/aroma.cn/issues

0 errors | 0 warnings | 0 notes

## aroma.core (3.1.0)
Maintainer: Henrik Bengtsson <henrikb@braju.com>  
Bug reports: https://github.com/HenrikBengtsson/aroma.core/issues

0 errors | 0 warnings | 1 note 

```
checking package dependencies ... NOTE
Packages suggested but not available for checking:
  ‘expectile’ ‘HaarSeg’ ‘mpcbs’
```

## aroma.light (3.7.0)
Maintainer: Henrik Bengtsson <henrikb@braju.com>  
Bug reports: https://github.com/HenrikBengtsson/aroma.light/issues

0 errors | 0 warnings | 2 notes

```
checking for hidden files and directories ... NOTE
Found the following hidden files and directories:
  inst/rsp/.rspPlugins
These were most likely included in error. See section ‘Package
structure’ in the ‘Writing R Extensions’ manual.

checking top-level files ... NOTE
Non-standard file/directory found at top level:
  ‘revdep’
```

## aSPU (1.48)
Maintainer: Il-Youp Kwak <ikwak@umn.edu>

0 errors | 0 warnings | 0 notes

## BatchQC (1.5.0)
Maintainer: Solaiappan Manimaran <manimaran_1975@hotmail.com>  
Bug reports: https://github.com/mani2012/BatchQC/issues

0 errors | 1 warning  | 1 note 

```
checking for missing documentation entries ... WARNING
Undocumented code objects:
  ‘lmFitC’
All user-level objects in a package should have documentation entries.
See chapter ‘Writing R documentation files’ in the ‘Writing R
Extensions’ manual.

checking R code for possible problems ... NOTE
gls.series.C: no visible global function definition for ‘lm.fit’
Undefined global functions or variables:
  lm.fit
Consider adding
  importFrom("stats", "lm.fit")
to your NAMESPACE file.
```

## BayesTwin (1.0)
Maintainer: Inga Schwabe <bayestwin@gmail.com>  
Bug reports: https://github.com/ingaschwabe/BayesTwin

0 errors | 0 warnings | 0 notes

## bdynsys (1.3)
Maintainer: Viktoria Spaiser <viktoria.sp@web.de>

0 errors | 0 warnings | 1 note 

```
checking R code for possible problems ... NOTE
bayfacfig: no visible global function definition for ‘dev.set’
bayfacfig: no visible global function definition for ‘postscript’
bayfacfig: no visible global function definition for ‘points’
bayfacfig: no visible global function definition for ‘legend’
bayfacfig: no visible global function definition for ‘dev.off’
dysymod: no visible global function definition for ‘write.table’
errorcorr: no visible global function definition for ‘sd’
phaseportdat: no visible global function definition for ‘dev.set’
phaseportdat: no visible global function definition for ‘postscript’
... 24 lines ...
polyfitreg: no visible binding for global variable ‘na.exclude’
polyfitreg: no visible global function definition for ‘var’
Undefined global functions or variables:
  dev.off dev.set grid legend lm matplot na.exclude plot points
  postscript runif sd var write.table
Consider adding
  importFrom("grDevices", "dev.off", "dev.set", "postscript")
  importFrom("graphics", "grid", "legend", "matplot", "plot", "points")
  importFrom("stats", "lm", "na.exclude", "runif", "sd", "var")
  importFrom("utils", "write.table")
to your NAMESPACE file.
```

## bigstep (0.7.4)
Maintainer: Piotr Szulc <piotr.michal.szulc@gmail.com>

0 errors | 0 warnings | 0 notes

## bingat (1.3)
Maintainer: Berkley Shands <rpackages@biorankings.com>

0 errors | 0 warnings | 1 note 

```
checking Rd cross-references ... NOTE
Package unavailable to check Rd xrefs: ‘genalg’
```

## bmrm (3.4)
Maintainer: Julien Prados <julien.prados@unige.ch>

0 errors | 0 warnings | 0 notes

## brms (1.9.0)
Maintainer: Paul-Christian Bürkner <paul.buerkner@gmail.com>  
Bug reports: https://github.com/paul-buerkner/brms/issues

0 errors | 0 warnings | 0 notes

## bsseq (1.13.6)
Maintainer: Kasper Daniel Hansen <kasperdanielhansen@gmail.com>  
Bug reports: https://github.com/kasperdanielhansen/bsseq/issues

0 errors | 0 warnings | 0 notes

## bumphunter (1.17.2)
Maintainer: Rafael A. Irizarry <rafa@jimmy.harvard.edu>

0 errors | 0 warnings | 5 notes

```
checking package dependencies ... NOTE
Depends: includes the non-default packages:
  ‘S4Vectors’ ‘IRanges’ ‘GenomeInfoDb’ ‘GenomicRanges’ ‘foreach’
  ‘iterators’ ‘parallel’ ‘locfit’
Adding so many packages to the search path is excessive and importing
selectively is preferable.

checking DESCRIPTION meta-information ... NOTE
Malformed Description field: should contain one or more complete sentences.

checking dependencies in R code ... NOTE
Unexported object imported by a ':::' call: ‘doParallel:::.options’
  See the note in ?`:::` about the use of this operator.

checking R code for possible problems ... NOTE
bumphunterEngine: no visible binding for global variable ‘bootstraps’
Undefined global functions or variables:
  bootstraps

checking Rd line widths ... NOTE
Rd file 'annotateTranscripts.Rd':
  \usage lines wider than 90 characters:
     annotateTranscripts(txdb, annotationPackage = NULL, by = c("tx","gene"), codingOnly=FALSE, verbose = TRUE, requireAnnotation = FALSE, m ... [TRUNCATED]

Rd file 'bumphunter.Rd':
  \usage lines wider than 90 characters:
     bumphunter(object, design, chr=NULL, pos, cluster=NULL,coef=2, cutoff=NULL, pickCutoff = FALSE, pickCutoffQ = 0.99, maxGap=500, nullMet ... [TRUNCATED]
     bumphunterEngine(mat, design, chr = NULL, pos, cluster = NULL, coef = 2, cutoff = NULL, pickCutoff = FALSE, pickCutoffQ = 0.99, maxGap  ... [TRUNCATED]

Rd file 'matchGenes.Rd':
  \usage lines wider than 90 characters:
     matchGenes(x, subject, type = c("any", "fiveprime"), promoterDist = 2500, skipExons = FALSE, verbose = TRUE)
  \examples lines wider than 100 characters:
         islands=makeGRangesFromDataFrame(read.delim("http://rafalab.jhsph.edu/CGI/model-based-cpg-islands-hg19.txt")[1:100,])

These lines will be truncated in the PDF manual.
```

## calmate (0.12.1)
Maintainer: Henrik Bengtsson <henrikb@braju.com>  
Bug reports: https://github.com/HenrikBengtsson/calmate/issues

0 errors | 0 warnings | 0 notes

## carx (0.6.2)
Maintainer: Chao Wang <chao-wang@uiowa.edu>

0 errors | 0 warnings | 0 notes

## CATALYST (1.1.5)
Maintainer: Helena Lucia Crowell <crowellh@student.ethz.ch>

0 errors | 0 warnings | 1 note 

```
checking installed package size ... NOTE
  installed size is 11.8Mb
  sub-directories of 1Mb or more:
    data   2.3Mb
    doc    9.2Mb
```

## cellWise (1.0.0)
Maintainer: Jakob Raymaekers <jakob.raymaekers@kuleuven.be>

0 errors | 0 warnings | 0 notes

## CEMiTool (0.99.9)
Maintainer: Helder Nakaya <hnakaya@usp.br>

0 errors | 0 warnings | 2 notes

```
checking installed package size ... NOTE
  installed size is  8.6Mb
  sub-directories of 1Mb or more:
    data      3.1Mb
    doc       2.3Mb
    extdata   2.8Mb

checking R code for possible problems ... NOTE
plot_interaction: no visible binding for global variable ‘cem’
read_gmt: no visible binding for global variable ‘stack’
filter_expr,CEMiTool: no visible global function definition for ‘var’
find_modules,CEMiTool: no visible global function definition for ‘tail’
find_modules,CEMiTool: no visible global function definition for ‘head’
find_modules,CEMiTool: no visible global function definition for
  ‘hclust’
find_modules,CEMiTool: no visible global function definition for
  ‘as.dist’
... 7 lines ...
save_plots,CEMiTool : <anonymous>: no visible global function
  definition for ‘dev.off’
save_plots,CEMiTool: no visible global function definition for
  ‘dev.off’
Undefined global functions or variables:
  as.dist cem dev.off dist hclust head stack tail var
Consider adding
  importFrom("grDevices", "dev.off")
  importFrom("stats", "as.dist", "dist", "hclust", "var")
  importFrom("utils", "head", "stack", "tail")
to your NAMESPACE file.
```

## ChAMP (2.9.9)
Maintainer: Yuan Tian  <champ450k@gmail.com>

0 errors | 1 warning  | 5 notes

```
checking package subdirectories ... WARNING
Problems with news in ‘inst/NEWS.Rd’:
  inst/NEWS.Rd:67: unexpected END_OF_INPUT '
  '

checking package dependencies ... NOTE
Depends: includes the non-default packages:
  ‘minfi’ ‘ChAMPdata’ ‘FEM’ ‘DMRcate’ ‘Illumina450ProbeVariants.db’
  ‘IlluminaHumanMethylationEPICmanifest’
Adding so many packages to the search path is excessive and importing
selectively is preferable.

checking installed package size ... NOTE
  installed size is 11.6Mb
  sub-directories of 1Mb or more:
    doc       9.5Mb
    extdata   1.9Mb

checking DESCRIPTION meta-information ... NOTE
Package listed in more than one of Depends, Imports, Suggests, Enhances:
  ‘rmarkdown’
A package should be listed in only one of these fields.

checking R code for possible problems ... NOTE
Found the following calls to data() loading into the global environment:
File ‘ChAMP/R/Block.GUI.R’:
  data(probe.features.epic)
  data(probe.features)
File ‘ChAMP/R/CpG.GUI.R’:
  data(probe.features.epic)
  data(probe.features)
File ‘ChAMP/R/DMP.GUI.R’:
  data(MatchGeneName)
... 46 lines ...
  data(probe.features.epic)
  data(probe.features)
File ‘ChAMP/R/champ.norm.R’:
  data(probeInfoALL.epic.lv)
  data(probeInfoALL.lv)
  data(probeInfoALL.epic.lv)
  data(probeInfoALL.lv)
File ‘ChAMP/R/champ.refbase.R’:
  data(CellTypeMeans27K)
  data(CellTypeMeans450K)
See section ‘Good practice’ in ‘?data’.

checking Rd line widths ... NOTE
Rd file 'champ.filter.Rd':
  \examples lines wider than 100 characters:
             myfilter <- champ.filter(beta=myImport$beta,pd=myImport$pd,detP=myImport$detP,beadcount=myImport$beadcount)

These lines will be truncated in the PDF manual.
```

## Chicago (1.5.0)
Maintainer: Mikhail Spivakov <spivakov@babraham.ac.uk>

1 error  | 1 warning  | 5 notes

```
checking examples ... ERROR
Running examples in ‘Chicago-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: exportResults
> ### Title: Export Results
> ### Aliases: exportResults exportToGI
> 
> ### ** Examples
... 30 lines ...
Preprocessing for WashU outputs...
Writing out text file for WashU browser upload...
> 
> GI <- exportToGI(cdUnitTest)
Reading the restriction map file...
Reading the bait map file...
Preparing the output table...
Error in loadNamespace(name) : 
  there is no package called ‘GenomicInteractions’
Calls: exportToGI ... tryCatch -> tryCatchList -> tryCatchOne -> <Anonymous>
Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...

Calculating p-values...
Approximating 3 very small p-values.

*** Running getScores...

Calculating p-value weights...
... 8 lines ...
Removing duplicated other-ends from significant interactions (same will happen with samples)...
Bin non-significant interactions according to distance from bait before drawing random samples...
Draw random samples...
Sum number of overlaps with feature in our significant interactions and in our samples...
Plotting barplot number of overlaps for features and samples...
Returning table with results...

Quitting from lines 277-280 (Chicago.Rmd) 
Error: processing vignette 'Chicago.Rmd' failed with diagnostics:
there is no package called 'GenomicInteractions'
Execution halted

checking package dependencies ... NOTE
Package suggested but not available for checking: ‘GenomicInteractions’

checking for hidden files and directories ... NOTE
Found the following hidden files and directories:
  .BBSoptions
These were most likely included in error. See section ‘Package
structure’ in the ‘Writing R Extensions’ manual.

checking top-level files ... NOTE
Non-standard file/directory found at top level:
  ‘session.txt’

checking Rd line widths ... NOTE
Rd file 'mergeSamples.Rd':
  \usage lines wider than 90 characters:
           NcolNormPrefix = "NNorm", mergeMethod = c("weightedMean", "mean")[1], repNormCounts = (mergeMethod=="mean"))

Rd file 'peakEnrichment4Features.Rd':
  \examples lines wider than 100 characters:
     peakEnrichment4Features(cdUnitTest, folder=ChIPdir, list_frag = featuresList, no_bins = 500, sample_number = 100)

These lines will be truncated in the PDF manual.

checking Rd cross-references ... NOTE
Package unavailable to check Rd xrefs: ‘GenomicInteractions’
```

## ChIPpeakAnno (3.11.4)
Maintainer: Lihua Julie Zhu <julie.zhu@umassmed.edu>,
 Jianhong Ou <Jianhong.ou@umassmed.edu>

0 errors | 0 warnings | 2 notes

```
checking package dependencies ... NOTE
Depends: includes the non-default packages:
  ‘grid’ ‘IRanges’ ‘Biostrings’ ‘GenomicRanges’ ‘S4Vectors’
  ‘VennDiagram’
Adding so many packages to the search path is excessive and importing
selectively is preferable.

checking installed package size ... NOTE
  installed size is 19.6Mb
  sub-directories of 1Mb or more:
    data     12.7Mb
    doc       1.1Mb
    extdata   5.2Mb
```

## Clomial (1.13.0)
Maintainer: Habil Zare <zare@txstate.edu>

0 errors | 0 warnings | 1 note 

```
checking R code for possible problems ... NOTE
Clomial: no visible global function definition for ‘runif’
Clomial.generate.data: no visible global function definition for
  ‘runif’
Clomial.generate.data: no visible global function definition for
  ‘rbinom’
Clomial.likelihood: no visible global function definition for ‘dbinom’
Phi: no visible global function definition for ‘dbinom’
choose.best: no visible global function definition for ‘tail’
compute.P.reparam : update.Wj: no visible global function definition
  for ‘optim’
compute.P.reparam : plot.obj: no visible global function definition for
  ‘plot’
compute.q: no visible global function definition for ‘dbinom’
Undefined global functions or variables:
  dbinom optim plot rbinom runif tail
Consider adding
  importFrom("graphics", "plot")
  importFrom("stats", "dbinom", "optim", "rbinom", "runif")
  importFrom("utils", "tail")
to your NAMESPACE file.
```

## clusterExperiment (1.3.2)
Maintainer: Elizabeth Purdom <epurdom@stat.berkeley.edu>  
Bug reports: https://github.com/epurdom/clusterExperiment/issues

2 errors | 1 warning  | 2 notes

```
checking examples ... ERROR
Running examples in ‘clusterExperiment-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: clusterContrasts,ClusterExperiment-method
> ### Title: Create contrasts for testing DE of a cluster
> ### Aliases: clusterContrasts,ClusterExperiment-method clusterContrasts
> ###   clusterContrasts,vector-method
> 
> ### ** Examples
> 
> data(simData)
> cl <- clusterMany(simData,nPCADims=c(5,10,50),  dimReduce="PCA",
+ clusterFunction="pam", ks=2:4, findBestK=c(FALSE), removeSil=TRUE,
+ subsample=FALSE)
> #Pairs:
> clusterContrasts(cl,contrastType="Pairs")
Error in if (outputType == "MAST" & !requireNamespace("MAST", quietly = TRUE)) stop("for outputType 'MAST', you must have package 'MAST' from Bioconductor installed.") : 
  the condition has length > 1
Calls: clusterContrasts ... .local -> clusterContrasts -> clusterContrasts -> .local
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’ [125s/125s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  Note: Merging will be done on ' clusterSingle ', with clustering index 1 
  Note: Merging will be done on ' nPCAFeatures=5,k=NA,findBestK=TRUE ', with clustering index 1 
  Note: Merging will be done on ' nPCAFeatures=5,k=NA,findBestK=TRUE ', with clustering index 1 
  Note: Merging will be done on ' nPCAFeatures=5,k=NA,findBestK=TRUE ', with clustering index 1 
  Note: Merging will be done on ' nPCAFeatures=5,k=NA,findBestK=TRUE ', with clustering index 1 
  Note: Merging will be done on ' nPCAFeatures=5,k=NA,findBestK=TRUE ', with clustering index 1 
  Note: Merging will be done on ' nPCAFeatures=5,k=NA,findBestK=TRUE ', with clustering index 1 
  Note: Merging will be done on ' nPCAFeatures=5,k=NA,findBestK=TRUE ', with clustering index 1 
  Note: Merging will be done on ' nPCAFeatures=5,k=NA,findBestK=TRUE ', with clustering index 1 
  testthat results ================================================================
  OK: 484 SKIPPED: 0 FAILED: 1
  1. Error: `clusterContrasts` works with matrix and ClusterExperiment objects (@test_getBestFeatures.R#6) 
  
  Error: testthat unit tests failed
  Execution halted

checking Rd files ... WARNING
mergeClusters.Rd: non-ASCII input and no declared encoding
plotClusters.Rd: non-ASCII input and no declared encoding
problems found in ‘mergeClusters.Rd’, ‘plotClusters.Rd’

checking dependencies in R code ... NOTE
':::' call which should be '::': ‘ape:::.PlotPhyloEnv’
  See the note in ?`:::` about the use of this operator.
Unexported object imported by a ':::' call: ‘ape:::.matchDataPhylo’
  See the note in ?`:::` about the use of this operator.

checking Rd cross-references ... NOTE
Package unavailable to check Rd xrefs: ‘ConsensusClusterPlus’
```

## CNPBayes (1.7.1)
Maintainer: Jacob Carey <jcarey15@jhu.edu>  
Bug reports: https://github.com/scristia/CNPBayes/issues

0 errors | 0 warnings | 4 notes

```
checking installed package size ... NOTE
  installed size is 12.7Mb
  sub-directories of 1Mb or more:
    libs  10.5Mb

checking DESCRIPTION meta-information ... NOTE
Package listed in more than one of Depends, Imports, Suggests, Enhances:
  ‘GenomicRanges’
A package should be listed in only one of these fields.

checking R code for possible problems ... NOTE
consensusRegion: no visible global function definition for
  ‘elementLengths’
Undefined global functions or variables:
  elementLengths

checking compiled code ... NOTE
File ‘CNPBayes/libs/CNPBayes.so’:
  Found no calls to: ‘R_registerRoutines’, ‘R_useDynamicSymbols’

It is good practice to register native routines and to disable symbol
search.

See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
```

## cointmonitoR (0.1.0)
Maintainer: Philipp Aschersleben <aschersleben@statistik.tu-dortmund.de>  
Bug reports: https://github.com/aschersleben/cointmonitoR/issues

0 errors | 0 warnings | 0 notes

## cointReg (0.2.0)
Maintainer: Philipp Aschersleben <aschersleben@statistik.tu-dortmund.de>  
Bug reports: https://github.com/aschersleben/cointReg/issues

0 errors | 0 warnings | 0 notes

## consensusOV (0.99.3)
Maintainer: Benjamin Haibe-Kains <benjamin.haibe.kains@utoronto.ca>  
Bug reports: https://github.com/bhklab/consensusOV/issues

0 errors | 1 warning  | 1 note 

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Loading required package: survcomp
Loading required package: survival
Loading required package: prodlim
Loading required package: mclust
Package 'mclust' version 5.3
Type 'citation("mclust")' for citing this R package in publications.
Loading required package: limma
... 8 lines ...
Loading required package: biomaRt
Loading required package: iC10
Loading required package: pamr
Loading required package: cluster
Loading required package: iC10TrainingData
Loading required package: AIMS
Loading required package: e1071
Quitting from lines 55-56 (consensusOV.Rmd) 
Error: processing vignette 'consensusOV.Rmd' failed with diagnostics:
4 simultaneous processes spawned
Execution halted

checking top-level files ... NOTE
Non-standard file/directory found at top level:
  ‘appveyor.yml’
```

## CopywriteR (2.9.0)
Maintainer: Thomas Kuilman <t.kuilman@nki.nl>

0 errors | 0 warnings | 1 note 

```
checking R code for possible problems ... NOTE
.peakCutoff: no visible global function definition for ‘dpois’
.peakCutoff: no visible global function definition for ‘ppois’
.tng: no visible global function definition for ‘png’
.tng: no visible global function definition for ‘par’
.tng: no visible global function definition for ‘dev.off’
.tng: no visible global function definition for ‘loess’
.tng: no visible global function definition for ‘predict’
.tng: no visible global function definition for ‘points’
.tng: no visible global function definition for ‘rgb’
... 41 lines ...
  par pdf plot png points ppois predict read.table rgb segments text
  write.table
Consider adding
  importFrom("grDevices", "dev.off", "pdf", "png", "rgb")
  importFrom("graphics", "abline", "axis", "lines", "par", "plot",
             "points", "segments", "text")
  importFrom("methods", "as", "getClass")
  importFrom("stats", "dpois", "ecdf", "loess", "ppois", "predict")
  importFrom("utils", "packageVersion", "read.table", "write.table")
to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
contains 'methods').
```

## cosinor2 (0.1.0)
Maintainer: Augustin Mutak <aumutak@ffzg.hr>

0 errors | 0 warnings | 0 notes

## CpGFilter (1.1)
Maintainer: Jun Chen <chen.jun2@mayo.edu>

0 errors | 0 warnings | 0 notes

## crlmm (1.35.0)
Maintainer: Benilton S Carvalho <benilton@unicamp.br>,
 Robert Scharpf <rscharpf@jhsph.edu>, Matt Ritchie
 <mritchie@wehi.EDU.AU>

0 errors | 0 warnings | 6 notes

```
checking installed package size ... NOTE
  installed size is  5.1Mb
  sub-directories of 1Mb or more:
    data   2.7Mb

checking DESCRIPTION meta-information ... NOTE
Versioned 'LinkingTo' value for ‘preprocessCore’ is only usable in R >= 3.0.2

checking dependencies in R code ... NOTE
'library' or 'require' call to ‘RUnit’ in package code.
  Please use :: or requireNamespace() instead.
  See section 'Suggested packages' in the 'Writing R Extensions' manual.
Unexported object imported by a ':::' call: ‘Biobase:::assayDataEnvLock’
  See the note in ?`:::` about the use of this operator.

checking R code for possible problems ... NOTE
.test: no visible global function definition for ‘defineTestSuite’
.test: no visible global function definition for ‘runTestSuite’
.test: no visible global function definition for ‘printTextProtocol’
calculateRBafCNSet : processByChromosome: no visible global function
  definition for ‘position’
genotypeInf: no visible binding for global variable ‘anno’
krlmm: no visible binding for global variable ‘VGLMparameters’
Undefined global functions or variables:
  VGLMparameters anno defineTestSuite position printTextProtocol
  runTestSuite

checking Rd line widths ... NOTE
Rd file 'constructInf.Rd':
  \usage lines wider than 90 characters:
            arrayInfoColNames = list(barcode="SentrixBarcode_A",position="SentrixPosition_A"), highDensity = FALSE, sep = "_",
            fileExt = list(green = "Grn.idat", red = "Red.idat"), XY, cdfName, anno, genome, verbose = FALSE, batch=NULL, saveDate = TRUE)

Rd file 'genotype.Illumina.Rd':
  \usage lines wider than 90 characters:
           highDensity=FALSE, sep="_", fileExt=list(green="Grn.idat", red="Red.idat"), XY=NULL, anno, genome, 
           call.method="crlmm", trueCalls=NULL, cdfName, copynumber=TRUE, batch=NULL, saveDate=FALSE, stripNorm=TRUE, 
... 20 lines ...
Rd file 'readGenCallOutput.Rd':
  \usage lines wider than 90 characters:
     readGenCallOutput(filenames, path=".", cdfName, colnames=list("SampleID"="Sample ID", "SNPID"="SNP Name", "XRaw"="X Raw", "YRaw"="Y Raw ... [TRUNCATED]
                       type=list("SampleID"="character", "SNPID"="character", "XRaw"="integer", "YRaw"="integer"), verbose=FALSE)

Rd file 'snprma.Rd':
  \usage lines wider than 90 characters:
     snprma(filenames, mixtureSampleSize = 10^5, fitMixture = FALSE, eps = 0.1, verbose = TRUE, seed = 1, cdfName, sns)
     snprma2(filenames, mixtureSampleSize = 10^5, fitMixture = FALSE, eps = 0.1, verbose = TRUE, seed = 1, cdfName, sns)

These lines will be truncated in the PDF manual.

checking compiled code ... NOTE
File ‘crlmm/libs/crlmm.so’:
  Found no call to: ‘R_useDynamicSymbols’

It is good practice to register native routines and to disable symbol
search.

See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
```

## crossmeta (1.3.0)
Maintainer: Alex Pickering <alexvpickering@gmail.com>

0 errors | 0 warnings | 3 notes

```
checking dependencies in R code ... NOTE
Missing or unexported objects:
  ‘crossmeta::gs.names’ ‘crossmeta::gslist’

checking R code for possible problems ... NOTE
explore_paths : server: no visible binding for global variable ‘gslist’
explore_paths : server: no visible binding for global variable
  ‘gs.names’
Undefined global functions or variables:
  gs.names gslist

checking Rd cross-references ... NOTE
Packages unavailable to check Rd xrefs: ‘PADOG’, ‘GeneMeta’
```

## DAMOCLES (1.1)
Maintainer: Rampal S. Etienne <r.s.etienne@rug.nl>

0 errors | 0 warnings | 1 note 

```
checking R code for possible problems ... NOTE
DAMOCLES_ML: no visible global function definition for ‘flush.console’
DAMOCLES_bootstrap: no visible global function definition for
  ‘cophenetic’
DAMOCLES_bootstrap: no visible global function definition for ‘sd’
DAMOCLES_bootstrap: no visible global function definition for
  ‘quantile’
DAMOCLES_sim: no visible global function definition for ‘na.omit’
DAMOCLES_sim: no visible global function definition for ‘runif’
DAMOCLES_sim: no visible global function definition for ‘lines’
DAMOCLES_sim: no visible global function definition for ‘rnorm’
DAMOCLES_sim: no visible global function definition for ‘dexp’
DAMOCLES_sim: no visible global function definition for ‘dist’
DAMOCLES_simplex: no visible global function definition for
  ‘flush.console’
Undefined global functions or variables:
  cophenetic dexp dist flush.console lines na.omit quantile rnorm runif
  sd
Consider adding
  importFrom("graphics", "lines")
  importFrom("stats", "cophenetic", "dexp", "dist", "na.omit",
             "quantile", "rnorm", "runif", "sd")
  importFrom("utils", "flush.console")
to your NAMESPACE file.
```

## DatabionicSwarm (0.9.7)
Maintainer: Michael Thrun <m.thrun@gmx.net>

0 errors | 0 warnings | 0 notes

## DeepBlueR (1.3.10)
Maintainer: Felipe Albrecht <felipe.albrecht@mpi-inf.mpg.de>, Markus List <markus.list@mpi-inf.mpg.de>

0 errors | 0 warnings | 1 note 

```
checking top-level files ... NOTE
Non-standard file/directory found at top level:
  ‘logo.png’
```

## DelayedArray (0.3.19)
Maintainer: Hervé Pagès <hpages@fredhutch.org>

1 error  | 0 warnings | 0 notes

```
checking package dependencies ... ERROR
Packages required and available but unsuitable versions:
  ‘S4Vectors’ ‘IRanges’

See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
```

## DGCA (1.0.1)
Maintainer: Andrew McKenzie <amckenz@gmail.com>

0 errors | 0 warnings | 0 notes

## diffloop (1.5.0)
Maintainer: Caleb Lareau <caleblareau@g.harvard.edu>  
Bug reports: https://github.com/aryeelab/diffloop/issues

0 errors | 0 warnings | 1 note 

```
checking for hidden files and directories ... NOTE
Found the following hidden files and directories:
  .travis.yml
These were most likely included in error. See section ‘Package
structure’ in the ‘Writing R Extensions’ manual.
```

## DisHet (0.1.0)
Maintainer: Tao Wang <tao.wang@utsouthwestern.edu>

0 errors | 0 warnings | 0 notes

## dplR (1.6.6)
Maintainer: Andy Bunn <andy.bunn@wwu.edu>

0 errors | 0 warnings | 0 notes

## DPWeibull (1.0)
Maintainer: Yushu Shi <shiyushu2006@gmail.com>

0 errors | 0 warnings | 0 notes

## dtree (0.2.3)
Maintainer: Ross Jacobucci <rcjacobuc@gmail.com>

0 errors | 0 warnings | 0 notes

## EasyqpcR (1.19.0)
Maintainer: Le Pape Sylvain <sylvain.le.pape@univ-poitiers.fr>

1 error  | 0 warnings | 1 note 

```
checking whether package ‘EasyqpcR’ can be installed ... ERROR
Installation failed.
See ‘/home/hb/repositories/matrixStats/revdep/checks/EasyqpcR.Rcheck/00install.out’ for details.

checking package namespace information ... NOTE
  Namespace with empty importFrom: ‘gWidgetsRGtk2’
```

## EMDomics (2.7.0)
Maintainer: Sadhika Malladi <contact@sadhikamalladi.com> and Daniel Schmolze <emd@schmolze.com>

0 errors | 0 warnings | 3 notes

```
checking DESCRIPTION meta-information ... NOTE
Authors@R field gives more than one person with maintainer role:
  Sadhika Malladi <contact@sadhikamalladi.com> [aut, cre]
  Daniel Schmolze <emd@schmolze.com> [aut, cre]

checking R code for possible problems ... NOTE
.cvm_pairwise_q: no visible global function definition for ‘combn’
.cvm_pairwise_q : <anonymous>: no visible global function definition
  for ‘median’
.emd_gene_pairwise: no visible global function definition for ‘hist’
.emd_pairwise_q: no visible global function definition for ‘combn’
.emd_pairwise_q : <anonymous>: no visible global function definition
  for ‘median’
.ks_pairwise_table: no visible global function definition for ‘ks.test’
calculate_cvm: no visible global function definition for ‘combn’
... 11 lines ...
calculate_ks : <anonymous>: no visible global function definition for
  ‘median’
calculate_ks_gene: no visible global function definition for ‘combn’
calculate_ks_gene: no visible global function definition for ‘ks.test’
Undefined global functions or variables:
  combn hist ks.test median p.adjust
Consider adding
  importFrom("graphics", "hist")
  importFrom("stats", "ks.test", "median", "p.adjust")
  importFrom("utils", "combn")
to your NAMESPACE file.

checking files in ‘vignettes’ ... NOTE
The following directory looks like a leftover from 'knitr':
  ‘figure’
Please remove from your package.
```

## EnrichedHeatmap (1.7.0)
Maintainer: Zuguang Gu <z.gu@dkfz.de>

0 errors | 0 warnings | 4 notes

```
checking installed package size ... NOTE
  installed size is  5.2Mb
  sub-directories of 1Mb or more:
    doc       2.5Mb
    extdata   2.5Mb

checking top-level files ... NOTE
Non-standard file/directory found at top level:
  ‘test’

checking S3 generic/method consistency ... NOTE
Found the following apparent S3 methods exported but not registered:
  +.AdditiveUnit [.normalizedMatrix print.normalizedMatrix
See section ‘Registering S3 methods’ in the ‘Writing R Extensions’
manual.

checking Rd line widths ... NOTE
Rd file 'anno_enriched.rd':
  \usage lines wider than 90 characters:
         yaxis = TRUE, ylim = NULL, value = c("mean", "sum", "abs_mean", "abs_sum"), yaxis_side = "right",

Rd file 'normalizeToMatrix.rd':
  \usage lines wider than 90 characters:
         mean_mode = c("absolute", "weighted", "w0", "coverage"), include_target = any(width(target) > 1),

These lines will be truncated in the PDF manual.
```

## EventPointer (1.1.0)
Maintainer: Juan Pablo Romero <jpromero@ceit.es>  
Bug reports: https://github.com/jpromeror/EventPointer/issues

0 errors | 0 warnings | 3 notes

```
checking package subdirectories ... NOTE
Found the following CITATION file in a non-standard place:
  inst/CITATION.orig
Most likely ‘inst/CITATION’ should be used instead.

checking dependencies in R code ... NOTE
Unexported objects imported by ':::' calls:
  ‘SGSeq:::edges’ ‘SGSeq:::exonGraph’ ‘SGSeq:::nodes’
  See the note in ?`:::` about the use of this operator.

checking Rd line widths ... NOTE
Rd file 'CDFfromGTF.Rd':
  \examples lines wider than 100 characters:
        CDFfromGTF(input='GTF',inputFile=DONSON_GTF,PSR=PSRProbes,Junc=JunctionProbes,PathCDF=Directory,microarray=microarray)

These lines will be truncated in the PDF manual.
```

## expss (0.8.1)
Maintainer: Gregory Demin <gdemin@gmail.com>  
Bug reports: https://github.com/gdemin/expss/issues

0 errors | 0 warnings | 0 notes

## FADA (1.3.2)
Maintainer: David Causeur <david.causeur@agrocampus-ouest.fr>

0 errors | 0 warnings | 0 notes

## FastHCS (0.0.5)
Maintainer: Kaveh Vakili <vakili.kaveh.email@gmail.com>

0 errors | 0 warnings | 3 notes

```
checking installed package size ... NOTE
  installed size is 16.8Mb
  sub-directories of 1Mb or more:
    data   1.3Mb
    libs  15.4Mb

checking R code for possible problems ... NOTE
FHCSpsdo: no visible global function definition for ‘is’
FastHCS: no visible global function definition for ‘complete.cases’
compPcaParams: no visible global function definition for ‘quantile’
compPcaParams: no visible global function definition for ‘qchisq’
compPcaParams: no visible global function definition for ‘qnorm’
compPcaParams: no visible global function definition for ‘sd’
plot.FastHCS: no visible global function definition for ‘plot’
plot.FastHCS: no visible global function definition for ‘abline’
Undefined global functions or variables:
  abline complete.cases is plot qchisq qnorm quantile sd
Consider adding
  importFrom("graphics", "abline", "plot")
  importFrom("methods", "is")
  importFrom("stats", "complete.cases", "qchisq", "qnorm", "quantile",
             "sd")
to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
contains 'methods').

checking compiled code ... NOTE
File ‘FastHCS/libs/FastHCS.so’:
  Found no calls to: ‘R_registerRoutines’, ‘R_useDynamicSymbols’

It is good practice to register native routines and to disable symbol
search.

See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
```

## FastPCS (0.1.2)
Maintainer: Kaveh Vakili <vakili.kaveh.email@gmail.com>

0 errors | 0 warnings | 4 notes

```
checking installed package size ... NOTE
  installed size is  9.2Mb
  sub-directories of 1Mb or more:
    libs   9.1Mb

checking S3 generic/method consistency ... NOTE
Found the following apparent S3 methods exported but not registered:
  plot.FastPCS
See section ‘Registering S3 methods’ in the ‘Writing R Extensions’
manual.

checking R code for possible problems ... NOTE
FastPCS: no visible global function definition for ‘complete.cases’
FastPCS: no visible global function definition for ‘cov’
FastPCS: no visible global function definition for ‘mahalanobis’
FastPCS: no visible global function definition for ‘qchisq’
FastPCS: no visible global function definition for ‘quantile’
plot.FastPCS: no visible global function definition for ‘plot’
plot.FastPCS: no visible global function definition for ‘abline’
plot.FastPCS: no visible global function definition for ‘qchisq’
Undefined global functions or variables:
  abline complete.cases cov mahalanobis plot qchisq quantile
Consider adding
  importFrom("graphics", "abline", "plot")
  importFrom("stats", "complete.cases", "cov", "mahalanobis", "qchisq",
             "quantile")
to your NAMESPACE file.

checking compiled code ... NOTE
File ‘FastPCS/libs/FastPCS.so’:
  Found no calls to: ‘R_registerRoutines’, ‘R_useDynamicSymbols’

It is good practice to register native routines and to disable symbol
search.

See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
```

## FastRCS (0.0.7)
Maintainer: Kaveh Vakili <vakili.kaveh.email@gmail.com>

0 errors | 0 warnings | 2 notes

```
checking installed package size ... NOTE
  installed size is 17.3Mb
  sub-directories of 1Mb or more:
    libs  17.2Mb

checking compiled code ... NOTE
File ‘FastRCS/libs/FastRCS.so’:
  Found no calls to: ‘R_registerRoutines’, ‘R_useDynamicSymbols’

It is good practice to register native routines and to disable symbol
search.

See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
```

## fergm (0.2.0)
Maintainer: Benjamin W. Campbell <campbell.1721@osu.edu>

0 errors | 0 warnings | 0 notes

## filesstrings (1.1.0)
Maintainer: Rory Nolan <rorynoolan@gmail.com>  
Bug reports: https://www.github.com/rorynolan/filesstrings/issues

0 errors | 0 warnings | 0 notes

## flowCore (1.43.5)
Maintainer: M.Jiang <wjiang2@fhcrc.org>

0 errors | 2 warnings | 10 notes

```
checking Rd \usage sections ... WARNING
Documented arguments not in \usage in documentation object 'read.flowSet':
  ‘truncate_max_range’

Functions with \usage entries need to have the appropriate \alias
entries, and all their arguments documented.
The \usage entries must correspond to syntactically valid R code.
See chapter ‘Writing R documentation files’ in the ‘Writing R
Extensions’ manual.

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Quitting from lines 301-303 (HowTo-flowCore.Rnw) 
Error: processing vignette 'HowTo-flowCore.Rnw' failed with diagnostics:
there is no package called 'ggcyto'
Execution halted


checking package dependencies ... NOTE
Packages suggested but not available for checking:
  ‘flowWorkspace’ ‘openCyto’ ‘ggcyto’

checking if this is a source package ... NOTE
Found the following apparent object files/libraries:
  src/boost_regex/c_regex_traits.o src/boost_regex/cpp_regex_traits.o
  src/boost_regex/cregex.o src/boost_regex/fileiter.o
  src/boost_regex/icu.o src/boost_regex/instances.o
  src/boost_regex/posix_api.o src/boost_regex/regex.o
  src/boost_regex/regex_debug.o src/boost_regex/regex_raw_buffer.o
  src/boost_regex/regex_traits_defaults.o
  src/boost_regex/static_mutex.o src/boost_regex/usinstances.o
  src/boost_regex/w32_regex_traits.o src/boost_regex/wc_regex_traits.o
  src/boost_regex/wide_posix_api.o src/boost_regex/winstances.o
Object files/libraries should not be included in a source package.

checking installed package size ... NOTE
  installed size is 34.1Mb
  sub-directories of 1Mb or more:
    data      5.0Mb
    extdata   1.1Mb
    lib      20.3Mb
    libs      5.6Mb

checking DESCRIPTION meta-information ... NOTE
Versioned 'LinkingTo' value for ‘BH’ is only usable in R >= 3.0.2

checking top-level files ... NOTE
Non-standard files/directories found at top level:
  ‘examples’ ‘hyperlog.notice.html’ ‘paper’ ‘unitTests’

checking dependencies in R code ... NOTE
'library' or 'require' call to ‘Rgraphviz’ in package code.
  Please use :: or requireNamespace() instead.
  See section 'Suggested packages' in the 'Writing R Extensions' manual.

checking S3 generic/method consistency ... NOTE
Found the following apparent S3 methods exported but not registered:
  as.data.frame.manyFilterResult
See section ‘Registering S3 methods’ in the ‘Writing R Extensions’
manual.

checking R code for possible problems ... NOTE
rbind2,flowSet-flowSet: warning in assign(i, x[[i]], env = env):
  partial argument match of 'env' to 'envir'
rbind2,flowSet-flowSet: warning in assign(i, y[[i]], env = env):
  partial argument match of 'env' to 'envir'
%in%,flowFrame-norm2Filter: no visible global function definition for
  ‘cov.rob’
spillover,flowSet: no visible global function definition for
  ‘devAskNewPage’
spillover,flowSet : <anonymous>: no visible global function definition
... 6 lines ...
spillover,flowSet : <anonymous> : <anonymous>: no visible global
  function definition for ‘density’
split,flowFrame-factor: no visible binding for global variable ‘i’
summary,workFlow: no visible binding for global variable ‘wf’
Undefined global functions or variables:
  abline cov.rob density devAskNewPage i rangeGate wf
Consider adding
  importFrom("grDevices", "devAskNewPage")
  importFrom("graphics", "abline")
  importFrom("stats", "density")
to your NAMESPACE file.

checking Rd line widths ... NOTE
Rd file 'FCSTransTransform.Rd':
  \usage lines wider than 90 characters:
     FCSTransTransform(transformationId = "defaultFCSTransTransform", channelrange = 2^18, channeldecade = 4.5, range = 4096, cutoff = -111, ... [TRUNCATED]

Rd file 'biexponentialTransform.Rd':
  \usage lines wider than 90 characters:
     biexponentialTransform(transformationId="defaultBiexponentialTransform", a = 0.5, b = 1, c = 0.5, d = 1, f = 0, w = 0, tol = .Machine$d ... [TRUNCATED]

Rd file 'compensation-class.Rd':
... 10 lines ...
                 decades=0, ncdf = FALSE, min.limit=NULL, truncate_max_range = TRUE, dataset=NULL, emptyValue=TRUE, ...)

Rd file 'spillover-flowSet.Rd':
  \usage lines wider than 90 characters:
             ssc = "SSC-A", method = "median", stain_match = c("intensity", "ordered", "regexpr"),

Rd file 'splitScaleTransform.Rd':
  \usage lines wider than 90 characters:
     splitScaleTransform(transformationId="defaultSplitscaleTransform", maxValue=1023, transitionChannel=64, r=192)

These lines will be truncated in the PDF manual.

checking for GNU extensions in Makefiles ... NOTE
GNU make is a SystemRequirements.
```

## fslr (2.12)
Maintainer: John Muschelli <muschellij2@gmail.com>  
Bug reports: https://github.com/muschellij2/fslr/issues

0 errors | 0 warnings | 0 notes

## funtooNorm (1.1.0)
Maintainer: Kathleen Klein <kathleen.klein@mail.mcgill.ca>

0 errors | 1 warning  | 0 notes

```
checking sizes of PDF files under ‘inst/doc’ ... WARNING
  ‘gs+qpdf’ made some significant size reductions:
     compacted ‘funtooNorm.pdf’ from 1239Kb to 199Kb
  consider running tools::compactPDF(gs_quality = "ebook") on these files
```

## GAD (1.1.1)
Maintainer: Leonardo Sandrini-Neto <leonardosandrini@gmail.com>

0 errors | 0 warnings | 0 notes

## gcapc (1.1.9)
Maintainer: Mingxiang Teng <tengmx@gmail.com>

0 errors | 0 warnings | 0 notes

## GeneralizedUmatrix (0.9.5)
Maintainer: Michael Thrun <mthrun@mathematik.uni-marburg.de>

0 errors | 0 warnings | 0 notes

## genomation (1.9.3)
Maintainer: Altuna Akalin <aakalin@gmail.com>, Vedran Franke
 <vedran.franke@gmail.com>  
Bug reports: https://github.com/BIMSBbioinfo/genomation/issues

0 errors | 0 warnings | 4 notes

```
checking installed package size ... NOTE
  installed size is  5.6Mb
  sub-directories of 1Mb or more:
    doc       3.6Mb
    extdata   1.2Mb

checking dependencies in R code ... NOTE
Unexported object imported by a ':::' call: ‘BiocGenerics:::testPackage’
  See the note in ?`:::` about the use of this operator.

checking R code for possible problems ... NOTE
ScoreMatrixBin,RleList-GRangesList: no visible binding for global
  variable ‘cvg’
ScoreMatrixBin,RleList-GRangesList: no visible binding for global
  variable ‘id’
ScoreMatrixBin,RleList-GRangesList: no visible global function
  definition for ‘:=’
Undefined global functions or variables:
  := cvg id

checking files in ‘vignettes’ ... NOTE
The following directory looks like a leftover from 'knitr':
  ‘cache’
Please remove from your package.
```

## GenRank (1.5.0)
Maintainer: Chakravarthi Kanduri <chakra.kanduri@gmail.com>  
Bug reports: https://github.com/chakri9/GenRank/issues

0 errors | 0 warnings | 1 note 

```
checking R code for possible problems ... NOTE
CombP: no visible global function definition for ‘read.table’
ComputeCE: no visible global function definition for ‘read.table’
ComputeCE: no visible global function definition for ‘complete.cases’
ComputeRP: no visible global function definition for ‘read.table’
ComputeRP: no visible global function definition for ‘complete.cases’
Undefined global functions or variables:
  complete.cases read.table
Consider adding
  importFrom("stats", "complete.cases")
  importFrom("utils", "read.table")
to your NAMESPACE file.
```

## GJRM (0.1-1)
Maintainer: Giampiero Marra <giampiero.marra@ucl.ac.uk>

0 errors | 0 warnings | 0 notes

## GPrank (0.1.2)
Maintainer: Hande Topa <hande.topa@helsinki.fi>  
Bug reports: https://github.com/PROBIC/GPrank/issues

0 errors | 0 warnings | 0 notes

## GUIDEseq (1.7.3)
Maintainer: Lihua Julie Zhu <julie.zhu@umassmed.edu>

1 error  | 1 warning  | 3 notes

```
checking examples ... ERROR
Running examples in ‘GUIDEseq-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: GUIDEseqAnalysis
> ### Title: Analysis pipeline for GUIDE-seq dataset
> ### Aliases: GUIDEseqAnalysis
> ### Keywords: misc
> 
... 50 lines ...
>>> DONE searching
>>> Finding all hits in sequence chr13+:39262927:39262939:chr13-:39262918:39262920 ...
>>> DONE searching
finish off-target search in sequence 2
finish off-target search in sequence 1
finish feature vector building
finish score calculation
[1] "Done!"
Error in if (!is.na(TS2)) { : the condition has length > 1
Calls: GUIDEseqAnalysis -> offTargetAnalysisOfPeakRegions
Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in searchHits(gRNAs = gRNAs1, PAM = PAM, PAM.pattern = PAM.pattern,  :
  No matching found, please check your input sequence, and make
            sure you are using the right genome. You can also alter your 
            search criteria such as increasing max.mismatch!
Warning in searchHits(gRNAs = gRNAs1, PAM = PAM, PAM.pattern = PAM.pattern,  :
  No matching found, please check your input sequence, and make
            sure you are using the right genome. You can also alter your 
... 8 lines ...
            sure you are using the right genome. You can also alter your 
            search criteria such as increasing max.mismatch!
Warning in searchHits(gRNAs = gRNAs1, PAM = PAM, PAM.pattern = PAM.pattern,  :
  No matching found, please check your input sequence, and make
            sure you are using the right genome. You can also alter your 
            search criteria such as increasing max.mismatch!

Error: processing vignette 'GUIDEseq.Rnw' failed with diagnostics:
 chunk 7 
Error in if (!is.na(TS2)) { : the condition has length > 1
Execution halted

checking installed package size ... NOTE
  installed size is 11.1Mb
  sub-directories of 1Mb or more:
    extdata  10.6Mb

checking R code for possible problems ... NOTE
annotateOffTargets: no visible binding for global variable
  'offTarget_Start'
annotateOffTargets: no visible global function definition for 'exons'
annotateOffTargets: no visible global function definition for
  'seqlevelsStyle<-'
annotateOffTargets: no visible global function definition for 'genes'
annotateOffTargets: no visible global function definition for 'toTable'
getPeaks: no visible binding for global variable 'adjusted.p.value'
getPeaks: no visible binding for global variable 'SNratio'
... 8 lines ...
  'strand.first'
getUniqueCleavageEvents: no visible binding for global variable
  'readName'
offTargetAnalysisOfPeakRegions: no visible binding for global variable
  'gRNAPlusPAM'
offTargetAnalysisOfPeakRegions: no visible binding for global variable
  'offTarget'
Undefined global functions or variables:
  SNratio adjusted.p.value exons gRNAPlusPAM genes offTarget
  offTarget_Start qwidth.first qwidth.last readName seqlevelsStyle<-
  strand.first strand.last toTable

checking Rd files ... NOTE
prepare_Rd: annotateOffTargets.Rd:33-35: Dropping empty section \details
prepare_Rd: annotateOffTargets.Rd:43-45: Dropping empty section \references
prepare_Rd: createBarcodeFasta.Rd:42-43: Dropping empty section \value
prepare_Rd: createBarcodeFasta.Rd:44-46: Dropping empty section \references
prepare_Rd: getUsedBarcodes.Rd:39-41: Dropping empty section \references
```

## Gviz (1.21.2)
Maintainer: Florian Hahne <florian.hahne@novartis.com>

0 errors | 2 warnings | 4 notes

```
checking for missing documentation entries ... WARNING
Undocumented S4 methods:
  generic '[' and siglist 'GenomeAxisTrack,ANY,ANY,ANY'
All user-level objects in a package (including S4 classes and methods)
should have documentation entries.
See chapter ‘Writing R documentation files’ in the ‘Writing R
Extensions’ manual.

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
  Running 'texi2dvi' on 'Gviz.tex' failed.
LaTeX errors:
! Undefined control sequence.
l.31 \soulregister
                  {\link}{1}% necessary for compatibility with LaTeX soul pa...
The control sequence at the end of the top line
... 8 lines ...
<to be read again> 
                   \par 
l.31 \soulregister{\link}
                         {1}% necessary for compatibility with LaTeX soul pa...
! LaTeX Error: Missing \begin{document}.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
Calls: buildVignettes -> texi2pdf -> texi2dvi
Execution halted

checking installed package size ... NOTE
  installed size is  6.0Mb
  sub-directories of 1Mb or more:
    doc       1.7Mb
    extdata   2.1Mb

checking top-level files ... NOTE
Non-standard file/directory found at top level:
  ‘Rplots.pdf’

checking R code for possible problems ... NOTE
.buildRange,TxDb-ANY-ANY-ANY: no visible global function definition for
  'cdsBy'
.buildRange,TxDb-ANY-ANY-ANY: no visible global function definition for
  'fiveUTRsByTranscript'
.buildRange,TxDb-ANY-ANY-ANY: no visible global function definition for
  'threeUTRsByTranscript'
Undefined global functions or variables:
  cdsBy fiveUTRsByTranscript threeUTRsByTranscript

checking Rd line widths ... NOTE
Rd file 'AlignmentsTrack-class.Rd':
  \usage lines wider than 90 characters:
                     stacking="squish", id, cigar, mapq, flag, isize, groupid, status, md, seqs,
                     name="AlignmentsTrack", isPaired=TRUE, importFunction, referenceSequence, ...)

Rd file 'AnnotationTrack-class.Rd':
  \examples lines wider than 100 characters:
     annTrack <- AnnotationTrack(start=st, end=ed, strand=str, chromosome=7, genome="hg19", feature="test",
                                 group=gr, id=paste("annTrack item", 1:4), name="generic annotation", stacking="squish")

Rd file 'CustomTrack-class.Rd':
  \usage lines wider than 90 characters:
     CustomTrack(plottingFunction=function(GdObject, prepare=FALSE, ...){}, variables=list(), name="CustomTrack",  ...)

Rd file 'HighlightTrack-class.Rd':
  \usage lines wider than 90 characters:
     HighlightTrack(trackList=list(), range=NULL, start=NULL, end=NULL, width=NULL, chromosome, genome,

These lines will be truncated in the PDF manual.
```

## haploReconstruct (0.1.2)
Maintainer: Susanne U. Franssen <susefranssen@googlemail.com>

0 errors | 0 warnings | 1 note 

```
checking installed package size ... NOTE
  installed size is  9.7Mb
  sub-directories of 1Mb or more:
    data   9.4Mb
```

## IMIFA (1.3.1)
Maintainer: Keefe Murphy <keefe.murphy@ucd.ie>  
Bug reports: https://github.com/Keefe-Murphy/IMIFA

0 errors | 0 warnings | 0 notes

## InfiniumPurify (1.3.1)
Maintainer: Yufang Qin <yfqin@shou.edu.cn>

0 errors | 0 warnings | 0 notes

## kernDeepStackNet (2.0.2)
Maintainer: Thomas Welchowski <welchow@imbie.meb.uni-bonn.de>

0 errors | 0 warnings | 1 note 

```
checking installed package size ... NOTE
  installed size is  5.9Mb
  sub-directories of 1Mb or more:
    libs   5.7Mb
```

## kgschart (1.3.5)
Maintainer: Kota Mori <kmori05@gmail.com>  
Bug reports: https://github.com/kota7/kgschart/issues

0 errors | 0 warnings | 0 notes

## kpmt (0.1.0)
Maintainer: Matthew M Parks <map2085@med.cornell.edu>

0 errors | 0 warnings | 0 notes

## localgauss (0.40)
Maintainer: Tore Selland Kleppe <tore.kleppe@uis.no>

0 errors | 0 warnings | 1 note 

```
checking compiled code ... NOTE
File ‘localgauss/libs/localgauss.so’:
  Found no calls to: ‘R_registerRoutines’, ‘R_useDynamicSymbols’

It is good practice to register native routines and to disable symbol
search.

See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
```

## loo (1.1.0)
Maintainer: Jonah Gabry <jsg2201@columbia.edu>  
Bug reports: https://github.com/stan-dev/loo/issues

0 errors | 0 warnings | 0 notes

## LS2Wstat (2.0-3)
Maintainer: Matt Nunes <m.nunes@lancaster.ac.uk>

0 errors | 1 warning  | 2 notes

```
checking whether package ‘LS2Wstat’ can be installed ... WARNING
Found the following significant warnings:
  Warning: no DISPLAY variable so Tk is not available
See ‘/home/hb/repositories/matrixStats/revdep/checks/LS2Wstat.Rcheck/00install.out’ for details.

checking DESCRIPTION meta-information ... NOTE
Malformed Description field: should contain one or more complete sentences.

checking R code for possible problems ... NOTE
TOS2D: no visible global function definition for ‘medpolish’
countTextures: no visible global function definition for ‘medpolish’
plot.imageQT: no visible global function definition for ‘segments’
simTexture: no visible global function definition for ‘rnorm’
Undefined global functions or variables:
  medpolish rnorm segments
Consider adding
  importFrom("graphics", "segments")
  importFrom("stats", "medpolish", "rnorm")
to your NAMESPACE file.
```

## ltmle (0.9-10)
Maintainer: Joshua Schwab <joshuaschwab@yahoo.com>  
Bug reports: https://github.com/joshuaschwab/ltmle/issues

0 errors | 0 warnings | 0 notes

## Luminescence (0.7.5)
Maintainer: Sebastian Kreutzer <sebastian.kreutzer@u-bordeaux-montaigne.fr>  
Bug reports: https://github.com/R-Lum/Luminescence/issues

0 errors | 0 warnings | 0 notes

## metagene (2.9.1)
Maintainer: Charles Joly Beauparlant <charles.joly-beauparlant@crchul.ulaval.ca>  
Bug reports: https://github.com/CharlesJB/metagene/issues

0 errors | 0 warnings | 1 note 

```
checking R code for possible problems ... NOTE
get_demo_design: no visible global function definition for ‘read.table’
get_promoters_txdb: no visible global function definition for ‘is’
plot_metagene: no visible binding for global variable ‘position’
plot_metagene: no visible binding for global variable ‘value’
plot_metagene: no visible binding for global variable ‘qinf’
plot_metagene: no visible binding for global variable ‘qsup’
plot_metagene: no visible binding for global variable ‘group’
Undefined global functions or variables:
  group is position qinf qsup read.table value
Consider adding
  importFrom("methods", "is")
  importFrom("utils", "read.table")
to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
contains 'methods').
```

## metagenomeSeq (1.19.0)
Maintainer: Joseph N. Paulson <jpaulson@jimmy.harvard.edu>  
Bug reports: https://github.com/nosson/metagenomeSeq/issues

0 errors | 1 warning  | 0 notes

```
checking sizes of PDF files under ‘inst/doc’ ... WARNING
  ‘gs+qpdf’ made some significant size reductions:
     compacted ‘metagenomeSeq.pdf’ from 976Kb to 673Kb
  consider running tools::compactPDF(gs_quality = "ebook") on these files
```

## MetaQC (0.1.13)
Maintainer: Don Kang <donkang75@gmail.com>

0 errors | 0 warnings | 3 notes

```
checking dependencies in R code ... NOTE
'library' or 'require' call to ‘survival’ in package code.
  Please use :: or requireNamespace() instead.
  See section 'Suggested packages' in the 'Writing R Extensions' manual.
Packages in Depends field not imported from:
  ‘foreach’ ‘iterators’ ‘proto’
  These packages need to be imported from (in the NAMESPACE file)
  for when this namespace is loaded but not attached.

checking S3 generic/method consistency ... NOTE
Found the following apparent S3 methods exported but not registered:
  plot.proto print.proto
See section ‘Registering S3 methods’ in the ‘Writing R Extensions’
manual.

checking R code for possible problems ... NOTE
Download: no visible global function definition for ‘download.file’
GMT2List: no visible global function definition for ‘%dopar%’
GMT2List: no visible global function definition for ‘foreach’
GMT2List: no visible global function definition for ‘iter’
GMT2List: no visible binding for global variable ‘x’
GetEWPval: no visible global function definition for ‘pchisq’
GetPVal : t.test2: no visible global function definition for ‘pt’
GetPVal : <anonymous>: no visible global function definition for
  ‘t.test’
... 137 lines ...
  registerDoMC registerDoSNOW sd t.test text w wilcox.test x
Consider adding
  importFrom("grDevices", "gray")
  importFrom("graphics", "abline", "arrows", "axis", "box", "plot",
             "points", "text")
  importFrom("stats", "as.dist", "cor", "cor.test", "fisher.test",
             "ks.test", "lm", "median", "na.omit", "p.adjust", "pchisq",
             "pnorm", "prcomp", "pt", "qnorm", "sd", "t.test",
             "wilcox.test")
  importFrom("utils", "download.file", "installed.packages")
to your NAMESPACE file.
```

## metavizr (1.1.2)
Maintainer: Hector Corrada Bravo <hcorrada@gmail.com>

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘metavizr-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: startMetavizStandalone
> ### Title: Start metaviz app in standalone (locally) and create
> ###   'MetavizApp' object to manage connection.
> ### Aliases: startMetavizStandalone
> 
> ### ** Examples
> 
> # see package vignette for example usage
> app <- startMetavizStandalone()
checking for updates to epiviz app...
done
Error in .wait_until_connected(app$server) : 
  [epivizrStandalone] Error starting app. UI unable to connect to websocket server.
Calls: startMetavizStandalone ... tryCatch -> tryCatchList -> tryCatchOne -> <Anonymous>
Execution halted
```

## MethylAid (1.11.1)
Maintainer: M. van Iterson <mviterson@gmail.com>

0 errors | 0 warnings | 1 note 

```
checking R code for possible problems ... NOTE
Found the following assignments to the global environment:
File ‘MethylAid/R/server.R’:
  assign("outliers", outliers, envir = globalenv())
  assign("background", as.background(background), envir = globalenv())
  assign("location", location, envir = globalenv())
  assign("location", location, envir = globalenv())
  assign("location", location, envir = globalenv())
  assign("location", location, envir = globalenv())
  assign("location", location, envir = globalenv())
File ‘MethylAid/R/util.R’:
  assign("highlight", highlight, envir = globalenv())
  assign("outliers", out, envir = globalenv())
```

## methylumi (2.23.0)
Maintainer: Sean Davis <sdavis2@mail.nih.gov>  
Bug reports: https://github.com/seandavi/methylumi/issues/new

0 errors | 2 warnings | 9 notes

```
checking for missing documentation entries ... WARNING
Undocumented code objects:
  ‘IDATsToMatrices’ ‘IDATtoMatrix’ ‘tcgaPipeline’
Undocumented S4 methods:
  generic '[' and siglist 'MethyLumiM,ANY,ANY,ANY'
  generic '[' and siglist 'MethyLumiSet,ANY,ANY,ANY'
All user-level objects in a package (including S4 classes and methods)
should have documentation entries.
See chapter ‘Writing R documentation files’ in the ‘Writing R
Extensions’ manual.

checking contents of ‘data’ directory ... WARNING
Files not of a type allowed in a ‘data’ directory:
  ‘5318317007_A_Grn.idat’ ‘5318317007_A_Red.idat’
  ‘5318317007_B_Grn.idat’ ‘5318317007_B_Red.idat’
  ‘5318317007_C_Grn.idat’ ‘5318317007_C_Red.idat’
Please use e.g. ‘inst/extdata’ for non-R data files

checking package dependencies ... NOTE
Depends: includes the non-default packages:
  ‘Biobase’ ‘scales’ ‘reshape2’ ‘ggplot2’ ‘matrixStats’
  ‘FDb.InfiniumMethylation.hg19’ ‘minfi’
Adding so many packages to the search path is excessive and importing
selectively is preferable.

checking for hidden files and directories ... NOTE
Found the following hidden files and directories:
  .travis.yml
These were most likely included in error. See section ‘Package
structure’ in the ‘Writing R Extensions’ manual.

checking installed package size ... NOTE
  installed size is  9.7Mb
  sub-directories of 1Mb or more:
    data      6.8Mb
    extdata   1.7Mb

checking DESCRIPTION meta-information ... NOTE
Packages listed in more than one of Depends, Imports, Suggests, Enhances:
  ‘Biobase’ ‘minfi’ ‘lattice’ ‘matrixStats’
A package should be listed in only one of these fields.

checking dependencies in R code ... NOTE
'library' or 'require' calls to packages already attached by Depends:
  ‘FDb.InfiniumMethylation.hg19’ ‘ggplot2’ ‘matrixStats’ ‘minfi’
  ‘reshape2’ ‘scales’
  Please remove these calls from your code.
'library' or 'require' calls in package code:
  ‘Biostrings’ ‘MASS’ ‘lumi’ ‘parallel’ ‘rtracklayer’
  Please use :: or requireNamespace() instead.
  See section 'Suggested packages' in the 'Writing R Extensions' manual.
Packages in Depends field not imported from:
  ‘FDb.InfiniumMethylation.hg19’ ‘ggplot2’ ‘matrixStats’ ‘methods’
  ‘reshape2’ ‘scales’
  These packages need to be imported from (in the NAMESPACE file)
  for when this namespace is loaded but not attached.
':::' call which should be '::': ‘lumi:::produceMethylationGEOSubmissionFile’
  See the note in ?`:::` about the use of this operator.
Unexported objects imported by ':::' calls:
  ‘Biobase:::unsafeSetSlot’ ‘genefilter:::.findCentralMap’
  See the note in ?`:::` about the use of this operator.

checking R code for possible problems ... NOTE
getBarcodes: warning in list.files(path = path, patt = "idat"): partial
  argument match of 'patt' to 'pattern'
.getFinalReportBlock: no visible global function definition for
  ‘read.table’
.mclapply: no visible global function definition for ‘mclapply’
.parallel : <anonymous>: no visible global function definition for
  ‘ecdf’
.readOldMethylationFile: no visible global function definition for
  ‘read.delim’
... 213 lines ...

Found the following calls to data() loading into the global environment:
File ‘methylumi/R/coercions.R’:
  data(hm450.controls)
  data(hm27.controls)
File ‘methylumi/R/methylumIDAT.R’:
  data(hm27.controls)
  data(hm450.controls)
  data(hm27.ordering)
  data(hm450.ordering)
See section ‘Good practice’ in ‘?data’.

checking Rd files ... NOTE
prepare_Rd: estimateM.Rd:34-36: Dropping empty section \seealso
prepare_Rd: estimateM.Rd:37-39: Dropping empty section \examples

checking Rd line widths ... NOTE
Rd file 'methylumIDAT.Rd':
  \examples lines wider than 100 characters:
       lumi450k <- methylumIDAT(barcodes,idatPath=system.file('extdata/idat',package='methylumi')) # no normalization done

These lines will be truncated in the PDF manual.

checking for unstated dependencies in vignettes ... NOTE
'library' or 'require' call not declared from: ‘TxDb.Hsapiens.UCSC.hg19.knownGene’
```

## MFHD (0.0.1)
Maintainer: K. Vakili <kaveh.vakili@wis.kuleuven.be>

0 errors | 0 warnings | 1 note 

```
checking R code for possible problems ... NOTE
MFHD: no visible global function definition for ‘complete.cases’
MFHD: no visible global function definition for ‘quantile’
PIT: no visible global function definition for ‘xy.coords’
PIT: no visible global function definition for ‘identify’
PIT: no visible global function definition for ‘points’
mfhd_compute.bagplot : cut.z.pg: no visible global function definition
  for ‘segments’
mfhd_compute.bagplot : cut.z.pg: no visible global function definition
  for ‘points’
... 35 lines ...
plot.bagplot: no visible global function definition for ‘lines’
plot.bagplot: no visible global function definition for ‘polygon’
Undefined global functions or variables:
  boxplot chull complete.cases identify lines plot points polygon
  prcomp quantile sd segments text xy.coords
Consider adding
  importFrom("grDevices", "chull", "xy.coords")
  importFrom("graphics", "boxplot", "identify", "lines", "plot",
             "points", "polygon", "segments", "text")
  importFrom("stats", "complete.cases", "prcomp", "quantile", "sd")
to your NAMESPACE file.
```

## MIGSA (1.1.1)
Maintainer: Juan C. Rodriguez <jcrodriguez@bdmg.com.ar>

1 error  | 1 warning  | 1 note 

```
checking tests ... ERROR
  Running ‘runTests.R’
Running the tests in ‘tests/runTests.R’ failed.
Last 13 lines of output:
  
      Filter, Find, Map, Position, Reduce, anyDuplicated, append,
      as.data.frame, cbind, colMeans, colSums, colnames, do.call,
      duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,
      lapply, lengths, mapply, match, mget, order, paste, pmax, pmax.int,
      pmin, pmin.int, rank, rbind, rowMeans, rowSums, rownames, sapply,
      setdiff, sort, table, tapply, union, unique, unsplit, which,
      which.max, which.min
  
  
  
  Error in library("RUnit", quietly = TRUE) : 
    there is no package called 'RUnit'
  Calls: <Anonymous> -> library
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...

See the pdftex.def package documentation for explanation.
Type  H <return>  for immediate help.
! Package pdftex.def Error: File `ClassDiagrams/geneSetsFunc-eps-converted-to.p
df' not found.

See the pdftex.def package documentation for explanation.
... 8 lines ...
' not found.

See the pdftex.def package documentation for explanation.
Type  H <return>  for immediate help.
! Package pdftex.def Error: File `ClassDiagrams/FitOptions-eps-converted-to.pdf
' not found.

See the pdftex.def package documentation for explanation.
Type  H <retur
Calls: buildVignettes -> texi2pdf -> texi2dvi
Execution halted

checking S3 generic/method consistency ... NOTE
Found the following apparent S3 methods exported but not registered:
  FitOptions.data.frame FitOptions.default summary.GSEAparams
  summary.IGSAinput summary.MIGSAres summary.SEAparams
See section ‘Registering S3 methods’ in the ‘Writing R Extensions’
manual.
```

## minfi (1.23.3)
Maintainer: Kasper Daniel Hansen <kasperdanielhansen@gmail.com>  
Bug reports: https://github.com/kasperdanielhansen/minfi/issues

0 errors | 0 warnings | 2 notes

```
checking dependencies in R code ... NOTE
Unexported object imported by a ':::' call: ‘bumphunter:::.getEstimate’
  See the note in ?`:::` about the use of this operator.

checking R code for possible problems ... NOTE
Found the following calls to data() loading into the global environment:
File ‘minfi/R/estimateCellCounts.R’:
  data(list = referencePkg)
See section ‘Good practice’ in ‘?data’.
```

## MinimumDistance (1.21.2)
Maintainer: Robert B Scharpf <rscharpf@jhu.edu>

0 errors | 0 warnings | 1 note 

```
checking R code for possible problems ... NOTE
callDenovoSegments: no visible global function definition for
  ‘read.bsfiles’
pruneTrioSet: no visible global function definition for
  ‘RangedDataList’
read.bsfiles2: no visible binding for global variable ‘read.bsfiles’
GenomeAnnotatedDataFrameFrom,character: no visible global function
  definition for ‘read.bsfiles’
calculateMindist,list: no visible binding for global variable ‘elt’
Undefined global functions or variables:
  RangedDataList elt read.bsfiles
```

## mmtfa (0.1)
Maintainer: Jeffrey L. Andrews <jeffrey.andrews@macewan.ca>

0 errors | 0 warnings | 1 note 

```
checking R code for possible problems ... NOTE
contrandz: no visible global function definition for ‘runif’
dfupdatefun: no visible global function definition for ‘uniroot’
kmeansz: no visible global function definition for ‘kmeans’
plot.mmtfa: no visible global function definition for ‘rainbow’
plot.mmtfa: no visible global function definition for ‘par’
plot.mmtfa: no visible global function definition for ‘plot’
plot.mmtfa: no visible global function definition for ‘title’
plot.mmtfa: no visible global function definition for ‘contour’
plot.mmtfa: no visible global function definition for ‘rgb’
... 7 lines ...
sgupdate: no visible global function definition for ‘cov.wt’
winit: no visible global function definition for ‘mahalanobis’
Undefined global functions or variables:
  contour cov.wt curve density dnorm kmeans mahalanobis par plot
  rainbow rgb runif title uniroot
Consider adding
  importFrom("grDevices", "rainbow", "rgb")
  importFrom("graphics", "contour", "curve", "par", "plot", "title")
  importFrom("stats", "cov.wt", "density", "dnorm", "kmeans",
             "mahalanobis", "runif", "uniroot")
to your NAMESPACE file.
```

## monocle (2.5.4)
Maintainer: Cole Trapnell <coletrap@uw.edu>

0 errors | 5 warnings | 3 notes

```
checking whether package ‘monocle’ can be installed ... WARNING
Found the following significant warnings:
  Warning: replacing previous import ‘dplyr::union’ by ‘igraph::union’ when loading ‘monocle’
  Warning: replacing previous import ‘dplyr::as_data_frame’ by ‘igraph::as_data_frame’ when loading ‘monocle’
  Warning: replacing previous import ‘dplyr::groups’ by ‘igraph::groups’ when loading ‘monocle’
  Warning: replacing previous import ‘dplyr::arrange’ by ‘plyr::arrange’ when loading ‘monocle’
See ‘/home/hb/repositories/matrixStats/revdep/checks/monocle.Rcheck/00install.out’ for details.

checking Rd files ... WARNING
checkRd: (5) detectGenes.Rd:0-27: Must have a \description
checkRd: (5) plot_cell_trajectory.Rd:0-60: Must have a \description
prepare_Rd: reducedDimK.Rd:16-18: Dropping empty section \description
checkRd: (5) reducedDimK.Rd:0-24: Must have a \description

checking for missing documentation entries ... WARNING
Undocumented code objects:
  ‘addCellType’ ‘plot_genes_violin’
All user-level objects in a package should have documentation entries.
See chapter ‘Writing R documentation files’ in the ‘Writing R
Extensions’ manual.

checking Rd \usage sections ... WARNING
Undocumented arguments in documentation object 'newCellTypeHierarchy'
  ‘cth’

Documented arguments not in \usage in documentation object 'plot_genes_branched_pseudotime':
  ‘multi_branch’

Functions with \usage entries need to have the appropriate \alias
entries, and all their arguments documented.
The \usage entries must correspond to syntactically valid R code.
See chapter ‘Writing R documentation files’ in the ‘Writing R
Extensions’ manual.

checking Rd contents ... WARNING
Argument items with no description in Rd object 'plot_cell_trajectory':
  ‘show_state_number’ ‘state_number_size’


checking top-level files ... NOTE
Non-standard files/directories found at top level:
  ‘actual_vignette_holder’ ‘examples’

checking R code for possible problems ... NOTE
assign_cell_lineage: no visible global function definition for ‘nei’
buildBranchCellDataSet: no visible global function definition for ‘nei’
clusterCells: no visible binding for global variable ‘rho’
clusterCells: no visible binding for global variable ‘delta’
count_leaf_descendents: no visible global function definition for ‘nei’
cth_classifier_cds: no visible global function definition for ‘nei’
cth_classifier_cell: no visible global function definition for ‘nei’
diff_test_helper: no visible binding for global variable ‘Size_Factor’
extract_good_ordering: no visible global function definition for ‘nei’
... 14 lines ...
plot_multiple_branches_pseudotime: no visible binding for global
  variable ‘pseudocount’
plot_multiple_branches_pseudotime: no visible binding for global
  variable ‘Branch’
project2MST: no visible global function definition for ‘nei’
Undefined global functions or variables:
  Branch Size_Factor delta esApply exprs<- lowess nei next_node
  pseudocount rho
Consider adding
  importFrom("stats", "lowess")
to your NAMESPACE file.

checking files in ‘vignettes’ ... NOTE
The following files look like leftovers/mistakes:
  ‘monocle-vignette-knitr.bbl’, ‘monocle-vignette-knitr.log’,
  ‘monocle-vignette-knitr.toc’
Please remove them from your package.
The following directory looks like a leftover from 'knitr':
  ‘figure’
Please remove from your package.
```

## motifbreakR (1.7.0)
Maintainer: Simon Gert Coetzee <Simon.Coetzee@cshs.org>  
Bug reports: https://github.com/Simon-Coetzee/motifbreakR/issues

0 errors | 0 warnings | 2 notes

```
checking dependencies in R code ... NOTE
Packages in Depends field not imported from:
  'MotifDb' 'grid'
  These packages need to be imported from (in the NAMESPACE file)
  for when this namespace is loaded but not attached.

checking R code for possible problems ... NOTE
plotMotifLogoStack.2: no visible global function definition for 'par'
snps.from.file: no visible global function definition for
  'elementLengths'
snps.from.file: no visible global function definition for 'info'
snps.from.file: no visible global function definition for 'rowRanges'
snps.from.rsid: no visible global function definition for 'snpsById'
Undefined global functions or variables:
  elementLengths info par rowRanges snpsById
Consider adding
  importFrom("graphics", "par")
to your NAMESPACE file.

Found the following assignments to the global environment:
File ‘motifbreakR/R/scoreMotif.R’:
  assign("tmp_motifStack_symbolsCache", list(), pos = ".GlobalEnv")
```

## MPAgenomics (1.1.2)
Maintainer: Samuel Blanck <samuel.blanck@inria.fr>

0 errors | 0 warnings | 2 notes

```
checking dependencies in R code ... NOTE
'library' or 'require' calls in package code:
  ‘R.devices’ ‘R.filesets’ ‘R.methodsS3’ ‘R.oo’ ‘aroma.affymetrix’
  ‘aroma.cn’ ‘aroma.core’ ‘aroma.light’ ‘matrixStats’ ‘snowfall’
  Please use :: or requireNamespace() instead.
  See section 'Suggested packages' in the 'Writing R Extensions' manual.
Unexported object imported by a ':::' call: ‘cghseg:::segmeanCO’
  See the note in ?`:::` about the use of this operator.

checking R code for possible problems ... NOTE
.varregtimescount: no visible global function definition for ‘var’
CGHSEGaroma: no visible global function definition for ‘read.csv’
CGHSEGaroma : <anonymous>: no visible global function definition for
  ‘points’
CGHSEGaroma : <anonymous>: no visible global function definition for
  ‘lines’
CGHSEGaroma : <anonymous>: no visible global function definition for
  ‘write.table’
CGHcall: no visible global function definition for ‘mad’
... 43 lines ...
tumorboostPlot: no visible global function definition for ‘par’
tumorboostPlot: no visible global function definition for ‘axis’
tumorboostPlot: no visible global function definition for ‘points’
Undefined global functions or variables:
  axis head lines lm mad median optim par points read.csv sd var
  write.table
Consider adding
  importFrom("graphics", "axis", "lines", "par", "points")
  importFrom("stats", "lm", "mad", "median", "optim", "sd", "var")
  importFrom("utils", "head", "read.csv", "write.table")
to your NAMESPACE file.
```

## mrfDepth (1.0.4)
Maintainer: Pieter Segaert <Pieter.Segaert@kuleuven.be>  
Bug reports: https://github.com/PSegaert/mrfDepth/issues

0 errors | 0 warnings | 1 note 

```
checking installed package size ... NOTE
  installed size is 15.9Mb
  sub-directories of 1Mb or more:
    libs  15.1Mb
```

## MultiBD (0.2.0)
Maintainer: Marc A. Suchard <msuchard@ucla.edu>

0 errors | 0 warnings | 2 notes

```
checking installed package size ... NOTE
  installed size is  9.5Mb
  sub-directories of 1Mb or more:
    libs   7.9Mb

checking compiled code ... NOTE
File ‘MultiBD/libs/MultiBD.so’:
  Found no calls to: ‘R_registerRoutines’, ‘R_useDynamicSymbols’

It is good practice to register native routines and to disable symbol
search.

See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
```

## nandb (0.2.1)
Maintainer: Rory Nolan <rorynoolan@gmail.com>  
Bug reports: https://github.com/rorynolan/nandb/issues

0 errors | 0 warnings | 0 notes

## NanoStringDiff (1.7.0)
Maintainer: hong wang <hong.wang@uky.edu>

0 errors | 0 warnings | 1 note 

```
checking R code for possible problems ... NOTE
compute.baseSigma: no visible global function definition for ‘IQR’
createNanoStringSetFromCsv: no visible global function definition for
  ‘read.table’
est.dispersion : get.phihat: no visible global function definition for
  ‘optimize’
estNormalizationFactors: no visible global function definition for
  ‘glm’
estNormalizationFactors: no visible global function definition for
  ‘poisson’
... 16 lines ...
  for ‘optim’
rnegbinom: no visible global function definition for ‘rpois’
rnegbinom: no visible global function definition for ‘rgamma’
Undefined global functions or variables:
  IQR glm lm median optim optimize p.adjust pchisq poisson read.table
  rgamma rpois
Consider adding
  importFrom("stats", "IQR", "glm", "lm", "median", "optim", "optimize",
             "p.adjust", "pchisq", "poisson", "rgamma", "rpois")
  importFrom("utils", "read.table")
to your NAMESPACE file.
```

## neurobase (1.13.2)
Maintainer: John Muschelli <muschellij2@gmail.com>  
Bug reports: https://github.com/muschellij2/nifti/issues

0 errors | 0 warnings | 0 notes

## NSA (0.0.32)
Maintainer: Maria Ortiz-Estevez <mortizest@gmail.com>

0 errors | 0 warnings | 6 notes

```
checking package dependencies ... NOTE
Depends: includes the non-default packages:
  ‘R.methodsS3’ ‘MASS’ ‘matrixStats’ ‘R.oo’ ‘R.utils’ ‘aroma.core’
  ‘aroma.affymetrix’ ‘DNAcopy’
Adding so many packages to the search path is excessive and importing
selectively is preferable.

checking top-level files ... NOTE
Non-standard file/directory found at top level:
  ‘incl’

checking dependencies in R code ... NOTE
Packages in Depends field not imported from:
  ‘DNAcopy’ ‘MASS’ ‘R.methodsS3’ ‘R.oo’ ‘aroma.affymetrix’ ‘aroma.core’
  ‘matrixStats’
  These packages need to be imported from (in the NAMESPACE file)
  for when this namespace is loaded but not attached.

checking S3 generic/method consistency ... NOTE
Found the following apparent S3 methods exported but not registered:
  NSAByTotalAndFracB.matrix allocateOutputDataSets.NSANormalization
  allocateOutputDataSets.SNPsNormalization
  allocateOutputDataSets.SampleNormalization
  as.character.NSANormalization as.character.SNPsNormalization
  as.character.SampleNormalization findArraysTodo.NSANormalization
  findArraysTodo.SampleNormalization findUnitsTodo.SNPsNormalization
  fitNSA.matrix fitNSAcnPs.matrix getDataSets.NSANormalization
  getDataSets.SNPsNormalization getDataSets.SampleNormalization
  getName.NSANormalization getName.SNPsNormalization
  getName.SampleNormalization getOutputDataSets.NSANormalization
  getOutputDataSets.SNPsNormalization
  getOutputDataSets.SampleNormalization getPath.NSANormalization
  getPath.SNPsNormalization getPath.SampleNormalization
  getRootPath.NSANormalization getRootPath.SNPsNormalization
  getRootPath.SampleNormalization process.NSANormalization
  process.SNPsNormalization process.SampleNormalization
  sampleNByTotalAndFracB.numeric snpsNByTotalAndFracB.matrix
See section ‘Registering S3 methods’ in the ‘Writing R Extensions’
manual.

checking R code for possible problems ... NOTE
NB: .First.lib is obsolete and will not be used in R >= 3.0.0

.First.lib: no visible global function definition for
  ‘packageDescription’
NSAByTotalAndFracB.matrix: no visible global function definition for
  ‘throw’
NSAByTotalAndFracB.matrix: no visible global function definition for
  ‘str’
NSANormalization: no visible global function definition for ‘throw’
... 279 lines ...
  extractMatrix findUnitsTodo getAsteriskTags getChipType getFile
  getFullName getFullNames getGenomeInformation getName getNames
  getPath getPathname getPathnames getPositions getRam getRootPath
  getTags getUnitsOnChromosome hist median nbrOfFiles newInstance
  packageDescription rowAlls rowMedians segment setTags str throw trim
  verbose
Consider adding
  importFrom("graphics", "hist")
  importFrom("stats", "approxfun", "median")
  importFrom("utils", "packageDescription", "str")
to your NAMESPACE file.

checking Rd line widths ... NOTE
Rd file 'NSANormalization.Rd':
  \examples lines wider than 100 characters:
     by <- 50e3; # 50kb bins; you may want to try with other amounts of smoothing xOut <- seq(from=xRange[1], to=xRange[2], by=by);
     plot(getSignals(cnCNPS), getSignals(cnSNPS), xlim=Clim, ylim=Clim); abline(a=0, b=1, col="red", lwd=2);

These lines will be truncated in the PDF manual.
```

## pandaR (1.9.0)
Maintainer: Joseph N. Paulson <jpaulson@jimmy.harvard.edu>, Dan Schlauch <dschlauch@fas.harvard.edu>

0 errors | 0 warnings | 4 notes

```
checking installed package size ... NOTE
  installed size is  9.3Mb
  sub-directories of 1Mb or more:
    data   9.0Mb

checking S3 generic/method consistency ... NOTE
Found the following apparent S3 methods exported but not registered:
  plot.panda
See section ‘Registering S3 methods’ in the ‘Writing R Extensions’
manual.

checking R code for possible problems ... NOTE
dFunction: no visible global function definition for ‘cor’
importPandaMatlab: no visible global function definition for
  ‘read.delim’
panda: no visible global function definition for ‘cor’
panda: no visible global function definition for ‘aggregate’
plot.panda: no visible global function definition for ‘hist’
plotCommunityDetection: no visible global function definition for
  ‘title’
plotZbyTF: no visible global function definition for ‘aggregate’
prepResult: no visible global function definition for ‘pnorm’
Undefined global functions or variables:
  aggregate cor hist pnorm read.delim title
Consider adding
  importFrom("graphics", "hist", "title")
  importFrom("stats", "aggregate", "cor", "pnorm")
  importFrom("utils", "read.delim")
to your NAMESPACE file.

checking Rd \usage sections ... NOTE
S3 methods shown with full name in documentation object 'plot.panda':
  ‘plot.panda’

The \usage entries for S3 methods should use the \method markup and not
their full name.
See chapter ‘Writing R documentation files’ in the ‘Writing R
Extensions’ manual.
```

## PathoStat (1.3.0)
Maintainer: Solaiappan Manimaran <manimaran_1975@hotmail.com>  
Bug reports: https://github.com/mani2012/PathoStat/issues

0 errors | 1 warning  | 0 notes

```
checking sizes of PDF files under ‘inst/doc’ ... WARNING
  ‘gs+qpdf’ made some significant size reductions:
     compacted ‘PathoStatUserManual.pdf’ from 1.9Mb to 1.0Mb
  consider running tools::compactPDF(gs_quality = "ebook") on these files
```

## pcaMethods (1.69.0)
Maintainer: Henning Redestig <henning.red@gmail.com>  
Bug reports: https://github.com/hredestig/pcamethods/issues

0 errors | 0 warnings | 3 notes

```
checking top-level files ... NOTE
Non-standard file/directory found at top level:
  ‘extra’

checking R code for possible problems ... NOTE
BPCA_initmodel: no visible global function definition for ‘cov’
Q2: no visible global function definition for ‘txtProgressBar’
Q2: no visible global function definition for ‘setTxtProgressBar’
Q2: no visible global function definition for ‘cor’
RnipalsPca: no visible global function definition for ‘na.omit’
cvseg : <anonymous>: no visible global function definition for
  ‘na.omit’
llsImpute: no visible global function definition for ‘cor’
nlpca: no visible global function definition for ‘runif’
... 24 lines ...
  abline barplot cor cov gray layout legend lines median na.omit pairs
  par points prcomp qf rnorm runif setTxtProgressBar text
  txtProgressBar
Consider adding
  importFrom("grDevices", "gray")
  importFrom("graphics", "abline", "barplot", "layout", "legend",
             "lines", "pairs", "par", "points", "text")
  importFrom("stats", "cor", "cov", "median", "na.omit", "prcomp", "qf",
             "rnorm", "runif")
  importFrom("utils", "setTxtProgressBar", "txtProgressBar")
to your NAMESPACE file.

checking compiled code ... NOTE
File ‘pcaMethods/libs/pcaMethods.so’:
  Found no calls to: ‘R_registerRoutines’, ‘R_useDynamicSymbols’

It is good practice to register native routines and to disable symbol
search.

See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
```

## peakPick (0.11)
Maintainer: Paal O. Westermark <pal-olof.westermark@charite.de>

0 errors | 0 warnings | 0 notes

## phosphonormalizer (1.1.7)
Maintainer: Sohrab Saraei <sohrab.saraei@utu.fi>

0 errors | 0 warnings | 1 note 

```
checking package dependencies ... NOTE
Package which this enhances but not available for checking: ‘MSnbase’
```

## Pigengene (1.3.6)
Maintainer: Habil Zare <zare@txstate.edu>

0 errors | 0 warnings | 2 notes

```
checking dependencies in R code ... NOTE
'library' or 'require' calls in package code:
  ‘AnnotationDbi’ ‘biomaRt’ ‘energy’ ‘org.Hs.eg.db’ ‘org.Mm.eg.db’
  Please use :: or requireNamespace() instead.
  See section 'Suggested packages' in the 'Writing R Extensions' manual.

checking R code for possible problems ... NOTE
Found the following possibly unsafe calls:
File ‘Pigengene/R/bn.calculation.R’:
  assignInNamespace("supported.clusters", fixArgs, "bnlearn")

check.pigengene.input: no visible global function definition for
  ‘setNames’
welch.pvalue: no visible global function definition for ‘as.formula’
welch.pvalue: no visible global function definition for ‘oneway.test’
Undefined global functions or variables:
  as.formula oneway.test setNames
Consider adding
  importFrom("stats", "as.formula", "oneway.test", "setNames")
to your NAMESPACE file.
```

## POMaSPU (1.0.0)
Maintainer: Junghi Kim <junghikim0@gmail.com>

0 errors | 0 warnings | 0 notes

## PSCBS (0.63.0)
Maintainer: Henrik Bengtsson <henrikb@braju.com>  
Bug reports: https://github.com/HenrikBengtsson/PSCBS/issues

0 errors | 0 warnings | 0 notes

## QDNAseq (1.13.0)
Maintainer: Daoud Sie <d.sie@vumc.nl>  
Bug reports: https://github.com/ccagc/QDNAseq/issues

0 errors | 1 warning  | 0 notes

```
checking for missing documentation entries ... WARNING
Undocumented code objects:
  ‘exportVCF’
All user-level objects in a package should have documentation entries.
See chapter ‘Writing R documentation files’ in the ‘Writing R
Extensions’ manual.
```

## r2dRue (1.0.4)
Maintainer: Alberto Ruiz Moreno <aruiz@eeza.csic.es>

0 errors | 0 warnings | 2 notes

```
checking dependencies in R code ... NOTE
'library' or 'require' call to ‘rgdal’ which was already attached by Depends.
  Please remove these calls from your code.
Packages in Depends field not imported from:
  ‘matrixStats’ ‘rgdal’ ‘sp’
  These packages need to be imported from (in the NAMESPACE file)
  for when this namespace is loaded but not attached.

checking R code for possible problems ... NOTE
.gdalWriteFormats: no visible global function definition for
  ‘gdalDrivers’
aiObsEx: no visible global function definition for ‘readGDAL’
aiObsEx: no visible global function definition for ‘txtProgressBar’
aiObsEx: no visible global function definition for ‘setTxtProgressBar’
aiObsMe: no visible global function definition for ‘txtProgressBar’
aiObsMe: no visible global function definition for ‘readGDAL’
aiObsMe: no visible global function definition for ‘setTxtProgressBar’
assessment: no visible global function definition for ‘writeGDAL’
... 86 lines ...
  layout layout.show lines na.omit pf plot pt read.table readGDAL rect
  rowCounts rowMedians rowRanges rowSds rowVars setTxtProgressBar
  spplot title txtProgressBar write.table writeGDAL
Consider adding
  importFrom("graphics", "abline", "axis.Date", "barplot", "boxplot",
             "hist", "image", "layout", "layout.show", "lines", "plot",
             "rect", "title")
  importFrom("stats", "cor", "density", "na.omit", "pf", "pt")
  importFrom("utils", "head", "read.table", "setTxtProgressBar",
             "txtProgressBar", "write.table")
to your NAMESPACE file.
```

## RAC (1.1.1)
Maintainer: Baldan D. <damiano.baldan91@gmail.com>

0 errors | 0 warnings | 0 notes

## randomizationInference (1.0.3)
Maintainer: Joseph J. Lee <joseph.j.lee@post.harvard.edu>

0 errors | 0 warnings | 1 note 

```
checking R code for possible problems ... NOTE
anovaF: no visible global function definition for ‘anova’
anovaF: no visible global function definition for ‘lm’
randInterval: no visible global function definition for ‘quantile’
randInterval : <anonymous>: no visible global function definition for
  ‘quantile’
randPlot: no visible global function definition for ‘par’
randPlot: no visible global function definition for ‘hist’
randPlot: no visible global function definition for ‘abline’
Undefined global functions or variables:
  abline anova hist lm par quantile
Consider adding
  importFrom("graphics", "abline", "hist", "par")
  importFrom("stats", "anova", "lm", "quantile")
to your NAMESPACE file.
```

## RnBeads (1.9.0)
Maintainer: Fabian Mueller <rnbeads@mpi-inf.mpg.de>

1 error  | 3 warnings | 5 notes

```
checking tests ... ERROR
  Running ‘runTests.R’
Running the tests in ‘tests/runTests.R’ failed.
Last 13 lines of output:
  
      ozone
  
  The following object is masked from 'package:IRanges':
  
      desc
  
  The following object is masked from 'package:S4Vectors':
  
      rename
  
  Error in library("RUnit", quietly = TRUE) : 
    there is no package called 'RUnit'
  Calls: <Anonymous> -> library
  Execution halted

checking for executable files ... WARNING
Found the following executable files:
  inst/bin/linux_x86.64/bedGraphToBigWig
  inst/bin/linux_x86.64/bedToBigBed
  inst/bin/macOSX.i386/bedGraphToBigWig
  inst/bin/macOSX.i386/bedToBigBed
Source packages should not contain undeclared executable files.
See section ‘Package structure’ in the ‘Writing R Extensions’ manual.

checking dependencies in R code ... WARNING
Unexported objects imported by ':::' calls:
  'Gviz:::.getBMFeatureMap' 'doParallel:::.options'
  'grDevices:::.smoothScatterCalcDensity'
  'minfi:::.default.450k.annotation' 'minfi:::.extractFromRGSet450k'
  'minfi:::.normalizeFunnorm450k'
  See the note in ?`:::` about the use of this operator.
  Including base/recommended package(s):
  'grDevices'

checking sizes of PDF files under ‘inst/doc’ ... WARNING
  ‘gs+qpdf’ made some significant size reductions:
     compacted ‘RnBeads.pdf’ from 2.8Mb to 1.5Mb
  consider running tools::compactPDF(gs_quality = "ebook") on these files

checking package dependencies ... NOTE
Package suggested but not available for checking: ‘ggbio’

Depends: includes the non-default packages:
  ‘BiocGenerics’ ‘S4Vectors’ ‘GenomicRanges’ ‘MASS’ ‘cluster’ ‘ff’
  ‘fields’ ‘ggplot2’ ‘gplots’ ‘gridExtra’ ‘limma’ ‘matrixStats’
  ‘illuminaio’ ‘methylumi’ ‘plyr’
Adding so many packages to the search path is excessive and importing
selectively is preferable.

checking installed package size ... NOTE
  installed size is  6.2Mb
  sub-directories of 1Mb or more:
    R     1.1Mb
    bin   1.0Mb
    doc   3.1Mb

checking top-level files ... NOTE
Non-standard file/directory found at top level:
  ‘BinaryFiles’

checking whether the namespace can be loaded with stated dependencies ... NOTE
Warning: no function found corresponding to methods exports from ‘RnBeads’ for: ‘samples’

A namespace must be able to be loaded with just the base namespace
loaded: otherwise if the namespace gets loaded by a saved object, the
session will be unable to start.

Probably some imports need to be declared in the NAMESPACE file.

checking R code for possible problems ... NOTE
.stopImplicitCluster: no visible global function definition for
  ‘stopCluster’
MethyLumiSet2RnBeadSet: no visible global function definition for
  ‘phenoData’
MethyLumiSet2RnBeadSet: no visible global function definition for
  ‘assayDataElementNames’
MethyLumiSet2RnBeadSet: no visible global function definition for
  ‘featureNames’
MethyLumiSet2RnBeadSet: no visible global function definition for
... 486 lines ...
  foreach geneCounts genome<- getCN getDoParWorkers getGreen
  getManifest getMeth getRed getSex getUnmeth getVarCov glmnet
  grid.draw grid.newpage group group1 group2 i impute.knn intensities
  is.subsegmentation k letterFrequency lme mapToGenome mean.diff
  mean.g1 mean.g2 mean.mean.g1 mean.mean.g2 mean.quot.log2 melt muted
  n.sites num.sites numSites numeric.names oddsRatios pData
  percent_format phenoData phenoData<- plotOrder plotTracks
  preprocessSWAN pvalues refText reg.type region.size
  registerDoParallel relative.coord report samples seqlengths
  seqlevels<- sigCategories sites2ignore size solve.QP stopCluster sva
  target tsne type types universeCounts useMart v varLabels x y yint
```

## RTransProb (0.1.0)
Maintainer: Ab NDiaye <pabdndiaye@gmail.com>

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘RTransProb-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: TransitionProb
> ### Title: Estimation of credit transition probabilities
> ### Aliases: TransitionProb
> 
> ### ** Examples
> 
> 
> #Example 1:
> #When start date and end date are not specified, the entire dataset is used and the package
> #performs TTC calculations. Equally when snapshots and interval are not specified the defaults
> #are 1.
> snapshots <- 0
> interval <- 0
> startDate  <- 0
> endDate    <- 0
> Example1<-TransitionProb(data,startDate,endDate,'cohort', snapshots, interval)
Error in if (is.nan(rating)) { : the condition has length > 1
Calls: TransitionProb -> getidTotCntCohort
Execution halted
```

## samr (2.0)
Maintainer: Rob Tibshirani <tibs@stanford.edu>

0 errors | 0 warnings | 6 notes

```
checking DESCRIPTION meta-information ... NOTE
Malformed Description field: should contain one or more complete sentences.

checking dependencies in R code ... NOTE
'library' or 'require' calls to packages already attached by Depends:
  ‘impute’ ‘matrixStats’
  Please remove these calls from your code.
Packages in Depends field not imported from:
  ‘impute’ ‘matrixStats’
  These packages need to be imported from (in the NAMESPACE file)
  for when this namespace is loaded but not attached.

checking S3 generic/method consistency ... NOTE
Found the following apparent S3 methods exported but not registered:
  plot.SAMoutput print.SAMoutput
See section ‘Registering S3 methods’ in the ‘Writing R Extensions’
manual.

checking R code for possible problems ... NOTE
Found an obsolete/platform-specific call in the following function:
  ‘samr.assess.samplesize.plot’
Found the platform-specific device:
  ‘win.metafile’
dev.new() is the preferred way to open a new device, in the unlikely
event one is needed.
est.s0: no visible global function definition for ‘quantile’
est.s0: no visible global function definition for ‘mad’
est.s0: no visible global function definition for ‘var’
... 62 lines ...
Undefined global functions or variables:
  abline dev.off impute.knn lines mad median mtext par plot points
  predict quantile rowMedians rpois runif samr.xl.var.knn.neighbors
  smooth.spline title var
Consider adding
  importFrom("grDevices", "dev.off")
  importFrom("graphics", "abline", "lines", "mtext", "par", "plot",
             "points", "title")
  importFrom("stats", "mad", "median", "predict", "quantile", "rpois",
             "runif", "smooth.spline", "var")
to your NAMESPACE file.

checking Rd \usage sections ... NOTE
S3 methods shown with full name in documentation object 'samr-internal':
  ‘plot.SAMoutput’ ‘print.SAMoutput’

The \usage entries for S3 methods should use the \method markup and not
their full name.
See chapter ‘Writing R documentation files’ in the ‘Writing R
Extensions’ manual.

checking compiled code ... NOTE
File ‘samr/libs/samr.so’:
  Found no calls to: ‘R_registerRoutines’, ‘R_useDynamicSymbols’

It is good practice to register native routines and to disable symbol
search.

See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
```

## scater (1.5.0)
Maintainer: Davis McCarthy <davis@ebi.ac.uk>  
Bug reports: https://support.bioconductor.org/

0 errors | 0 warnings | 3 notes

```
checking installed package size ... NOTE
  installed size is  8.6Mb
  sub-directories of 1Mb or more:
    doc   5.7Mb

checking S3 generic/method consistency ... NOTE
Found the following apparent S3 methods exported but not registered:
  arrange.SCESet filter.SCESet mutate.SCESet rename.SCESet
See section ‘Registering S3 methods’ in the ‘Writing R Extensions’
manual.

checking Rd \usage sections ... NOTE
S3 methods shown with full name in documentation object 'arrange':
  ‘arrange.SCESet’

S3 methods shown with full name in documentation object 'filter':
  ‘filter.SCESet’

S3 methods shown with full name in documentation object 'mutate':
  ‘mutate.SCESet’

S3 methods shown with full name in documentation object 'rename':
  ‘rename.SCESet’

The \usage entries for S3 methods should use the \method markup and not
their full name.
See chapter ‘Writing R documentation files’ in the ‘Writing R
Extensions’ manual.
```

## scmap (0.99.2)
Maintainer: Vladimir Kiselev <vladimir.yu.kiselev@gmail.com>  
Bug reports: https://support.bioconductor.org/t/scmap/

0 errors | 0 warnings | 1 note 

```
checking top-level files ... NOTE
File
  LICENSE
is not mentioned in the DESCRIPTION file.
```

## scone (1.1.2)
Maintainer: Michael Cole <mbeloc@gmail.com>  
Bug reports: https://github.com/YosefLab/scone/issues

2 errors | 0 warnings | 4 notes

```
checking examples ... ERROR
Running examples in ‘scone-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: SCRAN_FN
> ### Title: Simple deconvolution normalization wrapper
> ### Aliases: SCRAN_FN
> 
> ### ** Examples
> 
> ei <- matrix(0:76,nrow = 7)
> eo <- SCRAN_FN(ei)
Error in SCRAN_FN(ei) : scran package needed for SCRAN_FN()
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  testthat results ================================================================
  OK: 7 SKIPPED: 0 FAILED: 16
  1. Error: all back-ends work (@test_bpparam.R#17) 
  2. Error: get_normalized works in all three modes (@test_get_design.R#18) 
  3. Error: get_normalized works in all three modes (@test_get_norm.R#18) 
  4. Error: get_normalized works in all three modes with nested model (@test_get_norm.R#76) 
  5. Error: return_norm in memory (@test_hdf5.R#44) 
  6. Error: do not return_norm (@test_hdf5.R#65) 
  7. Error: Upper-quartile normalization works the same as in the edgeR package (@test_norm.R#15) 
  8. Error: Test with no real method (only identity) (@test_scone.R#12) 
  9. Error: Test imputation and scaling (@test_scone.R#129) 
  1. ...
  
  Error: testthat unit tests failed
  Execution halted

checking package dependencies ... NOTE
Package suggested but not available for checking: ‘scran’

checking dependencies in R code ... NOTE
'library' or 'require' calls in package code:
  ‘ggplot2’ ‘plotly’ ‘visNetwork’
  Please use :: or requireNamespace() instead.
  See section 'Suggested packages' in the 'Writing R Extensions' manual.
Missing or unexported object: ‘shiny::renderPlotly’

checking R code for possible problems ... NOTE
sconeReport: no visible global function definition for
  ‘visNetworkOutput’
sconeReport: no visible global function definition for ‘plotlyOutput’
sconeReport : server: no visible global function definition for
  ‘renderVisNetwork’
sconeReport : server: no visible global function definition for ‘%>%’
sconeReport : server: no visible global function definition for
  ‘visNetwork’
sconeReport : server: no visible global function definition for
... 46 lines ...
Undefined global functions or variables:
  %>% aes boxplot coord_cartesian element_blank geom_bar geom_point
  geom_violin ggplot ggplotly guides labs plot_ly plotlyOutput
  renderVisNetwork scale_fill_manual setNames theme var visEdges
  visGroups visHierarchicalLayout visLegend visNetwork visNetworkOutput
  visNetworkProxy visOptions visSelectNodes write.csv ylim
Consider adding
  importFrom("graphics", "boxplot")
  importFrom("stats", "setNames", "var")
  importFrom("utils", "write.csv")
to your NAMESPACE file.

checking Rd cross-references ... NOTE
Package unavailable to check Rd xrefs: ‘scran’
```

## SemiParSampleSel (1.5)
Maintainer: Giampiero Marra <giampiero.marra@ucl.ac.uk>

0 errors | 0 warnings | 0 notes

## SGP (1.7-0.0)
Maintainer: Damian W. Betebenner <dbetebenner@nciea.org>  
Bug reports: https://github.com/CenterForAssessment/SGP/issues

0 errors | 0 warnings | 0 notes

## shinyMethyl (1.13.0)
Maintainer: Jean-Philippe Fortin <jfortin@jhsph.edu>

0 errors | 0 warnings | 2 notes

```
checking for hidden files and directories ... NOTE
Found the following hidden files and directories:
  .travis.yml
These were most likely included in error. See section ‘Package
structure’ in the ‘Writing R Extensions’ manual.

checking R code for possible problems ... NOTE
addHoverDensity: no visible global function definition for ‘lines’
addHoverPoints: no visible global function definition for ‘points’
addHoverQC: no visible global function definition for ‘points’
densitiesPlot: no visible global function definition for ‘plot’
densitiesPlot: no visible global function definition for ‘lines’
plotDesign450k: no visible global function definition for ‘par’
plotDesign450k: no visible global function definition for ‘plot’
plotDiscrepancyGenders: no visible global function definition for
  ‘legend’
... 60 lines ...
Undefined global functions or variables:
  abline complete.cases current.control.type current.density.type
  current.probe.type genderCutoff grid legend lines lm
  mouse.click.indices palette par plot points prcomp rect write.csv
Consider adding
  importFrom("grDevices", "palette")
  importFrom("graphics", "abline", "grid", "legend", "lines", "par",
             "plot", "points", "rect")
  importFrom("stats", "complete.cases", "lm", "prcomp")
  importFrom("utils", "write.csv")
to your NAMESPACE file.
```

## SICtools (1.7.0)
Maintainer: Xiaobin Xing <xiaobinxing0316@gmail.com>

0 errors | 1 warning  | 4 notes

```
checking whether package ‘SICtools’ can be installed ... WARNING
Found the following significant warnings:
  Warning: replacing previous import ‘plyr::count’ by ‘matrixStats::count’ when loading ‘SICtools’
See ‘/home/hb/repositories/matrixStats/revdep/checks/SICtools.Rcheck/00install.out’ for details.

checking package dependencies ... NOTE
Depends: includes the non-default packages:
  ‘Rsamtools’ ‘doParallel’ ‘Biostrings’ ‘stringr’ ‘matrixStats’ ‘plyr’
  ‘GenomicRanges’ ‘IRanges’
Adding so many packages to the search path is excessive and importing
selectively is preferable.

checking for hidden files and directories ... NOTE
Found the following hidden files and directories:
  .BBSoptions
These were most likely included in error. See section ‘Package
structure’ in the ‘Writing R Extensions’ manual.

checking R code for possible problems ... NOTE
.indelDiffFunc: no visible global function definition for ‘fisher.test’
.indelDiffFunc: no visible global function definition for ‘dist’
indelDiff: no visible global function definition for ‘read.delim’
snpDiff : calcInfoRange : <anonymous>: no visible global function
  definition for ‘fisher.test’
snpDiff : calcInfoRange : <anonymous>: no visible global function
  definition for ‘dist’
Undefined global functions or variables:
  dist fisher.test read.delim
Consider adding
  importFrom("stats", "dist", "fisher.test")
  importFrom("utils", "read.delim")
to your NAMESPACE file.

checking Rd line widths ... NOTE
Rd file 'indelDiff.Rd':
  \usage lines wider than 90 characters:
     indelDiff(bam1, bam2, refFsa, regChr, regStart, regEnd, minBaseQuality = 13, minMapQuality = 0, nCores = 1, pValueCutOff= 0.05,gtDistCu ... [TRUNCATED]

Rd file 'snpDiff.Rd':
  \usage lines wider than 90 characters:
     snpDiff(bam1, bam2, refFsa, regChr, regStart, regEnd, minBaseQuality = 13, minMapQuality = 0, nCores = 1, pValueCutOff = 0.05, baseDist ... [TRUNCATED]

These lines will be truncated in the PDF manual.
```

## sizeMat (0.3.0)
Maintainer: Josymar Torrejon-Magallanes <ejosymart@gmail.com>

0 errors | 0 warnings | 0 notes

## splatter (1.1.4)
Maintainer: Luke Zappia <luke.zappia@mcri.edu.au>  
Bug reports: https://github.com/Oshlack/splatter/issues

1 error  | 0 warnings | 1 note 

```
checking tests ... ERROR
  Running ‘testthat.R’ [19s/19s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  2: stop("The scDD simulation requires the 'scDD' package.")
  
  3. Error: scDD simulation output is valid (@test-scDD-simulate.R#4) ------------
  The scDD simulation requires the 'scDD' package.
  1: newSCDDParams(nDE = 5, nDP = 5, nDM = 5, nDB = 5, nEE = 5, nEP = 5) at testthat/test-scDD-simulate.R:4
  2: stop("The scDD simulation requires the 'scDD' package.")
  
  testthat results ================================================================
  OK: 38 SKIPPED: 0 FAILED: 3
  1. Error: constructor is valid (@test-SCDDParams.R#4) 
  2. Error: nGenes checks work (@test-SCDDParams.R#8) 
  3. Error: scDD simulation output is valid (@test-scDD-simulate.R#4) 
  
  Error: testthat unit tests failed
  Execution halted

checking package dependencies ... NOTE
Package suggested but not available for checking: ‘scran’
```

## SRGnet (1.3.0)
Maintainer: Isar Nassiri <isar_nassiri@urmc.rochester.edu>

0 errors | 0 warnings | 2 notes

```
checking package dependencies ... NOTE
Depends: includes the non-default packages:
  ‘EBcoexpress’ ‘MASS’ ‘igraph’ ‘pvclust’ ‘gbm’ ‘limma’ ‘DMwR’
  ‘matrixStats’ ‘Hmisc’
Adding so many packages to the search path is excessive and importing
selectively is preferable.

checking R code for possible problems ... NOTE
SRGnet: no visible binding for global variable
  ‘Differentially_expressed_genes’
SRGnet: no visible binding for global variable ‘Transcriptomics’
SRGnet: no visible binding for global variable ‘PLCRG’
SRGnet: no visible global function definition for ‘ebPatterns’
SRGnet: no visible global function definition for ‘par’
SRGnet: no visible global function definition for ‘boxplot’
SRGnet: no visible global function definition for ‘plot’
SRGnet: no visible global function definition for ‘predict’
SRGnet: no visible global function definition for ‘terrain.colors’
SRGnet: no visible global function definition for ‘na.omit’
SRGnet: no visible global function definition for ‘write.table’
Undefined global functions or variables:
  Differentially_expressed_genes PLCRG Transcriptomics boxplot
  ebPatterns na.omit par plot predict terrain.colors write.table
Consider adding
  importFrom("grDevices", "terrain.colors")
  importFrom("graphics", "boxplot", "par", "plot")
  importFrom("stats", "na.omit", "predict")
  importFrom("utils", "write.table")
to your NAMESPACE file.
```

## statar (0.6.5)
Maintainer: Matthieu Gomez <mattg@princeton.edu>  
Bug reports: https://github.com/matthieugomez/statar/issues

0 errors | 0 warnings | 0 notes

## stm (1.2.2)
Maintainer: Brandon Stewart <bms4@princeton.edu>  
Bug reports: https://github.com/bstewart/stm/issues

0 errors | 0 warnings | 0 notes

## STROMA4 (1.1.0)
Maintainer: Sadiq Saleh <sadiq.mehdiismailsaleh@mail.mcgill.ca>

0 errors | 0 warnings | 1 note 

```
checking top-level files ... NOTE
Non-standard files/directories found at top level:
  ‘S4.Rproj’ ‘Stroma4.Rproj’
```

## StructFDR (1.2)
Maintainer: Jun Chen <chen.jun2@mayo.edu>

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘StructFDR-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: StructFDR
> ### Title: False Discovery Rate (FDR) Control Integrating a General Prior
> ###   Structure
> ### Aliases: StructFDR
> ### Keywords: Multiple testing False discovery rate Genetics Metagenomics
... 39 lines ...
> 
> # Call StructFDR
> tree.fdr.obj <- StructFDR(X, Y, D, test.func, perm.func)
Warning in StructFDR(X, Y, D, test.func, perm.func) :
  Both the data matrix and the distance matrix should have labels (rownames) to avoid potential errors!

Test on original data sets  ...
Test on permuted data sets  ...
Error in if (alt.FDR == "Permutation") { : the condition has length > 1
Calls: StructFDR
Execution halted
```

## summarytools (0.6.5)
Maintainer: Dominic Comtois <dominic.comtois@gmail.com>  
Bug reports: https://github.com/dcomtois/summarytools/issues

0 errors | 0 warnings | 0 notes

## sva (3.25.4)
Maintainer: Jeffrey T. Leek <jtleek@gmail.com>, John D. Storey
 <jstorey@princeton.edu>, W. Evan Johnson <wej@bu.edu>

0 errors | 0 warnings | 1 note 

```
checking compiled code ... NOTE
File ‘sva/libs/sva.so’:
  Found no call to: ‘R_useDynamicSymbols’

It is good practice to register native routines and to disable symbol
search.

See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
```

## tmlenet (0.1.0)
Maintainer: Oleg Sofrygin <oleg.sofrygin@gmail.com>  
Bug reports: https://github.com/osofr/tmlenet/issues

0 errors | 0 warnings | 1 note 

```
checking compiled code ... NOTE
File ‘tmlenet/libs/tmlenet.so’:
  Found no calls to: ‘R_registerRoutines’, ‘R_useDynamicSymbols’

It is good practice to register native routines and to disable symbol
search.

See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
```

## topGO (2.29.0)
Maintainer: Adrian Alexa <adrian.alexa@gmail.com>

0 errors | 0 warnings | 4 notes

```
checking package dependencies ... NOTE
Depends: includes the non-default packages:
  ‘BiocGenerics’ ‘graph’ ‘Biobase’ ‘GO.db’ ‘AnnotationDbi’ ‘SparseM’
Adding so many packages to the search path is excessive and importing
selectively is preferable.

checking dependencies in R code ... NOTE
'library' or 'require' calls in package code:
  ‘Rgraphviz’ ‘multtest’
  Please use :: or requireNamespace() instead.
  See section 'Suggested packages' in the 'Writing R Extensions' manual.
Missing object imported by a ':::' call: ‘globaltest:::globaltest’

checking R code for possible problems ... NOTE
GOplot: no visible global function definition for ‘getDefaultAttrs’
GOplot: no visible global function definition for ‘agopen’
GOplot.counts : plotSigChart : buildDrawing : <anonymous>: no visible
  global function definition for ‘getNodeCenter’
GOplot.counts : plotSigChart : buildDrawing : <anonymous>: no visible
  global function definition for ‘pieGlyph’
GOplot.counts : plotSigChart : buildDrawing : <anonymous>: no visible
  global function definition for ‘getX’
GOplot.counts : plotSigChart : buildDrawing : <anonymous>: no visible
... 35 lines ...
  getDefaultAttrs getNodeCenter getNodeLW getNodeXY getX getY
  mt.rawp2adjp mt.teststat name pieGlyph scoreInNode toDot txtLabel

Found the following assignments to the global environment:
File ‘topGO/R/topGOtests.R’:
  assign(".PERMSUM.LOOKUP", sizeLookUp, .GlobalEnv)
  assign(".PERMSUM.MAT", sapply(1:N, function(x) cumsum(sample(scoreVec, 
    maxSample))[goSize]), .GlobalEnv)
  assign(".PERMSUM.LOOKUP", sizeLookUp, .GlobalEnv)
  assign(".PERMSUM.MAT", sapply(1:N, function(x) cumsum(sample(scoreVec))[goSize]), 
    .GlobalEnv)

checking Rd line widths ... NOTE
Rd file 'diagnosticMethods.Rd':
  \examples lines wider than 100 characters:
     sig.tab <- GenTable(GOdata, Fis = resultFisher, KS = resultKS, orderBy = "KS", ranksOf = "Fis", topNodes = 20)

Rd file 'getPvalues.Rd':
  \usage lines wider than 90 characters:
        getPvalues(edata, classlabel, test = "t", alternative = c("greater", "two.sided", "less")[1],
        genesID = NULL, correction = c("none", "Bonferroni", "Holm", "Hochberg", "SidakSS", "SidakSD",

Rd file 'printGraph-methods.Rd':
  \usage lines wider than 90 characters:
                    useFullNames = TRUE, oldSigNodes = NULL, useInfo = c("none", "pval", "counts", "def", "np", "all")[1],
  \examples lines wider than 100 characters:
     printGraph(GOdata, resultFisher, firstSigNodes = 5, fn.prefix = "sampleFile", useInfo = "all", pdfSW = TRUE)

Rd file 'topGOdata-class.Rd':
  \examples lines wider than 100 characters:
                   description = "GO analysis of ALL data: Differential Expression between B-cell and T-cell",

These lines will be truncated in the PDF manual.
```

## ttScreening (1.5)
Maintainer: Meredith Ray <mere2110@yahoo.com>

0 errors | 0 warnings | 2 notes

```
checking DESCRIPTION meta-information ... NOTE
Malformed Title field: should not end in a period.

checking R code for possible problems ... NOTE
ttScreening: no visible global function definition for ‘gaussian’
ttScreening: no visible global function definition for ‘glm.control’
ttScreening: no visible global function definition for ‘model.response’
ttScreening: no visible global function definition for ‘is.empty.model’
ttScreening: no visible global function definition for ‘model.matrix’
ttScreening: no visible global function definition for ‘na.omit’
ttScreening: no visible global function definition for ‘terms’
ttScreening: no visible global function definition for ‘capture.output’
ttScreening: no visible global function definition for ‘glm’
ttScreening: no visible binding for global variable ‘gaussian’
ttScreening: no visible global function definition for ‘pt’
ttScreening: no visible global function definition for ‘p.adjust’
Undefined global functions or variables:
  capture.output gaussian glm glm.control is.empty.model model.matrix
  model.response na.omit p.adjust pt terms
Consider adding
  importFrom("stats", "gaussian", "glm", "glm.control", "is.empty.model",
             "model.matrix", "model.response", "na.omit", "p.adjust",
             "pt", "terms")
  importFrom("utils", "capture.output")
to your NAMESPACE file.
```

## VanillaICE (1.39.2)
Maintainer: Robert Scharpf <rscharpf@jhsph.edu>

0 errors | 0 warnings | 4 notes

```
checking package dependencies ... NOTE
Package which this enhances but not available for checking: ‘doRedis’

checking top-level files ... NOTE
Non-standard file/directory found at top level:
  ‘changelog’

checking dependencies in R code ... NOTE
Unexported object imported by a ':::' call: ‘SummarizedExperiment:::.ShallowSimpleListAssays0’
  See the note in ?`:::` about the use of this operator.

checking compiled code ... NOTE
File ‘VanillaICE/libs/VanillaICE.so’:
  Found no call to: ‘R_useDynamicSymbols’

It is good practice to register native routines and to disable symbol
search.

See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
```

## visualFields (0.4.3)
Maintainer: Ivan Marin-Franch <ivan.marin@uv.es>

0 errors | 0 warnings | 0 notes

## wateRmelon (1.21.2)
Maintainer: Leo <lschal@essex.ac.uk>

0 errors | 0 warnings | 5 notes

```
checking package dependencies ... NOTE
Depends: includes the non-default packages:
  ‘Biobase’ ‘limma’ ‘matrixStats’ ‘methylumi’ ‘lumi’ ‘ROC’
  ‘IlluminaHumanMethylation450kanno.ilmn12.hg19’ ‘illuminaio’
Adding so many packages to the search path is excessive and importing
selectively is preferable.

checking DESCRIPTION meta-information ... NOTE
Package listed in more than one of Depends, Imports, Suggests, Enhances:
  ‘Biobase’
A package should be listed in only one of these fields.

checking dependencies in R code ... NOTE
Packages in Depends field not imported from:
  ‘IlluminaHumanMethylation450kanno.ilmn12.hg19’ ‘ROC’ ‘illuminaio’
  ‘limma’ ‘lumi’ ‘matrixStats’ ‘methods’ ‘methylumi’
  These packages need to be imported from (in the NAMESPACE file)
  for when this namespace is loaded but not attached.
Unexported object imported by a ':::' call: ‘minfi:::.isRGOrStop’
  See the note in ?`:::` about the use of this operator.

checking R code for possible problems ... NOTE
DataToNChannelSet2: no visible global function definition for ‘is’
DataToNChannelSet2: no visible global function definition for ‘new’
IDATsToMatrices2: no visible global function definition for ‘.mclapply’
IDATtoMatrix2 : <anonymous>: no visible global function definition for
  ‘readIDAT’
NChannelSetToMethyLumiSet2: no visible global function definition for
  ‘new’
NChannelSetToMethyLumiSet2: no visible global function definition for
  ‘betas’
... 422 lines ...
Found the following calls to data() loading into the global environment:
File ‘wateRmelon/R/horv.R’:
  data(coef)
File ‘wateRmelon/R/readEPIC.R’:
  data(hm27.ordering)
  data(hm450.ordering)
  data(epic.ordering)
  data(hm27.controls)
  data(hm450.controls)
  data(epic.controls)
See section ‘Good practice’ in ‘?data’.

checking Rd line widths ... NOTE
Rd file 'BMIQ.Rd':
  \usage lines wider than 90 characters:
     BMIQ(beta.v, design.v, nL = 3, doH = TRUE, nfit = 50000, th1.v = c(0.2, 0.75), th2.v = NULL, niter = 5, tol = 0.001, plots = TRUE, samp ... [TRUNCATED]
     BMIQ(beta.v, nL=3, doH=TRUE, nfit=5000, th1.v=c(0.2,0.75), th2.v=NULL, niter=5, tol=0.001, plots=FALSE,  pri=FALSE ) 

Rd file 'wm_internal.Rd':
  \usage lines wider than 90 characters:
     DataToNChannelSet2(mats, chans = c(Cy3 = "GRN", Cy5 = "RED"), parallel = F, protocol.data = F, IDAT = TRUE, force=F)

These lines will be truncated in the PDF manual.
```

## WGCNA (1.61)
Maintainer: Peter Langfelder <Peter.Langfelder@gmail.com>

0 errors | 0 warnings | 0 notes

## XBSeq (1.7.0)
Maintainer: Yuanhang Liu <liuy12@uthscsa.edu>

1 error  | 0 warnings | 3 notes

```
checking tests ... ERROR
  Running ‘testthat.R’ [146s/146s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
      colMaxs, colMins, colRanges, rowMaxs, rowMins, rowRanges
  
  The following object is masked from 'package:base':
  
      apply
  
      Welcome to 'XBSeq'.
  > 
  > test_check("XBSeq")
  estimating parameters using MLE for group one 
  estimating parameters using MLE for group two 
  Error: XBplot(XB, Samplenum = "Sample_54_WT") did not throw an error.
  testthat results ================================================================
  OK: 0 SKIPPED: 0 FAILED: 0
  Execution halted

checking whether the namespace can be loaded with stated dependencies ... NOTE
Warning: no function found corresponding to methods exports from ‘XBSeq’ for: ‘conditions’, ‘conditions<-’, ‘dispTable’

A namespace must be able to be loaded with just the base namespace
loaded: otherwise if the namespace gets loaded by a saved object, the
session will be unable to start.

Probably some imports need to be declared in the NAMESPACE file.

checking R code for possible problems ... NOTE
Loglikhood : <anonymous>: no visible global function definition for
  ‘ddelap’
Loglikhood : <anonymous>: no visible global function definition for
  ‘dpois’
Loglikhood_NB : <anonymous>: no visible global function definition for
  ‘dnbinom’
MAplot: no visible binding for global variable ‘baseMean’
MAplot: no visible global function definition for ‘quantile’
MAplot: no visible binding for global variable ‘log2FoldChange’
... 49 lines ...
  importFrom("stats", "Gamma", "coefficients", "complete.cases", "cor",
             "dnbinom", "dpois", "formula", "glm", "median", "optim",
             "p.adjust", "pbeta", "predict", "qbeta", "quantile",
             "rnbinom")
  importFrom("utils", "data")
to your NAMESPACE file.

Found the following calls to data() loading into the global environment:
File ‘XBSeq/R/methods.R’:
  data("scvBiasCorrectionFits")
See section ‘Good practice’ in ‘?data’.

checking Rd line widths ... NOTE
Rd file 'XBSeq.Rd':
  \usage lines wider than 90 characters:
        sharingMode = "maximum", fitType = "local", pvals_only = FALSE, paraMethod='NP', big_count = 900)

Rd file 'XBplot.Rd':
  \usage lines wider than 90 characters:
     XBplot(XB, Samplenum = NULL, unit = c('counts', 'LogTPM'), Libsize = NULL, Genelength = NULL, xlab = 'log2 TPM', ylab = 'Frequencies',  ... [TRUNCATED]

Rd file 'estimateSCV.Rd':
  \usage lines wider than 90 characters:
     estimateSCV( object, method = c( "pooled", "per-condition", "blind" ), sharingMode = c( "maximum", "fit-only", "gene-est-only" ),

These lines will be truncated in the PDF manual.
```

## yarn (1.3.0)
Maintainer: Joseph N Paulson <jpaulson@jimmy.harvard.edu>

0 errors | 0 warnings | 0 notes

## zinbwave (0.99.6)
Maintainer: Davide Risso <risso.davide@gmail.com>  
Bug reports: https://github.com/drisso/zinbwave/issues

0 errors | 0 warnings | 0 notes

